{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "gpuType": "T4",
      "authorship_tag": "ABX9TyO0pGAn9PRYOhC7SgajQ69z",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU"
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/gurveervirk/florabert/blob/main/FloraBERT.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!unzip florabert.zip"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "seU4QqVXFfpL",
        "outputId": "b8d6db85-da8c-4379-9237-100b23de44b1"
      },
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Archive:  florabert.zip\n",
            "   creating: florabert/.git/\n",
            "   creating: florabert/.git/branches/\n",
            "  inflating: florabert/.git/config   \n",
            "  inflating: florabert/.git/description  \n",
            " extracting: florabert/.git/HEAD     \n",
            "   creating: florabert/.git/hooks/\n",
            "  inflating: florabert/.git/hooks/applypatch-msg.sample  \n",
            "  inflating: florabert/.git/hooks/commit-msg.sample  \n",
            "  inflating: florabert/.git/hooks/fsmonitor-watchman.sample  \n",
            "  inflating: florabert/.git/hooks/post-update.sample  \n",
            "  inflating: florabert/.git/hooks/pre-applypatch.sample  \n",
            "  inflating: florabert/.git/hooks/pre-commit.sample  \n",
            "  inflating: florabert/.git/hooks/pre-merge-commit.sample  \n",
            "  inflating: florabert/.git/hooks/prepare-commit-msg.sample  \n",
            "  inflating: florabert/.git/hooks/pre-push.sample  \n",
            "  inflating: florabert/.git/hooks/pre-rebase.sample  \n",
            "  inflating: florabert/.git/hooks/pre-receive.sample  \n",
            "  inflating: florabert/.git/hooks/push-to-checkout.sample  \n",
            "  inflating: florabert/.git/hooks/update.sample  \n",
            "  inflating: florabert/.git/index    \n",
            "   creating: florabert/.git/info/\n",
            "  inflating: florabert/.git/info/exclude  \n",
            "   creating: florabert/.git/logs/\n",
            "  inflating: florabert/.git/logs/HEAD  \n",
            "   creating: florabert/.git/logs/refs/\n",
            "   creating: florabert/.git/logs/refs/heads/\n",
            "  inflating: florabert/.git/logs/refs/heads/main  \n",
            "   creating: florabert/.git/logs/refs/remotes/\n",
            "   creating: florabert/.git/logs/refs/remotes/origin/\n",
            "  inflating: florabert/.git/logs/refs/remotes/origin/HEAD  \n",
            "   creating: florabert/.git/objects/\n",
            "   creating: florabert/.git/objects/info/\n",
            "   creating: florabert/.git/objects/pack/\n",
            "  inflating: florabert/.git/objects/pack/pack-59b2e9fa235af592a8283a1bf8095f6e1793cc82.idx  \n",
            "  inflating: florabert/.git/objects/pack/pack-59b2e9fa235af592a8283a1bf8095f6e1793cc82.pack  \n",
            "  inflating: florabert/.git/packed-refs  \n",
            "   creating: florabert/.git/refs/\n",
            "   creating: florabert/.git/refs/heads/\n",
            " extracting: florabert/.git/refs/heads/main  \n",
            "   creating: florabert/.git/refs/remotes/\n",
            "   creating: florabert/.git/refs/remotes/origin/\n",
            " extracting: florabert/.git/refs/remotes/origin/HEAD  \n",
            "   creating: florabert/.git/refs/tags/\n",
            " extracting: florabert/.gitkeep      \n",
            "  inflating: florabert/config.yaml   \n",
            "   creating: florabert/data/\n",
            "   creating: florabert/data/final/\n",
            " extracting: florabert/data/final/.gitkeep  \n",
            "   creating: florabert/data/processed/\n",
            " extracting: florabert/data/processed/.gitkeep  \n",
            "   creating: florabert/data/raw/\n",
            " extracting: florabert/data/raw/.gitkeep  \n",
            "   creating: florabert/data/raw/gz_link/\n",
            "  inflating: florabert/data/raw/gz_link/ensembl_link.csv  \n",
            "  inflating: florabert/data/raw/gz_link/refseq_link.csv  \n",
            "   creating: florabert/libs/\n",
            " extracting: florabert/libs/.gitkeep  \n",
            "  inflating: florabert/LICENSE       \n",
            "  inflating: florabert/Makefile      \n",
            "   creating: florabert/module/\n",
            "  inflating: florabert/module/.DS_Store  \n",
            "   creating: florabert/module/.ipynb_checkpoints/\n",
            "   creating: florabert/module/florabert/\n",
            " extracting: florabert/module/florabert/__init__.py  \n",
            "  inflating: florabert/module/florabert/config.py  \n",
            "  inflating: florabert/module/florabert/dataio.py  \n",
            "  inflating: florabert/module/florabert/exceptions.py  \n",
            "  inflating: florabert/module/florabert/gene_db_io.py  \n",
            "  inflating: florabert/module/florabert/metrics.py  \n",
            "  inflating: florabert/module/florabert/models.py  \n",
            "  inflating: florabert/module/florabert/nlp.py  \n",
            "  inflating: florabert/module/florabert/training.py  \n",
            "  inflating: florabert/module/florabert/transformers.py  \n",
            "  inflating: florabert/module/florabert/utils.py  \n",
            "  inflating: florabert/module/florabert/visualization.py  \n",
            "  inflating: florabert/module/setup.py  \n",
            "   creating: florabert/output/\n",
            " extracting: florabert/output/.gitkeep  \n",
            "  inflating: florabert/README.md     \n",
            "  inflating: florabert/requirements.txt  \n",
            "   creating: florabert/scripts/\n",
            " extracting: florabert/scripts/__init__.py  \n",
            "   creating: florabert/scripts/0-data-loading-processing/\n",
            " extracting: florabert/scripts/0-data-loading-processing/__init__.py  \n",
            "  inflating: florabert/scripts/0-data-loading-processing/01-gene-expression.py  \n",
            "  inflating: florabert/scripts/0-data-loading-processing/02-download-process-db-data.py  \n",
            "  inflating: florabert/scripts/0-data-loading-processing/03-combine-databases.py  \n",
            "  inflating: florabert/scripts/0-data-loading-processing/04-process-genex-nam.py  \n",
            "  inflating: florabert/scripts/0-data-loading-processing/05-cluster-maize_seq.sh  \n",
            "  inflating: florabert/scripts/0-data-loading-processing/06-train-test-split.py  \n",
            "  inflating: florabert/scripts/0-data-loading-processing/07_train_tokenizer.py  \n",
            "  inflating: florabert/scripts/0-data-loading-processing/08-train-preprocessor.py  \n",
            "  inflating: florabert/scripts/0-data-loading-processing/09-prepare-nam_metadata.py  \n",
            "   creating: florabert/scripts/1-modeling/\n",
            "  inflating: florabert/scripts/1-modeling/evaluate.py  \n",
            " extracting: florabert/scripts/1-modeling/evaluate_lm.py  \n",
            "  inflating: florabert/scripts/1-modeling/finetune.py  \n",
            "  inflating: florabert/scripts/1-modeling/pretrain.py  \n",
            "   creating: florabert/scripts/2-feature-visualization/\n",
            "  inflating: florabert/scripts/2-feature-visualization/embedding_vis.py  \n",
            "  inflating: florabert/scripts/2-feature-visualization/get_species_embeddings.py  \n",
            "  inflating: florabert/scripts/2-feature-visualization/positional_importance.py  \n",
            "  inflating: florabert/scripts/2-feature-visualization/top_kmers_by_position.py  \n",
            "   creating: florabert/scripts/3-RNAseq-quantification/\n",
            "  inflating: florabert/scripts/3-RNAseq-quantification/alignment_script.sh  \n",
            "  inflating: florabert/scripts/3-RNAseq-quantification/cmds_to_dl_NAM_and_B73v5_genomes_and_create_HiSat_indices.txt  \n",
            "  inflating: florabert/scripts/3-RNAseq-quantification/input_genome_names.txt  \n",
            "   creating: florabert/tests/\n",
            " extracting: florabert/tests/.gitkeep  \n",
            "   creating: florabert/tests/test_data_quality/\n",
            " extracting: florabert/tests/test_data_quality/.gitkeep  \n",
            "  inflating: florabert/tests/test_data_quality/01-check-ensembel-refseq-links.py  \n",
            "   creating: florabert/tests/test_module/\n",
            "  inflating: florabert/tests/test_module/test_tokenizers.py  \n",
            "  inflating: florabert/tests/test_module/test_transformers.py  \n",
            "   creating: florabert/tests/test_scripts/\n",
            "   creating: florabert/tests/test_scripts/0-data-loading-processing/\n",
            "  inflating: florabert/tests/test_scripts/0-data-loading-processing/test-BPE-encode.py  \n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%cd /content/florabert"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Tx9iN7QFesCD",
        "outputId": "3424ef39-c000-4512-de8d-a8bcf73bc841"
      },
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/florabert\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install virtualenv"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "mD_hX_sKfRCc",
        "outputId": "02e5d2cd-4768-4446-c9ac-53d70e3527ba"
      },
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting virtualenv\n",
            "  Downloading virtualenv-20.24.5-py3-none-any.whl (3.7 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.7/3.7 MB\u001b[0m \u001b[31m37.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting distlib<1,>=0.3.7 (from virtualenv)\n",
            "  Downloading distlib-0.3.7-py2.py3-none-any.whl (468 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m468.9/468.9 kB\u001b[0m \u001b[31m36.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: filelock<4,>=3.12.2 in /usr/local/lib/python3.10/dist-packages (from virtualenv) (3.12.2)\n",
            "Requirement already satisfied: platformdirs<4,>=3.9.1 in /usr/local/lib/python3.10/dist-packages (from virtualenv) (3.10.0)\n",
            "Installing collected packages: distlib, virtualenv\n",
            "Successfully installed distlib-0.3.7 virtualenv-20.24.5\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!make"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "_PKkZH57fnJH",
        "outputId": "e3d50ab3-5a90-4f21-822f-391589063815"
      },
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "virtualenv env \\\n",
            "&& source ./env/bin/activate \\\n",
            "&& pip install -r requirements.txt\n",
            "created virtual environment CPython3.10.12.final.0-64 in 1003ms\n",
            "  creator CPython3Posix(dest=/content/florabert/env, clear=False, no_vcs_ignore=False, global=False)\n",
            "  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/root/.local/share/virtualenv)\n",
            "    added seed packages: pip==23.2.1, setuptools==68.2.0, wheel==0.41.2\n",
            "  activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator\n",
            "Obtaining file:///content/florabert/module (from -r requirements.txt (line 5))\n",
            "  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Collecting beautifulsoup4~=4.9 (from -r requirements.txt (line 1))\n",
            "  Downloading beautifulsoup4-4.12.2-py3-none-any.whl (142 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m143.0/143.0 kB\u001b[0m \u001b[31m285.5 kB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting biopython~=1.78 (from -r requirements.txt (line 2))\n",
            "  Downloading biopython-1.81-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.1/3.1 MB\u001b[0m \u001b[31m10.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting boto3 (from -r requirements.txt (line 3))\n",
            "  Obtaining dependency information for boto3 from https://files.pythonhosted.org/packages/fe/15/fa88dc3bf239fe047fcd9f7c1c38b655f533d453d4d6c3f72ab5b145d44d/boto3-1.28.53-py3-none-any.whl.metadata\n",
            "  Downloading boto3-1.28.53-py3-none-any.whl.metadata (6.7 kB)\n",
            "Collecting click~=7.1 (from -r requirements.txt (line 4))\n",
            "  Downloading click-7.1.2-py2.py3-none-any.whl (82 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m82.8/82.8 kB\u001b[0m \u001b[31m8.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting ipython~=7.18 (from -r requirements.txt (line 6))\n",
            "  Downloading ipython-7.34.0-py3-none-any.whl (793 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m793.8/793.8 kB\u001b[0m \u001b[31m41.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting joblib~=0.16 (from -r requirements.txt (line 7))\n",
            "  Downloading joblib-0.17.0-py3-none-any.whl (301 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m301.5/301.5 kB\u001b[0m \u001b[31m25.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting matplotlib~=3.3 (from -r requirements.txt (line 8))\n",
            "  Obtaining dependency information for matplotlib~=3.3 from https://files.pythonhosted.org/packages/b5/24/aaccf324ce862bb82277e8814d2aebbb2a2c160d04e95aa2b8c9dc3137a9/matplotlib-3.8.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading matplotlib-3.8.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (5.8 kB)\n",
            "Collecting numpy~=1.19 (from -r requirements.txt (line 9))\n",
            "  Obtaining dependency information for numpy~=1.19 from https://files.pythonhosted.org/packages/9b/5a/f265a1ba3641d16b5480a217a6aed08cceef09cd173b568cd5351053472a/numpy-1.26.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading numpy-1.26.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (58 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m58.5/58.5 kB\u001b[0m \u001b[31m5.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting pandas~=1.1 (from -r requirements.txt (line 10))\n",
            "  Downloading pandas-1.5.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m12.1/12.1 MB\u001b[0m \u001b[31m63.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting PyYAML (from -r requirements.txt (line 11))\n",
            "  Obtaining dependency information for PyYAML from https://files.pythonhosted.org/packages/29/61/bf33c6c85c55bc45a29eee3195848ff2d518d84735eb0e2d8cb42e0d285e/PyYAML-6.0.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading PyYAML-6.0.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (2.1 kB)\n",
            "Collecting regex (from -r requirements.txt (line 12))\n",
            "  Obtaining dependency information for regex from https://files.pythonhosted.org/packages/d1/df/460ca6171a8494fcf37af43f52f6fac23e38784bb4a26563f6fa01ef6faf/regex-2023.8.8-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading regex-2023.8.8-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (40 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m40.9/40.9 kB\u001b[0m \u001b[31m3.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting requests (from -r requirements.txt (line 13))\n",
            "  Obtaining dependency information for requests from https://files.pythonhosted.org/packages/70/8e/0e2d847013cb52cd35b38c009bb167a1a26b2ce6cd6965bf26b47bc0bf44/requests-2.31.0-py3-none-any.whl.metadata\n",
            "  Downloading requests-2.31.0-py3-none-any.whl.metadata (4.6 kB)\n",
            "Collecting scikit-learn (from -r requirements.txt (line 14))\n",
            "  Obtaining dependency information for scikit-learn from https://files.pythonhosted.org/packages/7d/af/03d3a7d5719d00486c296ddd876e6f07a681bc4e079cb45348d2f261a748/scikit_learn-1.3.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading scikit_learn-1.3.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (11 kB)\n",
            "Collecting scipy~=1.5 (from -r requirements.txt (line 15))\n",
            "  Obtaining dependency information for scipy~=1.5 from https://files.pythonhosted.org/packages/a8/cc/c36f3439f5d47c3b13833ce6687b43a040cc7638c502ac46b41e2d4f3d6f/scipy-1.11.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading scipy-1.11.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (59 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m59.1/59.1 kB\u001b[0m \u001b[31m5.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting tensorboard (from -r requirements.txt (line 16))\n",
            "  Obtaining dependency information for tensorboard from https://files.pythonhosted.org/packages/bc/a2/ff5f4c299eb37c95299a76015da3f30211468e29d8d6f1d011683279baee/tensorboard-2.14.0-py3-none-any.whl.metadata\n",
            "  Downloading tensorboard-2.14.0-py3-none-any.whl.metadata (1.8 kB)\n",
            "Collecting tokenizers (from -r requirements.txt (line 17))\n",
            "  Obtaining dependency information for tokenizers from https://files.pythonhosted.org/packages/57/bd/45b5ef6b088880779f70acf60027f7043ca5fa1b98f4a4345cf3aea09044/tokenizers-0.14.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading tokenizers-0.14.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (6.7 kB)\n",
            "Collecting torch (from -r requirements.txt (line 18))\n",
            "  Downloading torch-2.0.1-cp310-cp310-manylinux1_x86_64.whl (619.9 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m619.9/619.9 MB\u001b[0m \u001b[31m2.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting tqdm (from -r requirements.txt (line 19))\n",
            "  Obtaining dependency information for tqdm from https://files.pythonhosted.org/packages/00/e5/f12a80907d0884e6dff9c16d0c0114d81b8cd07dc3ae54c5e962cc83037e/tqdm-4.66.1-py3-none-any.whl.metadata\n",
            "  Downloading tqdm-4.66.1-py3-none-any.whl.metadata (57 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m57.6/57.6 kB\u001b[0m \u001b[31m5.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting transformers (from -r requirements.txt (line 20))\n",
            "  Obtaining dependency information for transformers from https://files.pythonhosted.org/packages/1a/06/3817f9bb923437ead9a794f0ac0d03b8b5e0478ab112db4c413dd37c09da/transformers-4.33.2-py3-none-any.whl.metadata\n",
            "  Downloading transformers-4.33.2-py3-none-any.whl.metadata (119 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m119.9/119.9 kB\u001b[0m \u001b[31m11.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting urllib3 (from -r requirements.txt (line 21))\n",
            "  Obtaining dependency information for urllib3 from https://files.pythonhosted.org/packages/37/dc/399e63f5d1d96bb643404ee830657f4dfcf8503f5ba8fa3c6d465d0c57fe/urllib3-2.0.5-py3-none-any.whl.metadata\n",
            "  Downloading urllib3-2.0.5-py3-none-any.whl.metadata (6.6 kB)\n",
            "Collecting wget (from -r requirements.txt (line 22))\n",
            "  Downloading wget-3.2.zip (10 kB)\n",
            "  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Collecting soupsieve>1.2 (from beautifulsoup4~=4.9->-r requirements.txt (line 1))\n",
            "  Obtaining dependency information for soupsieve>1.2 from https://files.pythonhosted.org/packages/4c/f3/038b302fdfbe3be7da016777069f26ceefe11a681055ea1f7817546508e3/soupsieve-2.5-py3-none-any.whl.metadata\n",
            "  Downloading soupsieve-2.5-py3-none-any.whl.metadata (4.7 kB)\n",
            "Collecting botocore<1.32.0,>=1.31.53 (from boto3->-r requirements.txt (line 3))\n",
            "  Obtaining dependency information for botocore<1.32.0,>=1.31.53 from https://files.pythonhosted.org/packages/21/fd/d8591332f5a6b3a94a55dc88a8374518ef7caa24302562558e4eac07ec2e/botocore-1.31.53-py3-none-any.whl.metadata\n",
            "  Downloading botocore-1.31.53-py3-none-any.whl.metadata (6.0 kB)\n",
            "Collecting jmespath<2.0.0,>=0.7.1 (from boto3->-r requirements.txt (line 3))\n",
            "  Downloading jmespath-1.0.1-py3-none-any.whl (20 kB)\n",
            "Collecting s3transfer<0.7.0,>=0.6.0 (from boto3->-r requirements.txt (line 3))\n",
            "  Obtaining dependency information for s3transfer<0.7.0,>=0.6.0 from https://files.pythonhosted.org/packages/d9/17/a3b666f5ef9543cfd3c661d39d1e193abb9649d0cfbbfee3cf3b51d5af02/s3transfer-0.6.2-py3-none-any.whl.metadata\n",
            "  Downloading s3transfer-0.6.2-py3-none-any.whl.metadata (1.8 kB)\n",
            "Requirement already satisfied: setuptools>=18.5 in ./env/lib/python3.10/site-packages (from ipython~=7.18->-r requirements.txt (line 6)) (68.2.0)\n",
            "Collecting jedi>=0.16 (from ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Obtaining dependency information for jedi>=0.16 from https://files.pythonhosted.org/packages/8e/46/7e3ae3aa2dcfcffc5138c6cef5448523218658411c84a2000bf75c8d3ec1/jedi-0.19.0-py2.py3-none-any.whl.metadata\n",
            "  Downloading jedi-0.19.0-py2.py3-none-any.whl.metadata (22 kB)\n",
            "Collecting decorator (from ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Downloading decorator-5.1.1-py3-none-any.whl (9.1 kB)\n",
            "Collecting pickleshare (from ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Downloading pickleshare-0.7.5-py2.py3-none-any.whl (6.9 kB)\n",
            "Collecting traitlets>=4.2 (from ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Obtaining dependency information for traitlets>=4.2 from https://files.pythonhosted.org/packages/fb/00/78472b256929614443c3fa3be31ee60777e5a9e3c6770d8d934154aa2cab/traitlets-5.10.0-py3-none-any.whl.metadata\n",
            "  Downloading traitlets-5.10.0-py3-none-any.whl.metadata (10 kB)\n",
            "Collecting prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0 (from ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Obtaining dependency information for prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0 from https://files.pythonhosted.org/packages/a9/b4/ba77c84edf499877317225d7b7bc047a81f7c2eed9628eeb6bab0ac2e6c9/prompt_toolkit-3.0.39-py3-none-any.whl.metadata\n",
            "  Downloading prompt_toolkit-3.0.39-py3-none-any.whl.metadata (6.4 kB)\n",
            "Collecting pygments (from ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Obtaining dependency information for pygments from https://files.pythonhosted.org/packages/43/88/29adf0b44ba6ac85045e63734ae0997d3c58d8b1a91c914d240828d0d73d/Pygments-2.16.1-py3-none-any.whl.metadata\n",
            "  Downloading Pygments-2.16.1-py3-none-any.whl.metadata (2.5 kB)\n",
            "Collecting backcall (from ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Downloading backcall-0.2.0-py2.py3-none-any.whl (11 kB)\n",
            "Collecting matplotlib-inline (from ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Downloading matplotlib_inline-0.1.6-py3-none-any.whl (9.4 kB)\n",
            "Collecting pexpect>4.3 (from ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Downloading pexpect-4.8.0-py2.py3-none-any.whl (59 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m59.0/59.0 kB\u001b[0m \u001b[31m6.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting contourpy>=1.0.1 (from matplotlib~=3.3->-r requirements.txt (line 8))\n",
            "  Obtaining dependency information for contourpy>=1.0.1 from https://files.pythonhosted.org/packages/f1/6b/e4b0f8708f22dd7c321f87eadbb98708975e115ac6582eb46d1f32197ce6/contourpy-1.1.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading contourpy-1.1.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (5.9 kB)\n",
            "Collecting cycler>=0.10 (from matplotlib~=3.3->-r requirements.txt (line 8))\n",
            "  Downloading cycler-0.11.0-py3-none-any.whl (6.4 kB)\n",
            "Collecting fonttools>=4.22.0 (from matplotlib~=3.3->-r requirements.txt (line 8))\n",
            "  Obtaining dependency information for fonttools>=4.22.0 from https://files.pythonhosted.org/packages/2b/e8/61b8525acf26ec222518bdff127ae502bfa3408981fb5e5493f2b037d7fb/fonttools-4.42.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading fonttools-4.42.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (150 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m151.0/151.0 kB\u001b[0m \u001b[31m14.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting kiwisolver>=1.0.1 (from matplotlib~=3.3->-r requirements.txt (line 8))\n",
            "  Obtaining dependency information for kiwisolver>=1.0.1 from https://files.pythonhosted.org/packages/6f/40/4ab1fdb57fced80ce5903f04ae1aed7c1d5939dda4fd0c0aa526c12fe28a/kiwisolver-1.4.5-cp310-cp310-manylinux_2_12_x86_64.manylinux2010_x86_64.whl.metadata\n",
            "  Downloading kiwisolver-1.4.5-cp310-cp310-manylinux_2_12_x86_64.manylinux2010_x86_64.whl.metadata (6.4 kB)\n",
            "Collecting packaging>=20.0 (from matplotlib~=3.3->-r requirements.txt (line 8))\n",
            "  Downloading packaging-23.1-py3-none-any.whl (48 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m48.9/48.9 kB\u001b[0m \u001b[31m5.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting pillow>=6.2.0 (from matplotlib~=3.3->-r requirements.txt (line 8))\n",
            "  Obtaining dependency information for pillow>=6.2.0 from https://files.pythonhosted.org/packages/7a/07/e896b096a77375e78e02ce222ae4fd6014928cd76c691d312060a1645dfa/Pillow-10.0.1-cp310-cp310-manylinux_2_28_x86_64.whl.metadata\n",
            "  Downloading Pillow-10.0.1-cp310-cp310-manylinux_2_28_x86_64.whl.metadata (9.5 kB)\n",
            "Collecting pyparsing>=2.3.1 (from matplotlib~=3.3->-r requirements.txt (line 8))\n",
            "  Obtaining dependency information for pyparsing>=2.3.1 from https://files.pythonhosted.org/packages/39/92/8486ede85fcc088f1b3dba4ce92dd29d126fd96b0008ea213167940a2475/pyparsing-3.1.1-py3-none-any.whl.metadata\n",
            "  Downloading pyparsing-3.1.1-py3-none-any.whl.metadata (5.1 kB)\n",
            "Collecting python-dateutil>=2.7 (from matplotlib~=3.3->-r requirements.txt (line 8))\n",
            "  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m247.7/247.7 kB\u001b[0m \u001b[31m20.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting pytz>=2020.1 (from pandas~=1.1->-r requirements.txt (line 10))\n",
            "  Obtaining dependency information for pytz>=2020.1 from https://files.pythonhosted.org/packages/32/4d/aaf7eff5deb402fd9a24a1449a8119f00d74ae9c2efa79f8ef9994261fc2/pytz-2023.3.post1-py2.py3-none-any.whl.metadata\n",
            "  Downloading pytz-2023.3.post1-py2.py3-none-any.whl.metadata (22 kB)\n",
            "Collecting charset-normalizer<4,>=2 (from requests->-r requirements.txt (line 13))\n",
            "  Obtaining dependency information for charset-normalizer<4,>=2 from https://files.pythonhosted.org/packages/a4/65/057bf29660aae6ade0816457f8db4e749e5c0bfa2366eb5f67db9912fa4c/charset_normalizer-3.2.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading charset_normalizer-3.2.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (31 kB)\n",
            "Collecting idna<4,>=2.5 (from requests->-r requirements.txt (line 13))\n",
            "  Downloading idna-3.4-py3-none-any.whl (61 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m61.5/61.5 kB\u001b[0m \u001b[31m6.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting certifi>=2017.4.17 (from requests->-r requirements.txt (line 13))\n",
            "  Obtaining dependency information for certifi>=2017.4.17 from https://files.pythonhosted.org/packages/4c/dd/2234eab22353ffc7d94e8d13177aaa050113286e93e7b40eae01fbf7c3d9/certifi-2023.7.22-py3-none-any.whl.metadata\n",
            "  Downloading certifi-2023.7.22-py3-none-any.whl.metadata (2.2 kB)\n",
            "INFO: pip is looking at multiple versions of scikit-learn to determine which version is compatible with other requirements. This could take a while.\n",
            "Collecting scikit-learn (from -r requirements.txt (line 14))\n",
            "  Obtaining dependency information for scikit-learn from https://files.pythonhosted.org/packages/5c/e9/ee572691a3fb05555bcde41826faad29ae4bc1fb07982e7f53d54a176879/scikit_learn-1.3.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading scikit_learn-1.3.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (11 kB)\n",
            "  Downloading scikit_learn-1.2.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (9.6 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m9.6/9.6 MB\u001b[0m \u001b[31m100.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Downloading scikit_learn-1.2.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (9.6 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m9.6/9.6 MB\u001b[0m \u001b[31m78.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Downloading scikit_learn-1.2.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (9.5 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m9.5/9.5 MB\u001b[0m \u001b[31m69.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Downloading scikit_learn-1.1.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (30.5 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m30.5/30.5 MB\u001b[0m \u001b[31m17.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Downloading scikit_learn-1.1.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (30.5 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m30.5/30.5 MB\u001b[0m \u001b[31m39.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Downloading scikit_learn-1.1.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (30.4 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m30.4/30.4 MB\u001b[0m \u001b[31m17.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hINFO: pip is still looking at multiple versions of scikit-learn to determine which version is compatible with other requirements. This could take a while.\n",
            "  Downloading scikit_learn-1.1.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (30.7 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m30.7/30.7 MB\u001b[0m \u001b[31m48.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Downloading scikit_learn-1.0.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (26.5 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m26.5/26.5 MB\u001b[0m \u001b[31m65.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting threadpoolctl>=2.0.0 (from scikit-learn->-r requirements.txt (line 14))\n",
            "  Obtaining dependency information for threadpoolctl>=2.0.0 from https://files.pythonhosted.org/packages/81/12/fd4dea011af9d69e1cad05c75f3f7202cdcbeac9b712eea58ca779a72865/threadpoolctl-3.2.0-py3-none-any.whl.metadata\n",
            "  Downloading threadpoolctl-3.2.0-py3-none-any.whl.metadata (10.0 kB)\n",
            "Collecting absl-py>=0.4 (from tensorboard->-r requirements.txt (line 16))\n",
            "  Obtaining dependency information for absl-py>=0.4 from https://files.pythonhosted.org/packages/01/e4/dc0a1dcc4e74e08d7abedab278c795eef54a224363bb18f5692f416d834f/absl_py-2.0.0-py3-none-any.whl.metadata\n",
            "  Downloading absl_py-2.0.0-py3-none-any.whl.metadata (2.3 kB)\n",
            "Collecting grpcio>=1.48.2 (from tensorboard->-r requirements.txt (line 16))\n",
            "  Obtaining dependency information for grpcio>=1.48.2 from https://files.pythonhosted.org/packages/28/94/63bef715d0afea96662c66075d72249bf9b55b018b91b6937045bdc1470c/grpcio-1.58.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading grpcio-1.58.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (4.0 kB)\n",
            "Collecting google-auth<3,>=1.6.3 (from tensorboard->-r requirements.txt (line 16))\n",
            "  Obtaining dependency information for google-auth<3,>=1.6.3 from https://files.pythonhosted.org/packages/9d/44/5a992cb9d7bf8aaae73bc5adaf721ad08731c9d00c1c17999a8691404b0c/google_auth-2.23.0-py2.py3-none-any.whl.metadata\n",
            "  Downloading google_auth-2.23.0-py2.py3-none-any.whl.metadata (4.2 kB)\n",
            "Collecting google-auth-oauthlib<1.1,>=0.5 (from tensorboard->-r requirements.txt (line 16))\n",
            "  Downloading google_auth_oauthlib-1.0.0-py2.py3-none-any.whl (18 kB)\n",
            "Collecting markdown>=2.6.8 (from tensorboard->-r requirements.txt (line 16))\n",
            "  Obtaining dependency information for markdown>=2.6.8 from https://files.pythonhosted.org/packages/1a/b5/228c1cdcfe138f1a8e01ab1b54284c8b83735476cb22b6ba251656ed13ad/Markdown-3.4.4-py3-none-any.whl.metadata\n",
            "  Downloading Markdown-3.4.4-py3-none-any.whl.metadata (6.9 kB)\n",
            "Collecting protobuf>=3.19.6 (from tensorboard->-r requirements.txt (line 16))\n",
            "  Obtaining dependency information for protobuf>=3.19.6 from https://files.pythonhosted.org/packages/bb/c3/6a06208ecf0934ecaf509b51c52a6cf688586f54ae81ac65c56124571494/protobuf-4.24.3-cp37-abi3-manylinux2014_x86_64.whl.metadata\n",
            "  Downloading protobuf-4.24.3-cp37-abi3-manylinux2014_x86_64.whl.metadata (540 bytes)\n",
            "Collecting tensorboard-data-server<0.8.0,>=0.7.0 (from tensorboard->-r requirements.txt (line 16))\n",
            "  Obtaining dependency information for tensorboard-data-server<0.8.0,>=0.7.0 from https://files.pythonhosted.org/packages/02/52/fb9e51fba47951aabd7a6b25e41d73eae94208ccf62d886168096941a781/tensorboard_data_server-0.7.1-py3-none-manylinux2014_x86_64.whl.metadata\n",
            "  Downloading tensorboard_data_server-0.7.1-py3-none-manylinux2014_x86_64.whl.metadata (1.1 kB)\n",
            "Collecting werkzeug>=1.0.1 (from tensorboard->-r requirements.txt (line 16))\n",
            "  Obtaining dependency information for werkzeug>=1.0.1 from https://files.pythonhosted.org/packages/9b/59/a7c32e3d8d0e546a206e0552a2c04444544f15c1da4a01df8938d20c6ffc/werkzeug-2.3.7-py3-none-any.whl.metadata\n",
            "  Downloading werkzeug-2.3.7-py3-none-any.whl.metadata (4.1 kB)\n",
            "Requirement already satisfied: wheel>=0.26 in ./env/lib/python3.10/site-packages (from tensorboard->-r requirements.txt (line 16)) (0.41.2)\n",
            "Collecting huggingface_hub<0.17,>=0.16.4 (from tokenizers->-r requirements.txt (line 17))\n",
            "  Obtaining dependency information for huggingface_hub<0.17,>=0.16.4 from https://files.pythonhosted.org/packages/7f/c4/adcbe9a696c135578cabcbdd7331332daad4d49b7c43688bc2d36b3a47d2/huggingface_hub-0.16.4-py3-none-any.whl.metadata\n",
            "  Downloading huggingface_hub-0.16.4-py3-none-any.whl.metadata (12 kB)\n",
            "Collecting filelock (from torch->-r requirements.txt (line 18))\n",
            "  Obtaining dependency information for filelock from https://files.pythonhosted.org/packages/5e/5d/97afbafd9d584ff1b45fcb354a479a3609bd97f912f8f1f6c563cb1fae21/filelock-3.12.4-py3-none-any.whl.metadata\n",
            "  Downloading filelock-3.12.4-py3-none-any.whl.metadata (2.8 kB)\n",
            "Collecting typing-extensions (from torch->-r requirements.txt (line 18))\n",
            "  Obtaining dependency information for typing-extensions from https://files.pythonhosted.org/packages/24/21/7d397a4b7934ff4028987914ac1044d3b7d52712f30e2ac7a2ae5bc86dd0/typing_extensions-4.8.0-py3-none-any.whl.metadata\n",
            "  Downloading typing_extensions-4.8.0-py3-none-any.whl.metadata (3.0 kB)\n",
            "Collecting sympy (from torch->-r requirements.txt (line 18))\n",
            "  Downloading sympy-1.12-py3-none-any.whl (5.7 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m5.7/5.7 MB\u001b[0m \u001b[31m66.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting networkx (from torch->-r requirements.txt (line 18))\n",
            "  Downloading networkx-3.1-py3-none-any.whl (2.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.1/2.1 MB\u001b[0m \u001b[31m54.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting jinja2 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading Jinja2-3.1.2-py3-none-any.whl (133 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m133.1/133.1 kB\u001b[0m \u001b[31m12.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-cuda-nvrtc-cu11==11.7.99 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading nvidia_cuda_nvrtc_cu11-11.7.99-2-py3-none-manylinux1_x86_64.whl (21.0 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m21.0/21.0 MB\u001b[0m \u001b[31m55.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-cuda-runtime-cu11==11.7.99 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading nvidia_cuda_runtime_cu11-11.7.99-py3-none-manylinux1_x86_64.whl (849 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m849.3/849.3 kB\u001b[0m \u001b[31m45.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-cuda-cupti-cu11==11.7.101 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading nvidia_cuda_cupti_cu11-11.7.101-py3-none-manylinux1_x86_64.whl (11.8 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m11.8/11.8 MB\u001b[0m \u001b[31m69.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-cudnn-cu11==8.5.0.96 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading nvidia_cudnn_cu11-8.5.0.96-2-py3-none-manylinux1_x86_64.whl (557.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m557.1/557.1 MB\u001b[0m \u001b[31m1.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-cublas-cu11==11.10.3.66 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading nvidia_cublas_cu11-11.10.3.66-py3-none-manylinux1_x86_64.whl (317.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m317.1/317.1 MB\u001b[0m \u001b[31m2.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-cufft-cu11==10.9.0.58 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading nvidia_cufft_cu11-10.9.0.58-py3-none-manylinux1_x86_64.whl (168.4 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m168.4/168.4 MB\u001b[0m \u001b[31m7.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-curand-cu11==10.2.10.91 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading nvidia_curand_cu11-10.2.10.91-py3-none-manylinux1_x86_64.whl (54.6 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m54.6/54.6 MB\u001b[0m \u001b[31m10.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-cusolver-cu11==11.4.0.1 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading nvidia_cusolver_cu11-11.4.0.1-2-py3-none-manylinux1_x86_64.whl (102.6 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m102.6/102.6 MB\u001b[0m \u001b[31m6.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-cusparse-cu11==11.7.4.91 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading nvidia_cusparse_cu11-11.7.4.91-py3-none-manylinux1_x86_64.whl (173.2 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m173.2/173.2 MB\u001b[0m \u001b[31m2.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-nccl-cu11==2.14.3 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading nvidia_nccl_cu11-2.14.3-py3-none-manylinux1_x86_64.whl (177.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m177.1/177.1 MB\u001b[0m \u001b[31m2.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting nvidia-nvtx-cu11==11.7.91 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading nvidia_nvtx_cu11-11.7.91-py3-none-manylinux1_x86_64.whl (98 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m98.6/98.6 kB\u001b[0m \u001b[31m10.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting triton==2.0.0 (from torch->-r requirements.txt (line 18))\n",
            "  Downloading triton-2.0.0-1-cp310-cp310-manylinux2014_x86_64.manylinux_2_17_x86_64.whl (63.3 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m63.3/63.3 MB\u001b[0m \u001b[31m10.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting cmake (from triton==2.0.0->torch->-r requirements.txt (line 18))\n",
            "  Obtaining dependency information for cmake from https://files.pythonhosted.org/packages/de/94/cba4b3ddc0d4555967cce8fd6e9fbced98a6bf62857db71c2400a7b6e183/cmake-3.27.5-py2.py3-none-manylinux2014_x86_64.manylinux_2_17_x86_64.whl.metadata\n",
            "  Downloading cmake-3.27.5-py2.py3-none-manylinux2014_x86_64.manylinux_2_17_x86_64.whl.metadata (6.7 kB)\n",
            "Collecting lit (from triton==2.0.0->torch->-r requirements.txt (line 18))\n",
            "  Downloading lit-16.0.6.tar.gz (153 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m153.7/153.7 kB\u001b[0m \u001b[31m15.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Installing build dependencies ... \u001b[?25l\u001b[?25hdone\n",
            "  Getting requirements to build wheel ... \u001b[?25l\u001b[?25hdone\n",
            "  Installing backend dependencies ... \u001b[?25l\u001b[?25hdone\n",
            "  Preparing metadata (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n",
            "Collecting tokenizers (from -r requirements.txt (line 17))\n",
            "  Downloading tokenizers-0.13.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (7.8 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m7.8/7.8 MB\u001b[0m \u001b[31m84.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting safetensors>=0.3.1 (from transformers->-r requirements.txt (line 20))\n",
            "  Obtaining dependency information for safetensors>=0.3.1 from https://files.pythonhosted.org/packages/6c/f0/c17bbdb1e5f9dab29d44cade445135789f75f8f08ea2728d04493ea8412b/safetensors-0.3.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading safetensors-0.3.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (4.7 kB)\n",
            "Collecting urllib3 (from -r requirements.txt (line 21))\n",
            "  Obtaining dependency information for urllib3 from https://files.pythonhosted.org/packages/c5/05/c214b32d21c0b465506f95c4f28ccbcba15022e000b043b72b3df7728471/urllib3-1.26.16-py2.py3-none-any.whl.metadata\n",
            "  Downloading urllib3-1.26.16-py2.py3-none-any.whl.metadata (48 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m48.4/48.4 kB\u001b[0m \u001b[31m5.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting cachetools<6.0,>=2.0.0 (from google-auth<3,>=1.6.3->tensorboard->-r requirements.txt (line 16))\n",
            "  Obtaining dependency information for cachetools<6.0,>=2.0.0 from https://files.pythonhosted.org/packages/a9/c9/c8a7710f2cedcb1db9224fdd4d8307c9e48cbddc46c18b515fefc0f1abbe/cachetools-5.3.1-py3-none-any.whl.metadata\n",
            "  Downloading cachetools-5.3.1-py3-none-any.whl.metadata (5.2 kB)\n",
            "Collecting pyasn1-modules>=0.2.1 (from google-auth<3,>=1.6.3->tensorboard->-r requirements.txt (line 16))\n",
            "  Downloading pyasn1_modules-0.3.0-py2.py3-none-any.whl (181 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m181.3/181.3 kB\u001b[0m \u001b[31m20.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting rsa<5,>=3.1.4 (from google-auth<3,>=1.6.3->tensorboard->-r requirements.txt (line 16))\n",
            "  Downloading rsa-4.9-py3-none-any.whl (34 kB)\n",
            "Collecting requests-oauthlib>=0.7.0 (from google-auth-oauthlib<1.1,>=0.5->tensorboard->-r requirements.txt (line 16))\n",
            "  Downloading requests_oauthlib-1.3.1-py2.py3-none-any.whl (23 kB)\n",
            "Collecting fsspec (from huggingface_hub<0.17,>=0.16.4->tokenizers->-r requirements.txt (line 17))\n",
            "  Obtaining dependency information for fsspec from https://files.pythonhosted.org/packages/fe/d3/e1aa96437d944fbb9cc95d0316e25583886e9cd9e6adc07baad943524eda/fsspec-2023.9.2-py3-none-any.whl.metadata\n",
            "  Downloading fsspec-2023.9.2-py3-none-any.whl.metadata (6.7 kB)\n",
            "Collecting parso<0.9.0,>=0.8.3 (from jedi>=0.16->ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Downloading parso-0.8.3-py2.py3-none-any.whl (100 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m100.8/100.8 kB\u001b[0m \u001b[31m11.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting ptyprocess>=0.5 (from pexpect>4.3->ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Downloading ptyprocess-0.7.0-py2.py3-none-any.whl (13 kB)\n",
            "Collecting wcwidth (from prompt-toolkit!=3.0.0,!=3.0.1,<3.1.0,>=2.0.0->ipython~=7.18->-r requirements.txt (line 6))\n",
            "  Downloading wcwidth-0.2.6-py2.py3-none-any.whl (29 kB)\n",
            "Collecting six>=1.5 (from python-dateutil>=2.7->matplotlib~=3.3->-r requirements.txt (line 8))\n",
            "  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)\n",
            "Collecting MarkupSafe>=2.1.1 (from werkzeug>=1.0.1->tensorboard->-r requirements.txt (line 16))\n",
            "  Obtaining dependency information for MarkupSafe>=2.1.1 from https://files.pythonhosted.org/packages/12/b3/d9ed2c0971e1435b8a62354b18d3060b66c8cb1d368399ec0b9baa7c0ee5/MarkupSafe-2.1.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata\n",
            "  Downloading MarkupSafe-2.1.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (3.0 kB)\n",
            "Collecting mpmath>=0.19 (from sympy->torch->-r requirements.txt (line 18))\n",
            "  Downloading mpmath-1.3.0-py3-none-any.whl (536 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m536.2/536.2 kB\u001b[0m \u001b[31m32.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting pyasn1<0.6.0,>=0.4.6 (from pyasn1-modules>=0.2.1->google-auth<3,>=1.6.3->tensorboard->-r requirements.txt (line 16))\n",
            "  Downloading pyasn1-0.5.0-py2.py3-none-any.whl (83 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m83.9/83.9 kB\u001b[0m \u001b[31m8.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting oauthlib>=3.0.0 (from requests-oauthlib>=0.7.0->google-auth-oauthlib<1.1,>=0.5->tensorboard->-r requirements.txt (line 16))\n",
            "  Downloading oauthlib-3.2.2-py3-none-any.whl (151 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m151.7/151.7 kB\u001b[0m \u001b[31m14.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading boto3-1.28.53-py3-none-any.whl (135 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m135.8/135.8 kB\u001b[0m \u001b[31m12.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading matplotlib-3.8.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (11.6 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m11.6/11.6 MB\u001b[0m \u001b[31m62.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading numpy-1.26.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (18.2 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m18.2/18.2 MB\u001b[0m \u001b[31m19.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading PyYAML-6.0.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (705 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m705.5/705.5 kB\u001b[0m \u001b[31m39.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading regex-2023.8.8-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (771 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m771.9/771.9 kB\u001b[0m \u001b[31m42.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading requests-2.31.0-py3-none-any.whl (62 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m62.6/62.6 kB\u001b[0m \u001b[31m5.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading scipy-1.11.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (36.3 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m36.3/36.3 MB\u001b[0m \u001b[31m9.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading tensorboard-2.14.0-py3-none-any.whl (5.5 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m5.5/5.5 MB\u001b[0m \u001b[31m65.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading tqdm-4.66.1-py3-none-any.whl (78 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m78.3/78.3 kB\u001b[0m \u001b[31m7.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading transformers-4.33.2-py3-none-any.whl (7.6 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m7.6/7.6 MB\u001b[0m \u001b[31m67.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading absl_py-2.0.0-py3-none-any.whl (130 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m130.2/130.2 kB\u001b[0m \u001b[31m10.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading botocore-1.31.53-py3-none-any.whl (11.2 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m11.2/11.2 MB\u001b[0m \u001b[31m99.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading urllib3-1.26.16-py2.py3-none-any.whl (143 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m143.1/143.1 kB\u001b[0m \u001b[31m14.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading certifi-2023.7.22-py3-none-any.whl (158 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m158.3/158.3 kB\u001b[0m \u001b[31m15.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading charset_normalizer-3.2.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (201 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m201.8/201.8 kB\u001b[0m \u001b[31m20.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading contourpy-1.1.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (301 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m301.7/301.7 kB\u001b[0m \u001b[31m27.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading fonttools-4.42.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.5 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m4.5/4.5 MB\u001b[0m \u001b[31m79.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading google_auth-2.23.0-py2.py3-none-any.whl (181 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m181.4/181.4 kB\u001b[0m \u001b[31m15.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading grpcio-1.58.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (5.3 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m5.3/5.3 MB\u001b[0m \u001b[31m67.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading huggingface_hub-0.16.4-py3-none-any.whl (268 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m268.8/268.8 kB\u001b[0m \u001b[31m23.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading jedi-0.19.0-py2.py3-none-any.whl (1.6 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.6/1.6 MB\u001b[0m \u001b[31m47.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading kiwisolver-1.4.5-cp310-cp310-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (1.6 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.6/1.6 MB\u001b[0m \u001b[31m52.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading Markdown-3.4.4-py3-none-any.whl (94 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m94.2/94.2 kB\u001b[0m \u001b[31m9.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading Pillow-10.0.1-cp310-cp310-manylinux_2_28_x86_64.whl (3.6 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.6/3.6 MB\u001b[0m \u001b[31m75.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading prompt_toolkit-3.0.39-py3-none-any.whl (385 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m385.2/385.2 kB\u001b[0m \u001b[31m31.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading protobuf-4.24.3-cp37-abi3-manylinux2014_x86_64.whl (311 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m311.6/311.6 kB\u001b[0m \u001b[31m28.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading pyparsing-3.1.1-py3-none-any.whl (103 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m103.1/103.1 kB\u001b[0m \u001b[31m10.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading pytz-2023.3.post1-py2.py3-none-any.whl (502 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m502.5/502.5 kB\u001b[0m \u001b[31m35.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading s3transfer-0.6.2-py3-none-any.whl (79 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m79.8/79.8 kB\u001b[0m \u001b[31m8.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading safetensors-0.3.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.3 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.3/1.3 MB\u001b[0m \u001b[31m54.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading soupsieve-2.5-py3-none-any.whl (36 kB)\n",
            "Downloading tensorboard_data_server-0.7.1-py3-none-manylinux2014_x86_64.whl (6.6 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m6.6/6.6 MB\u001b[0m \u001b[31m82.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading threadpoolctl-3.2.0-py3-none-any.whl (15 kB)\n",
            "Downloading traitlets-5.10.0-py3-none-any.whl (120 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m120.4/120.4 kB\u001b[0m \u001b[31m11.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading typing_extensions-4.8.0-py3-none-any.whl (31 kB)\n",
            "Downloading werkzeug-2.3.7-py3-none-any.whl (242 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m242.2/242.2 kB\u001b[0m \u001b[31m1.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading filelock-3.12.4-py3-none-any.whl (11 kB)\n",
            "Downloading Pygments-2.16.1-py3-none-any.whl (1.2 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.2/1.2 MB\u001b[0m \u001b[31m58.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading cachetools-5.3.1-py3-none-any.whl (9.3 kB)\n",
            "Downloading MarkupSafe-2.1.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (25 kB)\n",
            "Downloading cmake-3.27.5-py2.py3-none-manylinux2014_x86_64.manylinux_2_17_x86_64.whl (26.1 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m26.1/26.1 MB\u001b[0m \u001b[31m18.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading fsspec-2023.9.2-py3-none-any.whl (173 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m173.4/173.4 kB\u001b[0m \u001b[31m5.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hBuilding wheels for collected packages: wget, lit\n",
            "  Building wheel for wget (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for wget: filename=wget-3.2-py3-none-any.whl size=9655 sha256=e1924719ceba329a41edbf48aac3dc2b4e31a1f7b64ac4fc2fcf958d85096c43\n",
            "  Stored in directory: /root/.cache/pip/wheels/8b/f1/7f/5c94f0a7a505ca1c81cd1d9208ae2064675d97582078e6c769\n",
            "  Building wheel for lit (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for lit: filename=lit-16.0.6-py3-none-any.whl size=93584 sha256=d373762dc075bdcc16bf562977384f44c3c2637a34d2605e9d239d946c0e4343\n",
            "  Stored in directory: /root/.cache/pip/wheels/14/f9/07/bb2308587bc2f57158f905a2325f6a89a2befa7437b2d7e137\n",
            "Successfully built wget lit\n",
            "Installing collected packages: wget, wcwidth, tokenizers, safetensors, pytz, ptyprocess, pickleshare, mpmath, lit, florabert, cmake, backcall, urllib3, typing-extensions, traitlets, tqdm, threadpoolctl, tensorboard-data-server, sympy, soupsieve, six, regex, PyYAML, pyparsing, pygments, pyasn1, protobuf, prompt-toolkit, pillow, pexpect, parso, packaging, oauthlib, nvidia-nvtx-cu11, nvidia-nccl-cu11, nvidia-cusparse-cu11, nvidia-curand-cu11, nvidia-cufft-cu11, nvidia-cuda-runtime-cu11, nvidia-cuda-nvrtc-cu11, nvidia-cuda-cupti-cu11, nvidia-cublas-cu11, numpy, networkx, MarkupSafe, markdown, kiwisolver, joblib, jmespath, idna, grpcio, fsspec, fonttools, filelock, decorator, cycler, click, charset-normalizer, certifi, cachetools, absl-py, werkzeug, scipy, rsa, requests, python-dateutil, pyasn1-modules, nvidia-cusolver-cu11, nvidia-cudnn-cu11, matplotlib-inline, jinja2, jedi, contourpy, biopython, beautifulsoup4, scikit-learn, requests-oauthlib, pandas, matplotlib, ipython, huggingface_hub, google-auth, botocore, transformers, s3transfer, google-auth-oauthlib, tensorboard, boto3, triton, torch\n",
            "  Running setup.py develop for florabert\n",
            "Successfully installed MarkupSafe-2.1.3 PyYAML-6.0.1 absl-py-2.0.0 backcall-0.2.0 beautifulsoup4-4.12.2 biopython-1.81 boto3-1.28.53 botocore-1.31.53 cachetools-5.3.1 certifi-2023.7.22 charset-normalizer-3.2.0 click-7.1.2 cmake-3.27.5 contourpy-1.1.1 cycler-0.11.0 decorator-5.1.1 filelock-3.12.4 florabert-0.1 fonttools-4.42.1 fsspec-2023.9.2 google-auth-2.23.0 google-auth-oauthlib-1.0.0 grpcio-1.58.0 huggingface_hub-0.16.4 idna-3.4 ipython-7.34.0 jedi-0.19.0 jinja2-3.1.2 jmespath-1.0.1 joblib-0.17.0 kiwisolver-1.4.5 lit-16.0.6 markdown-3.4.4 matplotlib-3.8.0 matplotlib-inline-0.1.6 mpmath-1.3.0 networkx-3.1 numpy-1.26.0 nvidia-cublas-cu11-11.10.3.66 nvidia-cuda-cupti-cu11-11.7.101 nvidia-cuda-nvrtc-cu11-11.7.99 nvidia-cuda-runtime-cu11-11.7.99 nvidia-cudnn-cu11-8.5.0.96 nvidia-cufft-cu11-10.9.0.58 nvidia-curand-cu11-10.2.10.91 nvidia-cusolver-cu11-11.4.0.1 nvidia-cusparse-cu11-11.7.4.91 nvidia-nccl-cu11-2.14.3 nvidia-nvtx-cu11-11.7.91 oauthlib-3.2.2 packaging-23.1 pandas-1.5.3 parso-0.8.3 pexpect-4.8.0 pickleshare-0.7.5 pillow-10.0.1 prompt-toolkit-3.0.39 protobuf-4.24.3 ptyprocess-0.7.0 pyasn1-0.5.0 pyasn1-modules-0.3.0 pygments-2.16.1 pyparsing-3.1.1 python-dateutil-2.8.2 pytz-2023.3.post1 regex-2023.8.8 requests-2.31.0 requests-oauthlib-1.3.1 rsa-4.9 s3transfer-0.6.2 safetensors-0.3.3 scikit-learn-1.0.2 scipy-1.11.2 six-1.16.0 soupsieve-2.5 sympy-1.12 tensorboard-2.14.0 tensorboard-data-server-0.7.1 threadpoolctl-3.2.0 tokenizers-0.13.3 torch-2.0.1 tqdm-4.66.1 traitlets-5.10.0 transformers-4.33.2 triton-2.0.0 typing-extensions-4.8.0 urllib3-1.26.16 wcwidth-0.2.6 werkzeug-2.3.7 wget-3.2\n",
            "curl -L https://github.com/samtools/samtools/releases/download/1.11/samtools-1.11.tar.bz2 \\\n",
            "\t| tar -xj \\\n",
            "\t&& cd samtools-1.11 \\\n",
            "\t&& ./configure --prefix=/content/florabert --without-curses \\\n",
            "\t&& make \\\n",
            "\t&& make install \\\n",
            "\t&& cd /content/florabert \\\n",
            "\t&& ln -s /content/florabert/samtools-1.11/samtools /content/florabert/libs/samtools\n",
            "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
            "                                 Dload  Upload   Total   Spent    Left  Speed\n",
            "  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\n",
            "100 4645k  100 4645k    0     0  1189k      0  0:00:03  0:00:03 --:--:-- 1612k\n",
            "checking for gawk... no\n",
            "checking for mawk... mawk\n",
            "checking for gcc... gcc\n",
            "checking whether the C compiler works... yes\n",
            "checking for C compiler default output file name... a.out\n",
            "checking for suffix of executables... \n",
            "checking whether we are cross compiling... no\n",
            "checking for suffix of object files... o\n",
            "checking whether we are using the GNU C compiler... yes\n",
            "checking whether gcc accepts -g... yes\n",
            "checking for gcc option to accept ISO C89... none needed\n",
            "checking for grep that handles long lines and -e... /usr/bin/grep\n",
            "checking for C compiler warning flags... -Wall\n",
            "checking for special C compiler options needed for large files... no\n",
            "checking for _FILE_OFFSET_BITS value needed for large files... no\n",
            "checking location of HTSlib source tree... htslib-1.11\n",
            "checking for zlib.h... yes\n",
            "checking for inflate in -lz... yes\n",
            "checking for library containing regcomp... none required\n",
            "configure: creating ./config.status\n",
            "config.status: creating config.mk\n",
            "config.status: creating config.h\n",
            "=== configuring in htslib-1.11 (/content/florabert/samtools-1.11/htslib-1.11)\n",
            "configure: running /bin/bash ./configure --disable-option-checking '--prefix=/content/florabert'  '--without-curses' --cache-file=/dev/null --srcdir=.\n",
            "checking for gcc... gcc\n",
            "checking whether the C compiler works... yes\n",
            "checking for C compiler default output file name... a.out\n",
            "checking for suffix of executables... \n",
            "checking whether we are cross compiling... no\n",
            "checking for suffix of object files... o\n",
            "checking whether we are using the GNU C compiler... yes\n",
            "checking whether gcc accepts -g... yes\n",
            "checking for gcc option to accept ISO C89... none needed\n",
            "checking for ranlib... ranlib\n",
            "checking for grep that handles long lines and -e... /usr/bin/grep\n",
            "checking for C compiler warning flags... -Wall\n",
            "checking for pkg-config... /usr/bin/pkg-config\n",
            "checking pkg-config is at least version 0.9.0... yes\n",
            "checking for special C compiler options needed for large files... no\n",
            "checking for _FILE_OFFSET_BITS value needed for large files... no\n",
            "checking shared library type for unknown-Linux... plain .so\n",
            "checking whether the compiler accepts -fvisibility=hidden... yes\n",
            "checking how to run the C preprocessor... gcc -E\n",
            "checking for egrep... /usr/bin/grep -E\n",
            "checking for ANSI C header files... yes\n",
            "checking for sys/types.h... yes\n",
            "checking for sys/stat.h... yes\n",
            "checking for stdlib.h... yes\n",
            "checking for string.h... yes\n",
            "checking for memory.h... yes\n",
            "checking for strings.h... yes\n",
            "checking for inttypes.h... yes\n",
            "checking for stdint.h... yes\n",
            "checking for unistd.h... yes\n",
            "checking for stdlib.h... (cached) yes\n",
            "checking for unistd.h... (cached) yes\n",
            "checking for sys/param.h... yes\n",
            "checking for getpagesize... yes\n",
            "checking for working mmap... yes\n",
            "checking for gmtime_r... yes\n",
            "checking for fsync... yes\n",
            "checking for drand48... yes\n",
            "checking for srand48_deterministic... no\n",
            "checking whether fdatasync is declared... yes\n",
            "checking for fdatasync... yes\n",
            "checking for library containing log... -lm\n",
            "checking for zlib.h... yes\n",
            "checking for inflate in -lz... yes\n",
            "checking for library containing recv... none required\n",
            "checking for bzlib.h... yes\n",
            "checking for BZ2_bzBuffToBuffCompress in -lbz2... yes\n",
            "checking for lzma.h... yes\n",
            "checking for lzma_easy_buffer_encode in -llzma... yes\n",
            "checking for libdeflate.h... yes\n",
            "checking for libdeflate_deflate_compress in -ldeflate... yes\n",
            "checking for curl_easy_pause in -lcurl... yes\n",
            "checking for CCHmac... no\n",
            "checking for library containing HMAC... -lcrypto\n",
            "checking whether PTHREAD_MUTEX_RECURSIVE is declared... yes\n",
            "configure: creating ./config.status\n",
            "config.status: creating config.mk\n",
            "config.status: creating htslib.pc.tmp\n",
            "config.status: creating config.h\n",
            "make[1]: Entering directory '/content/florabert/samtools-1.11'\n",
            "cd htslib-1.11 && make htslib_static.mk\n",
            "make[2]: Entering directory '/content/florabert/samtools-1.11/htslib-1.11'\n",
            "sed -n '/^static_libs=/s/[^=]*=/HTSLIB_static_LIBS = /p;/^static_ldflags=/s/[^=]*=/HTSLIB_static_LDFLAGS = /p' htslib.pc.tmp > htslib_static.mk\n",
            "make[2]: Leaving directory '/content/florabert/samtools-1.11/htslib-1.11'\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_index.o bam_index.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_plcmd.o bam_plcmd.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o sam_view.o sam_view.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_fastq.o bam_fastq.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_cat.o bam_cat.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_md.o bam_md.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_reheader.o bam_reheader.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_sort.o bam_sort.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_rmdup.o bam_rmdup.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_rmdupse.o bam_rmdupse.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_mate.o bam_mate.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_stat.o bam_stat.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_color.o bam_color.c\n",
            "echo '#define SAMTOOLS_VERSION \"1.11\"' > version.h\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bamtk.o bamtk.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam2bcf.o bam2bcf.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam2bcf_indel.o bam2bcf_indel.c\n",
            "\u001b[01m\u001b[Kbam2bcf_indel.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbcf_call_gap_prep\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kbam2bcf_indel.c:411:26:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kargument 1 range [18446744071562067968, 18446744073709551615] exceeds maximum object size 9223372036854775807 [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Walloc-size-larger-than=\u0007-Walloc-size-larger-than=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  411 |                     qq = \u001b[01;35m\u001b[Kcalloc(qend - qbeg, 1)\u001b[m\u001b[K;\n",
            "      |                          \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In file included from \u001b[01m\u001b[Khtslib-1.11/htslib/kstring.h:30\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Khtslib-1.11/htslib/hts.h:37\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Kbam2bcf_indel.c:31\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/stdlib.h:543:14:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin a call to allocation function ‘\u001b[01m\u001b[Kcalloc\u001b[m\u001b[K’ declared here\n",
            "  543 | extern void *\u001b[01;36m\u001b[Kcalloc\u001b[m\u001b[K (size_t __nmemb, size_t __size)\n",
            "      |              \u001b[01;36m\u001b[K^~~~~~\u001b[m\u001b[K\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o sample.o sample.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o cut_target.o cut_target.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o phase.o phase.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam2depth.o bam2depth.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o coverage.o coverage.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o padding.o padding.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bedcov.o bedcov.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bamshuf.o bamshuf.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o faidx.o faidx.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o dict.o dict.c\n",
            "In file included from \u001b[01m\u001b[Kdict.c:32\u001b[m\u001b[K:\n",
            "In function ‘\u001b[01m\u001b[Kkseq_read\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kwrite_dict\u001b[m\u001b[K’ at \u001b[01m\u001b[Kdict.c:73:17\u001b[m\u001b[K,\n",
            "    inlined from ‘\u001b[01m\u001b[Kdict_main\u001b[m\u001b[K’ at \u001b[01m\u001b[Kdict.c:176:5\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:212:40:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kwriting 1 byte into a region of size 0 [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wstringop-overflow=\u0007-Wstringop-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  212 |                 \u001b[01;35m\u001b[Kseq->seq.s[seq->seq.l] = 0\u001b[m\u001b[K;     /* null terminated string */ \\\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:237:9:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[K__KSEQ_READ\u001b[m\u001b[K’\n",
            "  237 |         \u001b[01;36m\u001b[K__KSEQ_READ\u001b[m\u001b[K(SCOPE)\n",
            "      |         \u001b[01;36m\u001b[K^~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:239:35:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[KKSEQ_INIT2\u001b[m\u001b[K’\n",
            "  239 | #define KSEQ_INIT(type_t, __read) \u001b[01;36m\u001b[KKSEQ_INIT2\u001b[m\u001b[K(static, type_t, __read)\n",
            "      |                                   \u001b[01;36m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kdict.c:35:1:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[KKSEQ_INIT\u001b[m\u001b[K’\n",
            "   35 | \u001b[01;36m\u001b[KKSEQ_INIT\u001b[m\u001b[K(gzFile, gzread)\n",
            "      | \u001b[01;36m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kdict.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kdict_main\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:210:45:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kat offset -2 into destination object of size 0 allocated by ‘\u001b[01m\u001b[Krealloc\u001b[m\u001b[K’\n",
            "  210 |                         seq->seq.s = (char*)\u001b[01;36m\u001b[Krealloc(seq->seq.s, seq->seq.m)\u001b[m\u001b[K; \\\n",
            "      |                                             \u001b[01;36m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:237:9:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[K__KSEQ_READ\u001b[m\u001b[K’\n",
            "  237 |         \u001b[01;36m\u001b[K__KSEQ_READ\u001b[m\u001b[K(SCOPE)\n",
            "      |         \u001b[01;36m\u001b[K^~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:239:35:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[KKSEQ_INIT2\u001b[m\u001b[K’\n",
            "  239 | #define KSEQ_INIT(type_t, __read) \u001b[01;36m\u001b[KKSEQ_INIT2\u001b[m\u001b[K(static, type_t, __read)\n",
            "      |                                   \u001b[01;36m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kdict.c:35:1:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[KKSEQ_INIT\u001b[m\u001b[K’\n",
            "   35 | \u001b[01;36m\u001b[KKSEQ_INIT\u001b[m\u001b[K(gzFile, gzread)\n",
            "      | \u001b[01;36m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o stats.o stats.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o stats_isize.o stats_isize.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_flags.o bam_flags.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_split.o bam_split.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_tview.o bam_tview.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_tview_curses.o bam_tview_curses.c\n",
            "\u001b[01m\u001b[Kbam_tview_curses.c:48:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K#warning \"No curses library is available; tview with curses is disabled.\" [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wcpp\u0007-Wcpp\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   48 | #\u001b[01;35m\u001b[Kwarning\u001b[m\u001b[K \"No curses library is available; tview with curses is disabled.\"\n",
            "      |  \u001b[01;35m\u001b[K^~~~~~~\u001b[m\u001b[K\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_tview_html.o bam_tview_html.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_lpileup.o bam_lpileup.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_quickcheck.o bam_quickcheck.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_addrprg.o bam_addrprg.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_markdup.o bam_markdup.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o tmp_file.o tmp_file.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_ampliconclip.o bam_ampliconclip.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o amplicon_stats.o amplicon_stats.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o lz4/lz4.o lz4/lz4.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_aux.o bam_aux.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam.o bam.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o sam.o sam.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bam_plbuf.o bam_plbuf.c\n",
            "ar -csru libbam.a bam_aux.o bam.o sam.o bam_plbuf.o\n",
            "ar: `u' modifier ignored since `D' is the default (see `U')\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o sam_opts.o sam_opts.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o sam_utils.o sam_utils.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o bedidx.o bedidx.c\n",
            "ar -rcs libst.a sam_opts.o sam_utils.o bedidx.o\n",
            "cd htslib-1.11 && make hts-object-files\n",
            "make[2]: Entering directory '/content/florabert/samtools-1.11/htslib-1.11'\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o kfunc.o kfunc.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o knetfile.o knetfile.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o kstring.o kstring.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o bcf_sr_sort.o bcf_sr_sort.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o bgzf.o bgzf.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o errmod.o errmod.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o faidx.o faidx.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o header.o header.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o hfile.o hfile.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o hfile_net.o hfile_net.c\n",
            "echo '#define HTS_VERSION_TEXT \"1.11\"' > version.h\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o hts.o hts.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o hts_os.o hts_os.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o md5.o md5.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o multipart.o multipart.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o probaln.o probaln.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o realn.o realn.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o regidx.o regidx.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o region.o region.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o sam.o sam.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o synced_bcf_reader.o synced_bcf_reader.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o vcf_sweep.o vcf_sweep.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o tbx.o tbx.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o textutils.o textutils.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o thread_pool.o thread_pool.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o vcf.o vcf.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o vcfutils.o vcfutils.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/cram_codecs.o cram/cram_codecs.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/cram_decode.o cram/cram_decode.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/cram_encode.o cram/cram_encode.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/cram_external.o cram/cram_external.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/cram_index.o cram/cram_index.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/cram_io.o cram/cram_io.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/cram_samtools.o cram/cram_samtools.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/cram_stats.o cram/cram_stats.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/mFILE.o cram/mFILE.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/open_trace_file.o cram/open_trace_file.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/pooled_alloc.o cram/pooled_alloc.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/rANS_static.o cram/rANS_static.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o cram/string_alloc.o cram/string_alloc.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o hfile_libcurl.o hfile_libcurl.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o hfile_gcs.o hfile_gcs.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o hfile_s3.o hfile_s3.c\n",
            "gcc -Wall -g -O2 -fvisibility=hidden -I.  -c -o hfile_s3_write.o hfile_s3_write.c\n",
            "touch hts-object-files\n",
            "make[2]: Leaving directory '/content/florabert/samtools-1.11/htslib-1.11'\n",
            "cd htslib-1.11 && make lib-static\n",
            "make[2]: Entering directory '/content/florabert/samtools-1.11/htslib-1.11'\n",
            "ar -rc libhts.a kfunc.o knetfile.o kstring.o bcf_sr_sort.o bgzf.o errmod.o faidx.o header.o hfile.o hfile_net.o hts.o hts_os.o md5.o multipart.o probaln.o realn.o regidx.o region.o sam.o synced_bcf_reader.o vcf_sweep.o tbx.o textutils.o thread_pool.o vcf.o vcfutils.o cram/cram_codecs.o cram/cram_decode.o cram/cram_encode.o cram/cram_external.o cram/cram_index.o cram/cram_io.o cram/cram_samtools.o cram/cram_stats.o cram/mFILE.o cram/open_trace_file.o cram/pooled_alloc.o cram/rANS_static.o cram/string_alloc.o  hfile_libcurl.o hfile_gcs.o hfile_s3.o hfile_s3_write.o\n",
            "ranlib libhts.a\n",
            "make[2]: Leaving directory '/content/florabert/samtools-1.11/htslib-1.11'\n",
            "gcc  -L./lz4  -o samtools bam_index.o bam_plcmd.o sam_view.o bam_fastq.o bam_cat.o bam_md.o bam_reheader.o bam_sort.o bam_rmdup.o bam_rmdupse.o bam_mate.o bam_stat.o bam_color.o bamtk.o bam2bcf.o bam2bcf_indel.o sample.o cut_target.o phase.o bam2depth.o coverage.o padding.o bedcov.o bamshuf.o faidx.o dict.o stats.o stats_isize.o bam_flags.o bam_split.o bam_tview.o bam_tview_curses.o bam_tview_html.o bam_lpileup.o bam_quickcheck.o bam_addrprg.o bam_markdup.o tmp_file.o bam_ampliconclip.o amplicon_stats.o ./lz4/lz4.o libbam.a libst.a htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto  -lm -lz  -lpthread\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o misc/ace2sam.o misc/ace2sam.c\n",
            "gcc  -L./lz4  -o misc/ace2sam misc/ace2sam.o htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz \n",
            "gcc -Wall -g -O2 -DMAQ_LONGREADS -I. -Ihtslib-1.11 -I./lz4  -c -o misc/maq2sam-long.o misc/maq2sam.c\n",
            "gcc  -o misc/maq2sam-long misc/maq2sam-long.o -lz \n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o misc/maq2sam-short.o misc/maq2sam.c\n",
            "gcc  -o misc/maq2sam-short misc/maq2sam-short.o -lz \n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o misc/md5fa.o misc/md5fa.c\n",
            "In file included from \u001b[01m\u001b[Kmisc/md5fa.c:29\u001b[m\u001b[K:\n",
            "In function ‘\u001b[01m\u001b[Kkseq_read\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kmd5_one\u001b[m\u001b[K’ at \u001b[01m\u001b[Kmisc/md5fa.c:59:17\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:212:40:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kwriting 1 byte into a region of size 0 [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wstringop-overflow=\u0007-Wstringop-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  212 |                 \u001b[01;35m\u001b[Kseq->seq.s[seq->seq.l] = 0\u001b[m\u001b[K;     /* null terminated string */ \\\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:237:9:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[K__KSEQ_READ\u001b[m\u001b[K’\n",
            "  237 |         \u001b[01;36m\u001b[K__KSEQ_READ\u001b[m\u001b[K(SCOPE)\n",
            "      |         \u001b[01;36m\u001b[K^~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:239:35:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[KKSEQ_INIT2\u001b[m\u001b[K’\n",
            "  239 | #define KSEQ_INIT(type_t, __read) \u001b[01;36m\u001b[KKSEQ_INIT2\u001b[m\u001b[K(static, type_t, __read)\n",
            "      |                                   \u001b[01;36m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmisc/md5fa.c:32:1:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[KKSEQ_INIT\u001b[m\u001b[K’\n",
            "   32 | \u001b[01;36m\u001b[KKSEQ_INIT\u001b[m\u001b[K(gzFile, gzread)\n",
            "      | \u001b[01;36m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmisc/md5fa.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kmd5_one\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:210:45:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kat offset -2 into destination object of size 0 allocated by ‘\u001b[01m\u001b[Krealloc\u001b[m\u001b[K’\n",
            "  210 |                         seq->seq.s = (char*)\u001b[01;36m\u001b[Krealloc(seq->seq.s, seq->seq.m)\u001b[m\u001b[K; \\\n",
            "      |                                             \u001b[01;36m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:237:9:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[K__KSEQ_READ\u001b[m\u001b[K’\n",
            "  237 |         \u001b[01;36m\u001b[K__KSEQ_READ\u001b[m\u001b[K(SCOPE)\n",
            "      |         \u001b[01;36m\u001b[K^~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Khtslib-1.11/htslib/kseq.h:239:35:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[KKSEQ_INIT2\u001b[m\u001b[K’\n",
            "  239 | #define KSEQ_INIT(type_t, __read) \u001b[01;36m\u001b[KKSEQ_INIT2\u001b[m\u001b[K(static, type_t, __read)\n",
            "      |                                   \u001b[01;36m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kmisc/md5fa.c:32:1:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin expansion of macro ‘\u001b[01m\u001b[KKSEQ_INIT\u001b[m\u001b[K’\n",
            "   32 | \u001b[01;36m\u001b[KKSEQ_INIT\u001b[m\u001b[K(gzFile, gzread)\n",
            "      | \u001b[01;36m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "gcc  -L./lz4  -o misc/md5fa misc/md5fa.o htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz \n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o misc/md5sum-lite.o misc/md5sum-lite.c\n",
            "gcc  -L./lz4  -o misc/md5sum-lite misc/md5sum-lite.o htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz \n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o misc/wgsim.o misc/wgsim.c\n",
            "gcc  -L./lz4  -o misc/wgsim misc/wgsim.o -lm htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz \n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o test/merge/test_bam_translate.o test/merge/test_bam_translate.c\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o test/test.o test/test.c\n",
            "gcc  -L./lz4  -o test/merge/test_bam_translate test/merge/test_bam_translate.o test/test.o libst.a htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz  -lpthread\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o test/merge/test_rtrans_build.o test/merge/test_rtrans_build.c\n",
            "gcc  -L./lz4  -o test/merge/test_rtrans_build test/merge/test_rtrans_build.o test/test.o libst.a htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz  -lpthread\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o test/merge/test_trans_tbl_init.o test/merge/test_trans_tbl_init.c\n",
            "gcc  -L./lz4  -o test/merge/test_trans_tbl_init test/merge/test_trans_tbl_init.o test/test.o libst.a htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz  -lpthread\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o test/split/test_count_rg.o test/split/test_count_rg.c\n",
            "gcc  -L./lz4  -o test/split/test_count_rg test/split/test_count_rg.o test/test.o libst.a htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz  -lpthread\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o test/split/test_expand_format_string.o test/split/test_expand_format_string.c\n",
            "gcc  -L./lz4  -o test/split/test_expand_format_string test/split/test_expand_format_string.o test/test.o libst.a htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz  -lpthread\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o test/split/test_filter_header_rg.o test/split/test_filter_header_rg.c\n",
            "gcc  -L./lz4  -o test/split/test_filter_header_rg test/split/test_filter_header_rg.o test/test.o libst.a htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz  -lpthread\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o test/split/test_parse_args.o test/split/test_parse_args.c\n",
            "gcc  -L./lz4  -o test/split/test_parse_args test/split/test_parse_args.o test/test.o libst.a htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz  -lpthread\n",
            "gcc -Wall -g -O2 -I. -Ihtslib-1.11 -I./lz4  -c -o test/vcf-miniview.o test/vcf-miniview.c\n",
            "gcc  -L./lz4  -o test/vcf-miniview test/vcf-miniview.o htslib-1.11/libhts.a -lpthread -lz -lm -lbz2 -llzma -ldeflate -lcurl -lcrypto -lz  -lpthread\n",
            "make[1]: Leaving directory '/content/florabert/samtools-1.11'\n",
            "make[1]: Entering directory '/content/florabert/samtools-1.11'\n",
            "mkdir -p -m 755 /content/florabert/bin /content/florabert/bin /content/florabert/share/man/man1\n",
            "install -p samtools /content/florabert/bin\n",
            "install -p misc/ace2sam misc/maq2sam-long misc/maq2sam-short misc/md5fa misc/md5sum-lite misc/wgsim /content/florabert/bin\n",
            "install -p misc/blast2sam.pl misc/bowtie2sam.pl misc/export2sam.pl misc/interpolate_sam.pl misc/novo2sam.pl misc/plot-bamstats misc/psl2sam.pl misc/sam2vcf.pl misc/samtools.pl misc/seq_cache_populate.pl misc/soap2sam.pl misc/wgsim_eval.pl misc/zoom2sam.pl misc/plot-ampliconstats /content/florabert/bin\n",
            "install -p -m 644 doc/samtools*.1 misc/wgsim.1 /content/florabert/share/man/man1\n",
            "make[1]: Leaving directory '/content/florabert/samtools-1.11'\n",
            "curl -L https://github.com/arq5x/bedtools2/releases/download/v2.29.1/bedtools-2.29.1.tar.gz \\\n",
            "\t| tar zxv \\\n",
            "\t&& cd bedtools2 \\\n",
            "\t&& make \\\n",
            "\t&& cd /content/florabert \\\n",
            "\t&& ln -s /content/florabert/bedtools2/bin/bedtools /content/florabert/libs/bedtools\n",
            "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
            "                                 Dload  Upload   Total   Spent    Left  Speed\n",
            "  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\n",
            "bedtools2/\n",
            "bedtools2/obj/\n",
            "bedtools2/tutorial/\n",
            "bedtools2/LICENSE\n",
            "bedtools2/test/\n",
            "bedtools2/bin/\n",
            "bedtools2/Makefile\n",
            "bedtools2/docs/\n",
            "bedtools2/README.md\n",
            "bedtools2/genomes/\n",
            "bedtools2/.gitignore\n",
            "bedtools2/scripts/\n",
            "bedtools2/.github/\n",
            "bedtools2/data/\n",
            "bedtools2/src/\n",
            "bedtools2/src/spacingFile/\n",
            "bedtools2/src/sortBed/\n",
            "bedtools2/src/split/\n",
            "bedtools2/src/bedToBam/\n",
            "bedtools2/src/complementFile/\n",
            "bedtools2/src/expand/\n",
            "bedtools2/src/fisher/\n",
            "bedtools2/src/linksBed/\n",
            "bedtools2/src/pairToPair/\n",
            "bedtools2/src/bedtools.cpp\n",
            "bedtools2/src/mapFile/\n",
            "bedtools2/src/slopBed/\n",
            "bedtools2/src/annotateBed/\n",
            "bedtools2/src/randomBed/\n",
            "bedtools2/src/getOverlap/\n",
            "bedtools2/src/reldist/\n",
            "bedtools2/src/nucBed/\n",
            "bedtools2/src/bedToIgv/\n",
            "bedtools2/src/regressTest/\n",
            "bedtools2/src/maskFastaFromBed/\n",
            "bedtools2/src/genomeCoverageBed/\n",
            "bedtools2/src/utils/\n",
            "bedtools2/src/windowBed/\n",
            "bedtools2/src/jaccard/\n",
            "bedtools2/src/groupBy/\n",
            "bedtools2/src/bamToFastq/\n",
            "bedtools2/src/fastaFromBed/\n",
            "bedtools2/src/nekSandbox1/\n",
            "bedtools2/src/unionBedGraphs/\n",
            "bedtools2/src/bed12ToBed6/\n",
            "bedtools2/src/subtractFile/\n",
            "bedtools2/src/bedpeToBam/\n",
            "bedtools2/src/windowMaker/\n",
            "bedtools2/src/coverageFile/\n",
            "bedtools2/src/summaryFile/\n",
            "bedtools2/src/multiIntersectBed/\n",
            "bedtools2/src/intersectFile/\n",
            "bedtools2/src/closestFile/\n",
            "bedtools2/src/shuffleBed/\n",
            "bedtools2/src/flankBed/\n",
            "bedtools2/src/tagBam/\n",
            "bedtools2/src/multiBamCov/\n",
            "bedtools2/src/sampleFile/\n",
            "bedtools2/src/clusterBed/\n",
            "bedtools2/src/shiftBed/\n",
            "bedtools2/src/mergeFile/\n",
            "bedtools2/src/bamToBed/\n",
            "bedtools2/src/pairToBed/\n",
            "bedtools2/src/pairToBed/pairToBed.h\n",
            "bedtools2/src/pairToBed/pairToBedMain.cpp\n",
            "bedtools2/src/pairToBed/Makefile.frag\n",
            "bedtools2/src/pairToBed/pairToBed.cpp\n",
            "bedtools2/src/bamToBed/bamToBed.cpp\n",
            "bedtools2/src/bamToBed/Makefile.frag\n",
            "bedtools2/src/mergeFile/mergeFile.cpp\n",
            "bedtools2/src/mergeFile/Makefile.frag\n",
            "bedtools2/src/mergeFile/mergeHelp.cpp\n",
            "bedtools2/src/mergeFile/mergeFile.h\n",
            "bedtools2/src/shiftBed/shiftBed.cpp\n",
            "bedtools2/src/shiftBed/shiftBedMain.cpp\n",
            "bedtools2/src/shiftBed/Makefile.frag\n",
            "bedtools2/src/shiftBed/shiftBed.h\n",
            "bedtools2/src/clusterBed/clusterBed.h\n",
            "bedtools2/src/clusterBed/clusterBed.cpp\n",
            "bedtools2/src/clusterBed/clusterMain.cpp\n",
            "bedtools2/src/clusterBed/Makefile.frag\n",
            "bedtools2/src/sampleFile/sampleFile.cpp\n",
            "bedtools2/src/sampleFile/sampleFile.h\n",
            "bedtools2/src/sampleFile/Makefile.frag\n",
            "bedtools2/src/sampleFile/sampleHelp.cpp\n",
            "bedtools2/src/multiBamCov/multiBamCov.cpp\n",
            "bedtools2/src/multiBamCov/Makefile.frag\n",
            "bedtools2/src/multiBamCov/multiBamCov.h\n",
            "bedtools2/src/multiBamCov/multiBamCovMain.cpp\n",
            "bedtools2/src/tagBam/tagBam.cpp\n",
            "bedtools2/src/tagBam/tagBamMain.cpp\n",
            "bedtools2/src/tagBam/tagBam.h\n",
            "bedtools2/src/tagBam/Makefile.frag\n",
            "bedtools2/src/flankBed/flankBed.cpp\n",
            "bedtools2/src/flankBed/flankBedMain.cpp\n",
            "bedtools2/src/flankBed/test.fa.fai\n",
            "bedtools2/src/flankBed/test.gff\n",
            "bedtools2/src/flankBed/Makefile.frag\n",
            "bedtools2/src/flankBed/test.fa\n",
            "bedtools2/src/flankBed/flankBed.h\n",
            "bedtools2/src/shuffleBed/shuffleBed.h\n",
            "bedtools2/src/shuffleBed/shuffleBed.cpp\n",
            "bedtools2/src/shuffleBed/Makefile.frag\n",
            "bedtools2/src/shuffleBed/shuffleBedMain.cpp\n",
            "bedtools2/src/closestFile/closestHelp.cpp\n",
            "bedtools2/src/closestFile/closestFile.h\n",
            "bedtools2/src/closestFile/closestFile.cpp\n",
            "bedtools2/src/closestFile/Makefile.frag\n",
            "bedtools2/src/intersectFile/intersectHelp.cpp\n",
            "bedtools2/src/intersectFile/Makefile.frag\n",
            "bedtools2/src/intersectFile/intersectFile.cpp\n",
            "bedtools2/src/intersectFile/intersectFile.h\n",
            "bedtools2/src/multiIntersectBed/multiIntersectBed.cpp\n",
            "bedtools2/src/multiIntersectBed/multiIntersectBed.h\n",
            "bedtools2/src/multiIntersectBed/Makefile.frag\n",
            "bedtools2/src/multiIntersectBed/multiIntersectBedMain.cpp\n",
            "bedtools2/src/summaryFile/summaryHelp.cpp\n",
            "bedtools2/src/summaryFile/Makefile.frag\n",
            "bedtools2/src/summaryFile/summaryFile.h\n",
            "bedtools2/src/summaryFile/summaryFile.cpp\n",
            "bedtools2/src/coverageFile/coverageFile.cpp\n",
            "bedtools2/src/coverageFile/coverageFile.h\n",
            "bedtools2/src/coverageFile/Makefile.frag\n",
            "bedtools2/src/coverageFile/coverageHelp.cpp\n",
            "bedtools2/src/windowMaker/windowMakerMain.cpp\n",
            "bedtools2/src/windowMaker/windowMaker.h\n",
            "bedtools2/src/windowMaker/windowMaker.cpp\n",
            "bedtools2/src/windowMaker/Makefile.frag\n",
            "bedtools2/src/bedpeToBam/notes.txt\n",
            "bedtools2/src/bedpeToBam/bedpeToBam.cpp\n",
            "bedtools2/src/bedpeToBam/Makefile.frag\n",
            "bedtools2/src/subtractFile/subtractHelp.cpp\n",
            "bedtools2/src/subtractFile/subtractFile.h\n",
            "bedtools2/src/subtractFile/Makefile.frag\n",
            "bedtools2/src/subtractFile/subtractFile.cpp\n",
            "bedtools2/src/bed12ToBed6/bed12ToBed6.cpp\n",
            "bedtools2/src/bed12ToBed6/Makefile.frag\n",
            "bedtools2/src/unionBedGraphs/unionBedGraphs.cpp\n",
            "bedtools2/src/unionBedGraphs/unionBedGraphsMain.cpp\n",
            "bedtools2/src/unionBedGraphs/Makefile.frag\n",
            "bedtools2/src/unionBedGraphs/unionBedGraphs.h\n",
            "bedtools2/src/nekSandbox1/FormatGuess.h\n",
            "bedtools2/src/nekSandbox1/Makefile\n",
            "bedtools2/src/nekSandbox1/nekSandboxMain.cpp\n",
            "bedtools2/src/fastaFromBed/fastaFromBedMain.cpp\n",
            "bedtools2/src/fastaFromBed/fastaFromBed.cpp\n",
            "bedtools2/src/fastaFromBed/fastaFromBed.h\n",
            "bedtools2/src/fastaFromBed/Makefile.frag\n",
            "bedtools2/src/bamToFastq/bamToFastq.h\n",
            "bedtools2/src/bamToFastq/bamToFastq.cpp\n",
            "bedtools2/src/bamToFastq/bamToFastqMain.cpp\n",
            "bedtools2/src/bamToFastq/Makefile.frag\n",
            "bedtools2/src/groupBy/groupBy.cpp\n",
            "bedtools2/src/groupBy/groupBy.h\n",
            "bedtools2/src/groupBy/Makefile.frag\n",
            "bedtools2/src/groupBy/groupByHelp.cpp\n",
            "bedtools2/src/jaccard/jaccard.cpp\n",
            "bedtools2/src/jaccard/Makefile.frag\n",
            "bedtools2/src/jaccard/jaccardHelp.cpp\n",
            "bedtools2/src/jaccard/jaccard.h\n",
            "bedtools2/src/windowBed/windowBed.cpp\n",
            "bedtools2/src/windowBed/windowBed.h\n",
            "bedtools2/src/windowBed/windowMain.cpp\n",
            "bedtools2/src/windowBed/Makefile.frag\n",
            "bedtools2/src/utils/tabFile/\n",
            "bedtools2/src/utils/htslib/\n",
            "bedtools2/src/utils/NewChromsweep/\n",
            "bedtools2/src/utils/RecordOutputMgr/\n",
            "bedtools2/src/utils/bedFile/\n",
            "bedtools2/src/utils/Contexts/\n",
            "bedtools2/src/utils/BamTools/\n",
            "bedtools2/src/utils/BlockedIntervals/\n",
            "bedtools2/src/utils/general/\n",
            "bedtools2/src/utils/driver/\n",
            "bedtools2/src/utils/Point/\n",
            "bedtools2/src/utils/bedGraphFile/\n",
            "bedtools2/src/utils/sequenceUtilities/\n",
            "bedtools2/src/utils/Fasta/\n",
            "bedtools2/src/utils/gzstream/\n",
            "bedtools2/src/utils/fileType/\n",
            "bedtools2/src/utils/KeyListOps/\n",
            "bedtools2/src/utils/BamTools-Ancillary/\n",
            "bedtools2/src/utils/GenomeFile/\n",
            "bedtools2/src/utils/version/\n",
            "bedtools2/src/utils/ToolBase/\n",
            "bedtools2/src/utils/lineFileUtilities/\n",
            "bedtools2/src/utils/bedFilePE/\n",
            "bedtools2/src/utils/chromsweep/\n",
            "bedtools2/src/utils/FileRecordTools/\n",
            "bedtools2/src/utils/BinTree/\n",
            "bedtools2/src/utils/VectorOps/\n",
            "bedtools2/src/utils/stringUtilities/\n",
            "bedtools2/src/utils/stringUtilities/stringUtilities.h\n",
            "bedtools2/src/utils/VectorOps/VectorOps.cpp\n",
            "bedtools2/src/utils/VectorOps/Makefile.frag\n",
            "bedtools2/src/utils/VectorOps/VectorOps.h\n",
            "bedtools2/src/utils/BinTree/BinTree.cpp\n",
            "bedtools2/src/utils/BinTree/BinTree.h\n",
            "bedtools2/src/utils/BinTree/Makefile.frag\n",
            "bedtools2/src/utils/FileRecordTools/Records/\n",
            "bedtools2/src/utils/FileRecordTools/FileRecordMergeMgr.h\n",
            "bedtools2/src/utils/FileRecordTools/FileRecordMgr.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Makefile.frag\n",
            "bedtools2/src/utils/FileRecordTools/FileRecordMgr.h\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/\n",
            "bedtools2/src/utils/FileRecordTools/FileRecordMergeMgr.cpp\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/InputStreamMgr.h\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/BamFileReader.h\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/BufferedStreamMgr.cpp\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/SingleLineDelimTextFileReader.cpp\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/SingleLineDelimTransferBuffer.cpp\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/SingleLineDelimTransferBuffer.h\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/InputStreamMgr.cpp\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/FileReader.cpp\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/BamFileReader.cpp\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/Makefile.frag\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/SingleLineDelimTextFileReader.h\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/FileReader.h\n",
            "bedtools2/src/utils/FileRecordTools/FileReaders/BufferedStreamMgr.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/BedGraphInterval.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/RecordMgr.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/RecordList.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/Bed12Interval.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/GffRecord.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/Bed4Interval.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/BedGraphInterval.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/RecordList.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/BamRecord.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/RecordKeyVector.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/PlusFields.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/StrandQueue.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/Bed5Interval.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/Record.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/GffRecord.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/Bed4Interval.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/VcfRecord.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/Bed3Interval.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/RecordKeyList.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/VcfRecord.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/EmptyRecord.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/Bed3Interval.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/RecordKeyVector.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/Bed6Interval.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/BlockMgr.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/RecordMgr.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/Bed6Interval.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/GffPlusRecord.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/GffPlusRecord.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/BedPlusInterval.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/Makefile.frag\n",
            "bedtools2/src/utils/FileRecordTools/Records/StrandQueue.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/Record.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/Bed5Interval.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/NoPosPlusRecord.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/Bed12Interval.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/BlockMgr.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/RecordKeyList.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/PlusFields.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/BamRecord.h\n",
            "bedtools2/src/utils/FileRecordTools/Records/NoPosPlusRecord.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/EmptyRecord.cpp\n",
            "bedtools2/src/utils/FileRecordTools/Records/BedPlusInterval.h\n",
            "bedtools2/src/utils/chromsweep/chromsweep.cpp\n",
            "bedtools2/src/utils/chromsweep/Makefile.frag\n",
            "bedtools2/src/utils/chromsweep/chromsweep.h\n",
            "bedtools2/src/utils/bedFilePE/bedFilePE.cpp\n",
            "bedtools2/src/utils/bedFilePE/Makefile.frag\n",
            "bedtools2/src/utils/bedFilePE/bedFilePE.h\n",
            "bedtools2/src/utils/lineFileUtilities/lineFileUtilities.h\n",
            "bedtools2/src/utils/ToolBase/ToolBase.cpp\n",
            "bedtools2/src/utils/ToolBase/Makefile.frag\n",
            "bedtools2/src/utils/ToolBase/ToolBase.h\n",
            "bedtools2/src/utils/version/version.h\n",
            "bedtools2/src/utils/version/version.cpp\n",
            "bedtools2/src/utils/version/version_release.txt\n",
            "bedtools2/src/utils/version/Makefile.frag\n",
            "bedtools2/src/utils/GenomeFile/NewGenomeFile.cpp\n",
            "bedtools2/src/utils/GenomeFile/NewGenomeFile.h\n",
            "bedtools2/src/utils/GenomeFile/GenomeFile.cpp\n",
            "bedtools2/src/utils/GenomeFile/Makefile.frag\n",
            "bedtools2/src/utils/GenomeFile/GenomeFile.h\n",
            "bedtools2/src/utils/BamTools-Ancillary/BamAncillary.h\n",
            "bedtools2/src/utils/BamTools-Ancillary/BamAncillary.cpp\n",
            "bedtools2/src/utils/BamTools-Ancillary/Makefile.frag\n",
            "bedtools2/src/utils/KeyListOps/KeyListOps.h\n",
            "bedtools2/src/utils/KeyListOps/KeyListOps.cpp\n",
            "bedtools2/src/utils/KeyListOps/KeyListOpsMethods.cpp\n",
            "bedtools2/src/utils/KeyListOps/Makefile.frag\n",
            "bedtools2/src/utils/KeyListOps/KeyListOpsMethods.h\n",
            "bedtools2/src/utils/fileType/fileType.cpp\n",
            "bedtools2/src/utils/fileType/FileRecordTypeChecker.h\n",
            "bedtools2/src/utils/fileType/FileRecordTypeChecker.cpp\n",
            "bedtools2/src/utils/fileType/Makefile.frag\n",
            "bedtools2/src/utils/fileType/fileType.h\n",
            "bedtools2/src/utils/gzstream/gzstream.C\n",
            "bedtools2/src/utils/gzstream/README\n",
            "bedtools2/src/utils/gzstream/COPYING.LIB\n",
            "bedtools2/src/utils/gzstream/version\n",
            "bedtools2/src/utils/gzstream/Makefile.frag\n",
            "bedtools2/src/utils/gzstream/gzstream.h\n",
            "bedtools2/src/utils/Fasta/Fasta.h\n",
            "bedtools2/src/utils/Fasta/split.cpp\n",
            "bedtools2/src/utils/Fasta/Fasta.cpp\n",
            "bedtools2/src/utils/Fasta/LargeFileSupport.h\n",
            "bedtools2/src/utils/Fasta/Makefile.frag\n",
            "bedtools2/src/utils/Fasta/split.h\n",
            "bedtools2/src/utils/sequenceUtilities/sequenceUtils.h\n",
            "bedtools2/src/utils/sequenceUtilities/Makefile.frag\n",
            "bedtools2/src/utils/sequenceUtilities/sequenceUtils.cpp\n",
            "bedtools2/src/utils/bedGraphFile/bedGraphFile.h\n",
            "bedtools2/src/utils/bedGraphFile/Makefile.frag\n",
            "bedtools2/src/utils/bedGraphFile/bedGraphFile.cpp\n",
            "bedtools2/src/utils/Point/Point.h\n",
            "bedtools2/src/utils/driver/BedtoolsDriver.h\n",
            "bedtools2/src/utils/driver/BedtoolsDriver.cpp\n",
            "bedtools2/src/utils/driver/Makefile.frag\n",
            "bedtools2/src/utils/general/QuickString.cpp\n",
            "bedtools2/src/utils/general/ParseTools.cpp\n",
            "bedtools2/src/utils/general/CommonHelp.h\n",
            "bedtools2/src/utils/general/FreeList.h\n",
            "bedtools2/src/utils/general/CompressionTools.cpp\n",
            "bedtools2/src/utils/general/BedtoolsTypes.h\n",
            "bedtools2/src/utils/general/ErrorMsg.h\n",
            "bedtools2/src/utils/general/PushBackStreamBuf.h\n",
            "bedtools2/src/utils/general/Tokenizer.h\n",
            "bedtools2/src/utils/general/ParseTools.h\n",
            "bedtools2/src/utils/general/QuickString.h\n",
            "bedtools2/src/utils/general/Tokenizer.cpp\n",
            "bedtools2/src/utils/general/ErrorMsg.cpp\n",
            "bedtools2/src/utils/general/CommonHelp.cpp\n",
            "bedtools2/src/utils/general/Makefile.frag\n",
            "bedtools2/src/utils/general/BTlist.h\n",
            "bedtools2/src/utils/general/CompressionTools.h\n",
            "bedtools2/src/utils/general/PushBackStreamBuf.cpp\n",
            "bedtools2/src/utils/general/InflateStreamBuf.h\n",
            "bedtools2/src/utils/BlockedIntervals/BlockedIntervals.cpp\n",
            "bedtools2/src/utils/BlockedIntervals/BlockedIntervals.h\n",
            "bedtools2/src/utils/BlockedIntervals/Makefile.frag\n",
            "bedtools2/src/utils/BamTools/mapping/\n",
            "bedtools2/src/utils/BamTools/include/\n",
            "bedtools2/src/utils/BamTools/.gitignore\n",
            "bedtools2/src/utils/BamTools/Makefile.frag\n",
            "bedtools2/src/utils/BamTools/include/BamAncillary.h\n",
            "bedtools2/src/utils/BamTools/include/SamHeader.hpp\n",
            "bedtools2/src/utils/BamTools/include/BamReader.hpp\n",
            "bedtools2/src/utils/BamTools/include/api/\n",
            "bedtools2/src/utils/BamTools/include/BamWriter.hpp\n",
            "bedtools2/src/utils/BamTools/include/BamAlignment.mapping.hpp\n",
            "bedtools2/src/utils/BamTools/include/BamAlignment.hpp\n",
            "bedtools2/src/utils/BamTools/include/api/BamWriter.h\n",
            "bedtools2/src/utils/BamTools/include/api/BamMultiReader.h\n",
            "bedtools2/src/utils/BamTools/include/api/SamConstants.h\n",
            "bedtools2/src/utils/BamTools/include/api/BamAlignment.h\n",
            "bedtools2/src/utils/BamTools/include/api/BamReader.h\n",
            "bedtools2/src/utils/BamTools/include/api/BamAux.h\n",
            "bedtools2/src/utils/BamTools/mapping/BamAlignment.map\n",
            "bedtools2/src/utils/BamTools/mapping/BamAlignment.py\n",
            "bedtools2/src/utils/Contexts/ContextBase.h\n",
            "bedtools2/src/utils/Contexts/ContextClosest.h\n",
            "bedtools2/src/utils/Contexts/ContextIntersect.h\n",
            "bedtools2/src/utils/Contexts/ContextIntersect.cpp\n",
            "bedtools2/src/utils/Contexts/ContextFisher.cpp\n",
            "bedtools2/src/utils/Contexts/ContextMerge.h\n",
            "bedtools2/src/utils/Contexts/ContextGroupBy.h\n",
            "bedtools2/src/utils/Contexts/ContextComplement.h\n",
            "bedtools2/src/utils/Contexts/ContextSubtract.h\n",
            "bedtools2/src/utils/Contexts/ContextCoverage.cpp\n",
            "bedtools2/src/utils/Contexts/ContextBase.cpp\n",
            "bedtools2/src/utils/Contexts/ContextSummary.h\n",
            "bedtools2/src/utils/Contexts/ContextFisher.h\n",
            "bedtools2/src/utils/Contexts/ContextSpacing.cpp\n",
            "bedtools2/src/utils/Contexts/ContextMap.h\n",
            "bedtools2/src/utils/Contexts/ContextSubtract.cpp\n",
            "bedtools2/src/utils/Contexts/ContextJaccard.h\n",
            "bedtools2/src/utils/Contexts/ContextComplement.cpp\n",
            "bedtools2/src/utils/Contexts/ContextCoverage.h\n",
            "bedtools2/src/utils/Contexts/ContextSample.cpp\n",
            "bedtools2/src/utils/Contexts/ContextClosest.cpp\n",
            "bedtools2/src/utils/Contexts/Makefile.frag\n",
            "bedtools2/src/utils/Contexts/ContextJaccard.cpp\n",
            "bedtools2/src/utils/Contexts/ContextSample.h\n",
            "bedtools2/src/utils/Contexts/ContextSummary.cpp\n",
            "bedtools2/src/utils/Contexts/ContextGroupBy.cpp\n",
            "bedtools2/src/utils/Contexts/ContextMap.cpp\n",
            "bedtools2/src/utils/Contexts/ContextSpacing.h\n",
            "bedtools2/src/utils/Contexts/ContextMerge.cpp\n",
            "bedtools2/src/utils/bedFile/bedFile.h\n",
            "bedtools2/src/utils/bedFile/Makefile.frag\n",
            "bedtools2/src/utils/bedFile/bedFile.cpp\n",
            "bedtools2/src/utils/RecordOutputMgr/RecordOutputMgr.h\n",
            "bedtools2/src/utils/RecordOutputMgr/Makefile.frag\n",
            "bedtools2/src/utils/RecordOutputMgr/RecordOutputMgr.cpp\n",
            "bedtools2/src/utils/NewChromsweep/NewChromsweep.h\n",
            "bedtools2/src/utils/NewChromsweep/CloseSweep.h\n",
            "bedtools2/src/utils/NewChromsweep/Makefile.frag\n",
            "bedtools2/src/utils/NewChromsweep/NewChromsweep.cpp\n",
            "bedtools2/src/utils/NewChromsweep/CloseSweep.cpp\n",
            "bedtools2/src/utils/htslib/htslib.mk\n",
            "bedtools2/src/utils/htslib/cram/\n",
            "bedtools2/src/utils/htslib/htslib/\n",
            "bedtools2/src/utils/htslib/faidx.c\n",
            "bedtools2/src/utils/htslib/vcfutils.c\n",
            "bedtools2/src/utils/htslib/bcf_sr_sort.h\n",
            "bedtools2/src/utils/htslib/faidx.5\n",
            "bedtools2/src/utils/htslib/regidx.c\n",
            "bedtools2/src/utils/htslib/htslib.pc.in\n",
            "bedtools2/src/utils/htslib/configure.ac\n",
            "bedtools2/src/utils/htslib/INSTALL\n",
            "bedtools2/src/utils/htslib/hts_os.c\n",
            "bedtools2/src/utils/htslib/tbx.c\n",
            "bedtools2/src/utils/htslib/probaln.c\n",
            "bedtools2/src/utils/htslib/LICENSE\n",
            "bedtools2/src/utils/htslib/kfunc.c\n",
            "bedtools2/src/utils/htslib/hfile_gcs.c\n",
            "bedtools2/src/utils/htslib/errmod.c\n",
            "bedtools2/src/utils/htslib/multipart.c\n",
            "bedtools2/src/utils/htslib/hfile_internal.h\n",
            "bedtools2/src/utils/htslib/Makefile\n",
            "bedtools2/src/utils/htslib/htsfile.c\n",
            "bedtools2/src/utils/htslib/realn.c\n",
            "bedtools2/src/utils/htslib/bgzip.c\n",
            "bedtools2/src/utils/htslib/hfile_net.c\n",
            "bedtools2/src/utils/htslib/tabix.1\n",
            "bedtools2/src/utils/htslib/textutils.c\n",
            "bedtools2/src/utils/htslib/hts.c\n",
            "bedtools2/src/utils/htslib/README\n",
            "bedtools2/src/utils/htslib/vcf_sweep.c\n",
            "bedtools2/src/utils/htslib/README.md\n",
            "bedtools2/src/utils/htslib/sam.5\n",
            "bedtools2/src/utils/htslib/bcf_sr_sort.c\n",
            "bedtools2/src/utils/htslib/synced_bcf_reader.c\n",
            "bedtools2/src/utils/htslib/htslib_vars.mk\n",
            "bedtools2/src/utils/htslib/sam.c\n",
            "bedtools2/src/utils/htslib/NEWS\n",
            "bedtools2/src/utils/htslib/.gitignore\n",
            "bedtools2/src/utils/htslib/vcf.5\n",
            "bedtools2/src/utils/htslib/bgzf.c\n",
            "bedtools2/src/utils/htslib/.gitattributes\n",
            "bedtools2/src/utils/htslib/hts_internal.h\n",
            "bedtools2/src/utils/htslib/vcf.c\n",
            "bedtools2/src/utils/htslib/m4/\n",
            "bedtools2/src/utils/htslib/plugin.c\n",
            "bedtools2/src/utils/htslib/version.sh\n",
            "bedtools2/src/utils/htslib/hfile_libcurl.c\n",
            "bedtools2/src/utils/htslib/kstring.c\n",
            "bedtools2/src/utils/htslib/os/\n",
            "bedtools2/src/utils/htslib/md5.c\n",
            "bedtools2/src/utils/htslib/hfile_s3.c\n",
            "bedtools2/src/utils/htslib/htsfile.1\n",
            "bedtools2/src/utils/htslib/thread_pool.c\n",
            "bedtools2/src/utils/htslib/hfile.c\n",
            "bedtools2/src/utils/htslib/config.mk.in\n",
            "bedtools2/src/utils/htslib/tabix.c\n",
            "bedtools2/src/utils/htslib/textutils_internal.h\n",
            "bedtools2/src/utils/htslib/knetfile.c\n",
            "bedtools2/src/utils/htslib/bgzip.1\n",
            "bedtools2/src/utils/htslib/thread_pool_internal.h\n",
            "bedtools2/src/utils/htslib/os/rand.c\n",
            "bedtools2/src/utils/htslib/os/lzma_stub.h\n",
            "bedtools2/src/utils/htslib/m4/hts_prog_cc_warnings.m4\n",
            "bedtools2/src/utils/htslib/htslib/sam.h\n",
            "bedtools2/src/utils/htslib/htslib/synced_bcf_reader.h\n",
            "bedtools2/src/utils/htslib/htslib/vcf_sweep.h\n",
            "bedtools2/src/utils/htslib/htslib/vcf.h\n",
            "bedtools2/src/utils/htslib/htslib/khash_str2int.h\n",
            "bedtools2/src/utils/htslib/htslib/bgzf.h\n",
            "bedtools2/src/utils/htslib/htslib/cram.h\n",
            "bedtools2/src/utils/htslib/htslib/kstring.h\n",
            "bedtools2/src/utils/htslib/htslib/knetfile.h\n",
            "bedtools2/src/utils/htslib/htslib/hts_log.h\n",
            "bedtools2/src/utils/htslib/htslib/hfile.h\n",
            "bedtools2/src/utils/htslib/htslib/thread_pool.h\n",
            "bedtools2/src/utils/htslib/htslib/hts_defs.h\n",
            "bedtools2/src/utils/htslib/htslib/khash.h\n",
            "bedtools2/src/utils/htslib/htslib/regidx.h\n",
            "bedtools2/src/utils/htslib/htslib/faidx.h\n",
            "bedtools2/src/utils/htslib/htslib/vcfutils.h\n",
            "bedtools2/src/utils/htslib/htslib/kseq.h\n",
            "bedtools2/src/utils/htslib/htslib/kfunc.h\n",
            "bedtools2/src/utils/htslib/htslib/tbx.h\n",
            "bedtools2/src/utils/htslib/htslib/ksort.h\n",
            "bedtools2/src/utils/htslib/htslib/kbitset.h\n",
            "bedtools2/src/utils/htslib/htslib/hts_os.h\n",
            "bedtools2/src/utils/htslib/htslib/klist.h\n",
            "bedtools2/src/utils/htslib/htslib/hts.h\n",
            "bedtools2/src/utils/htslib/htslib/hts_endian.h\n",
            "bedtools2/src/utils/htslib/cram/cram_codecs.h\n",
            "bedtools2/src/utils/htslib/cram/sam_header.c\n",
            "bedtools2/src/utils/htslib/cram/cram_index.c\n",
            "bedtools2/src/utils/htslib/cram/open_trace_file.c\n",
            "bedtools2/src/utils/htslib/cram/mFILE.h\n",
            "bedtools2/src/utils/htslib/cram/cram_external.c\n",
            "bedtools2/src/utils/htslib/cram/cram.h\n",
            "bedtools2/src/utils/htslib/cram/misc.h\n",
            "bedtools2/src/utils/htslib/cram/files.c\n",
            "bedtools2/src/utils/htslib/cram/os.h\n",
            "bedtools2/src/utils/htslib/cram/rANS_static.h\n",
            "bedtools2/src/utils/htslib/cram/cram_samtools.c\n",
            "bedtools2/src/utils/htslib/cram/pooled_alloc.c\n",
            "bedtools2/src/utils/htslib/cram/string_alloc.c\n",
            "bedtools2/src/utils/htslib/cram/cram_io.h\n",
            "bedtools2/src/utils/htslib/cram/cram_decode.h\n",
            "bedtools2/src/utils/htslib/cram/cram_encode.h\n",
            "bedtools2/src/utils/htslib/cram/cram_stats.h\n",
            "bedtools2/src/utils/htslib/cram/mFILE.c\n",
            "bedtools2/src/utils/htslib/cram/cram_index.h\n",
            "bedtools2/src/utils/htslib/cram/sam_header.h\n",
            "bedtools2/src/utils/htslib/cram/open_trace_file.h\n",
            "bedtools2/src/utils/htslib/cram/cram_codecs.c\n",
            "bedtools2/src/utils/htslib/cram/rANS_byte.h\n",
            "bedtools2/src/utils/htslib/cram/string_alloc.h\n",
            "bedtools2/src/utils/htslib/cram/rANS_static.c\n",
            "bedtools2/src/utils/htslib/cram/pooled_alloc.h\n",
            "bedtools2/src/utils/htslib/cram/cram_samtools.h\n",
            "bedtools2/src/utils/htslib/cram/cram_stats.c\n",
            "bedtools2/src/utils/htslib/cram/cram_encode.c\n",
            "bedtools2/src/utils/htslib/cram/cram_structs.h\n",
            "bedtools2/src/utils/htslib/cram/cram_io.c\n",
            "bedtools2/src/utils/htslib/cram/cram_decode.c\n",
            "bedtools2/src/utils/tabFile/tabFile.cpp\n",
            "bedtools2/src/utils/tabFile/tabFile.h\n",
            "bedtools2/src/utils/tabFile/Makefile.frag\n",
            "bedtools2/src/genomeCoverageBed/genomeCoverageMain.cpp\n",
            "bedtools2/src/genomeCoverageBed/genomeCoverageBed.h\n",
            "bedtools2/src/genomeCoverageBed/Makefile.frag\n",
            "bedtools2/src/genomeCoverageBed/genomeCoverageBed.cpp\n",
            "bedtools2/src/maskFastaFromBed/maskFastaFromBed.cpp\n",
            "bedtools2/src/maskFastaFromBed/maskFastaFromBed.h\n",
            "bedtools2/src/maskFastaFromBed/maskFastaFromBedMain.cpp\n",
            "bedtools2/src/maskFastaFromBed/Makefile.frag\n",
            "bedtools2/src/regressTest/RegressTest.h\n",
            "bedtools2/src/regressTest/RegressTest.cpp\n",
            "bedtools2/src/regressTest/regressTestMain.cpp\n",
            "bedtools2/src/regressTest/Makefile.frag\n",
            "bedtools2/src/bedToIgv/bedToIgv.cpp\n",
            "bedtools2/src/bedToIgv/Makefile.frag\n",
            "bedtools2/src/nucBed/nucBed.h\n",
            "bedtools2/src/nucBed/nucBedMain.cpp\n",
            "bedtools2/src/nucBed/LargeFileSupport.h\n",
            "bedtools2/src/nucBed/Makefile.frag\n",
            "bedtools2/src/nucBed/nucBed.cpp\n",
            "bedtools2/src/reldist/reldist.cpp\n",
            "bedtools2/src/reldist/Makefile.frag\n",
            "bedtools2/src/reldist/reldistMain.cpp\n",
            "bedtools2/src/reldist/reldist.h\n",
            "bedtools2/src/getOverlap/getOverlap.cpp\n",
            "bedtools2/src/getOverlap/Makefile.frag\n",
            "bedtools2/src/randomBed/randomBedMain.cpp\n",
            "bedtools2/src/randomBed/randomBed.cpp\n",
            "bedtools2/src/randomBed/Makefile.frag\n",
            "bedtools2/src/randomBed/randomBed.h\n",
            "bedtools2/src/annotateBed/annotateBed.cpp\n",
            "bedtools2/src/annotateBed/annotateBed.h\n",
            "bedtools2/src/annotateBed/annotateMain.cpp\n",
            "bedtools2/src/annotateBed/Makefile.frag\n",
            "bedtools2/src/slopBed/slopBed.h\n",
            "bedtools2/src/slopBed/slopBed.cpp\n",
            "bedtools2/src/slopBed/Makefile.frag\n",
            "bedtools2/src/slopBed/slopBedMain.cpp\n",
            "bedtools2/src/mapFile/mapHelp.cpp\n",
            "bedtools2/src/mapFile/mapFile.cpp\n",
            "bedtools2/src/mapFile/Makefile.frag\n",
            "bedtools2/src/mapFile/mapFile.h\n",
            "bedtools2/src/pairToPair/pairToPair.h\n",
            "bedtools2/src/pairToPair/pairToPairMain.cpp\n",
            "bedtools2/src/pairToPair/pairToPair.cpp\n",
            "bedtools2/src/pairToPair/Makefile.frag\n",
            "bedtools2/src/linksBed/linksBed.cpp\n",
            "bedtools2/src/linksBed/Makefile.frag\n",
            "bedtools2/src/linksBed/linksBed.h\n",
            "bedtools2/src/linksBed/linksMain.cpp\n",
            "bedtools2/src/fisher/fisher.h\n",
            "bedtools2/src/fisher/kfunc.h\n",
            "bedtools2/src/fisher/kfunc.cpp\n",
            "bedtools2/src/fisher/Makefile.frag\n",
            "bedtools2/src/fisher/fisher.cpp\n",
            "bedtools2/src/fisher/fisherHelp.cpp\n",
            "bedtools2/src/expand/Makefile.frag\n",
            "bedtools2/src/expand/expand.cpp\n",
            "bedtools2/src/complementFile/complementHelp.cpp\n",
            "bedtools2/src/complementFile/complementFile.h\n",
            "bedtools2/src/complementFile/Makefile.frag\n",
            "bedtools2/src/complementFile/complementFile.cpp\n",
            "bedtools2/src/bedToBam/bedToBam.cpp\n",
            "bedtools2/src/bedToBam/Makefile.frag\n",
            "bedtools2/src/split/splitBed.h\n",
            "bedtools2/src/split/splitBed.cpp\n",
            "bedtools2/src/split/splitBedMain.cpp\n",
            "bedtools2/src/split/Makefile.frag\n",
            "bedtools2/src/sortBed/sortMain.cpp\n",
            "bedtools2/src/sortBed/sortBed.cpp\n",
            "bedtools2/src/sortBed/sortBed.h\n",
            "bedtools2/src/sortBed/Makefile.frag\n",
            "bedtools2/src/spacingFile/spacingHelp.cpp\n",
            "bedtools2/src/spacingFile/spacingFile.h\n",
            "bedtools2/src/spacingFile/Makefile.frag\n",
            "bedtools2/src/spacingFile/spacingFile.cpp\n",
            "bedtools2/data/gerp.chr1.bed.gz\n",
            "bedtools2/data/simpleRepeats.chr1.bed.gz\n",
            "bedtools2/data/aluY.chr1.bed.gz\n",
            "bedtools2/data/refseq.chr1.exons.bed.gz\n",
            "bedtools2/data/knownGene.hg18.chr21.short.bed\n",
            "bedtools2/data/knownGene.hg18.chr21.bed\n",
            "bedtools2/.github/workflows/\n",
            "bedtools2/.github/workflows/main.yml\n",
            "bedtools2/scripts/makeBashScripts.py\n",
            "bedtools2/scripts/vcfsort.sh\n",
            "bedtools2/genomes/human.hg38.genome\n",
            "bedtools2/genomes/mouse.mm10.genome\n",
            "bedtools2/genomes/mouse.mm8.genome\n",
            "bedtools2/genomes/human.hg18.genome\n",
            "bedtools2/genomes/human.hg19.genome\n",
            "bedtools2/genomes/mouse.mm9.genome\n",
            "bedtools2/docs/index.rst\n",
            "bedtools2/docs/Makefile\n",
            "bedtools2/docs/conf.py\n",
            "bedtools2/docs/_static/\n",
            "bedtools2/docs/content/\n",
            "bedtools2/docs/bedtools.swiss.png\n",
            "bedtools2/docs/build.sh\n",
            "bedtools2/docs/make.bat\n",
            "bedtools2/docs/bedtools.ai\n",
            "bedtools2/docs/templates/\n",
            "bedtools2/docs/themes/\n",
            "bedtools2/docs/bedtools.png\n",
            "bedtools2/docs/themes/rtd/\n",
            "bedtools2/docs/themes/rtd/theme.conf\n",
            "bedtools2/docs/themes/rtd/static/\n",
            "bedtools2/docs/themes/rtd/static/rtd.css_t\n",
            "bedtools2/docs/templates/page.html\n",
            "bedtools2/docs/templates/sidebar-intro.html\n",
            "bedtools2/docs/content/general-usage.rst\n",
            "bedtools2/docs/content/tools/\n",
            "bedtools2/docs/content/images/\n",
            "bedtools2/docs/content/quick-start.rst\n",
            "bedtools2/docs/content/example-usage.rst\n",
            "bedtools2/docs/content/overview.rst\n",
            "bedtools2/docs/content/advanced-usage.rst\n",
            "bedtools2/docs/content/history.rst\n",
            "bedtools2/docs/content/installation.rst\n",
            "bedtools2/docs/content/tips-and-tricks.rst\n",
            "bedtools2/docs/content/bedtools-suite.rst\n",
            "bedtools2/docs/content/faq.rst\n",
            "bedtools2/docs/content/related-tools.rst\n",
            "bedtools2/docs/content/images/memory-comparo.png\n",
            "bedtools2/docs/content/images/speed-comparo.png\n",
            "bedtools2/docs/content/images/fisher/\n",
            "bedtools2/docs/content/images/github-zip-button.png\n",
            "bedtools2/docs/content/images/galaxy-bedtools.png\n",
            "bedtools2/docs/content/images/bedtoolsMenu.png\n",
            "bedtools2/docs/content/images/tool-glyphs/\n",
            "bedtools2/docs/content/images/tool-glyphs/map-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/jaccard-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/merge-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/reldist-glyph.ai\n",
            "bedtools2/docs/content/images/tool-glyphs/getfasta-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/map-speed-comparo.png\n",
            "bedtools2/docs/content/images/tool-glyphs/reldist-plot.png\n",
            "bedtools2/docs/content/images/tool-glyphs/subtract-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/flank-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/jaccard-glyph.ai\n",
            "bedtools2/docs/content/images/tool-glyphs/map-glyph.svg\n",
            "bedtools2/docs/content/images/tool-glyphs/random-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/genomecov-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/cluster-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/maskfasta-glyph.svg\n",
            "bedtools2/docs/content/images/tool-glyphs/random-glyph.ai\n",
            "bedtools2/docs/content/images/tool-glyphs/complement-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/slop-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/window-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/shuffle-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/reldist-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/closest-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/maskfasta-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/intersect-glyph.png\n",
            "bedtools2/docs/content/images/tool-glyphs/barski_binding_site.png\n",
            "bedtools2/docs/content/images/fisher/res-genes.png\n",
            "bedtools2/docs/content/tools/expand.rst\n",
            "bedtools2/docs/content/tools/genomecov.rst\n",
            "bedtools2/docs/content/tools/makewindows.rst\n",
            "bedtools2/docs/content/tools/igv.rst\n",
            "bedtools2/docs/content/tools/overlap.rst\n",
            "bedtools2/docs/content/tools/nuc.rst\n",
            "bedtools2/docs/content/tools/fisher.rst\n",
            "bedtools2/docs/content/tools/cluster.rst\n",
            "bedtools2/docs/content/tools/pairtobed.rst\n",
            "bedtools2/docs/content/tools/shift.rst\n",
            "bedtools2/docs/content/tools/complement.rst\n",
            "bedtools2/docs/content/tools/bed12tobed6.rst\n",
            "bedtools2/docs/content/tools/map.rst\n",
            "bedtools2/docs/content/tools/tag.rst\n",
            "bedtools2/docs/content/tools/pairtopair.rst\n",
            "bedtools2/docs/content/tools/slop.rst\n",
            "bedtools2/docs/content/tools/sort.rst\n",
            "bedtools2/docs/content/tools/reldist.rst\n",
            "bedtools2/docs/content/tools/closest.rst\n",
            "bedtools2/docs/content/tools/bedtobam.rst\n",
            "bedtools2/docs/content/tools/maskfasta.rst\n",
            "bedtools2/docs/content/tools/bamtofastq.rst\n",
            "bedtools2/docs/content/tools/multicov.rst\n",
            "bedtools2/docs/content/tools/flank.rst\n",
            "bedtools2/docs/content/tools/random.rst\n",
            "bedtools2/docs/content/tools/merge.rst\n",
            "bedtools2/docs/content/tools/shuffle.rst\n",
            "bedtools2/docs/content/tools/window.rst\n",
            "bedtools2/docs/content/tools/bedpetobam.rst\n",
            "bedtools2/docs/content/tools/groupby.rst\n",
            "bedtools2/docs/content/tools/multiinter.rst\n",
            "bedtools2/docs/content/tools/bamtobed.rst\n",
            "bedtools2/docs/content/tools/getfasta.rst\n",
            "bedtools2/docs/content/tools/unionbedg.rst\n",
            "bedtools2/docs/content/tools/subtract.rst\n",
            "bedtools2/docs/content/tools/coverage.rst\n",
            "bedtools2/docs/content/tools/jaccard.rst\n",
            "bedtools2/docs/content/tools/links.rst\n",
            "bedtools2/docs/content/tools/annotate.rst\n",
            "bedtools2/docs/content/tools/intersect.rst\n",
            "bedtools2/docs/_static/labibi.css\n",
            "bedtools2/docs/_static/labibi.js\n",
            "bedtools2/test/split/\n",
            "bedtools2/test/bedtobam/\n",
            "bedtools2/test/cluster/\n",
            "bedtools2/test/expand/\n",
            "bedtools2/test/shuffle/\n",
            "bedtools2/test/fisher/\n",
            "bedtools2/test/complement/\n",
            "bedtools2/test/intersect/\n",
            "bedtools2/test/sort/\n",
            "bedtools2/test/subtract/\n",
            "bedtools2/test/merge/\n",
            "bedtools2/test/general/\n",
            "bedtools2/test/slop/\n",
            "bedtools2/test/reldist/\n",
            "bedtools2/test/shift/\n",
            "bedtools2/test/genomecov/\n",
            "bedtools2/test/jaccard/\n",
            "bedtools2/test/groupby/\n",
            "bedtools2/test/bamtofastq/\n",
            "bedtools2/test/bed12tobed6/\n",
            "bedtools2/test/getfasta/\n",
            "bedtools2/test/map/\n",
            "bedtools2/test/bigchroms/\n",
            "bedtools2/test/spacing/\n",
            "bedtools2/test/sample/\n",
            "bedtools2/test/flank/\n",
            "bedtools2/test/negativecontrol/\n",
            "bedtools2/test/multicov/\n",
            "bedtools2/test/closest/\n",
            "bedtools2/test/test.sh\n",
            "bedtools2/test/coverage/\n",
            "bedtools2/test/bamtobed/\n",
            "bedtools2/test/makewindows/\n",
            "bedtools2/test/makewindows/test-makewindows.sh\n",
            "bedtools2/test/makewindows/a.33bp.bed\n",
            "bedtools2/test/makewindows/a.20bp.bed\n",
            "bedtools2/test/makewindows/input.bed\n",
            "bedtools2/test/makewindows/a.19bp.bed\n",
            "bedtools2/test/bamtobed/two_blocks.sam\n",
            "bedtools2/test/bamtobed/three_blocks.sam\n",
            "bedtools2/test/bamtobed/test-bamtobed.sh\n",
            "bedtools2/test/bamtobed/numeric_tag.sam\n",
            "bedtools2/test/bamtobed/extra-long-header.sam.gz\n",
            "bedtools2/test/bamtobed/two_blocks_w_D.sam\n",
            "bedtools2/test/bamtobed/one_block.sam\n",
            "bedtools2/test/coverage/two_blocks.sam\n",
            "bedtools2/test/coverage/chr_0-100.bed\n",
            "bedtools2/test/coverage/three_blocks.bam.bai\n",
            "bedtools2/test/coverage/sam-w-del.sam\n",
            "bedtools2/test/coverage/three_blocks.bam\n",
            "bedtools2/test/coverage/three_blocks_nomatch.bed\n",
            "bedtools2/test/coverage/sam-w-del.bam\n",
            "bedtools2/test/coverage/two_blocks.bam.bai\n",
            "bedtools2/test/coverage/three_blocks.sam\n",
            "bedtools2/test/coverage/one_block.bam.bai\n",
            "bedtools2/test/coverage/test-coverage.sh\n",
            "bedtools2/test/coverage/x.bed\n",
            "bedtools2/test/coverage/two_blocks.bam\n",
            "bedtools2/test/coverage/y.bed\n",
            "bedtools2/test/coverage/a.bed\n",
            "bedtools2/test/coverage/c.bed\n",
            "bedtools2/test/coverage/one_block.bam\n",
            "bedtools2/test/coverage/three_blocks_match.bam\n",
            "bedtools2/test/coverage/sam-w-del.bam.bai\n",
            "bedtools2/test/coverage/b.bed\n",
            "bedtools2/test/coverage/three_blocks_match.bed\n",
            "bedtools2/test/coverage/one_block.sam\n",
            "bedtools2/test/coverage/three_blocks_match_1bp.bed\n",
            "bedtools2/test/closest/d_d1F.bed\n",
            "bedtools2/test/closest/d_q1.bed\n",
            "bedtools2/test/closest/d_iu.bed\n",
            "bedtools2/test/closest/d_d1R.bed\n",
            "bedtools2/test/closest/d_q2.bed\n",
            "bedtools2/test/closest/tfbs.bed\n",
            "bedtools2/test/closest/null_b.bed\n",
            "bedtools2/test/closest/bug281_a.medium.bed\n",
            "bedtools2/test/closest/null_c.bed\n",
            "bedtools2/test/closest/null_a.bed\n",
            "bedtools2/test/closest/d_id.bed\n",
            "bedtools2/test/closest/a2.bed\n",
            "bedtools2/test/closest/test-closest.sh\n",
            "bedtools2/test/closest/kclosest/\n",
            "bedtools2/test/closest/close-a.bed\n",
            "bedtools2/test/closest/b.names.bed\n",
            "bedtools2/test/closest/close-b.bed\n",
            "bedtools2/test/closest/strand-test-d.bed\n",
            "bedtools2/test/closest/b-one-bp-closer.bed\n",
            "bedtools2/test/closest/islands.2.bed\n",
            "bedtools2/test/closest/bug157_b.bed\n",
            "bedtools2/test/closest/shores.bed\n",
            "bedtools2/test/closest/strand-test-a.bed\n",
            "bedtools2/test/closest/strand-test-c.bed\n",
            "bedtools2/test/closest/bug157_a.bed\n",
            "bedtools2/test/closest/strand-test-b.bed\n",
            "bedtools2/test/closest/bug281_b.bed\n",
            "bedtools2/test/closest/dmr.bed\n",
            "bedtools2/test/closest/a.bed\n",
            "bedtools2/test/closest/b.bed\n",
            "bedtools2/test/closest/a.names.bed\n",
            "bedtools2/test/closest/mdb3.bed\n",
            "bedtools2/test/closest/mdb2.bed\n",
            "bedtools2/test/closest/b2.bed\n",
            "bedtools2/test/closest/mdb1.bed\n",
            "bedtools2/test/closest/mq1.bed\n",
            "bedtools2/test/closest/d.bed\n",
            "bedtools2/test/closest/d4.bed\n",
            "bedtools2/test/closest/sortAndNaming/\n",
            "bedtools2/test/closest/islands.bed\n",
            "bedtools2/test/closest/bug281_b.flip.bed\n",
            "bedtools2/test/closest/bug281_a.flip.medium.bed\n",
            "bedtools2/test/closest/d_d2R.bed\n",
            "bedtools2/test/closest/d_d2F.bed\n",
            "bedtools2/test/closest/sortAndNaming/num_all.bed\n",
            "bedtools2/test/closest/sortAndNaming/db1_noChr.bed\n",
            "bedtools2/test/closest/sortAndNaming/alpha_all.bed\n",
            "bedtools2/test/closest/sortAndNaming/test-sort-and-naming.sh\n",
            "bedtools2/test/closest/sortAndNaming/q1_gls.bed\n",
            "bedtools2/test/closest/sortAndNaming/db1_num.bed\n",
            "bedtools2/test/closest/sortAndNaming/db2_numBackwards.bed\n",
            "bedtools2/test/closest/sortAndNaming/num_missing.bed\n",
            "bedtools2/test/closest/sortAndNaming/q1_num.bed\n",
            "bedtools2/test/closest/sortAndNaming/db2_num.bed\n",
            "bedtools2/test/closest/sortAndNaming/db1_leadingZero.txt\n",
            "bedtools2/test/closest/sortAndNaming/sq1.bed\n",
            "bedtools2/test/closest/sortAndNaming/db1_gls.bed\n",
            "bedtools2/test/closest/sortAndNaming/alpha_missing.bed\n",
            "bedtools2/test/closest/sortAndNaming/sdb1.bed\n",
            "bedtools2/test/closest/sortAndNaming/db3_numBackwards.bed\n",
            "bedtools2/test/closest/sortAndNaming/q1a_num.bed\n",
            "bedtools2/test/closest/kclosest/test-kclosest.sh\n",
            "bedtools2/test/closest/kclosest/bug471_b.bed\n",
            "bedtools2/test/closest/kclosest/bug471_a.bed\n",
            "bedtools2/test/closest/kclosest/d2.bed\n",
            "bedtools2/test/closest/kclosest/q2.bed\n",
            "bedtools2/test/closest/kclosest/d3.bed\n",
            "bedtools2/test/closest/kclosest/d1.bed\n",
            "bedtools2/test/closest/kclosest/q1.bed\n",
            "bedtools2/test/multicov/two_blocks.sam\n",
            "bedtools2/test/multicov/test-multicov.sh\n",
            "bedtools2/test/multicov/test-multi.bed\n",
            "bedtools2/test/multicov/test-multi.2.sam\n",
            "bedtools2/test/multicov/multicov.bed\n",
            "bedtools2/test/multicov/test-multi.sam\n",
            "bedtools2/test/multicov/one_block.sam\n",
            "bedtools2/test/negativecontrol/test-negativecontrol.sh\n",
            "bedtools2/test/flank/test-flank.sh\n",
            "bedtools2/test/flank/a.bed\n",
            "bedtools2/test/flank/tiny.genome\n",
            "bedtools2/test/sample/test-sample.sh\n",
            "bedtools2/test/sample/human.hg19.genome\n",
            "bedtools2/test/spacing/a.bed\n",
            "bedtools2/test/spacing/test-spacing.sh\n",
            "bedtools2/test/bigchroms/test-bigchroms.sh\n",
            "bedtools2/test/bigchroms/big4.bed\n",
            "bedtools2/test/bigchroms/make-big-chrom.py\n",
            "bedtools2/test/bigchroms/big4c.bed\n",
            "bedtools2/test/bigchroms/bigx.bed\n",
            "bedtools2/test/bigchroms/abig.bed\n",
            "bedtools2/test/map/values.bam\n",
            "bedtools2/test/map/.Rhistory\n",
            "bedtools2/test/map/a.vsorted.bed\n",
            "bedtools2/test/map/three_blocks_nomatch.bed\n",
            "bedtools2/test/map/b.vsorted.bed\n",
            "bedtools2/test/map/test.vcf\n",
            "bedtools2/test/map/values.bed\n",
            "bedtools2/test/map/test.gff\n",
            "bedtools2/test/map/values5.bed\n",
            "bedtools2/test/map/ivls.bed\n",
            "bedtools2/test/map/values4.bed\n",
            "bedtools2/test/map/t.bed\n",
            "bedtools2/test/map/ivls2.bed\n",
            "bedtools2/test/map/values3.bed\n",
            "bedtools2/test/map/.RData\n",
            "bedtools2/test/map/values2.bed\n",
            "bedtools2/test/map/d.bed\n",
            "bedtools2/test/map/genome\n",
            "bedtools2/test/map/three_blocks_match.bed\n",
            "bedtools2/test/map/test-map.sh\n",
            "bedtools2/test/map/bug262_a.bed\n",
            "bedtools2/test/map/bug262_b.bed\n",
            "bedtools2/test/map/fullFields.bam\n",
            "bedtools2/test/getfasta/test.iupac.bed\n",
            "bedtools2/test/getfasta/t.fa\n",
            "bedtools2/test/getfasta/test.iupac.fa\n",
            "bedtools2/test/getfasta/blocks.bed\n",
            "bedtools2/test/getfasta/t_fH.fa.fai\n",
            "bedtools2/test/getfasta/t.bed\n",
            "bedtools2/test/getfasta/t.fa.fai\n",
            "bedtools2/test/getfasta/test-getfasta.sh\n",
            "bedtools2/test/getfasta/t_fH.fa\n",
            "bedtools2/test/bed12tobed6/two_blocks.bed\n",
            "bedtools2/test/bed12tobed6/bed6.bed\n",
            "bedtools2/test/bed12tobed6/three_blocks.bed\n",
            "bedtools2/test/bed12tobed6/test-bed12tobed6.sh\n",
            "bedtools2/test/bed12tobed6/one_blocks.bed\n",
            "bedtools2/test/bamtofastq/test-bamtofastq.sh\n",
            "bedtools2/test/bamtofastq/test2.sam\n",
            "bedtools2/test/bamtofastq/golden.fq\n",
            "bedtools2/test/bamtofastq/golden.fq2\n",
            "bedtools2/test/bamtofastq/test.sam\n",
            "bedtools2/test/groupby/gdc.bam\n",
            "bedtools2/test/groupby/values3_case.header.bed\n",
            "bedtools2/test/groupby/test-groupby.sh\n",
            "bedtools2/test/groupby/values3.unmarked_header.bed\n",
            "bedtools2/test/groupby/a_vcfSVtest.vcf\n",
            "bedtools2/test/groupby/values3.7fields.header.bed\n",
            "bedtools2/test/groupby/values3.unmarked_header.bed.2\n",
            "bedtools2/test/groupby/values3.no_header.bed\n",
            "bedtools2/test/groupby/values3.header.bed\n",
            "bedtools2/test/groupby/noPosvalues.header.bed\n",
            "bedtools2/test/groupby/test.bed\n",
            "bedtools2/test/groupby/bug569_problem.txt\n",
            "bedtools2/test/jaccard/a645.bed\n",
            "bedtools2/test/jaccard/short.bed\n",
            "bedtools2/test/jaccard/bMixedStrands.bed\n",
            "bedtools2/test/jaccard/a.bed\n",
            "bedtools2/test/jaccard/c.bed\n",
            "bedtools2/test/jaccard/three_blocks_match.bam\n",
            "bedtools2/test/jaccard/b.bed\n",
            "bedtools2/test/jaccard/b645.bed\n",
            "bedtools2/test/jaccard/e.bed\n",
            "bedtools2/test/jaccard/test-jaccard.sh\n",
            "bedtools2/test/jaccard/long.bed\n",
            "bedtools2/test/jaccard/three_blocks_match.bed\n",
            "bedtools2/test/jaccard/aMixedStrands.bed\n",
            "bedtools2/test/jaccard/a.bam\n",
            "bedtools2/test/genomecov/two_blocks.sam\n",
            "bedtools2/test/genomecov/test_ref.fa.fai\n",
            "bedtools2/test/genomecov/test-genomecov.sh\n",
            "bedtools2/test/genomecov/chip.sam\n",
            "bedtools2/test/genomecov/y.bam\n",
            "bedtools2/test/genomecov/test_ref.fa\n",
            "bedtools2/test/genomecov/sam-w-del.sam\n",
            "bedtools2/test/genomecov/three_blocks_nomatch.bed\n",
            "bedtools2/test/genomecov/three_blocks.sam\n",
            "bedtools2/test/genomecov/empty.bam\n",
            "bedtools2/test/genomecov/y.bed\n",
            "bedtools2/test/genomecov/empty.cram\n",
            "bedtools2/test/genomecov/genome.txt\n",
            "bedtools2/test/genomecov/three_blocks_match.bed\n",
            "bedtools2/test/genomecov/pair-chip.sam\n",
            "bedtools2/test/genomecov/one_block.sam\n",
            "bedtools2/test/genomecov/three_blocks_match_1bp.bed\n",
            "bedtools2/test/shift/test-shift.sh\n",
            "bedtools2/test/shift/a.bed\n",
            "bedtools2/test/shift/b.bed\n",
            "bedtools2/test/shift/tiny.genome\n",
            "bedtools2/test/shift/huge.genome\n",
            "bedtools2/test/reldist/issue_711.a.bed\n",
            "bedtools2/test/reldist/issue_711.b.bed\n",
            "bedtools2/test/reldist/test-reldist.sh\n",
            "bedtools2/test/slop/a.bed\n",
            "bedtools2/test/slop/tiny.genome\n",
            "bedtools2/test/slop/test-slop.sh\n",
            "bedtools2/test/general/hg19.fa.fai\n",
            "bedtools2/test/general/empty.bed\n",
            "bedtools2/test/general/a.chromheader.bed\n",
            "bedtools2/test/general/test-general.sh\n",
            "bedtools2/test/general/non-empty.bed.gz\n",
            "bedtools2/test/general/t.bed\n",
            "bedtools2/test/general/a.trackheader.bed\n",
            "bedtools2/test/merge/bamCol8Mean.txt\n",
            "bedtools2/test/merge/mergeBugTest.bed\n",
            "bedtools2/test/merge/expFormat.bed\n",
            "bedtools2/test/merge/bamCol9Mean.txt\n",
            "bedtools2/test/merge/unsorted.bed\n",
            "bedtools2/test/merge/bug254_d.bed\n",
            "bedtools2/test/merge/bug254_e.bed\n",
            "bedtools2/test/merge/bamCol6Collapse.txt\n",
            "bedtools2/test/merge/bamCol3Collapse.txt\n",
            "bedtools2/test/merge/vcfSVtest.vcf\n",
            "bedtools2/test/merge/bug254_c.bed\n",
            "bedtools2/test/merge/bamCol11Collapse.txt\n",
            "bedtools2/test/merge/a.full.bed\n",
            "bedtools2/test/merge/a.bed\n",
            "bedtools2/test/merge/test-merge.sh\n",
            "bedtools2/test/merge/precisionTest.bed\n",
            "bedtools2/test/merge/b.bed\n",
            "bedtools2/test/merge/a.names.bed\n",
            "bedtools2/test/merge/testA.vcf\n",
            "bedtools2/test/merge/bamCol1Collapse.txt\n",
            "bedtools2/test/merge/precisionTest2.bed\n",
            "bedtools2/test/merge/bamCol10Collapse.txt\n",
            "bedtools2/test/merge/bamCol5Mean.txt\n",
            "bedtools2/test/merge/a.gff\n",
            "bedtools2/test/merge/bamCol7Collapse.txt\n",
            "bedtools2/test/merge/bamCol4Mean.txt\n",
            "bedtools2/test/merge/a.full.bam\n",
            "bedtools2/test/merge/mixedStrands.bed\n",
            "bedtools2/test/merge/fullFields.bam\n",
            "bedtools2/test/subtract/test-subtract.sh\n",
            "bedtools2/test/subtract/a.bed\n",
            "bedtools2/test/subtract/c.bed\n",
            "bedtools2/test/subtract/b.bed\n",
            "bedtools2/test/subtract/b2.bed\n",
            "bedtools2/test/subtract/d.bed\n",
            "bedtools2/test/sort/names.txt\n",
            "bedtools2/test/sort/a.bed\n",
            "bedtools2/test/sort/test-sort.sh\n",
            "bedtools2/test/sort/b.bed\n",
            "bedtools2/test/intersect/two_blocks.sam\n",
            "bedtools2/test/intersect/a_with_bothUnmapped.bam\n",
            "bedtools2/test/intersect/new_test-intersect.sh~\n",
            "bedtools2/test/intersect/gdc.gff\n",
            "bedtools2/test/intersect/bug223_sv1_b.vcf\n",
            "bedtools2/test/intersect/bed3.bed\n",
            "bedtools2/test/intersect/bug150_a.bed\n",
            "bedtools2/test/intersect/test_ref.fa.fai\n",
            "bedtools2/test/intersect/x.bam\n",
            "bedtools2/test/intersect/exons.issue750.bed\n",
            "bedtools2/test/intersect/performanceTest.sh\n",
            "bedtools2/test/intersect/bed6.strand.bed\n",
            "bedtools2/test/intersect/queryUnmappedMateMappedCoordsInvalid.bam\n",
            "bedtools2/test/intersect/bug223_sv1_a.vcf\n",
            "bedtools2/test/intersect/bug150_b.bed\n",
            "bedtools2/test/intersect/headerOnly.vcf\n",
            "bedtools2/test/intersect/gdc_one.bed\n",
            "bedtools2/test/intersect/bed6.strand2.bed\n",
            "bedtools2/test/intersect/null_b.bed\n",
            "bedtools2/test/intersect/a_withLargeHeader.bed\n",
            "bedtools2/test/intersect/human.hg19.vSort.genome\n",
            "bedtools2/test/intersect/gdc.bam\n",
            "bedtools2/test/intersect/bed4.bed\n",
            "bedtools2/test/intersect/chromOrderB.bed\n",
            "bedtools2/test/intersect/bedplus.bed\n",
            "bedtools2/test/intersect/test_ref.fa\n",
            "bedtools2/test/intersect/bed5.bed\n",
            "bedtools2/test/intersect/oneRecordNoNewline.bed\n",
            "bedtools2/test/intersect/aWithHeaderVsB.txt\n",
            "bedtools2/test/intersect/null_c.bed\n",
            "bedtools2/test/intersect/j1.bed\n",
            "bedtools2/test/intersect/b.issue311.vcf\n",
            "bedtools2/test/intersect/null_a.bed\n",
            "bedtools2/test/intersect/oneUnmapped.bam\n",
            "bedtools2/test/intersect/chromOrderA.bed\n",
            "bedtools2/test/intersect/bed6.bed\n",
            "bedtools2/test/intersect/issue_773_b.bed\n",
            "bedtools2/test/intersect/new_test-intersect.sh\n",
            "bedtools2/test/intersect/chromsOutOfOrder.bed\n",
            "bedtools2/test/intersect/jim.bed\n",
            "bedtools2/test/intersect/three_blocks_nomatch.bed\n",
            "bedtools2/test/intersect/b_vcfSVtest.vcf\n",
            "bedtools2/test/intersect/bug223_f.vcf\n",
            "bedtools2/test/intersect/a.issue311.bed\n",
            "bedtools2/test/intersect/three_blocks.sam\n",
            "bedtools2/test/intersect/gdc_exp\n",
            "bedtools2/test/intersect/bug44_b.bed\n",
            "bedtools2/test/intersect/bug223_d.vcf\n",
            "bedtools2/test/intersect/bug223_e.vcf\n",
            "bedtools2/test/intersect/a_testZeroLen.bed\n",
            "bedtools2/test/intersect/a_bgzipped.bed.gz\n",
            "bedtools2/test/intersect/x.bed\n",
            "bedtools2/test/intersect/a_withLargeHeader_bgzipped.bed.gz\n",
            "bedtools2/test/intersect/dosLineChar_a.bed\n",
            "bedtools2/test/intersect/bug44_a.vcf.gz\n",
            "bedtools2/test/intersect/b.cram\n",
            "bedtools2/test/intersect/y.bed\n",
            "bedtools2/test/intersect/test-intersect.sh\n",
            "bedtools2/test/intersect/a_vcfSVtest.vcf\n",
            "bedtools2/test/intersect/blocks.bed12\n",
            "bedtools2/test/intersect/recordsOutOfOrder.bed\n",
            "bedtools2/test/intersect/a.bed\n",
            "bedtools2/test/intersect/jim.vcf\n",
            "bedtools2/test/intersect/c.bed\n",
            "bedtools2/test/intersect/three_blocks_match.bam\n",
            "bedtools2/test/intersect/dosLineCharWithExtraTab_b.bed\n",
            "bedtools2/test/intersect/a.cram\n",
            "bedtools2/test/intersect/b.bed\n",
            "bedtools2/test/intersect/two_blocks_partial.bam.bai\n",
            "bedtools2/test/intersect/foo\n",
            "bedtools2/test/intersect/split.issue750.bam\n",
            "bedtools2/test/intersect/mapped_and_unmapped.sam\n",
            "bedtools2/test/intersect/two_blocks_partial.bam\n",
            "bedtools2/test/intersect/e.bed\n",
            "bedtools2/test/intersect/a_withLargeHeader_gzipped.bed.gz\n",
            "bedtools2/test/intersect/svlen.vcf\n",
            "bedtools2/test/intersect/d.bed\n",
            "bedtools2/test/intersect/splitBug155_a.bed\n",
            "bedtools2/test/intersect/multi_intersect/\n",
            "bedtools2/test/intersect/notexthdr.bam\n",
            "bedtools2/test/intersect/large_b.bed\n",
            "bedtools2/test/intersect/nonamecheck_a.bed\n",
            "bedtools2/test/intersect/issue_773_y.bed\n",
            "bedtools2/test/intersect/sortAndNaming/\n",
            "bedtools2/test/intersect/split.issue750.sam\n",
            "bedtools2/test/intersect/a_gzipped.bed.gz\n",
            "bedtools2/test/intersect/splitBug155_b.bed\n",
            "bedtools2/test/intersect/bug167_strandSweep.bed\n",
            "bedtools2/test/intersect/large_a.bed\n",
            "bedtools2/test/intersect/nonamecheck_b.bed\n",
            "bedtools2/test/intersect/b.issue311.gff\n",
            "bedtools2/test/intersect/two_blocks_partial.bed\n",
            "bedtools2/test/intersect/three_blocks_match.bed\n",
            "bedtools2/test/intersect/human.hg19.genome\n",
            "bedtools2/test/intersect/bug187_a.bed\n",
            "bedtools2/test/intersect/b_testZeroLen.bed\n",
            "bedtools2/test/intersect/one_block.sam\n",
            "bedtools2/test/intersect/bug187_b.bed\n",
            "bedtools2/test/intersect/three_blocks_match_1bp.bed\n",
            "bedtools2/test/intersect/aVSb.bam\n",
            "bedtools2/test/intersect/a.bam\n",
            "bedtools2/test/intersect/sortAndNaming/num_all.bed\n",
            "bedtools2/test/intersect/sortAndNaming/diffs.txt\n",
            "bedtools2/test/intersect/sortAndNaming/db1_noChr.bed\n",
            "bedtools2/test/intersect/sortAndNaming/alpha_all.bed\n",
            "bedtools2/test/intersect/sortAndNaming/test-sort-and-naming.sh\n",
            "bedtools2/test/intersect/sortAndNaming/exp1\n",
            "bedtools2/test/intersect/sortAndNaming/q1_gls.bed\n",
            "bedtools2/test/intersect/sortAndNaming/db1_num.bed\n",
            "bedtools2/test/intersect/sortAndNaming/db2_numBackwards.bed\n",
            "bedtools2/test/intersect/sortAndNaming/num_missing.bed\n",
            "bedtools2/test/intersect/sortAndNaming/q1_num.bed\n",
            "bedtools2/test/intersect/sortAndNaming/db2_num.bed\n",
            "bedtools2/test/intersect/sortAndNaming/db1_leadingZero.txt\n",
            "bedtools2/test/intersect/sortAndNaming/sq1.bed\n",
            "bedtools2/test/intersect/sortAndNaming/db1_gls.bed\n",
            "bedtools2/test/intersect/sortAndNaming/alpha_missing.bed\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/\n",
            "bedtools2/test/intersect/sortAndNaming/sdb1.bed\n",
            "bedtools2/test/intersect/sortAndNaming/db3_numBackwards.bed\n",
            "bedtools2/test/intersect/sortAndNaming/q1a_num.bed\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/curr5v5.txt\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/2.21_500Kvs500K.txt\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/q500K.bed\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/dbchr1.bed\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/2.21_chr1.txt\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/diffs.txt\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/curr500Kvs500K.txt\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/qchr1.bed\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/2.21_5v5.txt\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/currChr1.txt\n",
            "bedtools2/test/intersect/sortAndNaming/bigTests/db500K.bed\n",
            "100 19.6M  100 19.6M    0     0  18.7M      0  0:00:01  0:00:01 --:--:-- 18.7M\n",
            "bedtools2/test/intersect/multi_intersect/query3.bed\n",
            "bedtools2/test/intersect/multi_intersect/query2.bed\n",
            "bedtools2/test/intersect/multi_intersect/d8.bed\n",
            "bedtools2/test/intersect/multi_intersect/d9.bed\n",
            "bedtools2/test/intersect/multi_intersect/query.bed\n",
            "bedtools2/test/intersect/multi_intersect/g.bed\n",
            "bedtools2/test/intersect/multi_intersect/test-multi_intersect.sh\n",
            "bedtools2/test/intersect/multi_intersect/d4.bed\n",
            "bedtools2/test/intersect/multi_intersect/d5.bed\n",
            "bedtools2/test/intersect/multi_intersect/d7.bed\n",
            "bedtools2/test/intersect/multi_intersect/d6.bed\n",
            "bedtools2/test/intersect/multi_intersect/d2.bed\n",
            "bedtools2/test/intersect/multi_intersect/d3.bed\n",
            "bedtools2/test/intersect/multi_intersect/d1.bed\n",
            "bedtools2/test/complement/test-complement.sh\n",
            "bedtools2/test/complement/issue_503.genome\n",
            "bedtools2/test/complement/issue_503.bed\n",
            "bedtools2/test/fisher/test-fisher.sh\n",
            "bedtools2/test/fisher/sim.py\n",
            "bedtools2/test/fisher/plot.py\n",
            "bedtools2/test/fisher/cmp.sh\n",
            "bedtools2/test/fisher/dm6.fai\n",
            "bedtools2/test/fisher/t.60.genome\n",
            "bedtools2/test/fisher/get.sh\n",
            "bedtools2/test/fisher/README.md\n",
            "bedtools2/test/fisher/a.bed\n",
            "bedtools2/test/fisher/b.bed\n",
            "bedtools2/test/fisher/tumor.gff\n",
            "bedtools2/test/fisher/t.500.genome\n",
            "bedtools2/test/fisher/a_merge.bed\n",
            "bedtools2/test/fisher/shuf.sh\n",
            "bedtools2/test/fisher/test.bed\n",
            "bedtools2/test/shuffle/incl.sizedif.bed\n",
            "bedtools2/test/shuffle/test-shuffle.sh\n",
            "bedtools2/test/shuffle/excl.bed\n",
            "bedtools2/test/shuffle/simrep.bed\n",
            "bedtools2/test/shuffle/incl.bed\n",
            "bedtools2/test/expand/expand.txt\n",
            "bedtools2/test/expand/test-expand.sh\n",
            "bedtools2/test/cluster/in.bed\n",
            "bedtools2/test/cluster/test-cluster.sh\n",
            "bedtools2/test/bedtobam/test-bedtobam.sh\n",
            "bedtools2/test/bedtobam/chrsize.tmp\n",
            "bedtools2/test/split/test-split.sh\n",
            "bedtools2/test/split/randData.bed\n",
            "bedtools2/tutorial/answers.md\n",
            "bedtools2/tutorial/template.class.html\n",
            "bedtools2/tutorial/answers.html\n",
            "bedtools2/tutorial/bootstrap.css\n",
            "bedtools2/tutorial/Makefile\n",
            "bedtools2/tutorial/bedtools.html\n",
            "bedtools2/tutorial/bedtools.md\n",
            "bedtools2/tutorial/template.html\n",
            "make[1]: Entering directory '/content/florabert/bedtools2'\n",
            "Building BEDTools:\n",
            "=========================================================\n",
            "CXXFLAGS is [-g -Wall -O2 -std=c++11 ]\n",
            "DETECTED_VERSION = v2.29.1\n",
            "CURRENT_VERSION  = \n",
            "Updating version file.\n",
            "  * compiling src/bedtools.cpp\n",
            "  * compiling src/annotateBed/annotateMain.cpp\n",
            "\u001b[01m\u001b[Ksrc/annotateBed/annotateMain.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint annotate_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/annotateBed/annotateMain.cpp:39:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveTitles\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   39 |     bool \u001b[01;35m\u001b[KhaveTitles\u001b[m\u001b[K     = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/annotateBed/annotateBed.cpp\n",
            "  * compiling src/bamToBed/bamToBed.cpp\n",
            "\u001b[01m\u001b[Ksrc/bamToBed/bamToBed.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint bamtobed_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/bamToBed/bamToBed.cpp:88:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KuseAlignmentScore\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   88 |     bool \u001b[01;35m\u001b[KuseAlignmentScore\u001b[m\u001b[K = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In file included from \u001b[01m\u001b[Ksrc/utils/BamTools/include/api/BamReader.h:1\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/bamToBed/bamToBed.cpp:12\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint htslib_future::sam_hdr_rebuild(bam_hdr_t*)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:32:30:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint sam_hdr_write(samFile*, const bam_hdr_t*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   32 |                 \u001b[01;35m\u001b[Ksam_hdr_write(fp, hdr)\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/bamToFastq/bamToFastqMain.cpp\n",
            "\u001b[01m\u001b[Ksrc/bamToFastq/bamToFastqMain.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint bamtofastq_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/bamToFastq/bamToFastqMain.cpp:38:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveFastq2\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   38 |     bool \u001b[01;35m\u001b[KhaveFastq2\u001b[m\u001b[K    = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/bamToFastq/bamToFastq.cpp\n",
            "In file included from \u001b[01m\u001b[Ksrc/utils/BamTools/include/api/BamReader.h:1\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/bamToFastq/bamToFastq.h:15\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/bamToFastq/bamToFastq.cpp:15\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint htslib_future::sam_hdr_rebuild(bam_hdr_t*)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:32:30:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint sam_hdr_write(samFile*, const bam_hdr_t*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   32 |                 \u001b[01;35m\u001b[Ksam_hdr_write(fp, hdr)\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/bedToBam/bedToBam.cpp\n",
            "\u001b[01m\u001b[Ksrc/bedToBam/bedToBam.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint bedtobam_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/bedToBam/bedToBam.cpp:60:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveMapQual\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   60 |     bool \u001b[01;35m\u001b[KhaveMapQual\u001b[m\u001b[K     = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/bedpeToBam/bedpeToBam.cpp\n",
            "\u001b[01m\u001b[Ksrc/bedpeToBam/bedpeToBam.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint bedpetobam_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/bedpeToBam/bedpeToBam.cpp:61:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveMapQual\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   61 |     bool \u001b[01;35m\u001b[KhaveMapQual\u001b[m\u001b[K     = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/bedToIgv/bedToIgv.cpp\n",
            "  * compiling src/bed12ToBed6/bed12ToBed6.cpp\n",
            "  * compiling src/closestFile/closestHelp.cpp\n",
            "  * compiling src/closestFile/closestFile.cpp\n",
            "  * compiling src/clusterBed/clusterMain.cpp\n",
            "\u001b[01m\u001b[Ksrc/clusterBed/clusterMain.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint cluster_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/clusterBed/clusterMain.cpp:38:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveMaxDistance\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   38 |     bool \u001b[01;35m\u001b[KhaveMaxDistance\u001b[m\u001b[K = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/clusterBed/clusterBed.cpp\n",
            "  * compiling src/complementFile/complementHelp.cpp\n",
            "  * compiling src/complementFile/complementFile.cpp\n",
            "  * compiling src/coverageFile/coverageHelp.cpp\n",
            "  * compiling src/coverageFile/coverageFile.cpp\n",
            "\u001b[01m\u001b[Ksrc/coverageFile/coverageFile.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual void CoverageFile::giveFinalReport(RecordOutputMgr*)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/coverageFile/coverageFile.cpp:100:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint asprintf(char**, const char*, ...)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  100 |                 \u001b[01;35m\u001b[Kasprintf(&depthPctString, \"%0.7f\", depthPct)\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/coverageFile/coverageFile.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid CoverageFile::doMean(RecordOutputMgr*, RecordKeyVector&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/coverageFile/coverageFile.cpp:181:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint asprintf(char**, const char*, ...)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  181 |         \u001b[01;35m\u001b[Kasprintf(&meanString, \"%0.7f\", mean)\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/coverageFile/coverageFile.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid CoverageFile::doHist(RecordOutputMgr*, RecordKeyVector&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/coverageFile/coverageFile.cpp:210:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint asprintf(char**, const char*, ...)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  210 |                 \u001b[01;35m\u001b[Kasprintf(&coveredFractionString, \"%0.7f\", coveredFraction)\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/coverageFile/coverageFile.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid CoverageFile::doDefault(RecordOutputMgr*, RecordKeyVector&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/coverageFile/coverageFile.cpp:230:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint asprintf(char**, const char*, ...)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  230 |         \u001b[01;35m\u001b[Kasprintf(&coveredFractionString, \"%0.7f\", coveredFraction)\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/expand/expand.cpp\n",
            "  * compiling src/fastaFromBed/fastaFromBedMain.cpp\n",
            "  * compiling src/fastaFromBed/fastaFromBed.cpp\n",
            "  * compiling src/flankBed/flankBedMain.cpp\n",
            "  * compiling src/flankBed/flankBed.cpp\n",
            "  * compiling src/genomeCoverageBed/genomeCoverageMain.cpp\n",
            "  * compiling src/genomeCoverageBed/genomeCoverageBed.cpp\n",
            "In file included from \u001b[01m\u001b[Ksrc/utils/BamTools/include/api/BamReader.h:1\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/utils/GenomeFile/NewGenomeFile.h:20\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/genomeCoverageBed/genomeCoverageBed.h:13\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/genomeCoverageBed/genomeCoverageBed.cpp:13\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint htslib_future::sam_hdr_rebuild(bam_hdr_t*)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:32:30:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint sam_hdr_write(samFile*, const bam_hdr_t*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   32 |                 \u001b[01;35m\u001b[Ksam_hdr_write(fp, hdr)\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/getOverlap/getOverlap.cpp\n",
            "\u001b[01m\u001b[Ksrc/getOverlap/getOverlap.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint getoverlap_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/getOverlap/getOverlap.cpp:44:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveColumns\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   44 |     bool \u001b[01;35m\u001b[KhaveColumns\u001b[m\u001b[K = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/groupBy/groupByHelp.cpp\n",
            "  * compiling src/groupBy/groupBy.cpp\n",
            "  * compiling src/intersectFile/intersectHelp.cpp\n",
            "  * compiling src/intersectFile/intersectFile.cpp\n",
            "  * compiling src/fisher/fisherHelp.cpp\n",
            "  * compiling src/fisher/fisher.cpp\n",
            "  * compiling src/fisher/kfunc.cpp\n",
            "  * compiling src/jaccard/jaccardHelp.cpp\n",
            "  * compiling src/jaccard/jaccard.cpp\n",
            "  * compiling src/linksBed/linksMain.cpp\n",
            "  * compiling src/linksBed/linksBed.cpp\n",
            "  * compiling src/maskFastaFromBed/maskFastaFromBedMain.cpp\n",
            "  * compiling src/maskFastaFromBed/maskFastaFromBed.cpp\n",
            "  * compiling src/mapFile/mapHelp.cpp\n",
            "  * compiling src/mapFile/mapFile.cpp\n",
            "  * compiling src/mergeFile/mergeHelp.cpp\n",
            "  * compiling src/mergeFile/mergeFile.cpp\n",
            "  * compiling src/multiBamCov/multiBamCovMain.cpp\n",
            "\u001b[01m\u001b[Ksrc/multiBamCov/multiBamCovMain.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint multibamcov_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/multiBamCov/multiBamCovMain.cpp:38:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveBed\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   38 |     bool \u001b[01;35m\u001b[KhaveBed\u001b[m\u001b[K           = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/multiBamCov/multiBamCovMain.cpp:39:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveBams\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   39 |     bool \u001b[01;35m\u001b[KhaveBams\u001b[m\u001b[K          = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/multiBamCov/multiBamCovMain.cpp:47:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveFraction\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   47 |     bool \u001b[01;35m\u001b[KhaveFraction\u001b[m\u001b[K       = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/multiBamCov/multiBamCov.cpp\n",
            "In file included from \u001b[01m\u001b[Ksrc/utils/BamTools/include/BamReader.hpp:4\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/utils/BamTools/include/api/BamMultiReader.h:4\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/multiBamCov/multiBamCov.h:16\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/multiBamCov/multiBamCov.cpp:13\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint htslib_future::sam_hdr_rebuild(bam_hdr_t*)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:32:30:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint sam_hdr_write(samFile*, const bam_hdr_t*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   32 |                 \u001b[01;35m\u001b[Ksam_hdr_write(fp, hdr)\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/multiIntersectBed/multiIntersectBedMain.cpp\n",
            "\u001b[01m\u001b[Ksrc/multiIntersectBed/multiIntersectBedMain.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint multiintersect_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/multiIntersectBed/multiIntersectBedMain.cpp:45:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveFiles\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   45 |     bool \u001b[01;35m\u001b[KhaveFiles\u001b[m\u001b[K         = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/multiIntersectBed/multiIntersectBedMain.cpp:47:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveGenome\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   47 |     bool \u001b[01;35m\u001b[KhaveGenome\u001b[m\u001b[K        = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/multiIntersectBed/multiIntersectBedMain.cpp:48:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveFiller\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   48 |     bool \u001b[01;35m\u001b[KhaveFiller\u001b[m\u001b[K        = true;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/multiIntersectBed/multiIntersectBed.cpp\n",
            "  * compiling src/nucBed/nucBedMain.cpp\n",
            "  * compiling src/nucBed/nucBed.cpp\n",
            "  * compiling src/pairToBed/pairToBedMain.cpp\n",
            "\u001b[01m\u001b[Ksrc/pairToBed/pairToBedMain.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint pairtobed_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/pairToBed/pairToBedMain.cpp:43:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveFraction\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   43 |     bool \u001b[01;35m\u001b[KhaveFraction\u001b[m\u001b[K       = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/pairToBed/pairToBed.cpp\n",
            "\u001b[01m\u001b[Ksrc/pairToBed/pairToBed.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid BedIntersectPE::FindSpanningOverlaps(const BEDPE&, std::vector<BED>&, const string&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/pairToBed/pairToBed.cpp:257:12:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KspanLength\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  257 |     CHRPOS \u001b[01;35m\u001b[KspanLength\u001b[m\u001b[K = 0;\n",
            "      |            \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/pairToBed/pairToBed.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool BedIntersectPE::FindOneOrMoreSpanningOverlaps(const BEDPE&, const string&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/pairToBed/pairToBed.cpp:310:12:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KspanLength\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  310 |     CHRPOS \u001b[01;35m\u001b[KspanLength\u001b[m\u001b[K = 0;\n",
            "      |            \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "In file included from \u001b[01m\u001b[Ksrc/utils/BamTools/include/api/BamReader.h:1\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/pairToBed/pairToBed.h:15\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/pairToBed/pairToBed.cpp:13\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint htslib_future::sam_hdr_rebuild(bam_hdr_t*)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:32:30:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint sam_hdr_write(samFile*, const bam_hdr_t*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   32 |                 \u001b[01;35m\u001b[Ksam_hdr_write(fp, hdr)\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/pairToPair/pairToPairMain.cpp\n",
            "\u001b[01m\u001b[Ksrc/pairToPair/pairToPairMain.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint pairtopair_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/pairToPair/pairToPairMain.cpp:44:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveFraction\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   44 |     bool \u001b[01;35m\u001b[KhaveFraction\u001b[m\u001b[K = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/pairToPair/pairToPair.cpp\n",
            "\u001b[01m\u001b[Ksrc/pairToPair/pairToPair.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid PairToPair::FindOverlaps(const BEDPE&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/pairToPair/pairToPair.cpp:112:14:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[Kfound1\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  112 |         bool \u001b[01;35m\u001b[Kfound1\u001b[m\u001b[K = false;\n",
            "      |              \u001b[01;35m\u001b[K^~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/pairToPair/pairToPair.cpp:113:14:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[Kfound2\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  113 |         bool \u001b[01;35m\u001b[Kfound2\u001b[m\u001b[K = false;\n",
            "      |              \u001b[01;35m\u001b[K^~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/randomBed/randomBedMain.cpp\n",
            "  * compiling src/randomBed/randomBed.cpp\n",
            "  * compiling src/regressTest/regressTestMain.cpp\n",
            "  * compiling src/regressTest/RegressTest.cpp\n",
            "\u001b[01m\u001b[Ksrc/regressTest/RegressTest.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool RegressTest::executeAndCompareCorrectness(const fileListType&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/regressTest/RegressTest.cpp:431:15:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  431 |         \u001b[01;35m\u001b[Ksystem(diffCmd.c_str())\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/regressTest/RegressTest.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool RegressTest::startMemoryProfile(bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/regressTest/RegressTest.cpp:573:14:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kchar* fgets(char*, int, FILE*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  573 |         \u001b[01;35m\u001b[Kfgets(sLine, 4192, fp)\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K~~~~~^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/regressTest/RegressTest.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool RegressTest::calcMemoryStats()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/regressTest/RegressTest.cpp:622:22:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kchar* fgets(char*, int, FILE*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  622 |                 \u001b[01;35m\u001b[Kfgets(sLine, 4192, fp)\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/reldist/reldistMain.cpp\n",
            "  * compiling src/reldist/reldist.cpp\n",
            "\u001b[01m\u001b[Ksrc/reldist/reldist.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid RelativeDistance::CalculateRelativeDistance()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/reldist/reldist.cpp:129:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison of integer expressions of different signedness: ‘\u001b[01m\u001b[KCHRPOS\u001b[m\u001b[K’ {aka ‘\u001b[01m\u001b[Klong int\u001b[m\u001b[K’} and ‘\u001b[01m\u001b[Kstd::vector<long int>::size_type\u001b[m\u001b[K’ {aka ‘\u001b[01m\u001b[Klong unsigned int\u001b[m\u001b[K’} [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wsign-compare\u0007-Wsign-compare\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  129 |             if (\u001b[01;35m\u001b[Klow_idx != chrom_mids->size() - 1\u001b[m\u001b[K)\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/sampleFile/sampleHelp.cpp\n",
            "  * compiling src/sampleFile/sampleFile.cpp\n",
            "  * compiling src/shiftBed/shiftBedMain.cpp\n",
            "  * compiling src/shiftBed/shiftBed.cpp\n",
            "  * compiling src/shuffleBed/shuffleBedMain.cpp\n",
            "  * compiling src/shuffleBed/shuffleBed.cpp\n",
            "  * compiling src/slopBed/slopBedMain.cpp\n",
            "  * compiling src/slopBed/slopBed.cpp\n",
            "  * compiling src/sortBed/sortMain.cpp\n",
            "  * compiling src/sortBed/sortBed.cpp\n",
            "  * compiling src/spacingFile/spacingHelp.cpp\n",
            "  * compiling src/spacingFile/spacingFile.cpp\n",
            "  * compiling src/split/splitBedMain.cpp\n",
            "  * compiling src/split/splitBed.cpp\n",
            "  * compiling src/subtractFile/subtractHelp.cpp\n",
            "  * compiling src/subtractFile/subtractFile.cpp\n",
            "  * compiling src/summaryFile/summaryHelp.cpp\n",
            "  * compiling src/summaryFile/summaryFile.cpp\n",
            "  * compiling src/tagBam/tagBamMain.cpp\n",
            "  * compiling src/tagBam/tagBam.cpp\n",
            "In file included from \u001b[01m\u001b[Ksrc/utils/BamTools/include/api/BamReader.h:1\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/tagBam/tagBam.h:17\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/tagBam/tagBam.cpp:13\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint htslib_future::sam_hdr_rebuild(bam_hdr_t*)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:32:30:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint sam_hdr_write(samFile*, const bam_hdr_t*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   32 |                 \u001b[01;35m\u001b[Ksam_hdr_write(fp, hdr)\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/unionBedGraphs/unionBedGraphsMain.cpp\n",
            "\u001b[01m\u001b[Ksrc/unionBedGraphs/unionBedGraphsMain.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint unionbedgraphs_main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/unionBedGraphs/unionBedGraphsMain.cpp:46:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveFiles\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   46 |     bool \u001b[01;35m\u001b[KhaveFiles\u001b[m\u001b[K         = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/unionBedGraphs/unionBedGraphsMain.cpp:48:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveGenome\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   48 |     bool \u001b[01;35m\u001b[KhaveGenome\u001b[m\u001b[K        = false;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/unionBedGraphs/unionBedGraphsMain.cpp:49:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KhaveFiller\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   49 |     bool \u001b[01;35m\u001b[KhaveFiller\u001b[m\u001b[K        = true;\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/unionBedGraphs/unionBedGraphs.cpp\n",
            "  * compiling src/windowBed/windowMain.cpp\n",
            "  * compiling src/windowBed/windowBed.cpp\n",
            "In file included from \u001b[01m\u001b[Ksrc/utils/BamTools/include/api/BamReader.h:1\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/windowBed/windowBed.h:15\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/windowBed/windowBed.cpp:13\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint htslib_future::sam_hdr_rebuild(bam_hdr_t*)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:32:30:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint sam_hdr_write(samFile*, const bam_hdr_t*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   32 |                 \u001b[01;35m\u001b[Ksam_hdr_write(fp, hdr)\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/windowMaker/windowMakerMain.cpp\n",
            "  * compiling src/windowMaker/windowMaker.cpp\n",
            "  * compiling src/utils/FileRecordTools/FileRecordMgr.cpp\n",
            "  * compiling src/utils/FileRecordTools/FileRecordMergeMgr.cpp\n",
            "  * compiling src/utils/FileRecordTools/FileReaders/FileReader.cpp\n",
            "  * compiling src/utils/FileRecordTools/FileReaders/SingleLineDelimTextFileReader.cpp\n",
            "  * compiling src/utils/FileRecordTools/FileReaders/BamFileReader.cpp\n",
            "  * compiling src/utils/FileRecordTools/FileReaders/BufferedStreamMgr.cpp\n",
            "\u001b[01m\u001b[Ksrc/utils/FileRecordTools/FileReaders/BufferedStreamMgr.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool BufferedStreamMgr::readFileChunk()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/FileRecordTools/FileReaders/BufferedStreamMgr.cpp:147:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison of integer expressions of different signedness: ‘\u001b[01m\u001b[Kint\u001b[m\u001b[K’ and ‘\u001b[01m\u001b[Ksize_t\u001b[m\u001b[K’ {aka ‘\u001b[01m\u001b[Klong unsigned int\u001b[m\u001b[K’} [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wsign-compare\u0007-Wsign-compare\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  147 |                 if (\u001b[01;35m\u001b[K_mainBufCurrLen < _useBufSize\u001b[m\u001b[K) {\n",
            "      |                     \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/utils/FileRecordTools/FileReaders/InputStreamMgr.cpp\n",
            "In file included from \u001b[01m\u001b[Ksrc/utils/BamTools/include/api/BamReader.h:1\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/utils/FileRecordTools/FileReaders/InputStreamMgr.h:14\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Ksrc/utils/FileRecordTools/FileReaders/InputStreamMgr.cpp:8\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint htslib_future::sam_hdr_rebuild(bam_hdr_t*)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/BamTools/include/SamHeader.hpp:32:30:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint sam_hdr_write(samFile*, const bam_hdr_t*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   32 |                 \u001b[01;35m\u001b[Ksam_hdr_write(fp, hdr)\u001b[m\u001b[K;\n",
            "      |                 \u001b[01;35m\u001b[K~~~~~~~~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/utils/FileRecordTools/Records/Record.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/EmptyRecord.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/Bed3Interval.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/Bed4Interval.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/BedGraphInterval.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/Bed5Interval.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/Bed6Interval.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/PlusFields.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/GffRecord.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/GffPlusRecord.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/NoPosPlusRecord.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/BedPlusInterval.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/Bed12Interval.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/BamRecord.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/VcfRecord.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/BlockMgr.cpp\n",
            "\u001b[01m\u001b[Ksrc/utils/FileRecordTools/Records/BlockMgr.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kint BlockMgr::getNonRedundantOverlap()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/FileRecordTools/Records/BlockMgr.cpp:194:35:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison of integer expressions of different signedness: ‘\u001b[01m\u001b[Kint\u001b[m\u001b[K’ and ‘\u001b[01m\u001b[Kstd::vector<long int>::size_type\u001b[m\u001b[K’ {aka ‘\u001b[01m\u001b[Klong unsigned int\u001b[m\u001b[K’} [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wsign-compare\u0007-Wsign-compare\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  194 |                 for (int i = 1; \u001b[01;35m\u001b[Ki < _overlapStarts.size()\u001b[m\u001b[K; ++i)\n",
            "      |                                 \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/utils/FileRecordTools/Records/StrandQueue.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/RecordMgr.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/RecordList.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/RecordKeyList.cpp\n",
            "  * compiling src/utils/FileRecordTools/Records/RecordKeyVector.cpp\n",
            "  * compiling src/utils/bedFile/bedFile.cpp\n",
            "  * compiling src/utils/BinTree/BinTree.cpp\n",
            "  * compiling src/utils/version/version.cpp\n",
            "  * compiling src/utils/bedGraphFile/bedGraphFile.cpp\n",
            "  * compiling src/utils/chromsweep/chromsweep.cpp\n",
            "  * compiling src/utils/Contexts/ContextBase.cpp\n",
            "  * compiling src/utils/Contexts/ContextIntersect.cpp\n",
            "  * compiling src/utils/Contexts/ContextFisher.cpp\n",
            "  * compiling src/utils/Contexts/ContextMap.cpp\n",
            "  * compiling src/utils/Contexts/ContextSample.cpp\n",
            "  * compiling src/utils/Contexts/ContextSpacing.cpp\n",
            "  * compiling src/utils/Contexts/ContextMerge.cpp\n",
            "  * compiling src/utils/Contexts/ContextJaccard.cpp\n",
            "  * compiling src/utils/Contexts/ContextClosest.cpp\n",
            "  * compiling src/utils/Contexts/ContextSubtract.cpp\n",
            "  * compiling src/utils/Contexts/ContextCoverage.cpp\n",
            "  * compiling src/utils/Contexts/ContextComplement.cpp\n",
            "  * compiling src/utils/Contexts/ContextGroupBy.cpp\n",
            "  * compiling src/utils/Contexts/ContextSummary.cpp\n",
            "  * compiling src/utils/general/ParseTools.cpp\n",
            "  * compiling src/utils/general/PushBackStreamBuf.cpp\n",
            "  * compiling src/utils/general/CompressionTools.cpp\n",
            "  * compiling src/utils/general/Tokenizer.cpp\n",
            "  * compiling src/utils/general/CommonHelp.cpp\n",
            "  * compiling src/utils/general/ErrorMsg.cpp\n",
            "g++ -g -Wall -O2 -std=c++11  -Isrc/utils/gzstream  -c -o obj/gzstream.o src/utils/gzstream/gzstream.C\n",
            "  * compiling src/utils/fileType/fileType.cpp\n",
            "  * compiling src/utils/fileType/FileRecordTypeChecker.cpp\n",
            "\u001b[01m\u001b[Ksrc/utils/fileType/FileRecordTypeChecker.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool FileRecordTypeChecker::isTextDelimtedFormat(const char*, size_t)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/fileType/FileRecordTypeChecker.cpp:366:33:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison of integer expressions of different signedness: ‘\u001b[01m\u001b[Kint\u001b[m\u001b[K’ and ‘\u001b[01m\u001b[Ksize_t\u001b[m\u001b[K’ {aka ‘\u001b[01m\u001b[Klong unsigned int\u001b[m\u001b[K’} [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wsign-compare\u0007-Wsign-compare\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  366 |                 for (int j=0; \u001b[01;35m\u001b[Kj < len\u001b[m\u001b[K; j++) {\n",
            "      |                               \u001b[01;35m\u001b[K~~^~~~~\u001b[m\u001b[K\n",
            "  * compiling src/utils/bedFilePE/bedFilePE.cpp\n",
            "  * compiling src/utils/KeyListOps/KeyListOps.cpp\n",
            "  * compiling src/utils/KeyListOps/KeyListOpsMethods.cpp\n",
            "  * compiling src/utils/NewChromsweep/NewChromsweep.cpp\n",
            "  * compiling src/utils/NewChromsweep/CloseSweep.cpp\n",
            "  * compiling src/utils/sequenceUtilities/sequenceUtils.cpp\n",
            "  * compiling src/utils/tabFile/tabFile.cpp\n",
            "  * compiling src/utils/BamTools-Ancillary/BamAncillary.cpp\n",
            "  * compiling src/utils/BlockedIntervals/BlockedIntervals.cpp\n",
            "  * compiling src/utils/Fasta/Fasta.cpp\n",
            "\u001b[01m\u001b[Ksrc/utils/Fasta/Fasta.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kstd::string FastaReference::getSequence(std::string)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/Fasta/Fasta.cpp:292:14:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksize_t fread(void*, size_t, size_t, FILE*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  292 |         \u001b[01;35m\u001b[Kfread(seq, sizeof(char), seqlen, file)\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/utils/Fasta/Fasta.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kstd::string FastaReference::getSubSequence(std::string, CHRPOS, CHRPOS)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/Fasta/Fasta.cpp:334:14:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksize_t fread(void*, size_t, size_t, FILE*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  334 |         \u001b[01;35m\u001b[Kfread(seq, sizeof(char), (off_t) seqlen, file)\u001b[m\u001b[K;\n",
            "      |         \u001b[01;35m\u001b[K~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/utils/Fasta/split.cpp\n",
            "  * compiling src/utils/VectorOps/VectorOps.cpp\n",
            "  * compiling src/utils/GenomeFile/NewGenomeFile.cpp\n",
            "  * compiling src/utils/GenomeFile/GenomeFile.cpp\n",
            "\u001b[01m\u001b[Ksrc/utils/GenomeFile/GenomeFile.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid GenomeFile::loadGenomeFileIntoMap()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/GenomeFile/GenomeFile.cpp:60:26:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[Kc2\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-but-set-variable\u0007-Wunused-but-set-variable\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   60 |                     long \u001b[01;35m\u001b[Kc2\u001b[m\u001b[K;\n",
            "      |                          \u001b[01;35m\u001b[K^~\u001b[m\u001b[K\n",
            "  * compiling src/utils/RecordOutputMgr/RecordOutputMgr.cpp\n",
            "\u001b[01m\u001b[Ksrc/utils/RecordOutputMgr/RecordOutputMgr.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid RecordOutputMgr::printClosest(RecordKeyVector&, const std::vector<long int>*)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/RecordOutputMgr/RecordOutputMgr.cpp:170:56:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison of integer expressions of different signedness: ‘\u001b[01m\u001b[Kint\u001b[m\u001b[K’ and ‘\u001b[01m\u001b[Ksize_t\u001b[m\u001b[K’ {aka ‘\u001b[01m\u001b[Klong unsigned int\u001b[m\u001b[K’} [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wsign-compare\u0007-Wsign-compare\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  170 |                 if (\u001b[01;35m\u001b[Kcontext->getNumClosestHitsWanted() > keyList.size()\u001b[m\u001b[K)\n",
            "      |                     \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Ksrc/utils/RecordOutputMgr/RecordOutputMgr.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid RecordOutputMgr::reportOverlapSummary(RecordKeyVector&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Ksrc/utils/RecordOutputMgr/RecordOutputMgr.cpp:455:38:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison of integer expressions of different signedness: ‘\u001b[01m\u001b[Ksize_t\u001b[m\u001b[K’ {aka ‘\u001b[01m\u001b[Klong unsigned int\u001b[m\u001b[K’} and ‘\u001b[01m\u001b[Kint\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wsign-compare\u0007-Wsign-compare\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  455 |                 for (size_t i = 1; \u001b[01;35m\u001b[Ki < _context->getNumInputFiles()\u001b[m\u001b[K; i++)\n",
            "      |                                    \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  * compiling src/utils/ToolBase/ToolBase.cpp\n",
            "  * compiling src/utils/driver/BedtoolsDriver.cpp\n",
            "cd src/utils/htslib && make config.h\n",
            "make[2]: Entering directory '/content/florabert/bedtools2/src/utils/htslib'\n",
            "echo '/* Default config.h generated by Makefile */' > config.h\n",
            "echo '#define HAVE_LIBBZ2 1' >> config.h\n",
            "echo '#define HAVE_LIBLZMA 1' >> config.h\n",
            "echo '#ifndef __APPLE__' >> config.h\n",
            "echo '#define HAVE_LZMA_H 1' >> config.h\n",
            "echo '#endif' >> config.h\n",
            "echo '#define HAVE_FSEEKO 1' >> config.h\n",
            "echo '#define HAVE_DRAND48 1' >> config.h\n",
            "make[2]: Leaving directory '/content/florabert/bedtools2/src/utils/htslib'\n",
            "cd src/utils/htslib && make lib-static\n",
            "make[2]: Entering directory '/content/florabert/bedtools2/src/utils/htslib'\n",
            "gcc -g -Wall -O2 -I.  -c -o kfunc.o kfunc.c\n",
            "gcc -g -Wall -O2 -I.  -c -o knetfile.o knetfile.c\n",
            "gcc -g -Wall -O2 -I.  -c -o kstring.o kstring.c\n",
            "gcc -g -Wall -O2 -I.  -c -o bcf_sr_sort.o bcf_sr_sort.c\n",
            "gcc -g -Wall -O2 -I.  -c -o bgzf.o bgzf.c\n",
            "gcc -g -Wall -O2 -I.  -c -o errmod.o errmod.c\n",
            "gcc -g -Wall -O2 -I.  -c -o faidx.o faidx.c\n",
            "gcc -g -Wall -O2 -I.  -c -o hfile.o hfile.c\n",
            "gcc -g -Wall -O2 -I.  -c -o hfile_net.o hfile_net.c\n",
            "echo '#define HTS_VERSION \"1.9\"' > version.h\n",
            "gcc -g -Wall -O2 -I.  -c -o hts.o hts.c\n",
            "\u001b[01m\u001b[Khts.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Khts_idx_init\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Khts.c:1397:59:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Koverflow in conversion from ‘\u001b[01m\u001b[Kuint32_t\u001b[m\u001b[K’ {aka ‘\u001b[01m\u001b[Kunsigned int\u001b[m\u001b[K’} to ‘\u001b[01m\u001b[Kint\u001b[m\u001b[K’ changes value from ‘\u001b[01m\u001b[Kidx->z.last_bin = 4294967295\u001b[m\u001b[K’ to ‘\u001b[01m\u001b[K-1\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Woverflow\u0007-Woverflow\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1397 |     idx->z.save_bin = idx->z.save_tid = idx->z.last_tid = \u001b[01;35m\u001b[Kidx\u001b[m\u001b[K->z.last_bin = 0xffffffffu;\n",
            "      |                                                           \u001b[01;35m\u001b[K^~~\u001b[m\u001b[K\n",
            "gcc -g -Wall -O2 -I.  -c -o hts_os.o hts_os.c\n",
            "gcc -g -Wall -O2 -I.  -c -o md5.o md5.c\n",
            "gcc -g -Wall -O2 -I.  -c -o multipart.o multipart.c\n",
            "gcc -g -Wall -O2 -I.  -c -o probaln.o probaln.c\n",
            "gcc -g -Wall -O2 -I.  -c -o realn.o realn.c\n",
            "gcc -g -Wall -O2 -I.  -c -o regidx.o regidx.c\n",
            "gcc -g -Wall -O2 -I.  -c -o sam.o sam.c\n",
            "gcc -g -Wall -O2 -I.  -c -o synced_bcf_reader.o synced_bcf_reader.c\n",
            "gcc -g -Wall -O2 -I.  -c -o vcf_sweep.o vcf_sweep.c\n",
            "gcc -g -Wall -O2 -I.  -c -o tbx.o tbx.c\n",
            "gcc -g -Wall -O2 -I.  -c -o textutils.o textutils.c\n",
            "gcc -g -Wall -O2 -I.  -c -o thread_pool.o thread_pool.c\n",
            "gcc -g -Wall -O2 -I.  -c -o vcf.o vcf.c\n",
            "gcc -g -Wall -O2 -I.  -c -o vcfutils.o vcfutils.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/cram_codecs.o cram/cram_codecs.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/cram_decode.o cram/cram_decode.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/cram_encode.o cram/cram_encode.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/cram_external.o cram/cram_external.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/cram_index.o cram/cram_index.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/cram_io.o cram/cram_io.c\n",
            "In file included from \u001b[01m\u001b[K/usr/include/string.h:535\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Kcram/cram_io.c:53\u001b[m\u001b[K:\n",
            "In function ‘\u001b[01m\u001b[Kstrncpy\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kcram_populate_ref\u001b[m\u001b[K’ at \u001b[01m\u001b[Kcram/cram_io.c:2095:9\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/string_fortified.h:95:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K__builtin_strncpy\u001b[m\u001b[K’ specified bound 4096 equals destination size [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wstringop-truncation\u0007-Wstringop-truncation\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   95 |   return \u001b[01;35m\u001b[K__builtin___strncpy_chk (__dest, __src, __len,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   96 | \u001b[01;35m\u001b[K                                  __glibc_objsize (__dest))\u001b[m\u001b[K;\n",
            "      |                                   \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In function ‘\u001b[01m\u001b[Kstrncpy\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kfull_path\u001b[m\u001b[K’ at \u001b[01m\u001b[Kcram/cram_io.c:3748:13\u001b[m\u001b[K,\n",
            "    inlined from ‘\u001b[01m\u001b[Kcram_write_SAM_hdr\u001b[m\u001b[K’ at \u001b[01m\u001b[Kcram/cram_io.c:3823:17\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/string_fortified.h:95:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K__builtin_strncpy\u001b[m\u001b[K’ specified bound 4096 equals destination size [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wstringop-truncation\u0007-Wstringop-truncation\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   95 |   return \u001b[01;35m\u001b[K__builtin___strncpy_chk (__dest, __src, __len,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   96 | \u001b[01;35m\u001b[K                                  __glibc_objsize (__dest))\u001b[m\u001b[K;\n",
            "      |                                   \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In function ‘\u001b[01m\u001b[Kstrncpy\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kcram_dopen\u001b[m\u001b[K’ at \u001b[01m\u001b[Kcram/cram_io.c:4126:9\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/string_fortified.h:95:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K__builtin_strncpy\u001b[m\u001b[K’ specified bound 20 equals destination size [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wstringop-truncation\u0007-Wstringop-truncation\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   95 |   return \u001b[01;35m\u001b[K__builtin___strncpy_chk (__dest, __src, __len,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   96 | \u001b[01;35m\u001b[K                                  __glibc_objsize (__dest))\u001b[m\u001b[K;\n",
            "      |                                   \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/cram_samtools.o cram/cram_samtools.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/cram_stats.o cram/cram_stats.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/files.o cram/files.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/mFILE.o cram/mFILE.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/open_trace_file.o cram/open_trace_file.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/pooled_alloc.o cram/pooled_alloc.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/rANS_static.o cram/rANS_static.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/sam_header.o cram/sam_header.c\n",
            "gcc -g -Wall -O2 -I.  -c -o cram/string_alloc.o cram/string_alloc.c\n",
            "In file included from \u001b[01m\u001b[K/usr/include/string.h:535\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[Kcram/string_alloc.c:41\u001b[m\u001b[K:\n",
            "In function ‘\u001b[01m\u001b[Kstrncpy\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kstring_ndup\u001b[m\u001b[K’ at \u001b[01m\u001b[Kcram/string_alloc.c:151:5\u001b[m\u001b[K,\n",
            "    inlined from ‘\u001b[01m\u001b[Kstring_dup\u001b[m\u001b[K’ at \u001b[01m\u001b[Kcram/string_alloc.c:143:12\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/string_fortified.h:95:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K__builtin_strncpy\u001b[m\u001b[K’ output truncated before terminating nul copying as many bytes from a string as its length [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wstringop-truncation\u0007-Wstringop-truncation\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   95 |   return \u001b[01;35m\u001b[K__builtin___strncpy_chk (__dest, __src, __len,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "   96 | \u001b[01;35m\u001b[K                                  __glibc_objsize (__dest))\u001b[m\u001b[K;\n",
            "      |                                   \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Kcram/string_alloc.c:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kstring_dup\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Kcram/string_alloc.c:143:12:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Klength computed here\n",
            "  143 |     return \u001b[01;36m\u001b[Kstring_ndup(a_str, instr, strlen(instr))\u001b[m\u001b[K;\n",
            "      |            \u001b[01;36m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "ar -rc libhts.a kfunc.o knetfile.o kstring.o bcf_sr_sort.o bgzf.o errmod.o faidx.o hfile.o hfile_net.o hts.o hts_os.o md5.o multipart.o probaln.o realn.o regidx.o sam.o synced_bcf_reader.o vcf_sweep.o tbx.o textutils.o thread_pool.o vcf.o vcfutils.o cram/cram_codecs.o cram/cram_decode.o cram/cram_encode.o cram/cram_external.o cram/cram_index.o cram/cram_io.o cram/cram_samtools.o cram/cram_stats.o cram/files.o cram/mFILE.o cram/open_trace_file.o cram/pooled_alloc.o cram/rANS_static.o cram/sam_header.o cram/string_alloc.o\n",
            "ranlib libhts.a\n",
            "make[2]: Leaving directory '/content/florabert/bedtools2/src/utils/htslib'\n",
            "- Building main bedtools binary.\n",
            "done.\n",
            "- Creating executables for old CLI.\n",
            "done.\n",
            "make[1]: Leaving directory '/content/florabert/bedtools2'\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# from module.florabert import config\n",
        "!pip install wget"
      ],
      "metadata": {
        "id": "E4ODl6eWf-tB",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "0309d60e-42a0-4462-b9c1-9b1af3bb31fa"
      },
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting wget\n",
            "  Using cached wget-3.2-py3-none-any.whl\n",
            "Installing collected packages: wget\n",
            "Successfully installed wget-3.2\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install biopython"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "HWbF2wa_eF8C",
        "outputId": "10cf4ac7-8fdc-4b72-8eb0-1b0fe010f8a0"
      },
      "execution_count": 10,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting biopython\n",
            "  Using cached biopython-1.81-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (from biopython) (1.23.5)\n",
            "Installing collected packages: biopython\n",
            "Successfully installed biopython-1.81\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%run /content/florabert/scripts/0-data-loading-processing/02-download-process-db-data.py Ensembl"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "hZ69NcordukG",
        "outputId": "a7ca813b-8093-418e-871e-1c872816e2f3"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Name of database to be processed: Ensembl\n",
            "\n",
            "Finished processing index 0: Actinidia_chinensis.Red5_PS1_1.69.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "CEY00_Acc00111::LG1:4212-5212(+) \n",
            " ATTTATTCAAAATATGAAAATTTGTTGAGAGTCAAAACTCTCTCGTTACACATAAGACAATAAAAATTTACGTGGTTCAGTCAATAGTGCATACATCCACACACAAAAAATTAATAATTCACTAATCGAACGAATGAGATTACAAAGAAGGAGATTCAATCTCTCAATTCAATTCTCTCTCACACATCCACTAAAGCTAAGCCCACTTTACCTAATTAGGTAAATTCACTGCCTGCCATCCCAACTATTCCCATCACGGTGCCCACAATTCCGTCCCTGTCACCACCCCCATCCACCGCTGATCAAGAATGCCACCATTTGATTCAGCTTGTCCCACTTTGATTATTCATTAATTTGCTCCATTTGATTCAATTTCACTGTTTCTTGGTCTCAGTGTTTCTGTTATTGATAAATAAATATCATTCCTTCCTTTTGAATTTATTGTTGAATTAGCTCCTGAAAACTGTTGATTGCAGCAATTAATTTCTCGGTAATATTACGATAATTACATTTGAGTACATGCTATTTTGAGTCAATTAGTGGCATCAGGGGCGAACCCATGTAGGTGTTGAGCCAATTAGCGGCATCAAGACAGACTCATGTAGGGAATGTAAGGGCAAACTTTTGTGTAATTTGCTGTATGCAATATAATTATGTACATATATATGCATATACGTGTGTGTACTCAATAAAAATTATTCGATTAGCATCTTTTCAAAATTTCTACATATTAATCTCACAACTCTACCATCCCTTTTATAGATTAAGAGAATGAGGTGGTAGATTTTTAATAGGATACTTAATTTTTATCATATTTTAACTAAGAAAATATATCATAAGATCGATAGATAGATTTGTGCAACAATTAATTGTATCAATTTACTATTTACTATTGTACTATATCATTTTAGAGGGCTTTATTTTATGTTTGTTGGCCTTTTCTATAGTCATCATCTTCTTGACCTTTTCTATAGTCACAATTGTTTGTTTTTGTTTTTGT\n",
            "\n",
            "Finished processing index 1: Aegilops_tauschii.Aet_v4.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "AET1Gv20000100::1D:36415-37415(-) \n",
            " ATTCGGCTGGCGTTTGTCTTTGGATCCACGTGGCCCTTGTCTTCGTTCGTCTATGTTTGTGTGTCTACAAGTTGAATCCTTTTGATCTACGCTTCTCTTCAGCGGCGACAATTGCTGTTGTGGTGTATGGGACCTTAGAGCGACGATTTTCCGACTGTTTATTACAATAAGGTTTGCCCGGCGCCTTTCAATCCACCTTAGTCTTGCATGTGAGATGCGTGCCTCCCAATCCACCTAAGTCTTCAAGTGAGCACGTGATCGAACGAACACCCGATGGTCTCCTGGCTTAACGACATGCGTTTATGTGGCCAACGGATCAGTCGGTTGATCGTAAGCCTTTTCCATTTTTGCCAGTCTCCTGGCTTTAAAAGGAGCACCCGATGGATAATCCTAGCAAAGCAGGAGTGTAGATGAGCAGAGAGTGTGGACGAAGCAAGATCTCACCATCACGCTCGCCCGAAAGTCCCGAAAGATCAACTCTGTGATGCGGTAAAATCGCCGCGTCAAAACGAAAGCCGCAAAGAAGGAGAAGCTCAGCAAACAGAAGGCCCGCGGAGTTCATGCTTGTGGCGTACGTGGGGACAAGGCAGCAAAACGTACGCAATCCGTCTTAAACTCAAGCCAAAGCCACGTTGTCCGTCTCGGACAGACGCACATCGTCTTATAATATTGAACCCGAGTCCACTTCGTACTACTCTCATCAAGCGATGTACCCTTAAAGTTAGCCCCCGGCGTGTCCCTCCAACAATCAAACGCCCCCGCCTGTCGCCGCCAGCATGAACTGGCTCAGGAACTCGCGCTTAATTTTCTCCAGGCTCCAGATCGTCCGACGCTACTGCAACCCTGCCCAGACACTACCAGGTCCTCGATCGCGTTCCAATATCATGGGCTCAATCCGACGGCTCTTTTTCCGCCACGTCAAGCCTGCCGCCTCCCATATGGCCCATGCGCCGTCGCTGCCGGCAGTCGTCTCACGGAGCCGACCGTGGTGGTACCCGAA\n",
            "\n",
            "Finished processing index 2: Amborella_trichopoda.AMTR1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "AMTR_s00001p00009420::AmTr_v1.0_scaffold00001:6049-7049(-) \n",
            " TCAAAATTAGAGAGCCCCAATAGCCCAGAGACATGTGTTTTAGAGAGAGAAGGATGGTGGATGATGCATGGACAGAGAAAAGGCTTTTAGAGAGAGAGAGAAACGAGTTTGTAGTCTCAACCTCATTTACGATTGACTTGGAATCTGGGAAAGATTCAGTAATTGAATGAAAGCCTGTGGGTGAAGAGAATCTTCGCCATGGTGACCAACCCACAGGGCTCTCTCTTGTTTCTCTCTCTTCACTCACCCACAGTCCCTCTCTTGTTTCTCTCTCTTCACTCACCCACAGTCCCTCTCTTGTTTCTCTCTCTTCTTAATCTCATCCTTTCCATTTCTAGCTGTGTGGGAATCTCGGACAACTTTCTCTCTCTTGTTTCTCTCTCTTCTTAATCTCATCCTTTCCATTTCTAGCTGTGTGGGAATCTCAGACAAGTGCTGCTCTATCTCTTTTGACTTAATTTATGTCCTTCTCTGGTACATGTTTGGGAATCCCAAGAATTAGGGACATAGAGCCTTTTCTCACTCTAGGACAAATCCTAAGAGCTTCTTCCTCCTTGAAATAATCCACAGTAGAGCTATAAAACACATGATGGGTTCTTTTGATTTTTCTGGGTCCTTGTAAAAGTCCTTTTCCATGGCTTTTTGAGGCCTGTTTTATGGATTTCTGCTGCAATAAAAGTGGGTTTTTTTGGGGGTGTTCTTGAGTGGTTTCTCAGTCTAGCAAATGGGTCTCTTTTATTTTAGAGCTCTAAGGGATTCTTGATGGGTTTTTTTTTAATTATTCTCTTGTGCAAGTGGGCCTTTTGATTGAGTTCTTTAGTGGGGTTTTTCTGGAGGATTACCCTGCAATTGGTTTTCCTGCCGAGATTTCTTTCCACTCAGTTTTTGATGGATTCTCTCTTTCTTTCTTTCTTGCCACTCAGTGTTTGATGGATTTTCTCTCTCTTCGTTCTCTCTAGTGAGAAGAGGAACTTGGCTTTCTGAAGAAGAAGACCCATCA\n",
            "\n",
            "Finished processing index 3: Ananas_comosus.F153.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Aco009737.1.path1::contig1:5785-6785(+) \n",
            " GCGAGATACTCTGTGGCCGTTTTAAGAATGTTAGTATCATATTAGATATAAATTAGATTTCAAACAATTACTTATTTGATAAACGTATGTTAAAGGGTATCAAAGTAGACGTTGTGTTCTGCCTATGCCATAAATTTGCGCTGTTTGGTAATTTTTGTTTTTTCTGTCATGTTTTGCTCCGCATCATCTTTTTCTGTCATGTTTTGCTCCGCATCACCAAAGATAAAATAGTTGTAGCTTTGCTTTGCATTTGTTTTCTTAGTAGTGCAATCAAGTATCAGGGTCCATTTGGTTCAGTTATTCAAGTTAGTGTATGATTCGAGTCTCAATTTGATAAATTTATTTATGAGTTCATGGAATTACAACTGTGCTTAGTATGTCTTAATAGTCATGCTCATAATTTCAGTTTAGTGGATTTTTTTACCAATGTAATGGTATTACTTGCTATTTAGTTGGTGCATTGTGTTAGGTTATGTTTGTCAATTAAAAGACTTGGAAAAAGAGAGGTATCTGACCCTCTTCAACTGCTTTTAGTTTGCATTTTGCAGAGTCATATGAGTGATTTATATTGCTTAGGAGAATCTACATGTAAAAGTGCAATCATTTTGGTGGGTAATGTTGGATGGAGTTCGAGTACTATGTTTCTTCATTTTGCACTCATTAAATTTCACAGAATGGTTTTTTTTCTTTTTTTGGTTTGTATTGGAAAGAATTGCAACTATATCATACTTTTATTGTTGTTTTAGTGCAGTTGTAAAGCCCTGTAAAAGCTGGGACTTTGGCTTCTGGTGCTACTATCACTGATATTTCACCTGTTGTGTGATCTTTGGTCATCTTTTCGTTGTTCCTCCCGCTAATCCGCTATATCATGCTAACAGCCTAATTAGTATACACCCATAAAAAAAAAAATTTACTTCAACGGTGCTTAATCCTGGGGATACCCTTTCGAGTTTTCAAAAAGAGGCCCAAATTTCGTGCTTGGGCTTTTCTTATTTGACAA\n",
            "\n",
            "Finished processing index 4: Arabidopsis_halleri.Ahal2.2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "g00001::FJVB01000001.1:2515-3515(+) \n",
            " GTTAGCCTCTTTGAGTGATCGCAGTCCAAGACCACCCTCCTGTTTTGGTCTACAAACGTTTTCCCAAGAGATCTTGACTTTATGTGAATTTAGATCAGTTCCCGACCAAAGAAAAGCAGAACAGAGTTTTTCGACTTCTCGGATACATGCGTTTGGTAGTCGATAGGCTGCTAACCAAAAATTGCATATGCTCTATAAGATTGAGGAAATTAGATTCAGTCGCCCTGCAAAAGAGAGAGAAAACGTTAGGTCCACGACTCGATTCTCTATAGGATTTGTTCGATCAATGGTCTATAGTCCGCAGCTGAGAGTCTCTTTGTTACTAATGGTAGCCCGAGATAACGGACTGGGAGATGGCCTATTTCAAAAGGAAAGAGTGATGCAATCTGATCAGTAGATGCAACCGTGTTGCCTGCCAAGTAAAGAGTGTACTTCTCCATGCTTATTTTTAAACCCGAGTATTTGGCAAAAGCATCAAAAACCTTTATGATGCCTTCGACAGAGCGAACCTGTCCATCCGATAAAACCATAAGGTCATCTGCAAAACTCAAGTGAGTGAGTCCCAACTTTTTGCACCGTGGTTGGTAGCCAAATTTCTTTTCTTCTGCAGCTCTATCGAGCATTTTTTAAAGCACATCTATACTTATTACAAATAGATATGGTGAAAGAGAACACCTCCACATTTCACCCTCTATTGGCTGGCTTTTCTCTCTCTCGCTCCAAGTGTTATGTTAATACCTCTCACCCTTATAATTTTTATTTTTCTTACTAGAGAATTAATCAGAAGTAACACTTGGATTTTTTTTTTAAAAAATTGCGCGTTGTTAATTTCCTTAAACAAACAAAACGACGCATTTCCTTTCTAAAATGTTTCCACATATGAGTGTATGTATACCCTTTTATATTACGTTGTCTCTCTATACACGTTTAGACCTACTTAACCCTTTTTCTCTTTCTCTCTCTTTCTCTCAGTATATATAGCCAAAAGCTTTGTAGTTTA\n",
            "\n",
            "Finished processing index 5: Arabidopsis_lyrata.v.1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "scaffold_100001.1::1:2523-3310(-) \n",
            " AAACAGTGATATGATCATTACATCTTAACTAAATAATATCTGCGTGCATATTTAATAAAAACCAAAAATATAACATATATATATATATATATATATTTGGGTGCATCATCAAATCAAAGTAGTTGCAAAAACTGGACGAGGTTTTACTTAAATGGTCCTTACTCATGCATGGTCCATTTGCTACCTAATTAAGGATTGGTAGGGTGCGTATACATCTTATATAATTTGTGGCGGTGGGAGATGGATGGAGTTACTAAAAACGAAACGTATACAAGTACTATTCAAGGCTCGCTTAAGGGGTTTGCCCTCTGAGTATTAGATCTAGATATTTTTACGTTTCTTTCATTTAATGTCGGAACAACAGACACTAGCTGGCGCTTCAACGTGCATGATCTTGATTGGCTAGTAGATTCCAAGCATCAATACCTATCACATGCCCAACTTGGCTATGCATTCATGGGTAAATACCCTTACTTTCAATTATATCTAGAAATAAATATAAGAAGCCATCCATTAACCAGTGCATTTCCTCAACGCATGGATATGATCAGAACATCGATGAAGGTGGGAGGGGGTAATTAGCTGAGTGTCATAAATGAGGATCCATGTGGAGAGCATCCAATGGTAGTAGTACATGTTTGGTCTTAGCTGGCCCTACCACAAGGAATTAAACTGGTGGGACGATAGGGGTGGTTTCGTCATTCCACATATCTACCGATTAAGGAGTTTGATACCTTGCTATATAATGTACCTTGGCTCACAAGAGTTGGAGAGACAGTGCACAGAC\n",
            "\n",
            "Finished processing index 6: Arabidopsis_thaliana.TAIR10.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "AT1G01010::1:2630-3630(+) \n",
            " ATATTGCTATTTCTGCCAATATTAAAACTTCACTTAGGAAGACTTGAACCTACCACACGTTAGTGACTAATGAGAGCCACTAGATAATTGCATGCATCCCACACTAGTACTAATTTTCTAGGGATATTAGAGTTTTCTAATCACCTACTTCCTACTATGTGTATGTTATCTACTGGCGTGGATGCTTTTAAAGATGTTACGTTATTATTTTGTTCGGTTTGGAAAACGGCTCAATCGTTATGAGTTCGTAAGACACATACATTGTTCCATGATAAAATGCAACCCCACGAACCATTTGCGACAAGCAAAACAACATGGTCAAAATTAAAAGCTAACAATTAGCCAGCGATTCAAAAAGTCAACCTTCTAGATGGATTTAACAACATATCGATAGGATTCAAGATTAAAAATAAGCACACTCTTATTAATGTTAAAAAACGAATGAGATGAAAATATTTGGCGTGTTCACACACATAATCTAGAAGACAGATTCGAGTTGCTCTCCTTTGTTTTGCTTTGGGAGGGACCCATTATTACCGCCCAGCAGCTTCCCAGCCTTCCTTTATAAGGCTTAATTTATATTTATTTAAATTTTATATGTTCTTCTATTATAATACTAAAAGGGGAATACAAATTTCTACAGAGGATGATATTCAATCCACGGTTCACCCAAACCGATTTTATAAAATTTATTATTAAATCTTTTTTAATTGTTAAATTGGTTTAAATCTGAACTCTGTTTACTTACATTGATTAAAATTCTAAACCATCATAAGTAAAAAATAATATGATTAAGACTAATAAATCTTAATAGTTAATACTACTCGGTTTACTACATGAAATTTCATACCATCAATTGTTTTAATAATCTTTAAAATTGTTAGGACCGGTAAAACCATACCAATTAAACCGGAGATCCATATTAATTTAATTAAGAAAATAAAAATAAAAGGAATAAATTGTCTTATTTAAACGCTGACTTCACTGTCTTCCTCCCTCC\n",
            "\n",
            "Finished processing index 7: Arabis_alpina.A_alpina_V4.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "AALP_AA1G000100::1:1105-2105(+) \n",
            " GCACGCGCCTTCGACGAGTTTCCCATTGAACGTTCTTTTTTAGTTTTTATCAACGAATGGTTCGTATTTTGGCGTGTATAATTTTGTTTCATCATGAGTATGTTCTAAATTTTCCTAGTGGTGACTCATGGGCCATCAAAGTATAATTAAAATGTTTGTTTATAAGATATTAAGATGGGTTGTAGCTTGTGGGGTAACTACTGGTCACTTGATAAATTGAATAGTTTGAAGAAGATATGTCCGCTAAATTCTATTATACTAAAAATATATATATGTCCACTTTACGTCGATCTTTGATCAAAATGCAAAATTGCAACTTACAAATTCCTTTCCCTTACTATAAGTAAAAAAAAGAAGTCGTACACACGGTAAGAGATAATTTTGGGAGGATATTTTTCTTCATGAATCAAAGATTGGTAAAAAGGTGAGGATTCCACATCAGAGAATTTACGATTAGAGAATCTATTTCAAAAATCAAAGCAAAGTTTTTGGAAACAACATAAAAATAAAATTAAAGAAAATGTAGTTTTATATATATTCTCTATGACTACGATCAAAGGAAAAAGAGAAAAGTTATAATGTGAGCTTTAAAGGTTATATTATACTACGAAAGACGAAGAAATTATTAGATTATCATTGGAAATTATTTGTTTATCATTAGCACAAAAACAAATAAAATTTAAACGATAATGTGAGTTCTCACAGAAGGACCATGTCATTATCTCGTGCACATACCACCATTACTTCTTCTTTTTTTGGGTTGGTCAAAAAACCACCAATACTTCTAAACATCACGTGGGGCGAATTTTTTTGTCAATATGTTCTAACACCAAGCTTTTATTAGGGGTAAAACTGTCAATATAACTGGCGAGAAACTCCAGCAGTAAACATACTGTCCCTCCTAATTTTTGTATTTATGATTGCTTTTCGTATTTTTTTATTATTTAATAATACAAAAAAACTACATAACCCCTATATAAACCCCTTTCCTTTTTCCATC\n",
            "\n",
            "Finished processing index 8: Asparagus_officinalis.Aspof.V1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "A4U43_C01F80::1:59390-60390(-) \n",
            " AAACGGGTTTAGAATTTATTTAACCCTTCATGATGGGTTAACTAAGTTAACCCTTCTCTTTAATCCCTCATCCTTATCCCCTTGCCAAACGCAGCCGTAGTTATTTACCTGCGATGTGAGCAGTTTAAAAATTGGTAGGTCATGGGCCTAAATACTTTTAGAATAAAACTCCACGCCCTTACAAACACTAAATGAGCCTTACAACACTAAAGCCCTTTCATCAATAGGAGCGCAAACTCGTCTATAAAAGGAGAGGCGCCATCTCCGGCTTCTCCTTCGTTCGGAATTCGATCTGAGGCTTCATCCAGGGTTAGGGTTCGAGAGTTCAGAGCCAGAGGTCAGGTATTTTATTTTATTACGCTTTAATCTAATGTTCGTGAAGACGAATTAGCCCAGGGTTTTTGGTTTCTTTGAAAGTGTTTCCCTCTGCAAATTAATTTCAAAATTAATAATAATTGGAACTCTGCGTGAGATAAAAGAGAGATGTTTTCCTCCTTTCTCGGGGACTTTTCGTTTAATTTTTGTCGTTTTCGTTTGTAATTCCCAATTTGTGAAAAATCTAGGGTATTTTTTTCGTTCAAGATGTTTTCGTATTTTGCAATTTTCACGGTAGACTCGGAACGAGAAAGTTTTTCCTTTATTCTTTTCATTTTCTTTTCAAATGCTCCCAAACCTGTAAAGATCGCTGCTTTAGAAGAAAATGCTGTAGATTTCTGTTGGAAGCTGTGTTTTTTTAACTGTTTTAATTAGGGTTCATCAATCATGCGCGACCTTCAATTTCCCCCAAATTAGAGTTTGGATTTTCTTCCTTTGATCTTGACATTTTCATGTTTAATACTTTCTTTTTAATACCAGGGACTCACTGAAACCTTCCCCCTGAGCTATAGAAGCTAAGAATGGTGATGCCTCCCTCATTGCAGAGACCATGACTGAAGAGGAGTAAAAGTTAATCGTAGCTCGAGAGCAATAAGAGCTCGAGGTCAAGTTGGAACGTGACTTC\n",
            "\n",
            "Finished processing index 9: Avena_sativa_ot3098.Oat_OT3098_v2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "AVESA.00001b.r3.1Ag0000001::1A:49162-50162(+) \n",
            " TTAATTATCATGCCATGTAGTTTGCATTGCATATACTTGCATGTTAGAGCAAGTTCAATAATATAGCCGGCTGACTATAAGAAAAATTTCAGGTCATCTATAGTTCACTTGTATAATAGTTAGCTACAAAATGTACTATTTAATTAATACATGGCCCACATTTCACTCACACAAACTGCCTATTAGCACGTGCTAAAGCCGACTGTTAACTTATAGCCCGCTTACGTTCTTTCTCCTCTTATCTCTCCTCCACATCACCAAAACTACAATATTTCATTTCTTATAGCCAGCTGATTGTACCTTATTATACTTGCTCTTACTAGCTAAGTTACTTCCACTATGACTAGCCTGATAATACTAATATGGTATAGATCCGGACACAAACAAGATTAGGCATTTTTTTTCATAATTGTTTCTCGGGGCCTAAATCTGTAGAAGTCTATCTCTGAGTGCTATAATCGATAATAAATTTAGACATTAAAAATGTCTAGATACATTTACTTTATAAACCTAAAACATGTTATATTAATGAACGGAGGGATTAGTACCGTAAATGCTTTGTACACATATAGTACTCATTAGAATGATATAAGTTGTTTTTTCTCTACCGTAAATTTATGAACGGTTTAAAGGGCTAACCGTAAATATGAATTTTTTTGCATATCTGTACATATTTATGAACGGAGTAGAATCTAGTAAGTTGTAGGCGATGTATGGAAATAATATCATCAATTATAATTGTTAATATGTCCCCATTTATCATTGTCACTTTCCGTAAATTATGTGCCCTAATTACCGAATATTTTCATGAAAATTGTTCGTTGATATATTGTGCATTTTGTGTGCAAATTTGAATGTATACGTGATTTTTTCCTCTGTTTTTGGGTTCTGGCCACATCGGCTCTGTTTTGTTCTCCATGTTTTTGTGAGCAGCCGGCTGGTGTGGACAGGCTGGAGTACCTCCTAGTATTTGAATGGTGGGATCGCCGGCCGTTATGGT\n",
            "\n",
            "Finished processing index 10: Avena_sativa_sang.Asativa_sang.v1.1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "AVESA.00010b.r2.1AG0077770::1A:529681-530681(+) \n",
            " CCGATCTTAGGGATCAATTAATGCAGATCTGAATAAACTAATTACATACGTAATACTACACCTGACCTCTGCAGATCCCGATCTTAGGGATCAATTAAGATATCAGGGTCAGATCATGTTTATTTCTTTTGAAAATGGGACAACACCTGACCTTTGCAGCTCAATTAGTGCAGATCCTAATTAAGGGATCAATTAATACATATCTCAATTAAGGGATCAATTAATGCAGATCCCTGACTTGGGATTAATTAATGCAAATCCCAATTAACTAGAAGTACACACTATAGTCCATAAAACCAAGTTCAAATTAAGCACAAGTGCAACATGAGTAAAAACAAAGAAAACAGAGCGTATGCACACGAAAAATAGAGTATAACTAAATACATAAAGAAATAAAGTATGGTGCCCAAAAACGTAAATTAGTTGGTGAGGGGGTACTAAGTTAGTTGAAGGGATTCTCTTTTTCCATGCATGCGAGGAGAATGTCCTCCCAAAAAGGAGGGGGGAAGTGATTGAGTTACAGATGTATAACTTGAGAGGAACTGGACATCGAGATCCGGGCTAGAGTTTGACCAAGTTTTTAGAGAATATTTATAATACGTATTCCATACCCTTTAATACATTATGAAATATATTTTTACATGGTATCTATATAATATAATAGTTGTTGATAGTCTTTTCTGAAAGTTTTCTAAAAATTTGGTCAAAAAGGTTATGGGTTGCGTCCGAAAAGAAGCATGTTTGCCACGACTTACTAGTACTCCCTCCGACCCATGTATATAGGTTCATCCAAATTTTGACAAATCCAAGTCAAGTAATATGGGTTGAAGGGAGTAGGTAGCGTGTAATTTTTACGGAAAGTAGCAAGTGATGGTAATAAGGCCAGTACGTGGGCGCCACACGTCTGCGCGAGAGGGAGAGGAAATACAGAACTAGCTAGATGGTGGACTACTACAAGCAGGTGATGATAAGAAACGCGATCGAGGTTGTTTTTTGTAAA\n",
            "\n",
            "Finished processing index 11: Beta_vulgaris.RefBeet-1.2.2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "BVRB_000010::0010.scaffold00043:0-270(+) \n",
            " TTACTTTCAGTTGGTCCATTTTTTGTTTCAGTTGGTCCATTATTTGTTTCAGTTGGTATACATATTATGAAACAAAGTCCAACTGAAAGAAAACAAAGTCCAACTGAAAAGAAAACAAAACACTAACCTCTGCTTTCTCTCACTTTCTCCCTCTTCTTCATTCCTCTTAGCTGATTTTTTTCTTCCAACCTAGAAGAAAGTGAAGTACGAAGTTGATATGGGCCGGCCCATCAGCAATACTACTCATTACGGATCCTTTGTTTTTTTCAG\n",
            "\n",
            "Finished processing index 12: Brachypodium_distachyon.Brachypodium_distachyon_v3.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "BRADI_1g00200v3::1:9580-10580(+) \n",
            " CTACCAAACACCCAAATTTCGGTTGGGTTGGGCAAATCTGAGGTCGGCCGACCTACCAAACACACCCTTATACTACATCTACACTTAGACAACACTGTGGAATGATAACTGCTGGCAAGGATAAACGGATAAATAAAGTTTACTACAATGCCAGGAACCCTAAACCCCAAAACCCTAAACCCTAAAAACCCTAAACCCTAAACCCTAAAATTCCTAAACCCTAAACTAGAAAGTGATCCACTGAAAAGTCGCAACCCTATTCTTTCTTTTCACCAATGAACTGATCGGTGACTTATGTGATTGTGTGATCCACTGAAAAGTCGCGACCCGGATATCCTTAATTATTGTTTCAGGATTCAATGCTTTATTTTAGAATTGGCTCGATCAAAGGTGCACATTAGTTTACTCCAATACCTACACTCATTATTCCTTGATTTGTCTTTTCGCGCAAGAAAAAGGAAGTCAAGAGGGAGCTCTTTTTATTTGTAAAGAAGGGCAACCCAGCATGCATGACTCGTTTCTATCTCCTTCGAAGCCCGTCCCAAATTTCGTGGACGATAAATGTGTGTGGGAATAGCTGCGAGTCTGTCAATCTGTTAAGTCGCCAGCTGCCTTGGTCTCCAAGCCATGACTAGTTACATTTATTGTTTCGTAGGTCGAAGAACAAAACATTGAAGTTAAATAATTTGTAAGCAACGACCAACTAGCCAGTCGTTGCACTACTTGCCGTGCAAGGTTGGAGATTCAAAATAAAATATAAGTAATCAAGTTCGTCAGTTCGTGCAGCAGTACAAGCTAGGCACTCGGTAGAGACTGAATAGCAAAGTGTCTCATGGATGATGCATCGGTTGCCTCAAATCCTCAATACAATACCCCGTCGACCACCCAGAGGCGTGCTATATATACCGTTCCCCTCGCTCTGGCTAGAAACAAGCGAACACAGAGAATACTGAAAAGTAAGTACACATACGTACATACAGCTCCATACCATATAGGACAA\n",
            "\n",
            "Finished processing index 13: Brassica_juncea.ASM1870372v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "BjuA01g00010S::A01:18705-19705(+) \n",
            " TTCATTTTTTTTCTATAATCAACAAATAAAAGCCACTTTCGACACAATTTGTTGACAAAAAAAAAAGACACACTTTGTGACAAAGAGAATGTTGACATTATGTTGTGTGTGTTTGCATGGAAATGTCGTCATAGTTTTGCCATTGCCGCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCACTTTTACATATAAAGTCGCAAAACCAACCCTTAAAGCATTTCTCGTTTGTCTTTCTTCTTCAGTTTTTTCCCCCTCTTTTAACATGTGAATCAAAAGCGGAAGAAGCGCGTACTTTACTCCCTAATTTGGCTTACATATGATTTTTAGCCATGGAATGATGGAAAATTTGAGAGAGAGAGAGATATACAATATATCTTTTAATGGCTTTTCTCTCTTTTAAGTCTTGTGATTCCTCTTTGTTACATGTTCATGCTAAGAGTTTTCACTTGTGTTTGTTGTTTCATAGAATATAACAGTCCATGCATAGGCCTCTCAATGTAAATCATTTTTTTGTGTGGTCATGTAGCCCTTTTTAGCCTCATGATGGACATCTTCTGAATTTCTGTAACAAGAGCATGTCTCTGGCCTTTGGAGCGCTTAGAGAAGTATCAGTTATTATGGACCACAAACAATGGCCTGGGATCTAAAGTGGATGGTGTTTATTAAAGATGAGAGCCTGAAGAGGGTAAGTGAAAGAGGCTGATCCTTGTTTTGGTGATAAGGGACGTTATCCTTTACACCCATTGAGACCTTTTATTTTCTAAAATTAAACAAAGAAAAAAAACAAAATGAAAGAAGAAGAGCATGCAACATTCCTCATCCATAAAAAGAAAAGAAACTTTCTTATTCACCTCACCCATCGCTTTCATTATCCCCGTTCCCAACCTCACCATTCCCATGAGACACCGTTCTTTTAAACGCTCAAATCTTTCACCTCAGATATATATAATAAAATTCACTTCAGTAGTAGCTACAACATT\n",
            "\n",
            "Finished processing index 14: Brassica_napus.AST_PRJEB5043_v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "GSBRNA2T00098480001::LK031787:2427-3427(-) \n",
            " TGCTTTAGTAACAAATTGGGTAGTAAAGTGAAATTTAGGCCTGGGCATTCGGGTCGTCGGGTCGGATTCGGGTCGGGTCCTTTCGAGTCCGGGTCTTTCGGGTCTAAGAGTTTAGGATCCGATAGGATAATTTCAAATTTTCGATTTCGGGTCGGTTCGGGTCGTGCCGGGTCTGGGTCGGGTCGGGTCTTAGATAGTTAGACCCATTCGGGTAATTAGAATTTATCGGTTCGGATTCGGTTCGGTTTCGGGTTCGATCTAAAAAAGACCTAAAAATACAAAAAAATATCTGAAAATATTTATATATCCAAAAATATTCAAAATTTTATTCAAAATTTGTGTTTTTATTATATTAAAATGTGTATATATACTAAACTATGCGAAATACAACAACAATTTGTTGTCTCCTAGTGGTTGACCCCCATATTCTCTAGACAAATAACCAGTCTTCGATTCTCCCTTGATGCATTTTATTTAATTTACATTATTAATTCGGGTATCTATCGGGTTTCGGGTTCGGGCCGGGTCGGGTCCAAGACCCGCGGGTCTTCTACAACAAGACCCGATAGGGTAATTTGATCGGGTCGGTTCCTACCCGAACCGAGTTTTTTCGGATCGGTTTCAGGTCGGATCTTCGGGTTCGGGTAAAATGCCCAGGCCTAGTGAAATTAATGGCTGGTGAGAGGATCGTCGGATACAATGTTTGTGGTTGATAAGTAATGAACCATGTTTGTTCTTATCAAACAACTGAGTTCTGAAAAGTCAGAAATATCTCAACTTGATCCAGACAGGAGCAAGGCAACAAGACACACAAAAAAAAAAGCTTGGGAAAAAGACAAGACACAAATTATTCTTTCTGTTCATGAGATGAATGATTTATTGATTTCTAAAGTCTTTTCAATAGTCAACTAATTCGCTTCTTTTAAGTCTCTCTGTATCCTTCCTTAGTCACGTTCTGTCTTTTCGTTTCTAATCACAGCAAAAACATTCTCTATTATCC\n",
            "\n",
            "Finished processing index 15: Brassica_oleracea.BOL.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Bo1g001000::C1:1625-1710(-) \n",
            " CTTTGCCTTAATCTCTTGAAGAAAGTAAGTAACAAAGATCTTCATTAAATCAAATATCAAGTCGAAATCCAACTAGAGATAATGG\n",
            "\n",
            "Finished processing index 16: Brassica_rapa.Brapa_1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Bra011902::A01:10051-10564(-) \n",
            " GAACAACCAGTGTGGTGTGGTTTGTTTGGGATCCTCAAAGTGGCAGACAAAAGCGATGCGTTTGAGAGGGTTTTTGGGGGTTCCTTTTGATCATAAGGCTCCCCTTTCAAGACTTGTGGGGATGATAGCGACAAAAGATTGCTCTGACATGTTTTTGACTTTGATAGTTCCCTTTCAGACCATCTCTTCTATAAAGAGAAAAAGAAGAAAAAAAGGACCAATACGTTTTCCAGGCCCATTTCATTTTCGACGGCCCAGTTTTCAATTCGGAAACAAGAGAAGCGAAACCATTGACATGATTGGATCCACCAGCTCCTCATTCGTTTGAGTGGATGTCATGGTGATTATCAAAAATCGTTAAAAGTTTCTTTCGCCCTTGACCTTGCTTTTATTCACAAAACTGCCCTCGATGGTCCGAGTCAATTTAATTTCGCATTATTGCTCTATCTCTCTCCAACCGCTGCTGCTGCTTGTGCTAGTGGTTGCAGAAGAATCTTCACCTTCCAGTGAGTG\n",
            "\n",
            "Finished processing index 17: Brassica_rapa_ro18.SCU_BraROA_2.3.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "A01p000050.1_BraROA::A01:10942-11465(-) \n",
            " GAACAACCAGTGTGGTGTGGTTTGTTTGGGATCCTCACAGTGGCAGACAAAAGCGATGCGTTTGAGAGGGTTTTTGGGGGTTCCTTTTGATCATAAGGCTCCCCTTTCAAGACTTGTGGGGATGATAGCGACAAAAGATTGCTCTGACATGTTTTTGACTTTGATAGTTCCCTTTCAGACCATCTCTTCTATAAAGAGAAAAAGAAGAAGAAAAGGACCAATACGTTTTCCAGGCCCATTTCATTTTCGACGGCCCAGTTTTCAATTCGGAAACAAGAGAAGCGAAACCATTGACATGATTGGATCCACCAGCTCCTCGTTCGTTTGAGTGGATGTCATGGTGATTATCAAAAATCGTTAAAAGTTTTGCTACGTTTCTTTCGCCCTTGACCTTGCTTTTATTCACAAAACTGCCCTCGATGGTCCGAGTCAATTTAATTTCGCATTATTGCTCTCTCTCTCTCCAACCGCTGCTGCTGCTTGTGCTAGTGGTTGCAGAAGAATCTTCACCTTCCAGTGAGTG\n",
            "\n",
            "Finished processing index 18: Cajanus_cajan.C.cajan_V1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "C.cajan_19181::1:12891-13891(+) \n",
            " TGTTGCAATTTCTGCTTCTGTTTTTGGTTTCGTTTCAGCGAGACCTCTCTTCTCACCTCTGATTCCACTTCGCAACCTTTGAATTTTTGTTTTTTGTTTTGTTTTTCGGTTTCCGATCTAAACTCGATTCGATTCGCCATTGCCGGAAATGAGTCCTCCGGCGAGTGTCCGGTGATGATCGCTCTCCCGAGAGGTTCTACTTCTTCTTCCACGTCATGGTTTTGTTTCGAGGTATTCGTTTTCCTTTTTTATTTTATTTTTGTTCATTTATTTGTTGTTTGGTTTCCGAGAAAATTGAACCGCCATAAATGAAACTGCGAACGGATTTACCTCATCTGGATTTGGATGTAACAGGTTCTGTCAGAACTCACAGCGCGACGCTGTTGAGTTGAATTTCGCTGGTTTTGAGTTGCTTTGTGATTATTTTTTCTCGGCAACCAAACAGGAGGATTCAAAATTTGCAGTGATTAGTAAAGTATTTGTGAAAAGTGAAGATAGAAATGGTTGTTATTGTTGATGTTGTGATGTGATTTTTTGTCTCTGTACTAATTGTAGTTTCCCTCTCTTTTGCGCGTTTGAAAGGTATTGCCGCTGTCGAGTTCATTTTGGTGAGGATTCTGATTAAGGAAATTTTCTTTTATTCATTCGGAGGAGAAGCGTCTCTCTCTCCCCCTCTATCTGTTGCGGTTGATTCTTGAAATTTGTTGAACTTTATTTCTTCTTTCTTCTTCGTCTTTTGTGTAATATTTTTCGTGTGACATGTGAATTGGAATGCTTATTCCAGCTTATACAAAACAGAAACGGATCTCTTCCTATTGTTCTCGCTTTGTTTCTTTTGCAGTAGCATCATCACAATACCTTGTTCAGATTCTGCTCACTTTCACCACGACGGCTTTACTATTTACTGTGTTACGTGTCACCAAAAATGAAAGGAGGTATTCCCTGTATCCACTACTTGTCAGGGAAGATCTGAAGCAGCGCTAGCTGCTCACGTCCTGTA\n",
            "\n",
            "Finished processing index 19: Camelina_sativa.Cs.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Csa01g001000::1:6504-7504(-) \n",
            " TAATCTAGACCCATGTGTACGGCGAAATTGATGTAATCCACCTTCTTGTTTTTTTTTTTTTTTTTTTTTTTTTTTTCTTTTTCGGCGAAATTGATGTTTGATTTTCTTCTTTTATCTGATGCTTTGGTTGGTTCATGTAATGTAATCCTCGAAAATACTTTTTGGTTGGACTAATATGGAACATCCAAATATATATTTTCTCATTTCTGTAATGATTTTATACTTTCTTCACCATCTTCACCAGCTAACATATACTGATTCTTTAAGAAATATATAGATAGTTGTACAAATTGAAATGGTTAACAAACGGATAACAACATAGATACATTGATAAAACGAACCGATGTAGTAGAGAGACTTGTAGTTAAATTGGACCGGCAGATGTACGTAGTTGTTTTTATATATATGTGTTTACAAACTTGGTCCACTACTGAATCGGTTGATTGTAATGGGGCTTAGTTTTGTAAGAGAAGAAGAAGAATACATACACACTGCTTTAATTAGAAAAAAAAGGGAAATTAATAGAGATTAGTAATATCATTAACGTGAGTATTACAACACCTTCCGGGAGCTACTTTCCTCGTGATGCCAAAATGGTTTTAAATTCTTCTTTTCCTTTCTTCTGACACAGTAACCATTTATTTGACTACGGAGCCTTACTTATCTTTATCGCGTTATATATCCAGGCAAATTTACCCTTTTTTTTTTTTTTTTTTTGTTAAACATAGACTATTTACTTTATAAACATAAGAGTAAAAAATTATTTTATTTTTATAAACTAACGATATAATACCCAAAAAAATAACAATAAATTAGCATCAGACATTAATAATTAACATTTTTTTTTGTTTGGGTGCGTTACGTGATCAGTAGCATCATGCCTCTTCGAAAATATTAGTATCGATTAAGATAAAAGATAATTCACGTTTTATTGTTTCTGTCTCCAAACAGAACTTCGGGAAAAGTGGGAAGAAAAAAAAAAAAAAAGAGAGAATTAATT\n",
            "\n",
            "Finished processing index 20: Cannabis_sativa_female.cs10.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "evm.TU.01.1::1:12598-13598(+) \n",
            " ATTAGGGGCCATTTCAACAAATTTTAACCCAAAACCATTCTTCATATTCATTTCTTAGTAGTACAAATTGATTGTGACTAGTGGCAAAAAAATTCGAGATTGGTCCCGCCACATTTCACAAAGGGTACCACTATCTAAATGTGAGATTTTCACTATTTCGATTAAAACGATCGTTAGTGCAAGAAAACTAATTTTTCTCTCAACGGTCCAAGAGAAAAGTGATGCACAACGGTCTTGGCTAGGGCCCCGTGAAATTTCATGATCTTTGGAGATGTGTAGGTTGAGATTTTAACAGATAACTAACGAGTGTAAGAGTAAGGAAAAACTTGTGTAAAATTTCCTAAAAGAAGAATATTTTCAATTAGGGGCCATTTCAACAAATTTTAACCCAAAACCATTCTTCATATTCATTTCTTAGTAGTACAAATTGATTGTGACTAGTGGCAAAAAAATTCGAGATTGGTCCCGCCACATTTCACAAAGGGTACCACTATCTAAATGTGAGATTTTCACTATTTCGATTAAAACGATCGTTAGTGCAAGAAAACTAATTTTTCTCTCAACGGTCCAAGAGAAAAGTGATGCACAACGGTCTTGGCTAGGCCCCGTGAAATTTCATGAGTAAAAGGGAAGTAGCTAAATATGAAATTAAAGCTGTAGGTTATAAGTTCGTTATTGTCTCATGAGAGTAAACTTGCACTAACAATTTGCACTATTGACTTGTAACTTCTGCATATAAAACTCATACTGTGCAATACTTTCCAGCTTTTTCCAGATTTGCTGCTCATTATGTTCCAGTGTATTTGTTTTGAATGAATCATTCATGGGACCTCTTTTTCTTCAGAAAATCTCAGATTGAGAATTTGTGTTGTTGAGTATTGGAGGATTTTAGAGAATGGAGAACATTTGTTTTCATGCTCTTTATTTATTTTAAAAGAAAATGGGTAATCTTTGTCTAATGCCTCCTGTGCTTTTTCAGGTAGAAGGTTTCTTAAGGTTA\n",
            "\n",
            "Finished processing index 21: Capsicum_annuum.ASM51225v2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "T459_00001::1:37617-38617(-) \n",
            " AATCTAGCATGAATGTTTAATGTTCATAATGCATGTAATGAAAGAGTGCGTGGATTTCCTAAAGGCACTAGACCACCATATGTTGATGAAGCTTTTGTATGATTTTGACTTGAGTTTTGAAACATGAAATTTTCTGTATCAGTTGCATATTTTTGGTGGTCTAAATTATGCTTTAAATGAAATATTTACCTTAATGTATTATGGCTCAGAAATCATGACTTGCATTGGTATGACGATACCAATTCAGACCAATGCCATATCAGACTTAGACTAATAGGCATCTCAGACTATCATGATTTTAGATATTCAGAATATTGCATGTACAAAAAAGGTTAAAAAATGGGCATACAGAGATGTTAGGTGGTTCCCTGAAGAGGGCTTGAGTTCAAGCAACTCATTGCCTAAAATCGTGGTTTGCCGACCCGGGTATATTATTATACGCTGACTTAAGTATTGTGTGGCGTATCAGATTTAGGGACTCCAACCCTGGTGGCATACTCGGGTTGGAGGCTTCTCCGCCGAGTCAATGGCGGATTCCATATTTCCCGTAGAATATCAGACTTGTAGGGGATACCACCTAACTCAGAAGTAATACACAGATCAGAGTTTGAGATTTACAGATAGGTTACATGTTTTCAGAAGGTGCCCATGTGTTTTTTAGAAACTGTTTTATACATAATGTTTGATGAACATGCTTCCTGTACAGAGAAAGTTAACCATAGATTATATCAAAGTGTAAACAGGTATATATCGTGGAGAAATTTTGAGCTTTAACTCCATATTGTGTTCATAAACATATATCTTAGACATATTTTTGTTGTACAGGGATAAGGAGAGTGTAAATTCATGTCAACCAGGCGTGAAACAGAATGGATCTTATTTCTGTTTTAAAACTGAAGAGCGGGCCAGGAAAAGGAAAGAGGCAAGGACCAATTTCTTGTGTCTAATAAAATGTCATATAAAACCTAAAAGAAGTTTTTGATGTTGATTTGAGTTCTTT\n",
            "\n",
            "Finished processing index 22: Chara_braunii.Cbr_1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "CBR_g1::BFEA01000001:0-4(+) \n",
            " GCAA\n",
            "\n",
            "Finished processing index 23: Chenopodium_quinoa.ASM168347v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "AUR62002649::C_Quinoa_Scaffold_1000:244044-245044(+) \n",
            " TATGTAATAGAAGCTACAAGCTTAAACAATGAAATGCGAATATCAAATCAACAAATGATATTGTTGTTATTTCCTTGAATAATTTATAATTAGAATTTCAATAAATTAGCACTCCATGAGAACTTCAATAATACTTGCTCACATGTCGCTAATTAGTAGTGTTATTTTTTCTTGCCATTTTTGTTGTTTTTTAGGATTCAATTGATATGAATGTATGTAACATACGAAAGCATATAATTTTAATTTTAAATACAGTTATTGAAAATTACCGTGCATGACACGGGACCTAACTAGTACGTCTACAATAGTAGCAACCCGGAACAACGTCCGGGGTCACTTACTAGTTACTCACTCTGTGTTTTTTTTAAGTGTTACAGTTTGACCAGACATAGTTTTAGGTAATTTAATTGACTTAAATAAAATAAGAGGTAAATGAGGTAGTTGGAGTATTAAAAAATTATAAAAGAAAGATAGTAGGGTTAATAATTTATTAAAAAGTACTCCCTTCGTTTTTAAATACTTCTCCATTTCTATTTTTTGCACTATTCATTTATCATATTTTGACTAATGTTTGTGATCTATACATAAGAAAAAATATAGTCGTATGGGTACTTGTTGGAATCATCTCAATGAGTAGTTTATTAATTTCAATTGTTAGAAGTTATACCTATTGATAATTAAAGAAATTAATGATCAAAGATTAGCATCGACATACGTGTCCAAGAGAAATGGAGCAAATATTTTAAAGAGGAAAAATATAATGAAAAAGTAATAGTGGATTTGGGTGGGGATCATCAGACGAGAGAGATATAATATTTTAATGTAACTTTTTACTAAAAAAAAAGAAGTATAACACTTAAAAAAAACACTAGAGAGTAATGAAAATTAAAACACGAGGTCACGAGTAGTACTGTAGCTTTTATCTCTCCTTTAGAAAGAAACGAGAGAACATATTCGTGTTTTCCTTAAAAAAAAAAAAGAAAAAAAAATAAATAAAGAA\n",
            "\n",
            "Finished processing index 24: Chlamydomonas_reinhardtii.Chlamydomonas_reinhardtii_v5.5.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "CHLRE_01g000017v5::1:17765-18765(+) \n",
            " ACATTCAGAGCACTGGGCAGAAATCACATTGTGTCAACATCCTTTAGGACCATCACAATGCTTTGTTTTAATTAAACAGTCGGATTCCCCTTGTCCGTACCAGTTCTGAGTTGGCTGTTCGTCGCCTAGGGAACGCCGAAGCTTCTACAGCCGTCCACCCAGGACACGCAGCAGTCCGCCCAGCCGTTTCCAGCTGGGTAGACCACCGCAGTCCCGAGCTTCGCAGCTGCAGACCCCTAGGCCCAGCCCTCAGAGCCAATCCTTTTCCCGAAGTTACGGATCCATTTTGCCGACTTCCCTTATCTACATTGTTCTATCGACTAGAGGCTGTTCACCTTGGAGACCTGATGCGGTTATGAGTACGACTTGGCAAGATCGGGAATGCTCCCCCGGATTTTCAAGGACCGTCAACGGCGCGCCGGACACCGCGAGAAGTGCGGTGCTTTACCAACGTCTGAGCCCTATCTCCGAATGATTCGATTCCAGGGCCTTCGCGTTGTTAAAAAGAAAAGAGAACTCTTCCCAGGGCCGATGCCGATGTCTCCGGGCTCGCTTGCGTTACCGCCAGCCGCCTTGTCCAAGTAAGGGAATCTTAACCCTTTTCCCTTTCGATGGGCAGCGCGAATCGCGCTCTTCACACAGGATTACCCCATCTCTTAGGATCGACTAACCCATGTCCAATTGCTGTTCACATGGAACCTTTCTCCACTTCAGTCTTCAAAGTTCTCATTTGAATATTTGCTACTACCACCAAGATCTGCACTAGATGCCGATTCACCCAGGCTCACGCCAGAGGCTTAGTCTCGACACCCACGCCCTCCTACTCATGGAAGCGTCGCACTTGCTTCCATGGCCGAGTATAGGTCACGCGCTTAAGCGCCATCCATTTTCGGGGCTAATTGATTCGGCAGGTGAGTTGTTACACACTCCTTAGCGGATTTCGACTTCCATGACCACCGTCCTGCTGTTTATATCAATCAACACCCTTTGTGGGATCTAG\n",
            "\n",
            "Finished processing index 25: Chondrus_crispus.ASM35022v2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "CHC_T00001315001::HG001459:44106-44681(-) \n",
            " TATTTTGTTAGCGTCTGCGTGGATGCGCGGAAGGGCGGAAGGGCGGCCGCCGCATGCAACAGGGCGAACAGCAATGCGGGCGCAGCCGAGGCGGGGGAGTGCGCGCCATTATGGCGCGAGACACTTGGACGATTCCCCGTGCGTCTCGAGGTTTACACCTCACGCCCCTGTGAAGCCGCTTTTGAATTCCCGCGGCGATGGCCGCCGTGACTGCCCCGGCTGCTCCCTGGTAATTCGGCAGTCTGCGGTGTAAGCAGCGCCCGCGATCTGCTCATTGAAAGACCTCCATTCGTCGTGGATTTCTGTAAAGCGGTTCACCAGTGACGCCCTTGAGGCAGTCTTCCTCCTCTCCTTACTGTACGGCCCCCGCCGTGAGTAACTTCCTCTCTTCCGAAATTTCCTCTCACGCGCTTCCCTTACCTCATCCCCCCGCAGCGTTACACCCAATCCCGTCACCCCTCACCTATGCCAACTTCTCACGTTCACCTCCACGAAATCCTCTCCTCCTTCCGCCGCACCTTCCCCTCTTCCTCGCTTCGCCCTTCGCCCGCCGCCGCCCCATCCCGCCTTCCGTC\n",
            "\n",
            "Finished processing index 26: Citrullus_lanatus.Cla97_v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Cla97C01G000010::1:1023-2023(+) \n",
            " TAAGAACTTGTGCTATTAAATGGATGAGAAAGAATAAAACACTTGTTTAAATGTAAAAATAAATAAAAATAAAAACAATAGCTAAATATTACAGATAAACTTTAAATGGAAATGACATACAATAAAACTCTTCAACCTATCTCTCGATCTAACTAAGTCCTCTTTTTTGAGATGAACAATAGCCTTTGTGCTTGTTCTTACAAAGCAGGCACCTCAAAGAAAATTACAAGGATGAAAAGAACCACTATAACTAGCCTTCAAGTGTGTTTACAATAAATGAAAATAGAAATACAGTAAAGATTCTATGTAGCTTGAGCTTGTTGATCGATTTTGTGGGGGAGTGCAAACAACGCCTTTTATAAACTTCGAAGGAACTAAGATTTCCTTAGCACACCTTGCAACCCTTTGTGAATTAGTTTGGACAAAATAAATGCATATTTGATATGATTTAGATCAAAGTTGAGGAATTTGCAAATTTCGAAAAATTTGAAGAAAAAATCTAAGGATTTTTGTTTTTGTGGGTTGCAACCGCATCAAATTGCAATCTAAACGCGATTTAAACTCTCCTATCCAACTTCGTGTTATATATCATATAAATTCGATAATTGATGACCCCCACATATTTCTTTGCAACCTAGCAAGATCATATTTATTTGTCTTAAATAATAGATTTCCAAACATGCAAATAGACATTTACACTTTAATATAAAAGTCATTCAAAATCCGGAACAGTCTTGAACAGGCAAAAGAAAAGGAAAAAAGAAAAAGAGAAAAAAAAAACGTCTGAGAAGGTTTTTTTTATATTAGGGTTAGCTTGGGATTGTAGAATTAAATAAGAACAAATGAAATATATATTTGTTATCCATCCACCGCTTTGTTGATCCGGCTCCAAGCTTCAGGTTTTCTGCGAAACTAACGGTTGCCGGAGTGAGGATAGAGGAGAAGGAAGAGGAACCGAAAAGAGATAGAGGCTCCATCCAGTTGACCTTACTTATCATAG\n",
            "\n",
            "Finished processing index 27: Citrus_clementina.Citrus_clementina_v1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "CICLE_v10010164mg::KI535697:33911-34911(-) \n",
            " CTGCGGCACCCCAGCAACCCAAGGAATGTCCCCAGGGTTGAAGAACGAGGTTGATTTTGGAGAAGAACCCAACTTCTCTCTTTGGGCCCCCAGGCCAAAGGCTATAACACACGATTTTGGGGCTTTTTTGTGGTCCCCCTGGGCTACCCTAGAATTTTGGAACAGGCTTCTGGGCTAACCTTCTCGCGGCCGGTGCAGTGCCACTTGAAAAGGCGAGAGTCCGTTTGGAGCCCTCGAGCTGTTTTGGCGAAATTCATTCCAAAATTCATGAAAAACCGCCTTTTTCGGCCTTTTTTTAAAATAAAGTCCCAAAACTATCAAAGACTCCCAGACGGCGGTGGGGCAAGGCCCAATGGTACCATGGTAGACCAAGCAATGTCCCTATCAGAGAAGAACGAGGTTCCTTCTAGAGAAGAACCTAAATTCTCTCCTCGGGCCGTCAGGCCAAATGATATAGATGGCGGTTGTTAGCCTTTTTCATTGGCGCTCCTGGGGCTACCCTGTAATTTTTGAATGGGCTTTTATGCGAACCTTTCCGCAGGTGCTGCGATACCACCGGAAAAGACGAGAGTCCATTTGGAGCCCGGTTTTTTATACTAGAAAAGTATATGGATTGGAGTCTGCTCCTTAGCAATTGATCCAAAAAGTCCCACAGCTCATTCTTAGGCGAGCGAGGTGACCTCCATTGGAAGAAAGAAGCGAATAGAGTCATAGAAGATCCCGAAAGAGGCGAATTGCCAAAAAGTTTATCAAATCCGTTCCTTGCATCATCTGATCTCGATCGGGTTAACTCCTAAATTCAGGCAGTGAGCATCAAATAAGCCAATTGACACTATCGCCTGTTTACAGCAAATCAAGATTAGATGGACAAAGAGAGTTTCAGTCAACACAGTCCTAGAAGCTCCATTCATGCCCACTTCTATTCCTAAGAGCCCATCATTCTACTTAAAAGACGACGAAGGCTACTTTCGGGACATTGGTTGCAGCGATTTGTAGACCA\n",
            "\n",
            "Finished processing index 28: Coffea_canephora.AUK_PRJEB4211_v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "GSCOC_T00012439001::1:1820-2820(-) \n",
            " ATCTTTGAAAACCAGTTAGTCACAAATGTATCTAATTACAAGAAAAAATTTTAAAAATGTATATTAATTTTACATATATGGATTCTTATTCATGAATTTTGTTGCGTATAAACTGCATTTAATTTACATATAAATATTCATATATTAGCTTACACATGTTGTTCGTCAAAAAATTTGAGCAAGCATTGAATTTTTATTGTACATAGAATTATACTAATTTGCAGTTTGAAAAATATAATAATCTTCCATTTCCACTTTAAAATTTTGATCTTCCTAACTTAAGTGTTATAAGACTCCACAGCAAAATCTTTAAGTTATAAAATCAAAATCTTTGATTAGGATTAGGGCAATAAAATCGTTTAGGTATCTCTTCCAAAAAATAAAGAGCATAGGAAAGAGTGAAAGAAAGTAAAAGGGCAGCTAGTGCGGTGAATTTGTATGGCCTTTCGTTTCACTTTTGCTACAAAATTTATGTCCCCAAATACATCCCATTTGCTGGCCCTTTTTTTTGCCGCCTGTCCCAGCCGTGCCCAGTCAGCACGGCTGGGGACAGGCTGATTTTTAAAAAAAAAGTGGGCCTCTGAGTCATCTTACAATATCGTTAAGCCAGTCAGAAAGAGATGATGAGTTTTAGCTGCAAGTTGAAGGTTGGCTATATGCTGCAGATGCGACGACTTGTCGGCAAGAAGGTGTGGCTACGAATTGTCGGCAACTTCCAGGAAGTCCAGGGTAACGCGGTAAAAGAAAAGTTCGTTGGTGACGTCTTTGGAGTGTAGCTTTGCAGAGTGAAGCCGGGTGGCAAATTGCACAAAGTGGAAAATTTTGGAAGTGCAAAAATTGTGTGGAAAAATTTCATTCCCTCAAGTTTTAGCCTCCACAGATTTTGCTGCCCTTAGTTGCCAAAGTTGAACCATATCCTTCCTTGTCTCATAGCAGAAAGCATCCCCCCATCACCATTGCAGAACACTTTCATCCTAATTCCAATCCCAAGCGAGAAAAA\n",
            "\n",
            "Finished processing index 29: Corchorus_capsularis.CCACVL1_1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "CCACVL1_00002::contig00002:0-860(+) \n",
            " AGTCGAGCCGTGAACGCCACTTCGCCTAAAGGCTCAGCTCTAAACGACCCTTCTATATCGGCTCACCTCTAAAGACAATTCGCCTTACGGCTCACCTCTTTCGCTCCTCTCCTTCTAGTCGAGCCGTGAACGCCTCAAAGCTGCTATGCTACGCAGCTCACCTCTCCGCTCCATAGAAAACGGCGATTCAGGCTATCTAACTTATCAGTAGTACTATGGGGTAACAATGCAGTCAACATATGAAAAGTAGGAATCGCAGACGAGACAGAACTCACCAAAGTCGACCGACCAGCCAAAGACAATGTCTTCGCTTTCCACCCGGATAACTTAGATTTAACCTTCTCACAATAAGTTCAAGAAAAGTAACCTTTGTAACTCGACCATGTATTAACGGTGTGCCCAAATACTTCCTTAGTCTAAACAGAAGGAGAAGCACAAACAACTCCATCGGAGTTTAGGCAACTTAGATAGTACACACATCCATATCATCTCGAAGCTTCCCGTTTGCCAGACCTATAGGATGACATTTCTCGTGTCAGTCAATCCTGTCTCTCTGAGCTTATGAGCGCGTCTGCTACGCCTTTGCCCACTGCCAATTGGATTCCAAGTTTGCTAGGTGGTTCTCTTTTAGTCATCATGAGAGAGCTGGCAGTTGGATTTAAACGGTGGCAAAGCTGCTAGTTTTAAATAGCCGTGAAGTCTAACACGAGATGTATGTCAGTTAAAAATGTACCTATCTCATGGTTAAAAAGGTAATCGAACCCTTTGCATAGTCACTGCACTCTGCACTCTGCACTCTGCTCTCTGCACTCTTCAATCAGAAGAAACTATGCTTGCCCCAATCGGCGTACCCACCATAG\n",
            "\n",
            "Finished processing index 30: Corylus_avellana.CavTom2PMs-1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Cav01g00010::ca1:7065-8065(+) \n",
            " GAAAGACTTGATTGCACAGCTCTCCCCAACAAATGCATACACTGTTTTAATTTGAGATATCGTCCGCGATAGGGAAATGAGAGAGCAGATTAGTCAAATAATTAGATCAGGTACAGGTTGAAGGTGGAGATGGTTAGCAATTAATTAATTGTGCTTTCGAGACTCTTACTTGTTCTACCATACACGTAGCTTCTGAATGGAAAAGCATTTTGGCAGCTGTAATAGTGTTCATTTTCTTTGTGTATGTGGTTCCGATTGCAAGAATTAATGGAACAACCAACAAGGTAAGCAGTGCTACTTCCCAACAGCATATAGCAGCAATCAGTATTCTAGAAAAGAAAGTTGCAAAGCTCGACAGAAAGTGACCCAACTGTACGTGACCGAGAGAAACTTTGTTAGCATTAATATCTTTTGTGATTATGAATGCCCAAGTGAAGTGCATGAAACATTTGAGCATAAATTAATTGCATACCTTTTCGCCGATGGCATCTTGTACGATACTCATGTGGTTAGTAATTCCATCAATGATTTTTCCACTTGTTAAATCTGTATGAAAAGCCCCAATCTCCTGGCTAAGAACTGCTCTTAGGAATGCTAATCTTAGGCGTGCTAACTGCCTTTCGCTTGCATACATCCAGCATCCAACCTCTATAAATCGAAAATTAAAGTGAGGACGAAGAGATGGTCATTAGTAAAAAAGGGTGAAAAGAAAACACAGTAATTAATGTTTGTTAATTTTTTTTTCCTCATACATCTGAAAATACAAAATATTTAAAGCTATTTTCAACTTTTTTGTCAAGTCAAAACAATTTTAAACCAAAAACAAAAGAAAAAAGAAAAAACANNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNAAAACGTGAAAACCAAAAAAAAAAAACGTATAAACTAAAACAAAACCAAAAAAACC\n",
            "\n",
            "Finished processing index 31: Corymbia_citriodora.Ccitriodora_v2_1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "gene-BT93_A0001::1:16518-17518(+) \n",
            " AGGTCGTCGGCGAGGGTCGTTGGGGCCGCGACGCTCGCCGGTGGCCCCTAGGTTTGGGCGAGGTCGCCGGCGAGGGCTTCGCGGCCCCCAGCGGCCTCGCCTAGACCTGGGCGAGGTCGTCGGCGAGGGTCGTTGGGGCCGCGACGCTCGCCGGTGGCCCCCAGGTTTGGGCGAGGTCGCCGGCGAGGGCTTCGCGGCCCTCGCACAGTGGCCGGCAGCCTCGTCAAAGCTGGCCGGCCACTGTCGCCGGCAGTGGGGGAGGGGAGGGGAGGGGAGGGGGAGAAAAAGAAAAGAAAACATAAAAATTTAAAAAATAATAAAAAATTCGGAAAATAATAAAAAAAATAAAAATAAAAATAAAAATATTATCGCCGGCGCCGGTCGGCGTCCACGTCAGCTCCGGCCGGCCAAATTTGGCCGGATGGACTCAATTGGCACAATCGCAAAAGGTTTAGGACTGAATTGGTCAAAAAAATGTTTATGACTGAATTGGCACAACTGCAATAGGTTTATGACTTTTTTGGCAATTCTCTCCAAAGTTTGATAGACTCAATTCAAGGCCCGTTGCTCTAGTGGCCACCCTTCTTATATGGGAAGGGGAGACGAGGAGTACGAATCCCTACCTTTTCAAGAAGTAGGGGTGGAGACAATTTGGTTTCAAATATGGATTCATAAGAGTTAGAGTAGGAGTATAGTATAATTCATAAAAAACATAAAAAAAATCCTGGCACGCCACTTATTTATATTTAGAATGATGCCCCAATAAAATAAACGCTGCAATCAAAGTCCTCATTTTCCAAGTATCTACGTTTTTAATTCTCTTTCCGCTTTGCATGGTCCAAATTGCTGAACGTGAAGCTCATGCAAAATGATGACAAGAAAAAGAATAAAAAAAAGCTCATGCAAATGCAAGTTGCCTTCCTCAATCACTATAAATATCCAAGCCTCACCCCATCGAAACTACACCTTGAAGCCATCATCATTCCTCCCTTTCGAAAAG\n",
            "\n",
            "Finished processing index 32: Cucumis_melo.Melonv4.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "MELO3C027031.2::contig1:4526-5526(+) \n",
            " TTTGCTCTAACTGAAGCTATTGCATTGTTTGCTCTAATGATGGCCTTTTTTATCTCAAATTCGTATTCCGATCGAAAAGGTTTCAGTCTCACATCTTGAACATTAGAAAGTCGTCCCTCTTCCTATGGTTGAGCTACTTCCGTACCTCATCTGGAGCTTTTGATTTGTCCATATGAACCTGTGGAATTCAAAGCTCAAATTCTTCGAGAATTCTTTGACTTTATTCATTGATTCAGTCGAGACCTCTAGAACTCCTTTGGAGTAATTGAATTGTCCTTATTAATTTCAACTTCTACTCCACCTCTATCTGAACTAGGTTCTACGTAGCTTAATTAGTCCGAATGACAGGTTCATATGGAATTAGATTCCTTCAGAGTGGTTCGAATGCTCAAAGAATTTGAGAAAAAGAAAGTCACTAGAATGAAACTAAAGAGGACGCTCGACTGAAAGGAGTCCAAGATTCAAGCTGAATTGATGAACAAGGGCTCGACTAAAAGGAGAGAAGGAGGCTCAATGAAATATGAGTTCGTTTGTCATTCCACTAGAAAAAAGGATTCCACATTCAAGAGGACTGAACAATAGCCTGAGTCGAGTATGGGAAGCGGGCTAGTCCCAAAAATGCTCATTCCTCTGACGTTGGGGCTCACAATGAACCTTATGACTCTAACTCTTCTTAAAAAAAAGTGTCTCTATATATATAGATTATTCTTGACTTATCTCAAGCAATGTCCATCCCATGCCTTTCTTGGTTGGACCAACCCAAACTGCGATTTTCGACAAGTCTTTCTTCATTTTTCTTGAAAGAAGCAGAACTACAGGGATGAAATGAAAAGTGTTTCTGACTCTGACACCCAACAGCCCACTTGAGCAATTTTCCATTCTCCCATTGATTTTTATGCATATAGGAAACTTTTATTTCTTATTCACAAATCCATCTTTGTTTATGCTGCTAACTCTCAGTTCAGTCCTACTTCTGCTTCATTTTGTTACTAAAATGGAG\n",
            "\n",
            "Finished processing index 33: Cucumis_sativus.ASM407v2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Csa_1G000010::1:4284-5284(-) \n",
            " TGTTGCCCATCGGGATAAATAAAGGGGATGGGAGAGTCGTGTTATGCACGTTGATGACCACGAAGGAGAGGAGGGGTTTCGTCCAGTAGCTTTAGCATTAGATGCACGTTGATGTGATATCTAAGCATGTTAGTCTCAAGAGGTCCAACGGATGCTTAATTATCTAAATCGGTATCAAATTTATAGTTTGTGCACGGAGTCTGATGACACAAATATAAGAACTCGATTGAAATGGAGGTTCCAAAAGTGCATCAAGAGTGGGTCCCTATGGCAGGGCTGTTGTGTTTGTTAATCAGTATTAGACATGTTGAAGTTGAACTCTGAGACTCTAGAGGTTACATCCGAACAAATGGGGCCATAACAAGGACATGATTTTGCCTCAATGAATTATGGAATCAATAAAAGGTCAATATGATTCTTAGAGGTGTTTTAACAAAAAGTGAGCAATTACCAAAATAGGAATGTCAGATTGCATGACTTGAAGAATAGTAGAACTACAAGAAACCTCAATGATCTTAATGTCTTATTGAAAAGTACATGTTGGAAATTTCAAATGAACTAACTAACCCATTGTAAAATAAACTGAAGTGTGACAATGGACCGAACACTATCCCATATTGAAAATGGAATTGATGAAATTTCTGCAAAGCATTTTGAAAGCATTGTTAAAACATGAATGAAAAGGATTAAATTAGTAAAGTAAAAAATGAGGTTAAAAGGGCTAATCAACCAAAAAACTTGAGACTAACATGCTTAGATATCGTATCAACGTGCATCTAATGCTAAAGTACAAATTACCCTCATCAATAGTTGTGATCGGCTACCAATATTCTCATCAAGCAAAATCTTAGTTGCAAATTGTTAGAATAATGGGCCGTTAAAGGCTTTTAAAGAACTAAGCCCAATCTTACTGAAGCCCAAACTTCATGTTTTTGGTAGTGGCATGAAAAACCCTAAATCCTAATCCCTCTTCTCTCAGTATATAATCAGATAGAGTAGT\n",
            "\n",
            "Finished processing index 34: Cyanidioschyzon_merolae.ASM9120v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "CMA001C::1:6125-7125(-) \n",
            " ATGTAAAAATACACTCCACGGAGGATTTTTCGGTAAATTGGGTGAAACGTGAGGAAGTCTCCGATTGTGAAAGTATTCTACGCGGATCATTGGTCTGCGCCAGCCCTGCAAATGCATGGTGATGCAAGGCTTGCCAGCGGCAGTTGAAATACATGCTCAAAGCTTACGCCGAATAAGGTTAGATGAAAAAGTGACTTACGAGTTCCCACTTCGTGGGTTGATAGCTGAAGCATGGGTACGGACATGTGCGGGTGTTTTCGAGCAACGAGGGAGGGTCCCGAATCAGAGACCACATTCCTCGCGAGCAATATTGTTTAGTATTCCTCGCAATAGAAATATTGGAGCTCACTAGTTAATGTTTCTTATCTAAGACAAATACCAATGGTACAGACCTACATAAACCTCCCAACTACTCAGCTTGGCTGTCACCACGAGAACAGGACTGCAACACAATAAATCTAACACCGAATACTTCCACTACAATCGTCTCCCTAAAATATAAAATAGAAAATACGCCTTAGAGATTCATCTGCCTTTCAGGTGGAAGACGAACCTCAGCAACCGATAGTTCGTTATTCGCTATCCTTCTTACCCAGAGCCGTGACCTGCAGAACCCTGCTGACTCGCACCTGGAAGGAGCAGTACTCGGGAGTTACTCATTCGAATGAAATCCCACTAGCCCATATATAGTTTTTAGTACCTCTTTCAACTTGGATATTTCGGACTATGTACCCCGAATTCCCTACGTCCGTTACGCTCCAGTCTCGACATCAACGTGAATTCTTCATCGCATCATGATGACCCCAATTCAAATTCCTCTGCAGTGCGCGCGCGCGTTTCGCGTTTTTGGCGCTATCACGTTGTGCGTTGTTGTTCTGGCTTTCACAACACACGGCGGTTACGTTCGTGTTGGCCATAACGAAAAAACAATAGTGAAGATATATTCACTCGAGAAAGAACAAATATCCGTATTTATTGGACTGAGAGAGCGTCGAACGAG\n",
            "\n",
            "Finished processing index 35: Cynara_cardunculus.CcrdV1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Ccrd_019280::scaffold_0:0-867(+) \n",
            " TATATATATATATATATATATTGATTTTTAATCCACCGACTGCCTAATTGAATAAGCAACACAGGATTAGAGCTTACACGTGTTAAGAAGATCCATCTTTTTCATGGAAATCCATGTGATTTGTGACCCTGAATATTCAAATAAATAATTGCTCTATGGAAAAGAAGTGGACGTGCATGTATCTAATAACAGTGAGGCCCATTCCATGCCGCCTCTACACTCTAAACTCCTGGTCAGCATATGAAGCGATTCACAGCAACACCTTTCATGGGCTCACTGGAAGTCCCATGCATTAGTGTCCATCACATCAGTAGCTCGATTCCCATCTTTTGTCTGTGTGTGGCCATGCCCATCTGGACTGAAGGAAGCATCCCGTATAGTTTAATTAAGGAAATTCACCAAAATAGCTATTTCAAATTCCCTAATTTATTTTATAGCTATTTTTCTACTTAATTGAACAAAATAGCGAAAACAAAATCTGCCTTTGATATATTTAATATTCATAAAGTGACACTTTAACAAATTGTTAGAATGCTATATACGAATATTTTCACATATTCTCACATTGTGTCAGAGTCAAAGTCTACATCTGGGAGTTTCATGTTCCTATAAATTGCTAAAAGGTTACTTGTTAAAATTTGGCTATTTTGATGAATTATATTAGAAAAGAGGCTTTAAAGTAAATTGGATGGCTTAAAAATGGCTATTTTGGACAATTCCATACACTAATAAACTATTGATCTGGGTTTGCATGTTAAGCAAACAAAAAAAAGAATAAAAGAATACTATGCTTCCCCAGTGGGTTTCTATTTGTCCATTATGCGAAATGACAACGTTTTTTTTTTTGATTATTTTGGTAAATGACAG\n",
            "\n",
            "Finished processing index 36: Daucus_carota.ASM162521v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "DCAR_000504::1:19808-20808(+) \n",
            " ATCCCATACATTTTTGAAGTAAAAGATTGTAAAATATATTACTTTATAAAGCATGCTTAACTTGTAGAGCATACATGTTCATGTTGCGTAACATACATATTATAATTGAAAATATAAGAGATTTGTAAGATTTTTTTGAAGTAATGATATTTGTGAAACACGTTTTACAAGATTTTTTATTTATAAAAATATAAATGGGACTCCAATAAATCCAATAAAATGGGGACTATAAATAATATATATGTATATAATATAATTTTATTTAAAAATCACTAATAATAATATATTTATATATGTTATTATATTACGGTGCGGTGCGCTTTGAACCGCATTAGTAAACTGTCAAACCGCAACCCGCACAATAGTACGAGGTTTGTGAAAAATTCAGACCGCAATCGCATCACGAAAATTTCAAACCACATTTTGCACTGCGGTGCGGCGGTTTGGAAGGTTTTTTATCACCCCTATCTTTTAGTATTAATGTATGTTATTGATAGCAGTTTTTTTTTAATAGTTATAAATTTAATTTTTAATATTTTTTATTTCTTTGTTTATAAGAAAATAAATTTATTTATCTTGTATTAGTATGATAAACGAGTTTTTTAGTAAGAAAATTTTACAATATTTATTTCCTTTTGTTTTTGAAATATAGAATAAATGAGTTTGATACAAAAATAATAATAATGAATCATCTAATAGAAGAAAAAAATGTAAACTTGCAAAAATAATAAATATAAAAAAATATTAGATGATAGAAGTTAAAAAGGAAAATAGCAAAAGTACAATATAAAGACGTTTAGTTGTATGGTCCCAGAAAACAGGGGAAAGGCCGAACGAGTGGCTCGAAGAAAAGCTGACGCCGGCCATATCCGCTGCTAATTGTTTGCTTCCACTGTCTCTGCCTTTTGCTAAGAAATACTCACAATATATCCAGATTATAAAACCCACTTTAGTTTTTCACCGCATTTCCTAATTTCTGTTTGTTGCAGGACACAGAGGG\n",
            "\n",
            "Finished processing index 37: Digitaria_exilis.Fonio_CM05836.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Dexi1A01G0000010::1A:284-1284(+) \n",
            " GCATGTTATTAACCATCAAAGACTAAAGAATGGACATTGGAACGGGAGACAAGAAGCCTGTGTGGTGCGAATGCCATGGAGCGAGAGAAGCAAGTGCCCGATTAATTTCACTAGTTGAAGTGCTATACGGATAAAAAAGGTCACTTTGTTTGCAAAGAAGAAAATTATTGCCTTTTACCTTGCGTTCAATTGATCTACTGTAAGAGGAGTTTTTTCTTTTAGCTTTCGTTCAATTGGAGGTGATTTTCGTTAAAAAAAGAAGAAGAAGGATCGGAGCAGGTGGCAGGGGGACCTTGGGCTTGTCGAAGGGGGCGGCCTTGGCGGCGGCAGGGATCTTGGCACCCAGACTGCAAAAGAGAGAGAGAAAAAGGGGGGGGGGNNNNNNNNNNGGGGGGGGGGTCTCAGATCAGAAGGCCGTGGAGGGCATACAGATCGAAAGAAGAAACAAAAATACCTCTCGAGGTAATCCCTGAAGAAGCCGAGGTTAGGGTCGGCGAGGAGAGAAGGGTCCTTCTTGCACGCCTCCACGAACTCCCGCAGCTCGCGGGCCTTGGCGGCATCCATCGTCCGTCTTTCGCTCTACTGGGATTGGATTGGATTGGGGGTTTTCGGCCGCTCGCTAGGGTTTTGAGGGAGGGAGGGACGGAGGAGAGGTGCCCTCTGTTCGTGCTCGGCGCTTGGCTGCTGCTGGGTGCTTCTTCTCCTTCTTCTAGTGCCTTCTCCACCCAGTCAGTGTGTCCGTTCATGGGCTTAGATGGACGCTCTGGGCTGGGCTGAGCTGACAAACGATCAGCCCACCACTAGCAAGCAGCGAGCGCCCTGGTGGGCTTGGGCTGGGGCACATGGTAACGGATCCGTCAGGCACGGCGGGCGTGCGTGCGTGGTGGGGAAGGGAGACCACGTGACCGGCGTGATGATGAGTCAGTCGGTTGACTCCGACGGACGGACGGACGGAGATCTCAGTCCACAGCTGCCTCTGCCGCTGCTGCAGCCGAGAACC\n",
            "\n",
            "Finished processing index 38: Dioscorea_rotundata.TDr96_F1_v2_PseudoChromosome.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "DRNTG_29170::1:4585-5585(+) \n",
            " ATATACAAATAGCTAAAAAAAAAAAAATCAAATTTATGAATATAACCTTATGAATACATATTTTCCTATACACCTTTTAAAAAAAAACTCATATAGTAATCTATGCCATTATAAATTATGTATTTCCACTCATAGCATCCATAAATATTTATGTTCATATAACCAAATATATATATATATATATATATATGTTAGGTCTGAAGACCTAGATTAGCATTATGTGACAAGCTAAGATATGACATGGCAAAGGGAGATGACGTGGTAGTGATGACTTGATAAAATAAGTGGTGACAAGGATAGGGATGTGGCTTATGCATGGTGACATGGCACATGAAGATATGAGGGCTACAAGATCATAATGGGAGATAATGTTTAGAGAAATTCCTCTCAATGGTTCATATGAAGAATAACTATTTTTGGAAAGAGTATCAAGATTAAAGGATCTTTCTAAGAGCGAGTTTATCTTAGGTATGATTTTATCTATCTTTTGTAAGATCCAAGATGGTGAATTTTATTTTTGGTAAACTCCTTTTATGAGAGATCTATTTCTGAAAGAGAGGAATAATCTATTGTTGGCAAGAATAAAAGATCATCATAAAGATTGCATGGAGCTTACTTTGGTGCGAAGCTGTTTGAAGCCACAAACTAAGAAAGAAAAGGGCAAGCTAACATGATTCTCAAGACCATAATTAGTAATGTAATTTAAAAGAAAGATCTAAAGCTATTTATGGGTGGAGCTATTCAAGGACACAATCATATATGTTGATATTTGATTGAAGATTGATTGAAGATTAATTGAAGATTAATTGAAGATTTGATGACCCAAGCAGAGATGATTGGAGATCATGCTTGGAAGATATTGAAGGCTAAACTTCGATGAAAAGAGATCAAGTTTAGTAACAATATATCCATGATTCAAATGAAGATCATATGGGAAGACCAAACTTGGATCATGTTTGGAAAGAAGATCGAGAGATCTTTTGGAACCATCTTTGGTGAA\n",
            "\n",
            "Finished processing index 39: Echinochloa_crusgalli.ec_v3.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "scaffold3.1036::AH01:104930-105930(+) \n",
            " GCTAATAATATTTACTCTATTTCCTACTCCAAAGTTACTACATATTATTTATTATTGCTTGACAATAAGAGGTTCTTCAGCACACATGAATTTTGCCCGGAGCATATACCCTAGTATCTGCACATTTGCATTGTGTTTTTTTAGTTATATGCCGATTTGCATTGTGTTGACGACCGAGGCAATCCAAATCTTTTAAAACCTCGATTTTCCTAGGTTCCTGAGGAAAGATAGTATATCTATCCAAATCTTAAAACTTCAACTTGCCCCTAGGTTACCGAGGCAGGATAGTATCTCTCCAAATCTTAAAAACTCCAGATTTTATAGGCGAGAGTGCGTGGAGGCAGGGCGCATGGGTCCAAATGGCCAATCACGCAGCGGATAAAACTAGTCGCAGGAGCACATTTCCCCTGCCCGCACACTTGTTTAGAGTTTCGTCGCGCCGGCGCGTCGGGGGAAATGATGGTGTCAGCGCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCACACACACACACACACACACACGCCGTCTCCACTCCGTGGTCGCGTAACCAATCGCCGCGGGGCAGGCCACATGAGCCATGAGCGCCTTCGCCTTGACGAGCGTACGGTAGCGAGCCCATGGCGAGGGAGGGAAACGTAAAGCGCAACGCGAGAAATCGCTGGGCAGGGGTTAACGACGGCGGCGCCAGCGCCCATCACCGGGCGTCAAGCCTGACCTGACCGGCTGGCTGGTGCTTCCGCCGCCCCACCGGGCCACACCAGGGCCCACCAGCAGACGACACCGGCCCTTTATTCCGCCCGCTTTTCCCCTTTTGCCCTTGCGACGGACCCCATCCATCCCCTTGGCCCTCGCTCTCGCCCATGTTTCACTCTCCCGACTCCGCTTCCTGCCGTGCCGTGCTCCCCCCCCCCCCCCATCCATCTTGAAACGAAACGAAACCCCTCAAACCCCAGCGGCTCAAAACCACGGGGAGATTCCGCGAGATCTGAGCGCC\n",
            "\n",
            "Finished processing index 40: Eragrostis_curvula.CERZOS_E.curvula1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "EJB05_19047::1:4988-5988(-) \n",
            " ATCGGCTTTGGGGGAAGAAGATGAGGAGAGGTGGGTTTCATAAAAAAATATTTTATGTGAGGTCCACATCAGGTCAAAACCGGTGGAATGGTTTGGAGGGGTGTATTTTAGACGGTTTTCAAAGTTAAAGGACTTGTTTTAAACGGTTTTGTAGTTTGGGGTTGAAACTTAGACTTTGGAAAGAGTTTAGGGGGGTAATATAGACTTGTTCCTTTTACAAAAGGGACTGTCTATACGTTGCCCGAGCAATCCGAAGTAGCAAAAATGCCCGAATTTTTCAACCGTCTGATCTGGATCGAATGGCTCTCTAATCTTCTTCCTCCTCCTGACGAAGCTGCATTCCACCGCCCCTGCCTCGGCGGGCTAGGGCCTCGCCGGAGCTCCGGGCGCGCGGCGCGCCCGGAGCTCCGGCGAGGCCCTAGCCTGCCGAGGCACCACCCACCGCCACCACCCGCCGTCGCCTCTTGCCCCCGCCGTCGTCGACCTTGCCCCCGCCCGACCCTGACCCAACCGCCATCTCCCGCTGCCCGCTCCGGCGGCGCCCCCGCCGCCTCCGCCTCCGCCGTCTCGCCCGCCGCCACCGCCGGGCCGGCCTGCCTCCCCCGACCCCCCTCTTCTAAGTCCGGTGGACACCGGTGACCTCCAAACCCGGAACCCTAACCCTAACCCCGATCACTAATCTCGTCGGAGAGGTCGCCGGACTTGAGGAAAGTGCTCGGAAGGCCTCGCCGGAGTTGGGGAAGAAGGAACCAGAGAAATAGCTGTAGGAATAAAATTCGGGTGCTGCGGCTGCCCTGGAATGCCCGGGCAATGTTTAGCAATACCGTTACAAAATGGTCCTTTCTCTACAACTCATGAATTCCCAAGTGGTCGTCTGCCCCTTCTTATTCTGCCACCGTAAATTCCGTATTCCGCACGCGCCTGCGATCCAAGAGCCACACGCGCGCCTCCGATTCAAAATCCACTCGCCCCTGCGCCACCCACCACGCCACCAACTACC\n",
            "\n",
            "Finished processing index 41: Eragrostis_tef.Salk_teff_dabbi_3.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Et_10A_000405::10A:13176-14176(+) \n",
            " TGCAATTTCAAATTTCTTTCATGTCCTGTTTGGCTTTCATAAGGACCTATGTGTAGATTGTTTCTAAGGGTCGTTTTGTAAGAATTTCGATGTATGTAATGGAGATCTTTCTAGACTCTTTTAGGGTGTGTTGCCTTCAAAATAAAATAAAAAAAAACTCAAAGATTTTCTTTTATGCGACATCACACTTAATATTATTAAAAAAAGCTTTATACTCCTAGGTATTGTAGCTTGAAGATCTGCCTATATGTATGATTTTTTCAGTGTAACTCGCAAATATAAAGTTCTCAGTAAATAGACAAGGGATTGAAATATGCGTAACTATATGAGCAACTATGCCTTTCTATTACGAAAAGTAATAATGTAGTTTGTTATAATTGTTTCTGTTTCAACTCATATTGACACGTTATTGCAAGTAGGCCAATAGGTAGTTGTTGATGGTAAAGTTGACGGTGGCGACTGAACAAATTATTCCTTCACTTTTTGTGGAGATGCATGACTAGTTTTCAGGCAAATTAGCTAGTTAGAAGTGATAAATCTTCAATTACGATGTTTTAGACAAGAATGACAAGGATAACTCTTGCCCTATGCACCAGCATGCAATTGATTTTGATATACTCCTAATTAACAAAGTCTACGAGGTATAGAGAAGTCTAGACTAATATGTTCTCGTGTATAAGATTGGAGATATTTTTTATTTACGGAAGGGGAGACGTGGTCGATGATTTGTGGCTTGAGAAAAGAAATGGTAGTACAAAGACACGGAGGCACATGTCGTGGAACGGGGGCGAGTCGGCTCCATTCCTGCCGTGGGCCACCGTGACAGCGGTTGCCGTGGCATCTAGCGAAGAGGGAAAAAGTTTTGCTTCCTTGCTTCTTGCTGTCCTTCTTCTTCTTTCTCTGTCAACAAAATTCAATCAACTGAATTCATTAGAGGCTTACCACCTGCCCTCCTCTTCCTCTACTCTGATCGAATCAGAGCAGCAAAGGGGAGGAAGGG\n",
            "\n",
            "Finished processing index 42: Eucalyptus_grandis.Egrandis1_0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "EUGRSUZ_A00001::KK198753:11023-12023(+) \n",
            " TCTTTATTTAACTTAAAATAATATAAGATGAGACGAAATGTTAGGCCAAAGAAGGGGTTAAAATTTTTTCTTTTGTTGGACGGTTTTTCAATTGGGCAAATATGGCAAAAAATCCAAAATCATATCCAGTACAACATAAATACCTCAATTTTTTTTGTATCGCAAAAAAAAAAAAAAACCTAAACTAGTATAAATGTGACATATGTACTTTTTATTTCAATTTCATTTGAGATATATTTATCGTATCGGTATAAATTCAGGGTTTATTTTTTATTTTTTTATTAGGGATGAGTGTATTAAACCAGTTAAATTGAGAATTGCTCGAACTGGACCGAACTGGCTGATTTGGTTTGGCTTCTTGGGGCATAGGTCCGATTCCTAGTTCCACAAAAATGGAATCGGTGATTTATGATCTAGTTTCCAGTTCTAGGGGAGAATTGGACTAGATCGACTTTTTTTTTAAATATATATTTTTAATTTTTAAAAATAGCAATAATCATTAAAAGTGTAACAAATCTAGTATTTTTAAAATTTTTTAATATTCTTTTTGTTAAGTGATTTGTTTCACTGGATAGGATCGAACCAGTAGGTCCACTTCGATTCCCTATCCAAGACCGTTTGGCCAGGTCCTCGATCTCAAAAGCTGGGAATCGAGAATCGGACGAGAAATTGAATACTCGTAATTTTTATGACACAAACTAATAAGATTTTTAATGATATAAACAAGGTTCGAGATATCTACCATAATAAGTAAAATCCGAGATTTTTTTTGTGTCATTAAGCCTTTTGAAAGTATCCAAATTGCGCGTATGCAAAAACTGTCCTAGATGTTGACTAAAGATCTTCAACGAGTGGACGAAAATGCCCTCCACGTGCCGTCGTGTTCCCATGGCTAAAATCGGTAATCCACCGCAAAGCCCTACTGTTACGACTGCTACATTTCCATCATCATCTTCTTCATTTCGTAGATCTCAATCCGAAATACAGAAAACCCCACGGA\n",
            "\n",
            "Finished processing index 43: Eutrema_salsugineum.Eutsalg1_0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "EUTSA_v10024269mg::KI517384:8499-8731(-) \n",
            " TTTGACGATCGAGGAAATGCGAGTTTTTGGTGTCTAAAGAACATCTAACTTACCTCCAAAATGGGCTGAGCCCAACGAATTCTGGATCTAGACGTACCATTAACTTATATTGACCAGCCCAAGTCAGTGAGCCCAAAGGTTTACCTTTCCTTATGTCGGAAACCGTTTCGTGTTCGACCGCAGCACATCTCCTCTCTCTCTCTCGTCTTTCTCTGATTACTCGGTAACTGAA\n",
            "\n",
            "Finished processing index 44: Ficus_carica.UNIPI_FiCari_1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "FCD_00026666::1:32922-33922(-) \n",
            " GGGTTCGAAATATAATAAGACATAGTAAAAAAAAGGACAAGCACATCAAATTCCTTAAAGATACAGGTACTATAAATTGTTTCCTCACATACCGAGACGTAGGGCATAAAAAAGATCTTAGAGATCTGGTCCCGCCTCATCACCAATAGGAATATATAAATATGAGAGTCCTTTTAGGCGTTGGTCCACATATTTTGGACTTTATCAGGATTACAAAATGGTAAAATTTATACACAAGTTCTCAAATTTGGTGCTGGATGAGAAACGACAGTGAAATGGTAATAGGTCTCATTGCAAGACAGGTGTTGAATGGTAATGATTTTTGAGGGCAATGCTGATTGATTAGACAAGTTGGAAGGAAAGTAGTTGGGAAGAAAAAATCATATCATATGAAAGGTCAGGGAAAAAAAGTCAATAAAAAATAGCAATAAGTGAATTGCATGGGAATGTGATCTTGTGAGGTTGGATGTGAGTGTGTAATAAAGTTAATGTAAATACAAAGAAGCAAATGTAGTAAATGTGAAAATTAATGAAAAACAACAAGGTATGTGTTATTTCAAATAGATAGCTCACTTCAAATCCTAACCGAGAAAAATAGATGTTGTCTCCCCGTTGTAGCGAAACCTAGCCTTGTGAGTTAGATGTGAAGATGGCGTTGGCTGCTAGTGGGATTCAGTTGTCTGGAGTCCGACTTGGTGTTAGTGTTTCGACCTTATACTTTTGCTCTTCTCCTGCTTCTCTTGCTTCTCTTTCATGACTCATTTTAGTCGCGATCATTTGCTTTAATGCTTTCGTCTTTGTTAATTATGGGAATCACTATCCCTATTATGTTCTTTTGGATTACATGAGGTGGCCAGAAGGTTCTCATTTAACCTTCTCAGTATTGTGGTTGGCTGCACTTTGGTACCTCTGTTTCTTGAAGCTCCCTTTTATTAAAAACCATCAAAGATACCAAAAAAATATATCTATATAACAAACTAACCCTGCCTATTTTTCGCTG\n",
            "\n",
            "Finished processing index 45: Fraxinus_excelsior.BATG-0.5.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "FRAEX38873_v2_000000010::FTPI01000001.1:15314-16314(+) \n",
            " GGCAAAAGGGTTTTACAGAATGTGATAGAAGATTTGTATCCCGAAGCTCAACATCGATATTGTGTTAGACACATGTATACCAATTTCTTCAAACTGTGGAATCGAACTTGCTCTTGATGAGCGAGTTGCTTGCTAGAATGAATAGATGAATTATGGGAATACTCTTGTCTAATTCCCCCTGGAAATACGTACAAACAGTGGAATGTGAAGAAAAATGATTCAAGTCTTCGAGATCAAGCTTTTGCCAGAGGACCGGTATGTAAATGCGAAATTGTGCTTAGACAGAATGAATAGCGGAATAATACAACTTTAATCTTGATTTTGATGGAAATCGATTTGTAAGGAGAAGAAATCGATTCACAGTTAGGCTCATTGTAAAGAATTGGAAAAATTGAATTTGATTTGGGTAAGAGAAAAAAATAGTTTCTTGACACTGAGAATATCGGAAGGTTCGCAGCAGAAGGCCGGCTACAGTGAGGAAGATTCAAGAAAATGGGCTGGAGCTTAGTGTCAAGATTTGCATCAAGGAACAGTCGCCTGAAGACCTAAAGAACCGGGAAAATTGAATTTGATTTGGGTAAGGGAAAAAAAGGAACATTTCTTGCATATTTGAAGGGAAAAATGTAAGTTAAATTAATTGTTTTTGTCTAAGTGGCAATAATTAACGAAGTTTGTTGACACCTAGGATTTTCATCGGCATACATAACTTTTTGTTCAGATAGATGGATGAAAAAAATCACGTGATGAATGGATAACTTTTTTGTAAAAGATAGTGAGCAGTCAGCAACCTTTTTGTTTTTGAGTGAGTAAGTAGTAATCAGAGTTAAATGTCAGTGAAGAGTGATTTAATTAACTCGGATGGTGAGTAGGGCACTGTGGGGAAAGGGTTTTTCCAAAACTGAAGCTGAATTAGATAATAATGTGGGTGGGGCAACCCGGTAAATCGAAAACCATCTCATTCTTTTAAAAAAAAAAACGCTGCCACGTCATTCGGTGAACA\n",
            "\n",
            "Finished processing index 46: Galdieria_sulphuraria.ASM34128v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Gasu_00010::scaf_0:0-1(+) \n",
            " G\n",
            "\n",
            "Finished processing index 47: Glycine_max.Glycine_max_v2.1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "GLYMA_01G000100::1:28320-29320(-) \n",
            " CTCCAATTAACTCGATTGTAGCTTGGTTTATGACCATTCCTAATTCCGGCATGATTCTGGTGGGAATCCTCTCTCCAGCTCTCACTGTATCTCTCTTTCTTCTCTATCCTGAACCTCACTCTCACCCACCCATTCTCTCTCTCTCTCTCTCTCTCTACTCATCTCTCTTTTTCCTCTACTAGGTTGATCTCTCCTCACAAGGCACTTCTATATGGATGGACACACATTTAGATTATTAAAAGAAAATAAAAAAATAAAAAAATAAAAAATAAAAAAATAAAAAATTCATTTTTTAACTTTTGGAAAAGTCAGCTTAAAATATAATATAAAATTAGCAAAAATAATGTCTCTTGACTATTTTAAGTCTTGTTATTATTAAAAAACATAGAAGGTTGATTTAAAAAGATGAATATACCATCAATTTAGATATTGTTATATATTACTCATATTTCATTTCATTAGAGTCTCTCTTTGTGTGTGTATGCGATGCCCCTCTGTCCTATATTTTAGACCTTAGTCGAGTCGGAATGTCTGTGTCCGTATCCGTGTCTGGTGTTTGTGTCAAAGTGCGTGTTTTATAGACTGAAAGTGACAAGAAACTGAAATCTAAACAAAAAAAAAGGTAAGGTATGCTATATACAGCTGAGTAGACAACTAGCCCTAGGATGCATAGTTGTATGGGATGGTTCTCTCTATTAAGCAAAAATGTTTTAATCTATCTATCTAATCTACATACACACATTAAATTTTGTATTTTATTTCAAATTAAATATTTTTCCCAATTATTTTGATATTCACGCCATGTTCTACCAATGCTGCCTTTTCTTTCCTAAGGAAAAAATCAATATACTTTCAGCATCAAACATTTGCATGTGAAGACAAAGGCAGAACTATTGAGCAAAAATGTTGTAATCTATCTATCTAATCTACACACACACATATTAAATTTTGTATTTTCTTTGAAATTAAATATTTTTCCCAATTGTTTTGATATTCACGC\n",
            "\n",
            "Finished processing index 48: Gossypium_raimondii.Graimondii2_0_v6.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "B456_001G000100::1:6456-6986(-) \n",
            " TATAATTGTGTTTTAAAGGTATCATGATCTTGTTGTGTTTTTGTTTCATAATTGCATTTGATGTATTCAAGAAGGTATCCAGCAGGTCATGGAGGAAAAGGAAGATTCTTCTAAAGTACAGCAAGAAGGCAGTACTGATAAAAATTCAACTTGATGGCTGATAAGTGCGTAATCGGAGATGCATGTGAATACTTCTCCAACATACCTATTGTAGATACAAGTTTAGAGATTTGGTGGAGCTGGCCTGGCATGAAATCACAAGAATGATACATGTCCTGAGAATTTTCCTGTATTGAGAAAGTGACCTTTAGAAACTAAAAATATAATGGGTATCTCCATTGACAATGTTGACGCCCTTTTCCTATTTATATGCATATATATATTAATGAATTTTGGTCTTCCTGGTATTCTAGGTGATAGTTTATTCTTAATGGAACTTTGGATATGTGGTAAATATTTTCGTTTTGTGGAGTAAATTTTAGTACAAAAATTGCATATTAGTCCAACATCATCTGTGTATTCAATTGAAT\n",
            "\n",
            "Finished processing index 49: Helianthus_annuus.HanXRQr2.0-SUNRISE.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "HanXRQr2_Chr01g0000001::1:116942-117942(+) \n",
            " GTTTGGGTTGTGTTCTTAACATACATACACGTAACACATAAAGTACATAACATAATCACATAATTATTCAATTAATAACATTCTCATAGCCACGTATCGTTACACAAAGTAGGTCTAAAGTTCGAGTTGTCACAGTTATCTCCTGAAAGTTTTCAATTTTATTTTGCAGATCACTTGGAGAAACTGAATGAGAGACGAGCAACAAAAGAGCAGAAAAAGAAGAAATGTGAAAGTGTTGATCAAGAAAGAAATAACATCGCATGAAAAACAAAACTGACCTGGGTTTTGATCGGGGTTTGCGGTTTTGGTTGCAGGGGAGGCGCTCAGGTATGTCGATTTTGAACAACAAAGGGGTTCAAAAGCCCACCAAGAATGACAGGCATAGATCGCCGCTATCGAATCTTGTCTTCAACGACATGTCTCTGAACCTCAATTAACTTCTACGATGAATGAAGCTGGTCAAGTTATGGCAGAATGTAAACAGAAAGGTGAAGATACGATCCCAGCTCAACACCTCATCTCCCCTAATTGGGATTCTAAATCACTGATGGAGGTGGAAACGACTCATCTGGGTGCACATATGGATCTGGCGGTGGCAAGTTCATTTGATGAAGCTGTTTTTTCGATTAAACAACCAAAAGCGCCCACCCAGCAGGTAATTACCCCATCAAAATCTCCCTGTGTGTTGAATGCCTCGTGTTATGCTGTTCCATCTCCAGTGAAAGTCTTGGATTTTGATATGTGTGCTGAAGATTATGGGGAAGAAAATGTGGAAAATGGGGTTAGGGGTAGTGCCTGATGCTGAACAACAGATTAATCAGGCCAAAAATCTTGAACATTCTGTTCAAGTGAACTGGGGTACAGGGGTCAGAACTGGGTTTGAATGTGCAATTTTTATTTTTTTATTTTTAAAATGAAATAGGTGAAATTACAATAGTACCATCATGTGCCTTGCACATGACATAAGTTAACAGAAAATTATAATTTGGTTTGGCCTAAA\n",
            "\n",
            "Finished processing index 50: Hordeum_vulgare.MorexV3_pseudomolecules_assembly.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "HORVU.MOREX.r3.1HG0000030::1H:75743-76743(+) \n",
            " CTTTTGAAGATAGTTTGGTCCATAAGGGTCTATTTTCCGGGTCATAGAGGTCTATTTTCCATAAATAGTAAATGGTCCAGGTTGTTTTTTGCAACAAAAAAAAGGGCACGCGTGTTAGACACACGCCAGCGAGTCATACGAACCCAGACGGGATTTCCGATAGCATGTGCATGAAAATACAAGTTATGTGACCGTGACAACGTACTTTGTAAGTTTCGGCATTAAAATGAACACCTAATGTGGCAAGTTTTGTGACCGACAGTGTTATTACCTCTTGAAATATACCGTGGCATGCAACTTGAGTGTGATGGTGATGGCAATTTATGTGCGGCGAACATGACAACTTTCAAACCGAAAAAAAAGTCCCACATCGAGATGAGGTGAAACCCAAGAGAGGAATGGGTCCACCTCCCCACCCCACTCCACTACACTCTAAACTTATTGTGTTGTAATGTGGAGTTGCACGTACTACCACTCACTCACTCACTACACGTGGAGACGAGTGGAGGCAGCATTACTCCGGCCGGATCCATCCCCACAGACACAGATCGCCGTCGTCTCCCTCCGGCCGGATCCATCCCCACAGATCGCCGTCGTCTCCCTCATCCTCATGGACACCCCACCCGCGGAGGACAAGGCCAAGCGCGCGCGCAAGCGGAGCAGGTACCTCGACCACCACTACCAAACCCCCACCCTCCTCGACGCCCTACACGGCGGCGGCGGCCACCACACCCCACCTCCACCTCCACCTCCACCTCCAACAGACCTGCTCGCCGCGCTCCACCAGGCCGCGGCCTGCCCCTACGACCTCCCGGGGGGCGTCGATGCCGGCGCGCTCCTCGCCTTCTTCGCCCTCCACAGGGACTCTGTCACTGCCTCTGCCTCTGCCTCTGCGCCCGCCCTCCACAAGGCCAAGGCCAAGGCCAAGGATACCACTACTTCTATCAGCAGCAGCAGCAGCAGCAGCTCCTCCTCTCCTCGCCCGAATCCATGCCGGGCC\n",
            "\n",
            "Finished processing index 51: Hordeum_vulgare_goldenpromise.GPv1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "HORVU.MOREX.r2.1HG0000220.1::contig1:88963-89963(+) \n",
            " AAGTACCATCTTTGATACAGTGGCAACAGTTAGATACTCGTATAGGATACTACCATATGGATGTAATTAATGACACCCATCCTGCCCATAACAGTTGGTGAAAGCTACCATCATGGTGTCATGAGTGTATTTTATAAAGCATCTTAACATATAACAGCCGCTAAAACCCCATGTCATCTCCAAAGCTCGAGTCAAGCATTTCAGATACGATCACAAAATACAATAGCGTATACCTCAATACTTTTCTACTACTTGGTTTTCCTCTTTGTAAAATGGATTATGTATTGGGAAAATGGAATATAAGGCGAGATACGCAAAGGAAGAGGTATATCTTATATACCAATCATCTCGTTGAGTCTTTTAACTGTCGGATTGTCCATCGGGTTGAAATTATGAAGCATTGCATGGAGACAACTGTCAATAGAATCTAAAGCAAATTGAAATGGGATTAATCATTAGTCTTTATGACTGCTAAATTGTTACCAGAGAACACATGGACCCATTACCAGAATTTCACTGCCGCTTCTAGATACTACAAACAGTATGTAATTTACAACAATGATATTCAGTAAAATGTGCCTGTAATTAGATTGCTACCAAGTTTTAGACATGTCTATTTAGACAAGAATGGCGTGCATTAATTAATATTATCTTACAACAGTCTCAATGACAGTGAGCGGTGTACTACTTTTATGCTTACCAACTATCGGGAGCCGGGGTGTACGTAAGGCTATATATAGGTGGCTTTCGATCGCTCAACGAAAACATCCAGCAAACCAAGTCACTCTTCTAGCCCCATACATAGGTTGCTGCTTTGGGTGGATTTGGTAAAGCGGGACCCAAAATGGTATGTGTTTGATCCTCTTATGTAGTTCTTAGTTTTCTTTTCTTCAAAAGAGTGGACGAGTTTGACCAAATATGGAATGGTATGTGTTTAGATGTTGTCAATATCTTCAATTTGGGAGTAGATTGATGGTAATACTCTCCTCTAATTAATTCA\n",
            "\n",
            "Finished processing index 52: Hordeum_vulgare_tritex.Morex_V2_scaf.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "HORVU.MOREX.r2.1HG0000020::1H:141444-142444(+) \n",
            " GGAGAGCAAAGCAGAGCACTTTCTCCTCTCCTTCCTCTTCCTCTTCTTCTTCCTCCTTCCACCCTTCACACACACCAGCCAAGCCAAGACCAGACCAGAGCACACGGAGAAGAAAAAGATTGAGTCTTGAGGGCCTGCGGAGCTTACAAAGCAGCATCCAGCCCTCCTCCCCTTCTCCTTCCTTTTTTAATTCACTTGTCCTCTTTCTTGAGGGCCGGCCGGAGAAGATCGGGTCCTCTTTCTTGCCTGCAAATCAATACTCCATATATATACTCCCACATCCCGCCGGCGATTCTTCCATTGGATCTTCTTCTTCTGNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNCATTGCCGCCCTCCCAATCCTCCTGCAAATAACCAACCAACCAACCAACCACTATAGCTGTAGCAGCCACCACCACCAGCAGCAGCCCTGCTCTACCTAAGCTCCGCCAGAACCAAGTACTAGTACTGCTCCAGGACACACAAGGACAGCAAAGATTGCTTCTTTCCACTCCCCGGCGCGCTCGCCGCAGCCGCTG\n",
            "\n",
            "Finished processing index 53: Ipomoea_triloba.ASM357664v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "itb01g00040::1:12660-13660(+) \n",
            " GTATCTCCTTTCCTTCTATTCATCAGATTTTTTGCTTTGACCAATTCAAGGTGAAAAAAGGGGTTGGCTAAAAAGGGGGAGAGAGGAGAGCCTTAGGGTACGCTCACTTCTGCATTTTTGTTTAGGTTCTCGAGATTCTCAATCGCTCTTTTTAGTGGGGAGGAATAGTACGGGAATGGCGGTTCTCAGACGAGTTAATCCTTCCTCTCCCTATCGGTCGAGCTACTCCGTTCCTCTATAAATAAAAATAGGCTAGAATGCTTCTATTGAGCAAGAAAACAAGTAAAAAAGGCAAGCAACGCGCATCCGTTTTCTTGGTCCGCTACATATTATATGTTATTATAAGCCGTTGTGCGTGAGTATAAGTCTAAAGACTATAAGCGTCTTCCTTGTTCAGAACGCTTTGACCCTTTGCCATAGTATAGTTATGGAGATTCCAGATTCGCGGGAAACAATAGAAATCATTTCTTTGTGGACAGAAAGAAAGAAGATTACGACTATTTCCGATAGATAACCAAAGAACAGACAGGAGCAAGCCAGAGACCTATTCCATATGAGCTAGCTAGCCAATGAGATGAGCTACCCATTGAGGGAGCTTAAAGCGATGTCCCTATGTGCCAGCCTTTATCTTCCTTTCATATAGGTCCTTCCTTGCGTACTATACTTATAGAATTCCAGTTTAAATAAACGGCCTATCTTTTGCATTTGAAGGGACCTATCGTGACCCCGTAAACTCGCGATTTCGCAAATAGGCCATGAAGAACGTTTTCTATAAACCGAGATGAGAGATAAGCGGGATGGGCTAAGCTAAACGGATGCGAGAGTTAACGGATGACCGATCGATACAGTACGAGAGATTCCCGGGGAAGTTTTTATAAAAAGGAATGAGCGCTTACCCTATCGACCGAACGACCGAAGCTACTATCCGCAAATCATTAACTCCTAAACTCCAGAACTAAAGGAATGGTAATCTACTGAGCTAGCCTCCCGTTATGCGCCA\n",
            "\n",
            "Finished processing index 54: Juglans_regia.Walnut_2.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Jr01_00020::1:23304-24304(-) \n",
            " AACGATCTAAATCCATTTATACTTAGTCATTTATGCTTAACCCACTAATAATGCAAAGTACCATTCTACTATTGGCTTCTTGGTAGTAAATCCTAAAATAAAATGCTAAAACTCTTTCAATAATTATAGTTAAACTCAATGTGTGATTCTTAAGTTAAAAAAAAACAATCCCTCAAAATTTTCAAAAGGAAAATAAATAAATAAATATATATATATATATAATCATCATCTTACACTAACGAAAATTAAGAAGCTAAGGTCAAATTTTAATGGACCTTTAAAGTCGGGTGTTAAATAAACATGACAAATAATGAATTAGAGTTTCATATTAATTTTAGTACTATTATATAAAAATTAAAATATAATTAAATTTGTATTATAATTGATGTCGAAATTTTTTACCCATGTGTTGAGTTTATTATTAATTAGTCTTAATTTTGATGGTATTGTCATCAATTATGCTTGTAGACTTTCAGGGGGTAAGACTGGCTTCCCAAAAGACAAATTTGTTTTTAGAGATAGATTAAGTAAGATGAGTTGTGATTAAAATTAAAAAATTAAATAAAATATTATTAGAATATATTTTTTAATATTATTTTTATTTTAAGATTTAAAAAAGTTAAATTATTTATTTTATTTTATATGAAAATTATAAAAATTTATAATAATAAGATGTGATTATATAAGAGGTTTTCAAAGTTTTTAATTTTATCCCTGAATTAATCGAGGAATCACCCTTTATATTTAAAAGGATGAAGGTGAAAAGATTAGGTGTCGTTTGAATATAAAAATATTCTCAATTCATTTTATCTCATCTTATTATTATAATTTTTTAAATTTTTAAGTAAAATATAATAAATAATTTATTTTTTAAAAATTTTAAATAATAATAATATTAAAAATAATATTTTATTTAATTCATAAAAAATTATTTCATTTCTTCTGAAAAACCGGAAAAAATGAGAAATATTGAAAATGTAAAAAGATATGGGGAGGAAAT\n",
            "\n",
            "Finished processing index 55: Kalanchoe_fedtschenkoi.K_fedtschenkoi_M2_v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Kaladp0001s0002.v1.1::Scaffold_1:8386-9386(+) \n",
            " GAGCAAAATTTTATTTATTTTGAATCCTAATCAGACCAGGATTAGAAAGTCCTTTAAAATTACGGACGAACAAAATCATATGATCTAATTCAATCCAAATTGATTTGAGTCTAAATTTCGATCCGGACCTTTATTTGAATACCCTAATAGCCACACAAGACATATGACCATTTTTACACGTCGAATAGTTGTCCTTCTGACCATAACGATGCTCGTAACGGACCATCCCTAGTTGACTTGAGCTGATAGATGAAGAAGTAGAAGCTAAATTATAGTTCAATTAAAGCTGATAAAACCAAATTGGAATTAACAAAAATAAACTATATTTTTCTGTCCTAATTCTACCTCCTCCCTGTATTTCTTTAAAGAAGTCATATATTTTTATACTCTTGAAAAACAAAATATTAGGAAGCTAGCAGGAAACTTTGATGTAATTGTTTGGACATTGTCTCATAAGTCATAAATGAGGAATCATCCAGCTGGCAACAGTAATACTATGTTTAACTATGTTTGGCTTGGTTTAGGGAAAGGGAAAAAAATTCAAAAAGAGTTCTGATAATTTTCATAGAATGGATTTTATTTTTCTTTATTTGATTATGATGGAAAGTCTTATTAAAAAAATAGTAATTTACTATTGCATTGAAACAAATTATTTCTATCAATTTCCTATTCCTTATTTGGCTCTCACAAATCTTGAGAGGTGAAAAGAACATTTAGAAGAAAAAATTGACGTTTTGTGTTTGATATATTATGAAAAATTAAAATAAAATACTAGTATTTTGCAATATAAAATTCTCCTTCATAGTTTTGTTTTTTTCATGGACAAAACCATCTCACCACTAATTTTTCATTTCATTTTTTTTATCAGAGTTATCGAAATCAAAACAGAATATAAGAATTTCTCACGTAAAATCAAAAATGCAAAATTTGACAACTTTTTTCTTCCATTCTCCACGACTTTTTCTAATCCAAACAGTTCCTTAATGACTAATCCAAACAG\n",
            "\n",
            "Finished processing index 56: Lactuca_sativa.Lsat_Salinas_v7.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "gene-LSAT_1X1441::1:78440-79440(+) \n",
            " GACAAAATACTTCAATTGTTAATTTAAATATAACATTAAAGTGTCAACTTAAATATAAATTTCAGTTCCACTTAAAAAACCCAATGAATATTTTGTGAATAGTTAAAAGTGATAAATTGTAGGGCTACATGCAAAAACTAAATTATAATACCAATTTAACTAAAATATTTCTTAAAGATATTTATATAATCGTTAAAAGTTTTCAAATAAGTAGTTTAAAATAACTTACAATAATAATAGCTAAAATAACTCTATATAAATGATTGTATTATTACATTTAAAATAAAAAACAATGTCTGATGTTTTAAATAATTATAATAATAGAAATTAAAACGTTAAGAAAATTATTATAAAAAAATTAATTAACAATAACAAAAACTATAAATAACATTAAACCATATATAATATTTAATTTTATTCATGAGTAACTCGCGGATAGAAGTCATTCAAGCGGTTGAGATTATACAATTATAATAGATGTATATATTATTATATAATTCAAAATAATCAATATATTATATCACATTTAATAACAACGTTATTATTATATTTTAAAACTAGTTTATAACCCGAGGAAGTTACAAAATTAATTAATTTTTTATAGTAAAAGTTCAAAATTATTAATCAGTTATTTTAAATAAATTATTAATTAAGAGAGTTTTTTAGTTATATAAAATTATATTTTTAATGATTTAAATAAACATATAAAATTATATTATTTATAATTTTTATTTTAAAGTTATTAATTTAATGTAATTAAAGTGGAAATTTTATAATTTAAAATTTGAAATGAAAAATCAATAGATCGACAAGTGATATGTATTAATTAGAAGAGATAATATTGTGACACGTGATAAAAAAAATAATAATAATAAAAAAATATTCATTGATTAAGAGACGAAATAACTATTTTTTTTTATTAGAATAAGCCGATACATATATTTATAAAATTTCTAAGTATATAGGTATAAAATAAAAGAAACGACTATTTCCTTCGGCG\n",
            "\n",
            "Finished processing index 57: Leersia_perrieri.Lperr_V1.4.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "LPERR01G00010::1:4299-5299(+) \n",
            " GTCAGTTGTGAACCGAAAGAAGTAGATAATCTGGTCTGACAATGGCGAAATTATCCCGAACCAGCTAAGTGAAGTGGTCGAGGCTGCCGTTTGGCATGTACTCGTTGATAGCAACTTTTCCTCCTGGTGGATGCAGCAATCAACTAGCCGAACCAGGTTGGTGTGCTGCAGTTTTACCTCATTCTTAATTGCTCCAATCCTTGCTTGGAATACGAAGACGGCCGTTTCACTGCAATCTCAGCACCGCTCTGTAGCTTTCCCTGCAGCAGACTAATTCGAGGCAGAGTCGGTCATACCACTTAAAAAAAAATAGGGTATAACTAATGAATTTTTGGACTGAATGCTCGGAATTAAAATAATTATGGATAGTGGTGTCAACGGTCACGAATTGTTAGTCAATTCATGCTCAAAATTTAAAAATTATGGACAGTGGTCTTAACATTCTCGGGCTGCGTGTCAAACGGTACCAAGTCCTCGAAATTAAAAAAAATTGGATAGTTATCTAGAGTCACGGATTACGTGCCTATAGTTACACGACAAATACAGCTAAAGGTTATAACTTATAAAATTTTTGACCAAATGTTTGAATTCAAAAATTATGGATAGTGATATTAATAGTCACGGATTGTGTGTTAAACGGTTGCACAGCAAATACTCACGAAATTTTCGGTCGGAGTAAGAATTACGGCTATTTTTGTATCGGAGCAAGATTTACATTATACTTAGAAGTTTGGGACGAGGAGATTACTTGTAACCGCAGAGTATCCTATAAGCTTTTGGGTTGTGATTTTCCAAGGTAAAATTGAAGTTGACATGGCGTAGTACAACAAATACATAGTACAAGCATTTGGATTTAAACGCAACAAGTGGAATGAATCTGGACATTATGGGTCACATCTCAGCCAAAGGCCAAAAAGGTAAGCACGGAGGGTCGTCAGGGCGTCATCCGCAGAGACTGATCCCCCTGCCCTCCCCTCTCCCACCAAGCGAAGAAGCAAAG\n",
            "\n",
            "Finished processing index 58: Lolium_perenne.MPB_Lper_Kyuss_1697.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "KYUSg_chr1.5::1:52339-53339(+) \n",
            " GTTCATGCTCTGGCCCCATGTTTGGTTGTTTGTTTCCGCATAAAATCTCTCTGCCCCCTCCATTAATGCCATCGATCTGCCATTAGCATCACCTTTTGCAGGGGGATCTTTAGGGAATATGTTGCTATGAGCAAGCTTTTTGAGCCGCGACTCAAATCCATAAGCCCTTGACGGATATATTTGCTGCCCTGCGCTTGCATAGTTGAATTATACTACAAAGCAGTCAGCTTGTTTTCCAATTCTTAGATCCATAAGCCTCTGAATCAGTATGGGCTATTCACTTCTTCTTTAGTAGCAGCCACCAGGTGAACTATTGTTTCTTGGAAGATGCACTACAACAAGTTTTTATTGCATTGCTGTGCACATGTTTATTTACGTCTGTAGGGTCTGAAAGCTTGAAAATAGAAATGTTGATTATCATAATTCCTGACCTTTTAACTGGAAAGCATTCTGTAACTGTGGTTTCCCTCATGTTGTTAATGTAGAATTCTGTGCAGTAGCAATATTGGGTATTCATTTGCTTCTGTAATAGAAGCCACAAGGTGAACTAGTATTTCTTGGAAAATGCAATACTATGAGTTCCTATTTATTGTTTTGCACATGTTTATTTAAGTCTGCTATATAATCTGTGACCTATTTTTTAACTGGAGAGCATTGTACAATTTTGGGTGTTCTCATGTTGGTAATGCAGAATTCTGTGCAGAGCCAAATATTCATCATTCTGTTATGTAACACTTAACAAAAATGTACGAGGCATTTAAATTTCTAACTGGAAAGCATTCTTTGTTCTATATTTATGCAGTAATTTTGTTATTGCCTCATAATTCAATCCATTTAACATGTAGATAATACTGTACTGGAAAGCATTCACAGTAGTCAGTAGAGCCCAGCTGTTCTACTATTTCAGATTCCTGAAAACTTGCTTCCTTGAACATCCTTAGATGTTGTTGTTAGTTGACTTTTTTCCCCTTGTGATGGTATTTCAGGATTGGTGGTTGCT\n",
            "\n",
            "Finished processing index 59: Lupinus_angustifolius.LupAngTanjil_v1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "TanjilG_00076::KV862194:0-127(+) \n",
            " AGGATGAGGATGAGGACGAGAATGAGGATGAGGATAGGATGAGGGTAGGATGAGGAATAGAATGAGGATGAGGAAGTGGATGAGGATAGGTTGAGGATGAGAATATGGATGACGATAAGGATAGGAT\n",
            "\n",
            "Finished processing index 60: Malus_domestica_golden.ASM211411v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "MD01G0000000::1:19041-20041(-) \n",
            " CCATGCATCTGGGAGCCACCATCAGCAAGGCGATATGAGCCTGGGATTGGATTTTTTGGGGTTTGGGGTTCTGCCGACGGAAGAGATGTTGCATCAGGAAGAGAGGATGAGGTGATGGTATGGGGGCAAAATTGGGGATTGGGGGATGGGGTTGGTGACGGGAGAAGGGGTGACTCAGGTGGGCAAGGGGTGGAGGGATGTCGGGTAGGGGTGGGGAAGACGAAAGGGTTCTGGGTTTTGTTTTTTTTAATTTTTTTATTAATTTTTTTAATTTTTTAATTTTTTTTATTTTTAATTTCCACGTGGACCCTTAATGACACGTGATGCCCAATCATTGTCCACATAGGCGCCACGTCAGCAGTTAACTGCAGACTTAACAGTTTGACTAACGAATGTATGAGACTGTTCCAAAATTAACACTTTAGGTATGACTCTGAGACGAAAAAAACTTTATGTACTAAATGTTGAAAACCACGAAACATGAGGGTAGTAAACAGAGTTTAACCCAAAATTTTAAGTTGAGTATAGAGATACAAAACATGATTTCAAATAAAATTTTCTTAACAAGAAGTATAATGTAATGAACAAAAAAGTAAAGCTGGAAGGTAATGACTCATTTGGAAGTACATTTAAAATAACTGAAAGCACTTGTGATAAAAATATTTTTGAAACCAATCTTTAGTAAAAATTCAAGCGAATCTTGGAAAAACATTTTAAGTGCTTCCGATAATAATCAAATATCTGGTATTTTTTCTAAAAAAATACTGAAAATAACTTTTGGAACAAAAAATCAATTTCGCCAAATGAATTTTTAATTATTTTAAAAATACTTTTAAACGAGCCCATTTGCCTGGCCGGGATCATCTTTCATGGATTTTTCTTTCACTTGCTATTACCTTGGGCCCCAATCCAATAACCCTGACAATTTCGGAGCCCCTAAAGCCATGTGCACCGCACGTGTCTTGAGGTTTTGCATCAACCTTCCCCTCTTGCCCTAAAA\n",
            "\n",
            "Finished processing index 61: Manihot_esculenta.Manihot_esculenta_v6.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "MANES_01G000100::LG1:0-3(+) \n",
            " AAC\n",
            "\n",
            "Finished processing index 62: Marchantia_polymorpha.Marchanta_polymorpha_v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "MARPO_YA0001::Chr_Y_A:10935-11935(-) \n",
            " CTTGTGGGCACTAGATCCAAGCTCTCTTTCTGTTACAAAGGATTAGAGTTGGAACAAGTGAAAAGTTACAAATATTTGGGCATTGATTTCTCATCTGATTATAGTTGGGCACTTTGTGTGCAAAAGAGGGTAGAAGCTGGTTTCAAAGCTCTATATTCATTGCTAAATAGATGTAGAGGAGCTCATCTAGTGTCTTGGAGTCTTAGGAAATTCTTATTCAGGAGCTTAGTGCTTCCCGTTATCTTATATGAAACACAAGTGTGGGGTCCTGGTTGTAAACCCTAAACCCTAAACCCATTTACCATTTACCATTAAGAAGACAGGTTGGCAGCGTTTGGAGAGGGTGCAAAAGATCTTCTTACAAGAAGAATTGGGTATGAGGACCCAAACTTCTTATGAATCTTGTGGGGACCATATCTTCTTGCTGGAGTGTCAACAGGAGCTGAGATTTGGTGGGCTTGTTTACGACGGCCGGGAGCACATACCTGCAAATTTTCAGCCCCAACGGAGCTACTGTCAAGAAGTTGTCATTTCGAAACTTTGAGTTTAGGGATTTGAGCATGAATCTTGTGGGGGCCATATCTTCTTGCTGGAGTGTCAACAGGAGCTGCGATTTGGTGAGCGTGTTCACTACAGCCGGACACACATACCTGCTAATTTTCAGTCCCAACGGAGCTGCTGTCAAGAAGTTGTCATTTCGAAACTTTGAGTTTAGGGATTTGAGCATGAATCTTGTGGGGGCCATATCTTCTTGCTGGAGTGTCAACAGGAGCTGAGATTTAATGGGCTTGTTCACGACGGCCGCGCGCACATACGTCCTAATTTTCAGCCCCAACGGAGCTGCTGTCAAGAAGTTGTCATTTCGAAACTTTGAGTTTAGGGATTTGAGCATGAATTTTGTGGTGGCCATATCTTCTTGCTGGAGTGTCAACAGGAGCTGAGATTTGGTGGGCTTGTTCACTACAGCCGGACGCACATACCTGCAAATTTTCAGCTACAA\n",
            "\n",
            "Finished processing index 63: Medicago_truncatula.MedtrA17_4.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "MTR_1g004930::1:7332-8332(-) \n",
            " AACACTTGATCGAACATGGAAATTGCCTAAGTTCTTCTTTATCTTTTGTGCTTAGCTAAGCAAATGCAACATTCATCTGAACCACAACACAACACGACAAACTTATTTAATTCAATAATCTATTACCCATACATATTGTTGAATGCAGAAAAAAAATTAGAATGAATGATACCAACGGTACTTAAGTGTCATGAAGTTTTTTAGCGAATGAAAATTAAGTAGCGTTGGAGGCATTTTAGTATTTTTGGGTGTTAAAGAGCAAAAGTCTAAGTGAAATGCACAAAATCTGACGTATATTGGGGAAAACACTAACTTCCTACCTAGGTGATGGTGATTTTGCCGCCCATGGTTACACCATTTTTATTCTAATCACCAACCAGAAGTTTCTTGAAGCAGCTGAAATCACGCAGGTGTTTTCAATGGCTTATTACCTATATTTGCCTTGAGAGACCAATGCATTTTCCAACTCGTGGATTTCTTTGTAGGATAAAATCTTCAAGAAAGTACTCCATCTCTGTCGTTCAATTCGCTTGATGCGAGGCTCATCTAGTGAAGAATCGACTCAACACATTGATCTTCTCAACATAATACTCCCTGATCGACCAAGAACTAGGGGTGTATCCCCCTTACTTACCAAGAAAGCTATTCATATTTTTCTTTTATGTTCTCTTTTAATTGAATCTTGTTTGTCATTTCTACCAATCTTATTACTTTTCGACGCTTTACTTTAGACAAGAAGCTTGATGTAGCCGATTTAGATGTTAGTAACGAAAAGCCATCACCAATTCCTAGAAAGAGGACATAATCTATCAGATTTCAAATTTACGAACATTTGGACTTGATGATGTCTAAAGTTTCAAATAGACATTCAGCTGACGTACCCCTCTCATCCGCGAAAAGGAAGGAAAAGTCCAAGGTAATTTTTTCTTCCCGTTTATATCATTTCTCCCTGACACAAAGGTGAAATTACCCAGAAACTGATTCCCAATAAGTTGTTGAC\n",
            "\n",
            "Finished processing index 64: Musa_acuminata.Musa_acuminata_v2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Ma01_g00010::CAJGYN010000001.1:2135-3135(+) \n",
            " GCTAACAACATGGCTGAGCAAGTATAATCAACCCAGGCTAATGTGAAGAAGTTGAAAGCTACTATTGCCAGGTACAAGGAAGCAGTTGATAAGCATAGGAGAGAGAAAGTGTTCGAGGAAAGAAATATAGTTATGGTGTACATGCGAAAAGAGAGATTTCCCATAGGCACTTACAACAAATTAAAGTCGAAGAAGTATGGTCCCCATAAATTCTTAAAGAAAATTAATAACAATGCTTATGTGATTGATTTACCAGATGATTGGGGAATTTCAAAAACCTTCAATGTTGTAGACTTATATGAATTTTTTTCAAATGAGAATCCATTGTATTCAGATATAAACTCGAGGACGAGATTTTTCTCAATTGGAGGTGATTAGTACGGAACAAGTAGAAGGATACCAGTTTCAAAGAGCCTAAAGCTACCCACTTATCACTTTATTTATCCCACATTGTTAATTTTTATGTTTCTTTCAATTATCTCACATTGTTAGTTGGTTAGTTTTCAAGGCCTAGAACTAGTATAAATGATGACCTCATCTAAGCATATGGAATCATAAAAAAAGCAATATCTTCAAGTACTGTAAAGGTTGTCTCCACTAGGAGTTGATTTCCCTCGTCCTCCCTTATAAGGTTAGTTTATTAGAATCATTTTTGGTAGTAATTTCTTAATTTTGTTTTTGTTCTGCGTCAAATAGTTATATCGGTTTATAACAGGTTGGGAGGTGATGAAGGTGTTTATCATAATGTATATGTAATTATTTGATGATTAGCAGTATTCAAGTGCTTGTGAAGAAGTATACGAATATATCACAACATATCAATAGCAAAACTCAAGAAAAGTTGAATGCTTATAAATAATTTGAAGATTTTTTATGGAATGTGTTTGATATTATGCTTGTTTTAAGTCCAAAATTTTCTTTCAAAGCATAGGAAGTTGAACTTTTCTCTGATTCATTTATCCATCCTCGTTTAACATGTTTTTCACTTGTTTATTGACTT\n",
            "\n",
            "Finished processing index 65: Nicotiana_attenuata.NIATTr2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "A4A49_26459::1:5155-6155(+) \n",
            " TCAAATCTTTTATATGTTTGGGACCTTTTGGGATCTATATATAGGGTTGTAGTTGGGGATTCTATAGATTGTACTGTGCTTTTTTTTCATTCATAGTGGAAAACTCTCTCTGCTTTTGCCCCGAGGATTAGGCTTAGCTGAACCTCGTTAAATCTTTGTGTTGATTTTTCTTCTCTATTTGCTTTGTGTATGATTGTTTGCTAGTAAACTCACGATATTCCGCAACAAATATTAATTGAACATTTAACAAAGTTGTTATACTTTAATCATTTATGAAGTTGATGGCTTTGATTCTTAAACTTCGCCCTTTTGGAGCAAATTAAGAACAGATGAATGAAGCTATATGGGATATTATTTAAGACGAAATATGAAATTGATCAGGTAAACTTGCTTCCTTAGTTAATTATTTAATAACCACAATTTCAGGCATAATGGTCAAATACTCCTTGTGATTGAAAAAGAAAAGAAATAATAATCAAATTAGTTTTGTGAATAAAGGGTTAGCAAAGCAGGTCGGCTACTTGTCTAAACTGTAAGGCTGCTCTATATTATTCAACTATGAAAGGCACGAGTTTCTCCCTAGTATTGATAATAGCCTAACATGTTGAAATTAAACTACCAAAGCCGAGGTGGTTGTGGCGTGGAATTGTTGTTACACGTGTAAAGAAGTTGACTAAAACGTAAAATCGTTCATCACGTCACCTCGTTTCCAAAATATACCTTGGAAGTCAATCTTTAATTTTGTCTTAGTCGCAGTAGAGGAAACAGTAAAGCTGGCTGTCATGCCTTGACCATATTTTTTGGAAAAAAACGATATCTAAAGTTTGTTCTACTTGGAGACTCTCTCAAGAAACTGCCGGAAAAGTTCCTACTTTTGAATTTTGTTAGTAAATCAATTCATTTGAATAATGCGATTAGCCTCCTTCAGTCCTTTGCCTGGTCAAAAGACTAGTCCCCTCGTTAACTAGTATAAAATGGCTTGAATCACCAAATAGTATTT\n",
            "\n",
            "Finished processing index 66: Nymphaea_colorata.ASM883128v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "NC1G0091270::1:8009-9009(-) \n",
            " TAATGATTGCTACAAAATTCGGACAAATAACAGAGCTTTGTTCCGATATTTTTCAAGAAGTCTTGAAGGCATTGCCGCACAATGGTATAAAGAAAATATCAATCCAATCGAGCTGAAGGAGTTTGACAAGTTGATTAACATGTTTGTTGAAAGATTCAAACAAAATGCTCCCATGGCTCCAACCCTCAGTACCATTTGCAGTCTTCGTCAGAAACAAGGAGAAAAGGCTCGAGAATTCATCCAAAAATGGAGAATGCAATGCAATAAGATGAAGGAGCCCATTTCTGAGACACAAGCCTTGTCACTCATTAGGAAAAATCTTGCCCAACCCTTGAAAAGTCTCATCCGCAATGCCCCGATCAAAACCTTTGCTGAGTTGATTGAACAAGCAAACTCGATAGAAGAAGGAATTGAGGAAGGAGATTTTGATGGAATTATTGCTCCTAAGTACAAAGAAGATGGAAAGAAAGGAGGAAGGACACAGTTGCCCGCACACTTCATAGCCAATGCTGGCATTAGAAACGACCATGGCAATAACTACAAGCATGACAAGGACGTCAGTTTCAAGAAGAACACGCAGTTTTTCAAAGGCGACGATGAAAGGAAAAACAAACACCCAGGTTGGAGCTACGACCGGAAATTCACTCCTCTGAGCCAGTCTCGGGAGAAGATCCTGGAATACCTTCTCGCCAAAGGAACTATCGTCTTGCCAAAAGTATCAGAGCCTCCAAAAATTATGGGAGGAAATAAAGAAAAACTGTGCAAGTTTCATCGTACCCCTGGTCATGACACTGAGGATTGTTTTGTCCTCAAAAACATTATTCAAGACTTCATCAACAAAGATCTCAAGATTTGTGATGATAACACCCCTGAGATACTAAGGAATCCATTCCCTGACCACGGAAAGGCGGCAGTGGCCATGATCACTACAGGCACTCCACTTCCGTACCATCCACAATAGCACATTCTACCTTATATTGGGAGCAGCTCACACAAAGTC\n",
            "\n",
            "Finished processing index 67: Olea_europaea.OLEA9.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OE9A017521::1:22035-23035(+) \n",
            " TTATGTGGGCTACGTTATGTGTCCAACAAAAAGTCTTACATTCTCTCACTTGTTTACAGTTTTCCGTATAGACTTATGTTGCAATAGAATTAGTCTAGCAACTAGTTTGCAAAACTAAAAATCTAAGGTTAATAACAAAAACCACTGCATGAACAGAAAAATAATCCCACCCAACAATTAATTAACAATGAACAATTTTAAAAAGCATTTTAGAAAAAATCAATTAGGAAAAGGAAAATATGAAATAGGAGTACATGTGCCAGAAAGTGAAAGAAATAACGAAAAAGAAATTTAAAAATCTAGACATATCTAAAACTAGTAAAACAGTATAGAAATTGAAAAAGGAGATGAAAAATTGAGAGGTGGATTGTTGATGGTAAATCGATGATGATGGGCACTACATCAGCTCTGACCTCCACCGTCGTTCTTCCATTTAGGTTTTGCCGTTGATTTCTCAAGCTTTCAGTGTGGAGATTTTGAAATTTTGAAGGTTTAGGTATGGATGGTTTTTTTTTTCCTTTAAAAAAATGAAAAGGAAAAAAAAAATTGGGCCTCCTAAAAAGCTGGGCCTGAAGCGAAGGCCTCTTGGGACTCTCCTTATGTCCGGCCCTACCTATGAGTTGACCGTGCCAATTGATGATTGACTTGCACAGAAGAACTTAGACTCTCTCACCCCCACCAATATATTTAGTTTATGTTCGTCTTTCATTAATATTTTCTTTCTTTTTTTTTTTTGTGAGAAAAATGAGTTTTGAAATAGTGAATTTTTAAACTTGTAATTTATTGTATTTAACATTGGTTGAAAATGAGATTAAGAAAAATGATTTTATTGGAAGAGAAACTGGGTTCAGATTCTAGTGTACAATCCCAACAATGAATAAAACTTTGTAGGCCAACGAAGTCGAAAAGAGTAGCACGGAACAAATAAATGGAGGAGTGAGACAAAAAGAAGGGAAAAAGGGAGGGAGAAATGGACTAAGTGATCGCGAATCTCACTGAA\n",
            "\n",
            "Finished processing index 68: Olea_europaea_sylvestris.O_europaea_v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Oeu061231::1:3692-4692(-) \n",
            " NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNAAATTGTTCTTTTGTATTTTCTTTGATTCTAGCTTTCTTTTTCTTTATTTTATTTTATTTTAACTTTTTCTATTTTGTATAACAGAGATGAAACGGGGTTGAAAATCAGGGGCGCGTTCCACCCTTACGCTCTAACCCAGCCACTTCTCCGTCCCCTGTTTATCCTCTCTATATTCTTTGTTCTCTGTCGATTTTGAATTTACAGATTCATGTAATAAATGTCTTCGTCGAGGGTTAATTCCTGTTATGAAAATTTATTCTAATTCACCGAGCGCTCATTTCTCACACACTAAATTTTAACAGAAGTAAAACATAAACGAAGAGAAAGA\n",
            "\n",
            "Finished processing index 69: Oryza_barthii.O.barthii_v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OBART01G00010::1:2175-3175(+) \n",
            " TTCTTCCTCTCTTGATTCAACGGGAGGAGTGGCAACTGACAGCAACAGAGCTCGCCTCATGGGCGATGGAACCAGCGGCGAAGCTCGGGGCGGAGGGGTTGAGCCACATGAGGCGATGGCGACAATGAGGCGAGACATGGCGTGGCTGGCTGTTACATTTTGTTTTGATGAAAAGCATAACCATGCGGATGATATTTTTATTATAGACTAGAGATGATTATTGAATAGACATGCTCTTAACCATTTTTAACTCTAATTCCACCTACACTATGAATATTAGAAAAAAATACAATGTATACTCTTAAGAAGTTTGCGCTATCATAGCCATCATGCTCATTAAAACATTTTTCTTCACCTGTTGTAACGCACTGCCATTTTGCTAGTTAGAGAATAAAAATAAATTTGTAGGTAAAACTTTTATATATGTGTTCTCGGTGACTTAAAAGCCAATGCTTAAAAAGAAACTACGTTAAAAATATCTTAAAATCAATATTAAAATTAAGTTTAAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCCGATGGAAGCTGAATAGGCCATCCGATGGGAGCTGTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGCATCCCAACAGCTCCTCTATCTAGTCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTTATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATAGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTTCCAAGATTTTTTTTTGAGCGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAAGATGTTGGTATATGTAGGATGTAATATAGATGATCTGTTGAAGTGAAAAGTTATATAGAGTAGGAATCTTGTTAGGCCATTCAAATGAGAGTATGGAGGATGAAATTTACACGAGAAGCACAG\n",
            "\n",
            "Finished processing index 70: Oryza_brachyantha.Oryza_brachyantha.v1.4b.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OB01G10010::1:13455-14455(+) \n",
            " ACCAAACAAGTCAAACATCTTACATTAGTAAAACAGAGAGTATTACGTTTTCCACGTCAGATAACTACACCACTTCTCCCTCCAGCTCCCTCCAGCTGAACGCAGCCCTAGTGTTGTAGCCTTGTAGGGAGGCATGACGCAAGGGAAGAGTCACTAGGCAGGAGATCTAGTGCTGCTTGACCTGCGGCCGCTAGATGTGTTTTTCACCGAGCTTGTTGCAGTTGAATATGACGGTCGACGTGGATGTTGAACCGTCGTCGTTTAAAGGGAGAGGAGATGGTGTAATATATGGTTTCTGTTGGTTTGGCTTCCATGAGAACACCTCTGTTATCCCAAGCTTGTTGCAGTGACGGTTGGTGTGGGCGTTGAATCGCCGCAATTCAAAAGGAGAGGAGAGGGTGTAATATGTTGTTTCTGCTGGTTTGGCTTCCAAGAGAACACAGAGATGAAGAAATTTAACTATACGCCACTCTATCTATATGTCTTTAATAAAATATCACTAGTATATTTTTATAAAAAATAACACACTTCATCTAATCCCTCCATCCTATTTGAAGGTGTATTTTCATTTTTTTCATTAAAACAACGTGAAATGTCCAGAATGCCCTCTTCTATTCTGAATAGATGTGAGCTGGCTGGACAGATCTAGCGTCGGCTCGTCCTCCCATCGCTCCATCAGGCAGCCACGCCCGACGCCGCCCTCCCCCAGTTCCCCGCCGAACTGCACGGTCGCNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTCGCCGCCTCGGGGCGGCGCCGGCAGTTCTCCTCCCCCGGCTGCGGCAGTACCCAGCCGTCGTGCTGCCAGCTCTCCTCCCGCGGTAAGCCCTATTT\n",
            "\n",
            "Finished processing index 71: Oryza_glaberrima.Oryza_glaberrima_V1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "ORGLA01G0000100::1:0-1(+) \n",
            " A\n",
            "\n",
            "Finished processing index 72: Oryza_glumipatula.Oryza_glumaepatula_v1.5.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OGLUM01G00010::1:7426-8426(-) \n",
            " AGTCTATAATAAAAATATCATCCGCATGGTTATGCTTTTCATCAAAACAAAATGTAACAGCCAGCCACGCCATGCCTCGCCTCATTGTCGCCATCGCCTCATGTGGCTCGACCCCTCCGCCCCGAGCTTCGCCGCTGGTTCCATCGCCCATGAGATGAGCTCTGTTGCCGTCAGTTGCCACTCCTCCCGTTGAATCAAGAGAGGAAGAAGAAGTTGAGGGAGAGGGGACAGCCGAATCTGCGGGTGGCGGATCCAACGGTGGCTCGGGTCATGGGCGGTGGATCCATTGTTGGCTAATTGACCAACCAGTCACACCATGTCACGCCACGCCTTGCCTCGTTTTGCCCTCGCTCTCGCCTCCTTTTTCTCTTGTGGCAACACAGCTTGAGTTGTGGGCGATGGTATCCGTGGCGGAGCTCGGGTCGCGAGCGACAGTACTGGCAGTGAAGCTCAGGGCAGTGGCATCAATCCACAGGAGGCGTAGCTCGGTCGCGGGTTGTGGGTGGTGGATCCAACGATGACTCGGGTCGTGGGCGGCAGATCTGGTCGGCTCGAGACATGGGTGGTGGGGTAAGTCTCGTGGAACTTTTTTTACGATTGTTACTTTTTGTCTCAAAAAATATTAAAGTATAAGTTTCGGTTATTTTGATATCAACAATAAAAGATGTTCCATAAAAGATCTTAATACAAAGTTAAAAAGAAGAGAAAATTGTGTGGTCTGGTCTATATCCTTATCCTCACTCTCTTGTTCTCTCTCTTCCATTCTGTCTCTCTTGAGTTACTCTCCAAGCCGGCCGGCGTTGCATCTCGCTCAGAGCCGGTGCCGGTGCCCCTCTCGCGCGGACCTATTGTCGATGGCGATGGGAGGGGCGGATTCGGCGACTTCTTCCCCTCCCGCACGGATCCATGATCTCCGCTCCCCTCTCGCCCGGATCCGTCGCCGGCGCCTGGATCCCGTTGTCTCTCCGGTTCCTCTCGCGTGGATCCGTCGTCTGCTGCG\n",
            "\n",
            "Finished processing index 73: Oryza_indica.ASM465v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "BGIOSGA002568::1:17112-18112(+) \n",
            " TGCATTCCAAAAGAAACTGATGTTGAATCTAGGTAGTTGAATTTGGTAGGCATACAGGTTACAAAATCAGTCACTGACAACAAGAGCCCATAAGTCAGTGATGGAAGTAGTATAGATATCATAACGGTCTCCACGTCGTAATCCATCTAGACCTACGCCGTGCTAATATTACCTACATTAACGATTTTCACGTTGTTATCCATCGAGACTTACCCTATCTACTACTCCCTTCGTTTAAGCTTATAAGTAGTTTTGACTTTGGTCAAACTTAAACTATGTCAAGACTAAGAGAGTGTTTAAGATTAAAGCAGTGTCGGAGCCAATAATTTTGAGCCTGGGCAAACCTAATAAGAGAAAAACTATATATAGAAAATTGTACTATTTATCATATATGCAAGTTAAGTTAATTAGTGCCCAGCGCTAATTTATCAATGAGATGATGGCTGTTTATCGTCAGTGAGTTAGACTATAGGGCTGCAACTCTTTGCTTTGGCCTCATGGTTACAAACGTAATGATTTTGTCCTTGCTTTGCTTGTTTGTTGGCAAAACATATAATTAACATCCACTAGAACCAGGAAGCAAGGTAATTAGGTTCAAATTAAGTAGTAAATGATGTTTATGCCATGGAGCAATGTGTTGTTGTGTGGTACGGAGCCCGGGCACCTCCCGGTACAAGCCGAGGCGCCCTACTCCGATCCACCACGGGAGTAAAGGATTAAGAAGATCGAGAAAACACGTAAAACAAGTGAACCATTAGGATACAATTAACCAGGGGGTCCAATTATCAAGCAATAAAGTTATAGCTTTCTAATAAAATCTCGTACAGTATTTATTAATGTTGCAGTTGTCTGAAATGTCTAAATAAAGTTGCAGTAAATTGATGATTACTACAAATGGTCAATTAAACTAGATTATAAGAGAATGGAGGTACTAGATTATTTACTTATGATAATTAATTAAACTGCTAACTAGGTAGTAATATGTGGGTTTATGCATTGC\n",
            "\n",
            "Finished processing index 74: Oryza_longistaminata.O_longistaminata_v1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "AMDW01000314.1_FG001::AMDW01000314.1:0-41(+) \n",
            " CGTCCGCGTCCGCGTCCGCATCCGCGCCGCTCATCGCCGAC\n",
            "\n",
            "Finished processing index 75: Oryza_meridionalis.Oryza_meridionalis_v1.3.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OMERI01G00010::1:13404-14404(-) \n",
            " CAAACCATTATATTTACTATTCAGACTACTAAAAAATTGTGCTGATCTTGTCACCAAAACGAGTAACCAAAGAGGAGTACTATAAGTTTCACACATCTCATACAATAGCTTCTTTTGAAACGCAAGAATTTATAAGAATTTCATCGTTTTTAGTTTTTTTTGGATAATGGAATATAACATCCCGGTCTTTACTCAAGAGAGCTACAACCAATTATTACATAGTAGCACTCAAATAAGAGTGTAGTTCAATAAATCATTGAAAACACCCAATGAACAAAATNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNAAAACACCCAATGAACAAAATAAGTAACGAAACTAAAATAAGGAGAACACATTACTGAAGTCTATTTGTGAATCTCCAACTATAATTGACAAAAAGTTATATAATTGTCGT\n",
            "\n",
            "Finished processing index 76: Oryza_nivara.Oryza_nivara_v1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "ONIVA01G00010::1:3847-4847(+) \n",
            " ACATCGAATGTTCCGACAAAATTGGTTCATCACCCATGTGTAGATGCAGTCTACTTTTGTAGTTTTGTTTTGCTGCGGGAAATTCGGATTTGTTTTTTGGAAACAGTTTGAATAATCCGGTGCGTGAGACGGCATGGGGTCCAAATAAGTTACACGCGACTTATCCGGTTCTGTTACACGGAATAGAGATCGTGCAGGCGTCTTGATCAAAAGTCAAGCCTCCGCCTCCACGTAATACACGTCAAATCAATTTAGGGTTTGCTTTTTCCTCTGTATTGCACCGTCACTCGTAGTCTACTCCATCCCACTTGTCGGCGTGCACCGGTACGGTGATTGGGGAGGGGGGGCAGGTCTCCGGAACCTTCGCCTTCGACGTCCTGCACTGGGGTGAAACAGTGTGGAGTCCGAATAAGTTACACGCAACTCATCTGGTTCAGTTACGTGGAGTGGAGATCGTACATGCGTCCTGATCAAGCGTCCTGATATCTCTATATAAGTCAAGCCACAGCCTCCACGCAATACACGTGAAATCAATATAGGGTTTGGCTCTTCCTCTATACTGCACTGTCGCTCGTAGTCTACTCCACCCCGCTTGTCGACGTGCGCTGGTGATCGGGAGAGCAGGTCTCCGGAACCTTCGCCTTCGACGTCATGCAATGGGAGAAGGCGGTAATAAAGTTTATGGGAAGCGCTTCGTACGACTGCTCCCTGTTCGATCACAACGGCTTGTTTTCCTTTTCGCTTGCGTGCTGCGCGCCGTCGTTAAAGCCCGCAACCGCGAGTCTTTCCTGCTAAGCAGCCGGTTTTCCGATACCCTCAGTATGTGTTTAACATGTTACGCATATTTGGTCTGTTCATTCTTTACTGTCTAGTTTACATGTGTAGATCTTATGATGCGTTTATGCTAGTTTATCTCTGTAATATCATGATTTATTTATGAAATAAATTAAATCGTTATGTTATGTGCTTATAAATTCAACATGTACACACCCACGCTGCC\n",
            "\n",
            "Finished processing index 77: Oryza_punctata.Oryza_punctata_v1.2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OPUNC01G00010::1:815-1815(+) \n",
            " TGCCAAATTTAGGTATTAACATAGCTTGCCTCGATAGGGCGAAGGAGGAGGAGCCTGGACAATAGTTGGGGGCTCAAGTACGAGCTCGTCCACGTCGGGCTTGCCCCTCATAATCGCCACGGTGGCCTCCTGCCAATGCTCTCCCTAGAGGCAGGGGAGAGCGCGAGAAATTGAATCCGTGCATGCGCACCCCTACCTGCATGCTAGTTGTCGGAGCCCGAGCTCCTTAGGTGAGGAAGTGCGGTACCCCTTCCCCTTTGAGATTCGGCTAGGGTGCAGGTGCACGAGGTCGATTAATCACAACGCGGGACACGAAGAGGACAAACAATGTACGCACACATGTTTGAGCCACTCGAAAACGTAATACACAACTCCTATATTCTTTACTAGATTTGTGGTTGGTGAAAAATTACAATGCGTGAGCCTCTTCTCTTGGACTCTGACCACTACTCTAAATATTTCTTAGGGAGTCTAACAAGATGAACTCTTTTTGGATGTTGTAGTCAACAAGGGTGATTCCGGATGTCATAGTCAGCAAGGGTGCTCAATTTCTCTGCCGAAACTCTCAACCAGTCGAACCCCCTTCCTTGTGGCTTTGGCCTTTTTTTATAGCTATAAAGGGATGCCACAAGTGCCAACTGCCATACAAGTCCCACATCTACAAGAATGTGTAGGTTGGGTCTTGTCATGGCCCAACTGTAAGTATTATTGTCAGCCCTAAAAACCCCCGCCAGTTGTGTTGGCAGGAGGTATGAAAAATCTTCTCCACCTCCTCACTTCTATTGACGAGTGACACATGGAAGGACGATGCGGATGCGACATAGGATGCAAGGCCCACGAGCCATTCGCATAGACTGGGTCGACCTGCCCACGGAGATGCATATTGGACAACAAGTCAACCAATCATGTCACCCTCCTTTCCAAGATGCATGATGGCCGTAAAAAAATGTGCGATGATGCACGAAAGATATATATCTCACATCATTAAATGTTTTAGTAG\n",
            "\n",
            "Finished processing index 78: Oryza_rufipogon.OR_W1943.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "ORUFI01G00010::1:7125-7786(-) \n",
            " ACGAAAGAATTGGGTGCGTGACCCGGACATCCTGCGTGACCCAAACGAACGAATTTGGGTGAGGGCTGTTGGAGACAGTATCCTCTCTCAGTCACTCTCACTCTCTCGTTCTCTCTCTCCCACTCTCTCTCTCTCGTTACTCTCCAAGCCGGCTGATGGGGGCGAGGAGGGGAGGCGTCCAGGTGTAGATCGGGGTCGGAGACGGCAACTCTGGCAGCGGCTAACTGGCCTGGCAGGGCAGATCGATGAGATGGGTGTATGGGCGGTGTGGATGCTAGGGGACGGCGACTCCCTCGCGACGTCTTTACGGTCCCCTCTCGCACGGATCCGTCATCGGCTGCGACGGGAGGGGTTGGATCCGGCGACCTCTTCCTCTCCCGCTCAGATCCCGTCGTCTCTCCGTGCCCCGCTCACGCAGATCCGCCGCCGGCGTTGACGGGAGGGTCCAGATCCGGCGGTTCTACCGTGGTCGTGGCCTCCCTGCCGTCGCCGCCTCTGCACGCGCGGCCCCTACGCCCCCGCCCCTGCTGCCTCCTTCTCCCCCACGCCGTCTTCGGTGGCTGCGATAAGGGGCGGCGGCGGCGGCGGGAGGCGCTGGAGGGAGGGATATGAAGGGGAGGCGCATGGTCCGGTGGGTGAAGAATGAAGAAGTGGGCGGTCG\n",
            "\n",
            "Finished processing index 79: Oryza_sativa.IRGSP-1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Os01g0100100::1:1982-2982(+) \n",
            " ACCAGCGGCGAAGCTCGGGGCGGAGGGGTTGAGCCACATGAGGCGATGGCGACAATGAGGCGAGACATGGCGTGGCTGGCTGTTACATTTTGTTTTGATGAAAAGCATAACCATGCGGATGATATTTTTATTATAGACTAGAGATGATTATTGAATAGACATGCTCTTAACCATTTTTAACTCTAATTCCACCTACACTATGAATATTAGAAAAAAAATACAATGTATACTCTTAAGAAGTTTGCGCTATCATAGCCATCATACTCATTAAAACATTTTTCTTCACATGTTGCAACGCATGGGCATTTTGCTAGTTAGAGAATAAAAATAAATTTGTAGGTAAAACTTTTATATATGTGTTCTCGGTGACTTAAAAGCCAATGCTTAAAAATAAACTACGTTAAAAATATCTTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCACCGTCCTT\n",
            "\n",
            "Finished processing index 80: Oryza_sativa_arc.Os117425RS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsARC_01g0000010::1:1846-2846(+) \n",
            " TTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCACCGTCCTTGTTGGTAATGAAGAAGACGAGACGACGACTTCCCCACTAGGAAACACGACGGAGGCGGAGATGATCGACGGCGGAGAGAGCTACAGAAACATCGATGCCTCCTGTCCAATCCCCCCATCCCATTCGGTAGTTGGATTGAAGACTACCGAATAAGAGAAGCAGGCAGGCAGACAAACCCTTGAACCAAGGAGTCCTCGCTGAGGAAGCTTTGGATCCACGACGCAGCTATGGCCTCCCCGCCCACCAGGCCGCCAGCCACAACCAGCTGACTAGGTAGGCTTCCTAGGTAGGGATCCCATCCCTTCGATTCCCTACTCCCTCCCCCGATTGATTTGATTTGATTTGATTTAATTCGATTGCCTGCTTTTCAGGTCGCATGCATCATCAGATTTCAATCTCCCTTCGTTCCCT\n",
            "\n",
            "Finished processing index 81: Oryza_sativa_azucena.AzucenaRS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsAzu_01g0000010::1:1759-2759(+) \n",
            " TTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCACCGTCCTTGTTGGTAATGAAGAAGACGAGACGACGACTTCCCCACTAGGAAACACGACGGAGGCGGAGATGATCGACGGCGGAGAGAGCTACAGAAACATCGATGCCTCCTGTCCAATCCCCCCATCCCATTCGGTAGTTGGATTGAAGACTACCGAATAAGAGAAGCAGGCAGGCAGACAAACCCTTGAACCAAGGAGTCCTCGCTGAGGAAGCTTTGGATCCACGACGCAGCTATGGCCTCCCCGCCCACCAGGCCGCCAGCCACAACCAGCTGACTAGGTAGGCTTCCTAGGTAGGGATCCCATCCCTTCGATTCCCTACTCCCTCCCCCGATTGATTTGATTTGATTTGATTTAATTCGATTGCCTGCTTTTCAGGTCGCATGCATCATCAGATTTCAATCTCCCTTCGTTCCCT\n",
            "\n",
            "Finished processing index 82: Oryza_sativa_chaomeo.Os132278RS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsCMeo_01g0000010::1:1432-2432(+) \n",
            " TTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCACCGTCCTTGTTGGTAATGAAGAAGACGAGACGACGACTTCCCCACTAGGAAACACGACGGAGGCGGAGATGATCGACGGCGGAGAGAGCTACAGAAACATCGATGCCTCCTGTCCAATCCCCCCATCCCATTCGGTAGTTGGATTGAAGACTACCGAATAAGAGAAGCAGGCAGGCAGACAAACCCTTGAACCAAGGAGTCCTCGCTGAGGAAGCTTTGGATCCACGACGCAGCTATGGCCTCCCCGCCCACCAGGCCGCCAGCCACAACCAGCTGACTAGGTAGGCTTCCTAGGTAGGGATCCCATCCCTTCGATTCCCTACTCCCTCCCCCGATTGATTTGATTTGATTTGATTTAATTCGATTGCCTGCTTTTCAGGTCGCATGCATCATCAGATTTCAATCTCCCTTCGTTCCCT\n",
            "\n",
            "Finished processing index 83: Oryza_sativa_gobolsailbalam.Os132424RS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsGoSa_01g0000010::1:5660-6660(+) \n",
            " TTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCACCGTCCTTGTTGGTAATGAAGAAGACGAGACGACGACTTCCCCACTAGGAAACACGACGGAGGCGGAGATGATCGACGGCGGAGAGAGCTACAGAAACATCGATGCCTCCTGTCCAATCCCCCCATCCCATTCGGTAGTTGGATTGAAGACTACCGAATAAGAGAAGCAGGCAGGCAGACAAACCCTTGAACCAAGGAGTCCTCGCTGAGGAAGCTTTGGATCCACGACGCAGCTATGGCCTCCCCGCCCACCAGGCCGCCAGCCACAACCAGCTGACTAGGTAGGCTTCCTAGGTAGGGATCCCATCCCTTCGATTCCCTACTCCCTCCCCCGATTGATTTGATTTGATTTGATTTAATTCGATTGCCTGCTTTTCAGGTCGCATGCATCATCAGATTTCAATCTCCCTTCGTTCCCT\n",
            "\n",
            "Finished processing index 84: Oryza_sativa_ir64.OsIR64RS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsIR64_01g0000010::1:10464-11464(+) \n",
            " TTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCACCGTCCTTGTTGGTAATGAAGAAGACGAGACGACGACTTCCCCACTAGGAAACACGACGGAGGCGGAGATGATCGACGGCGGAGAGAGCTACAGAAACATCGATGCCTCCTGTCCAATCCCCCCATCCCATTCGGTAGTTGGATTGAAGACTACCGAATAAGAGAAGCAGGCAGGCAGACAAACCCTTGAACCAAGGAGTCCTCGCTGAGGAAGCTTTGGATCCACGACGCAGCTATGGCCTCCCCGCCCACCAGGCCGCCAGCCACAACCAGCTGACTAGGTAGGCTTCCTAGGTAGGGATCCCATCCCTTCGATTCCCTACTCCCTCCCCCGATTGATTTGATTTGATTTGATTTAATTCGATTGCCTGCTTTTCAGGTCGCATGCATCATCAGATTTCAATCTCCCTTCGTTCCCT\n",
            "\n",
            "Finished processing index 85: Oryza_sativa_ketannangka.Os128077RS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsKeNa_01g0000010::1:9134-10134(+) \n",
            " TTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCACCGTCCTTGTTGGTAATGAAGAAGACGAGACGACGACTTCCCCACTAGGAAACACGACGGAGGCGGAGATGATCGACGGCGGAGAGAGCTACAGAAACATCGATGCCTCCTGTCCAATCCCCCCATCCCATTCGGTAGTTGGATTGAAGACTACCGAATAAGAGAAGCAGGCAGGCAGACAAACCCTTGAACCAAGGAGTCCTCGCTGAGGAAGCTTTGGATCCACGACGCAGCTATGGCCTCCCCGCCCACCAGGCCGCCAGCCACAACCAGCTGACTAGGTAGGCTTCCTAGGTAGGGATCCCATCCCTTCGATTCCCTACTCCCTCCCCCGATTGATTTGATTTGATTTGATTTAATTCGATTGCCTGCTTTTCAGGTCGCATGCATCATCAGATTTCAATCTCCCTTCGTTCCCT\n",
            "\n",
            "Finished processing index 86: Oryza_sativa_khaoyaiguang.Os127518RS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsKYG_01g0000010::1:9970-10970(+) \n",
            " TTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCACCGTCCTTGTTGGTAATGAAGAAGACGAGACGACGACTTCCCCACTAGGAAACACGACGGAGGCGGAGATGATCGACGGCGGAGAGAGCTACAGAAACATCGATGCCTCCTGTCCAATCCCCCCATCCCATTCGGTAGTTGGATTGAAGACTACCGAATAAGAGAAGCAGGCAGGCAGACAAACCCTTGAACCAAGGAGTCCTCGCTGAGGAAGCTTTGGATCCACGACGCAGCTATGGCCTCCCCGCCCACCAGGCCGCCAGCCACAACCAGCTGACTAGGTAGGCTTCCTAGGTAGGGATCCCATCCCTTCGATTCCCTACTCCCTCCCCCGATTGATTTGATTTGATTTGATTTAATTCGATTGCCTGCTTTTCAGGTCGCATGCATCATCAGATTTCAATCTCCCTTCGTTCCCT\n",
            "\n",
            "Finished processing index 87: Oryza_sativa_larhamugad.Os125619RS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsLaMu_01g0000010::1:1431-2431(+) \n",
            " TTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCACCGTCCTTGTTGGTAATGAAGAAGACGAGACGACGACTTCCCCACTAGGAAACACGACGGAGGCGGAGATGATCGACGGCGGAGAGAGCTACAGAAACATCGATGCCTCCTGTCCAATCCCCCCATCCCATTCGGTAGTTGGATTGAAGACTACCGAATAAGAGAAGCAGGCAGGCAGACAAACCCTTGAACCAAGGAGTCCTCGCTGAGGAAGCTTTGGATCCACGACGCAGCTATGGCCTCCCCGCCCACCAGGCCGCCAGCCACAACCAGCTGACTAGGTAGGCTTCCTAGGTAGGGATCCCATCCCTTCGATTCCCTACTCCCTCCCCCGATTGATTTGATTTGATTTGATTTAATTCGATTGCCTGCTTTTCAGGTCGCATGCATCATCAGATTTCAATCTCCCTTCGTTCCCT\n",
            "\n",
            "Finished processing index 88: Oryza_sativa_lima.Os127564RS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsLima_01g0000010::1:9550-10550(+) \n",
            " TTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCACCGTCCTTGTTGGTAATGAAGAAGACGAGACGACGACTTCCCCACTAGGAAACACGACGGAGGCGGAGATGATCGACGGCGGAGAGAGCTACAGAAACATCGATGCCTCCTGTCCAATCCCCCCATCCCATTCGGTAGTTGGATTGAAGACTACCGAATAAGAGAAGCAGGCAGGCAGACAAACCCTTGAACCAAGGAGTCCTCGCTGAGGAAGCTTTGGATCCACGACGCAGCTATGGCCTCCCCGCCCACCAGGCCGCCAGCCACAACCAGCTGACTAGGTAGGCTTCCTAGGTAGGGATCCCATCCCTTCGATTCCCTACTCCCTCCCCCGATTGATTTGATTTGATTTGATTTAATTCGATTGCCTGCTTTTCAGGTCGCATGCATCATCAGATTTCAATCTCCCTTCGTTCCCT\n",
            "\n",
            "Finished processing index 89: Oryza_sativa_liuxu.Os125827RS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsLiXu_01g0000010::1:1406-2406(+) \n",
            " TTATGGGTGATGGAACTTTATATATTCATTAGAAGTCCAAAAAAAATCCGTATACTCACGGTATGGATTCCCTCTTTCTGTTTATGTTTTGTTAGAGATGGATGACGTACATAATCTGCTTGTCTTGTTTCAGGGAATCTGTACATATGCACAGTATACTTGAAAACAAGAGACGAAGCGGTGGAAGGATAATAGAGAAGACCGCCATGTTGGTGTAGCATCGATCGATGTTGTCATTGTTTTTGTGGTGCCAGCCCATCTGCCTTGGCACTTTCCGTGGTGATTTCTGATAAAAGTGGAGTACTAGTTGTTTTTTCTTTTTGCCAAACAATTTGCGTCAGCTATTGGTGGAGAGAGAAGGAATGAATGAGGAATGTGTATAAAGCACTAGTGGTAATCACGCAGTCGTTTCAATACAGAACAGTAAAAAATGGAGGAATTCTGCCACTTTGTTCTTTTTTTTCCTGGCACCTGTTCTTTTTGCTAGAGGGAAAAACAGCCCTGGCTGGGCACTGGGCAGCATACTTTGCGGCCCAATACGATAGAGCCCAACCAGGGTAGGTTTTCAACTAGGCCGGCGAGTCATCTTCCTTATAATGGATTCCTCAACCACAATATCAAATGCTCTAGATTTCCTAATTATTAAAAGGAAAAAGTCAAACTTTAACTTTCTTAAAAGTAGGCCGAATCTAATTCGTAACCCTTAACCAGAAAACCGGATAGGACGGATTCTTAAAAGTTTGGCGTTTATGTGGCATTAGAATTAAAAATATATGTGGAGCTCACATGATCATCTCTCTAATCCTTCTTCCTCCTCTCTCTCTCTTATCTCTCTCTTTTCTTTTCCCCTTCGGCCCTGGAGGCGGTGGGTAGGACGTTGACAGGTGCTGGAGCGGCGGCGGCGGCGAGCGGGTGAGGGCCAGAGCGGCAGCGGAGGGTGTGTCGGCCGATTCCCCTTCCCTTTCTTGGCTCGCCGAAATCCCTAATCCCCATGCCGCAT\n",
            "\n",
            "Finished processing index 90: Oryza_sativa_mh63.MH63RS2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsMH63_01G000010::1:6538-7538(+) \n",
            " CGATGGAACCAGCGGCGAAGCTCGGGGCGGAGGGGTTGAGCCACATGAGGCGATGGCGACAATGAGGCGAGACATGGCGTGGCTGGCTGTTACATTTTGTTTTGATGAAAAGCATAACCATGCGGATGATATTTTTATTATAGACTAGAGATGATTATTGAATAGACATGCTCTTAACCATTTTTAACTCTAATTCCACCTACACTATGAATATTAGAAAAAAAATACAATGTATACTCTTAAGAAGTTTGCGCTATCATAGCCATCATACTCATTAAAACATTTTTCTTCACATGTTGCAACGCATGGGCATTTTGCTAGTTAGAGAATAAAAATAAATTTGTAGGTAAAACTTTTATATATGTGTTCTCGGTGACTTAAAAGCCAATGCTTAAAAATAAACTACGTTAAAAATATCTTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCAC\n",
            "\n",
            "Finished processing index 91: Oryza_sativa_n22.OsN22RS2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsN22_01G000010::1:12520-13520(-) \n",
            " AGGTGGAATTAGAGTTAAAAATGGTTAAGAGCATGCCTATTCAATTATCATCTCTAGTCTATAATAAAAATATCATCCGCATGGTTATGCTTTTCATCAAAACAAAATGTAACAGCCAGCCACGCCATGTCTCGCCTCATTGTTGCCATCGCCTCATGTGGCTCAACCCCTCTGCCCCGAGCTTCGCCGCTGGTTCCATCGCCCATGAGGCGAGCTCTGTTGCTGTCAATTGCCACTCCTCCCGTTGAATCAAGAGAGGAAGAATAAGTTGAGGGAGAGGGGACAGCCGAATCTGCGGGTGGCGGATCCAACGGTGGCTCGGTTCATGGGCGGTGGATCCATTGTCGGCTAATTGACCAACCAGTCACACCATGTCACGCCATGCCTTGCCTCGTTTTGCCCTCGCTCTCGCCTCCTTTTTCTCTTGTGGCAACATAGCTTGATTTGCGGGCGACGGTACCAGCGGCGAAGCTCGGGTCGCGAGCGACAGTACCGGCAGTGAAGCTCAGGGCAGTGGCATCAATCCACAGGAGGCGTAGCTCGGTCGCGGGTTGTGGGTGGTGGATCCAACGATGACTCGGGTCGTGGGCGGCAGATCTGGTCAGCTCGAGACATGGGTGGTGGGGTAACTCTCGTGGAACTTTTTTTTTACGATTGTTACTTTTTGTCTCAAAAAATATTAAAGTATAAGTTTCGGTTATTTTGATATCAATAATAAAAGATGTTCCATAAAAGATCTTAATACAAAGTTAAAAAGAAGAGAAAACTGTGTGGTCTGGTCTATATCCTTATCCTCTCACTCTCTTGTTCTCTATCTTCCATTCTGTCTCTCTTGAGTTATTCTCCAAGCCGGCCGGCGTTGCATCTCGCTCAGAGCCGGTGCCGGTGCCCCCTCTCGCGTGGACCTACTGTCGATGGCGATGGGAGGGGCGGATTCGGCGACTTCTTCCCCTCCCGCACGGATCCATGATCTCCGCTCCCCTCTCGCCCGGATCCGTCG\n",
            "\n",
            "Finished processing index 92: Oryza_sativa_natelboro.Os127652RS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsNaBo_01g0000010::1:7607-8607(-) \n",
            " AGGTGGAATTAGAGTTAAAAATGGTTAAGAGCATGCCTATTCAATTATCATCTCTAGTCTATAATAAAAATATCATCCGCATGGTTATGCTTTTCATCAAAACAAAATGTAACAGCCAGCCACGCCATGTCTCGCCTCATTGTTGCCATCGCCTCATGTGGCTCAACCCCTCTGCCCCGAGCTTCGCCGCTGGTTCCATCGCCCATGAGGCGAGCTCTGTTGCTGTCAATTGCCACTCCTCCCGTTGAATCAAGAGAGGAAGAATAAGTTGAGGGAGAGGGGACAGCCGAATCTGCGGGTGGCGGATCCAACGGTGGCTCGGTTCATGGGCGGTGGATCCATTGTCGGCTAATTGACCAACCAGTCACACCATGTCACGCCATGCCTTGCCTCGTTTTGCCCTCGCTCTCGCCTCCTTTTTCTCTTGTGGCAACATAGCTTGATTTGCGGGCGACGGTACCAGCGGCGAAGCTCGGGTCGCGAGCGACAGTACCGGCAGTGAAGCTCAGGGCAGTGGCATCAATCCACAGGAGGCGTAGCTCGGTCGCGGGTTGTGGGTGGTGGATCCAACGATGACTCGGGTCGTGGGCGGCAGATCTGGTCAGCTCGAGACATGGGTGGTGGGGTAACTCTCGTGGAACTTTTTTTTTACGATTGTTACTTTTTGTCTCAAAAAATATTAAAGTATAAGTTTCGGTTATTTTGATATCAATAATAAAAGATGTTCCATAAAAGATCTTAATACAAAGTTAAAAAGAAGAGAAAACTGTGTGGTCTGGTCTATATCCTTATCCTCTCACTCTCTTGTTCTCTATCTTCCATTCTGTCTCTCTTGAGTTATTCTCCAAGCCGGCCGGCGTTGCATCTCGCTCAGAGCCGGTGCCGGTGCCCCCTCTCGCGTGGACCTACTGTCGATGGCGATGGGAGGGGCGGATTCGGCGACTTCTTCCCCTCCCGCACGGATCCATGATCTCCGCTCCCCTCTCGCCCGGATCCGTCG\n",
            "\n",
            "Finished processing index 93: Oryza_sativa_pr106.Os127742RS1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsPr106_01g0000010::1:9158-10158(+) \n",
            " TTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGCTTTTTCTTTGACTGAATAGGCCATCTGATGGAAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCCTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCCATCCTCAACTACCCACTTCCCAAGATTTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGAATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAAATGGCCATCCAAATGAGAGTATGAAATTTAGATGAGCTGGTGGGGATGCTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCACCGTCCTTGTTGGTAATGAAGAAGACGAGACGACGACTTCCCCACTAGGAAACACGACGGAGGCGGAGATGATCGACGGCGGAGAGAGCTACAGAAACATCGATGCCTCCTGTCCAATCCCCCCATCCCATTCGGTAGTTGGATTGAAGACTACCGAATAAGAGAAGCAGGCAGGCAGACAAACCCTTGAACCAAGGAGTCCTCGCTGAGGAAGCTTTGGATCCACGACGCAGCTATGGCCTCCCCGCCCACCAGGCCGCCAGCCACAACCAGCTGACTAGGTAGGCTTCCTAGGTAGGGATCCCATCCCTTCGATTCCCTACTCCCTCCCCCGATTGATTTGATTTGATTTGATTTAATTCGATTGCCTGCTTTTCAGGTCGCATGCATCATCAGATTTCAATCTCCCTTCGTTCCCT\n",
            "\n",
            "Finished processing index 94: Oryza_sativa_zs97.ZS97RS3.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OsZS97_01G000010::1:14551-15551(+) \n",
            " AACCAGCGGCGAAGCTCGGGGCAGAGGGGTTGAGCCACATGAGGCGATGGTGACAATGAGGCGAGACATGGCATGGCTGGCTGGTACATTTTGTTTTGATGAAAAGCATAACCATGCGGATGATATTTTTATTATAGACTAGAGATGATTATTGAATAGGCATGCTCTTAACCATTTTTAACTCTAATTCCACCTACACTATGAATATTAGAAAAAAAATACAATGTATACTCTTAAGAAGTTTGCGCTATCATAGCCATCATACTCATTAAAACATTTTTCTTCACATGTTGCAACGTACGGGCATTTTGCTAGTTAGAGAATAAAAATAAATTTGTAGGTAAAACTTTTATATATGTGTTCTCGGTAACTTAAAAGCCAATGCTTAAAAATAAACTACGTTAAAAATATCTTAAAATCAATATTAAAATTAAGTTTGAAAATTTAAATTTTGGTTTTTTTCTTTGACTGAATAGGCCATCCGATGGGAGCTGAATAAGCCATCCGATGGGAGCTGTGACAGTGATGAAAAGGAGAGCAGTAAGAGAGCAGTAAGAGCATCCCAGCAGCTCTTCTATCTAGGCATCCATCCGATATTTGGAGTATGGAGGAGAAAAACAGTGCTCCAGCAGAGTCTCCATCACATGCTTCATTTTTGGAGGTCCTCCAAATCTAGCCCTTCCCAAGCCAAATATGGAGGTTCTCTCTCCTCACTCTATCCTCAATTACCCACATCCCAAGATTTTTTTGAGTGGTGAAAGGTGAGAGGAATAGGAGTAAAAATGTGTGGTAGTTGAAAAATATTACTAAAATATAGGATGTTGGTATATGTAGGATGTAATATGGATGATCTGTTGGAGTGAAAAGTTATATAGAGTGGAAATCTTTTTAGATGGCCATCCAAATGAGAGTATGGAGGATGAAATTTAGATGAGCTGGTGGGATACTCTAAGAGAACGAGAGAAGCACAGAGCAGATAAACCACACCCACAGGCACCGC\n",
            "\n",
            "Finished processing index 95: Ostreococcus_lucimarinus.ASM9206v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "OSTLU_23817::1:3671-3906(-) \n",
            " GTCGCGGATTGGCGTCGCCCGCGCGTCTCGCGCAATTCAATCGACGTGTTTTAAAAATCATAGCCGTTACGCGTTTATTAGTGCAAAACGTCTTAATATATGTGAATAGCTCGTGATGTGCGTCAACACCCTTTTTGTCGGCGAGACTGTTGGGCGACGCGAGTCGGTCGTGACTTCGGTGCGCGCGCGCGTCGCGTGACCTCCGCGCCGAAGGATTGTCTCGTCGCGCGCAGTC\n",
            "\n",
            "Finished processing index 96: Panicum_hallii.PhalliiHAL_v2.1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "GQ55_1G000100::1:13201-14201(+) \n",
            " TCATCCATACCAACCATGTTGCTTTGTGGTCCAAAACTTGATTTATACTCGATCAATGATTGCCCTGTAAAGTTTGTAAAAATTGTTGCACTCATTTTTAACTTGCATTCATGCTTGCATACATTTAGAGCCTGCAGTAGAGGACGTGGTTGATAAATTAATTGCTGAACTGCAAGAGCAGCAAGAGCTAGGGCAATAAGCTAAACCAAAGAAAGACCCTAAGGCTCAGAAAATTAGCAAGCAGAACAAGAGTAGGAAGAAAGCAAGCCCTGGAGCATATCATTTAATTTCAAAACTATGCACCAAACTATTTACGCATTAAATTTCTAGGAGTTGATAGATATACGATCCCTATCGGGTGAAAACGATAAGGAAATGTGTGTAACAGCTCACGAGAGGCCAAAATCGGACTAACGGGTGCCCAACGGGAATAGCCAAGTTCTTTGAAAATTATTTTTAGTGTCACCAATATTTGGAATCATGTTCTAACGGGAGGAAAGAGGAAAAGGCTCATTAACGGGTGAAAACGGCTCCGGACGAAAACAACCAAAAAATCTGTGTTGTAGCTCATGGGAGGCAAAAATGGGTTGACAGGTGCTTGCCGCGAAAACGTCCAAGTTTTGCAGAAATGGTTATGAATATCACCAACATATGGAATCATATCTAAACAGGAGGAAATAGCTCATAAACTGGTGAGCAAATGGCTCAGGACAAAAATGATAAAAAAAAAATGTACTATAGCTCATGGAAAATTTAAAACAGACCGATGGGTGCCCACCACGAAAAAGTCCATGTTTTGTAGAAATGAATGTAGAGGAGGGAGAGAAGAGCCCCCTTTGCACCTGTCCTTGGTTAGGGATATGTTTTAGCTCCGTTATTATCTCACTGTAATACCGCATCCAAATCCAACGACTTTTCATTCTCCACGCAAAAAATACAAGAAAGCGGTCTGGCCGTACCGTGTTTACTTACCGGAAAAGTTTGTAATATAAGATCTGTT\n",
            "\n",
            "Finished processing index 97: Panicum_hallii_fil2.PHallii_v3.1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "PAHAL_1G000100::1:9252-10252(+) \n",
            " TACTCGATCAATGATTGCCCTGTAAAGTTTGTAAAAATTGTTGCACTTATTTTTAACTTGCATTCATGCTTGCATACATTTAGAGCCCGCAGTAGAGGACGTGGTTGATAAATTAATTGCTGAACTGCAAGAGCAGCAAGAGCTAAGGCAATAAGCTAAACCAAAGAAAGACCCCAAGGCTCAGAAAATTAGCAAGCAGAACAAGAGTAGGAAGAAAGCAAGCCCCGGAGCATATCATTTAATTTCAAAACTATGCACCAAACTATTTATGCATTAAATTTCTAGGAGTTGATAGATATACAATCCCTATCGGGTGAAAACGATAAGGAAATGTGTGTAACAGCTCACAAGAGGCCAAAATCGGACTAACGGGTGCCCAACAGGAATAGCCAAGTTCTTTGAAAATTGTTTTTAGTGTCACCAATATTTGGAATCATGCTCTAACGGGAGGAAAGAAGAAAAGGCTCATTAACGGGTGAAAACGGCTCCGGGCGAAAACAACCAAAAAATCTGTGTTGTAGCTCACGGGAGGCAAAAATGAGTCGACAGGTGCTTGCCATGAAAACGTCCAAGTTTTGCAGAAATGGTTATGAATATCACCAACATATGGAATCATATCTAAACATGAGGAAATAGCTCATAAACGGGTGAGAAAATGGCTCAAGACAAAAATGATAAAAAAAAATGTACTATAGCTCATGGGAAATTTAAAACAGACCAATGGGTGCCCACCACGAAAAAGTCCATGTTTTGTAGAAATGAATGTAGAGGAGGGGGAGAAGAGCCCCCTTTGCACCTGTCCTTGGTTAGGGATATGTTTTAGCTCCGTTATTATCTCACTGTAATACCGTATCCCATTTAACTTTGTCCATCTAACGACTTTTCATTCTCCACGCAAAAAATACAAGAAAGCGGTCTGGCCGTACCGTGTTTACTTACTGGAAAAATTTGTAATATAAGATCTGTTTTGGAAATAGCATCAGTTGCCTAGCTTTTTTTG\n",
            "\n",
            "Finished processing index 98: Papaver_somniferum.ASM357369v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "C5167_036300::1:90626-91626(+) \n",
            " AAGGAAATTACATGATAGTGCGTAAATTTTGGAATTCTCCTTAGCTTAGCGGATACTGCATAATTTATTACCACTGAAGGCTACTGCTGTAAAAATTTTGCGGTTATTGCATAAATTTTGATTTTTACTTGTAAGGTTATCTTAAAATTTATTACTCATGGTAAATGATAATTACTTGATCACTTAAAAAGAAGATAATTTCTAGATGAACCAGAAACAAGGAAAGAAAGATCATGACACACATGCTTGTCGAATAGGGACAGAGCTAAGTGTAGCTCAAGGGTAACCAGCGTCTTCCCTGTTTTCCATTAATTTATTAAAAATCCTTTGAAATTCTTATTATTTAGAAAGAAAATTACTTAACAAGCCTATCCTTGGATATGTTTTTTAATGACTATCCATCTTTGTAAAAGGTTTCTGGCTGGCTATGTTGATTTATTTGGAAACCGCTAGGTCAGTGATTTTCAGTCCATCACACTTTGACTAATATCATGGACTACTAGAATCAATCAATCCATAGTTGTTGCCAAAGAAAGAAAAAATCATAAACAAGAGATAGTGGATGATCATGTAGTCTAACGTGACATCCCAAACAAAAAGTGTAGATTTTCTTTCTGGTCAACACGAAAAGAGAGCAAAAGGAAAGAGAGACGGAAATCATGTAGTCAAATGTGACATCCCAAACAAGAAGTGTAGTGTAGGCTTCTCTTTCTGGTCAACACGAGAGTAAAAGGAAAGAGAGGCGAAAAGTCTCACGCGCCACCACTTCAACTACTATACTATATTACTATAAATAAAGGAGTAACAAACTCTCAAATTCATATTCCTCCACAAGCAGTTCCAGACATTTCATGGGTTCGGATGTATTCCAATCTCTAACCAAATTCCCATTCATTTTCCGATCAAAATTCTTTCTTGTTTCTCATTTTTGATTGTATTATTAATTGTTATATATCAGTAATAAATTATATATATACTTGTCCAAATTTTGGTTCTAGAG\n",
            "\n",
            "Finished processing index 99: Phaseolus_vulgaris.PhaVulg1_0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "PHAVU_001G0001001g::1:4760-5760(-) \n",
            " NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTCCAAAATATTTAGTTGATAATTTGTATTTCCTTTTTGATGTTTAATAATGATCTTATTGTTGATGGTTTAATAATTTTAGTTCAAAAGGGTGTTAGTTATGGGAACCCATTTCCTATGGTTCTTGAACAGCTTAAAAACATTTGAAGGCTTACACTAGGAGGCCCAAGTTCCTAAACACCAGCACAAACTTACAGGAATCACTATTAGGGAGAAGTGAGGAGTTAGTTATAGGAGAATGAAAGGAGATGCAATCTGATATATAGGGTAAGAGGGAATAAATGGGGGGTGGATATGATTTGAGATTATTGTGATTGGATTGGATTGGATCCAAGAGGCGAGTGTGGCTTGGGGTCTTGGCTCTTATTTTCTTGTAATCCCTTCCTTGTTAATAAACAATAATACAATTCAATTTCAGATAGGAATAGTTTATTTTATTTGTTGCTTTCTTTGAGAGTCGAATTCAAGTGCTTCTGACATGCAG\n",
            "\n",
            "Finished processing index 100: Physcomitrium_patens.Phypa_V3.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Pp3c1_20::1:7930-8930(+) \n",
            " AGGAGGTCAGAAAACCTTGGCACATCCTTGTAATCCTAAATCCTAAACTCTTAAATTTAAATTCTAAATCCCTAAATCCTAAACCTTTGTTCTATTTTTTTTGTAATCATAAAATAATATTATACTATTTTTTAATATAAAAAACAAAAAAAAAATAAAAAAAATTATTTAACTTATCTTCTTTTTTAAAAAAATTAGATAATATAAATTAGATTATTATTTATCATCTAAAATATAAATAAATAAAATCACCATCTATTGAAGGTCTTTACTAATTGACACTTAGTACTAATTAAATATATGGCTACCATGTTAAGAAAAAAAAAAAAAAAAAAAAAAAAAACTTATATAACTTCTTTTTCATTCACAATTTCTTTCTTTTGTAAAAAACCACTTATTAAAAGATATAAAAATAAAAAGATATAACGAGATATAAAAAGACATAACAATAAAACTCAATTAATGTTAATCCTTCATTCTAGGGTTGCTTAAAATTCATAATCCCAAATAAATGGATTTCATTGTGCATAAAGATATAAATATATAAATATATATATATATATGCACATAAACAAGTAAAAGTCTATCCTTCATTCATTTGACAGGAAAGTTCACTTTAAAAATTCCTGGTCAGACATTTAAGCTTTGGTTCGAGATTTAGCAGAGCCAATTGGGCCTGAACCTGGAAATAATCATTATATATACACACCTTGGTCCGAAGAGACTGCGACCCAACTGATCGGCCTGTCAAGAGGACAGACCCCAAATCCAGGCTGCCTGCCTGCGAAAAGGGCGTAGTAGAATGGTTGTGGGTCGGTGAGGTATGTTCTGGTCCATGCAGGAATGATTACCTTCCAGGTTGGTTCGTGGTGATGAGGGGCCACCTGAACAGCACGATGGTGCCATTAGATGTTATAATCATGGTGTGGAAATGGTGAATGAATGTGGACATCCATCCGTCGCTTCGTTACTACTACTCTTCTTCGTCTCTCCCCCCCTT\n",
            "\n",
            "Finished processing index 101: Pistacia_vera.PisVer_v2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "EVM0020043::Contig1:60115-61115(+) \n",
            " GTGTTTGGGCTTGGAAAAGTATTTGGGTTAGGCCTGGTTTGGTTGTCTTGATTGAGCTCTATATTTGGGCCTGTTTGTGTGTTTGGGTTGGCTTGGCTGTTTCAGTTTGGGTTTGGATCAATTTGGTTGTTTGTTGGATCAGACCCAATTGAGTTTGTTTGGCTTGGATCCATTTGGTTTTGAAGGACTAATCCTAGAATTTGTTTTGCAGGAAAAAAAGAAAGGGATAGGAGTTGGATTATTGGCACGTAACAAGGAGAGTGGCTGGCAGGGCAACCTGGCTCTGATACCATGTATGAGTCTGATATTGGAGTAGCACGAAGAGGCAACTGAGTTACCAAAGAGTGCAAGAAGAAGAGAGGCAAGAGAAGCAACTAAAGTTGTTGAGTATGCCTAGAGAGTATAAGCAGGGAAGAAAGAGAGAAGAATGATTTTGGTATATATATTTCATGTACAAGAGTTACAATTTATACAATTGCAAATGTAACATGTAAGGAAATAAAATAAAAGGAATAAATGTAAATCAATACGTACAAGGAAATAATATAATTCTACCTAAGGTACCATACTTGGACAATATTACAAATCTTCCAAATATGGCTTGTACAGTCATTCACATGTCAACACGGGCAATATAAGGAAAGAATACTTACATTAAAACATATATTTGCATTCCATATCTCAGCATGATCTGTCAAAGTCGTCCAACCGTTAGATTAATGCAATCTCTTAAGTCATAACATATACACCTTGTTGGCGTAGATCTTTCACTGTGCTTGAGATTTAATTTTTTAAAGCTTCTCACGAAGCCATTCTTTGTTCTTTAAGTTTGTGTTTCGTGGGAAATTTTTTTTTCCCTTGTCTTTTTTGGTTAAGTTTTTTTCTTGTTCTGCTTGGAAATCTCTGTCGTGTCCAAGCTCTGAACAAAAAAAATGAAGGGTTTCTGGTTCTTTACCGTTTTGGTTGTGTTTCTCTTTTACAATGCAGCTTCAGCTAAGTT\n",
            "\n",
            "Finished processing index 102: Pisum_sativum.Pisum_sativum_v1a.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Psat1g000040::1LG6:11467-12467(-) \n",
            " NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTGCTAGTGAAGGGAATGGAAGGATCCAGTTGGAGCTTCCCTGTGGCCAAAAGAGCAAGATGAAGAGATCCTGCATGCGCGGTAGAAGTAAGCGCGACCTGTTCGAGTTATCCCTTTCTTGAAGTGATATCTCGTGGGATCACTGGTCTTAAGCTGTATATGGTGTCCCCTTTTCTGGCGTGAGTACAACTTCATGTTTTTGATTTTATGGTTATGTCCCTTATGTTGTCAATGTTCAAAAAGATTGA\n",
            "\n",
            "Finished processing index 103: Populus_trichocarpa.Pop_tri_v4.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Potri.001G000400.v4.1::1:17702-18702(-) \n",
            " ATTGATTATTCATTCAATTTTCCATGCTTATTAATATTATCTTCATATAAACCTGTTTAAACTTATTTTCAAATAAGAATGTCATTTGTCTATTTTTCATTTAATTGTTATCATCAAAATATATAAAAACATTAATATGTGACCCAAAAATTAACAGAATAATGAAATTAAAAACCAATAAAACTTAACAAAAGAGTAAAGGAAAAAAATAGAAATAAAAAGAATAAGGATGGAATTAGAAAACATAATGCATGCCAAATTATAATTAAATGACTAAATTGAAAACAAACAAATTTTTTATAAAAGGGTCAAGAACAACAATTAGAAATCAAAAAAATAAGGACTTAAATTGAAATACTACAAATAAAAAGGACAAACTTGTACTCTTTTTTTTGGGGGGTGGGGGAAGAAAAGAAGAGGGAAAAAAAGAAGAAACAACCACCAACGACAAACCGTCACTCCTCCATCTACATGTGTTGTATCTATTGGAAGAGAACGACAACACTCCTGTTTGTTTTTGCGTTTCAAAAGCGCTTTTGAAAAAATTTGAAAGTTTTTTATTTTTTTATTAACTTTAAATTAATATGTTTTTAGTGTTTTCAAATTATTCTGATGTGCTGATATCAAAAATAATTTTTTAAAAATAAAAACATCATTGACATGTATTTTGACACAAAAAATTATTTAAAAAACAATCACAATTGATCGGATGCTATTTGAAGGGTGCAAATAGCTTTCACTCGTTGGCGTACGTTTGTGTTTGTCACTCGCTAATAACTTTTTTTTTATTATTATTTTATATTACTAAAAGATAAAAATGAAATTCCCCTGTAATGAAGCTTTAATTTTTTTATTACTAGGGAATTTAAATCTTTTTATTGTTCATAGAAAACAAAACGATGAAAAAATCCTTGAAACAAAGTTTAATTTTTTCTGTCAAAAAATTATTTCATTGTTTAAACAAAATATAAAAAGACCCCATGATCCCTTTGAATTTTAA\n",
            "\n",
            "Finished processing index 104: Prunus_avium.PAV_r1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Pav_co3988703.1_g010.1.br::Pav_co3988703.1.:195-1000(-) \n",
            " CCTTCCACCGGCGGAAGACGACCCTGTGATCGGCTTCCAGAAGAAAGATCTGATCGGCCTCGATATGCCACACAACGACGCCCTTGTCGTCAGCATTCAAATCGCCCAGGCCATGGTCGACCGAAACCATGTGGACGAAGGCAGCGCAGCCAATATCCTGCAGTTGACTATCATCCAGCAGATGGGCTTGGAAGCAAAGATCAACAAATCAGCCAGGTCGCTGACTGGCTTCAATGGCGCAACGACGGTTACTGTGGGCACAATAGATCTCGACGTCTACTCCCCACCTGTAATCAGCTCACAAACATTCATAGTCATTGATGAAGTCTCACCCTACAATGGCATCCTGGGTAGACCATGGATTGGTAAGATCAACGCCATCACCTCTGCCACTCATCAGAAAATCCGCTACCCAATTCCTGGGGGCGGTGTCTGCCAAATCAACAGCGACCAGGCAATGGCGAGGAAATGCTCAGCTCAAGGGCTGAAGAAAAGCAAGCCAATACAGTTCCTCCCTGTGAATCAGGCAGATCTGGAGAGAGTAGAACAACCAGATCTCGTTCCTAGCAACCGAGCAAGCCAGAAGGGGAAGGGGATAGATATCCCCTTATAGCAATCAAAGCATCGGGACCAAGTCGAGGAAATCCGTCCGGAGGTCTCCCCTGAAGAAGGAGGGAAACCCGAAGAGGACGTCGAGCTAATACCCTTGGATCCTTACCAGCCAGACAAAAAAGCGCGGATCGGCTCGCGCCTAAGCCCAGACGAGAAGGTGGAGCTGACCACCTTTCTCTAGAACAACAAGGAC\n",
            "\n",
            "Finished processing index 105: Prunus_dulcis.ALMONDv2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Prudul26B002130::pdulcis26_s0001:1870-2870(+) \n",
            " TCAAAATTCAAGCTTTCTTCATCTTATTATAGGGTATTTAATATACTGATACCGTGTTCTTCGATGAATTTTAAGAATCTAACAACATTCATCAAACTAAATCATTCAGAAAACAAATCTAAACTATATTATTTAGGGAGGCTAGTAAAATATCTAAGCGAGGGAAATAACAAAACCCAAAACTCTATCTCAATCAACATTACATTAGTGGACAAATTATAAAACAGTACCAAAATTAGAAACATTTTTGGGAACACCATTGGTATTCCCCAAGGGCCGATGTAATATCCTGTTTTTCACCTCGGTTTGACCGATTTACGACTCTTTGGGAACTCTTTTCGGCCTTGTCTTTGTTTCTTGCTCTACGGCCGCGGCTATTTGTACGCAGCTGGGTGCCTACGGTCAAAGCCTACGCACACCCTCCAACTCCAACCCCAAAAAGGCCCAACCTTTTCTTCTTTTCTCTGTAGCTCAGTCATATTTGCTTGCTTTTGTGCTCCAGATTGATGTGAATCTGCCACCTTTTTCAACCTAAACGGTGAGTTTTCAACATACTGTATGGATGCAGTAGGTACGAATATATGCTTATGTTAGTTTGTTGTTTGAATCTGGGAGGGCAATCGGCAATGTATGTTTGGTTGGTGAGAAAATGTGGGAATTCTAGTGTTTGAAACCTGGGAAAGTATCAATTTCGCCTATTGATTTCTGGGTATTTAACTTAGCTGAGATGAGAGTGTAGACAATTCGTAATCAGTCACGTTTTTGTATTTTCTATGCTTCCTGAGCCACCAAATGGAGGGTTAGAGTTAGAATTGTTTCATGTTCGTTTTTGGTTTTGCTTTTGAGCTAAGGTAGATTTAGGGTTTGTTTGAAAAGTTGTGATTTGGTTTTGTGTTTAAGAACATAATACAAGTTGTACCTCTTCTTTTGGCGCAGGGATAAAGGGATCTACTGAGATTTTTTGTTGAGATTTGGTTTGCACTTTGGTGTATGAGCTTTA\n",
            "\n",
            "Finished processing index 106: Prunus_persica.Prunus_persica_NCBIv2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "PRUPE_1G000100::G1:39339-40339(+) \n",
            " GAGGCACGGTATGACACGAGCACGAATATTAATGGGCAATGCTTGGCACGGCACAACACGAAAACTTAGGCTGGGCCGGGATCAAAAAATGAAGTCCAATGGACCGGGCTGGCCCATGGCCCAACATGGGCCCGGCTTGGCCCGAGCACGCTTCAAGCACGGCATGAGCCCAAGCACGGCCCACTTAACATTTTTTTCTTTCTCAAAAGTCAAAATACAATAATTTGACATCTAATGTCATTGTTTGTATGCTTTTTTTAAAAATCATAAATATATATTTGATGTCTTTCATTTTAGAGAAATTACTTGGGTCCTTAATTAATTAAACATATCAACTACAAATTAATTAAACAACATTGGACTTGATATCTAATCAGTACATATGACATATATGCTTCTCTTGTAATCAAAGCAAGACATTTTTTCACATATAAAATAATAAGTTGTTTAAAGATGAAAAGTTGAAGTGGACTTTGTGGAGTTGGACACTTTTTGCCCTCCGCCATTTCATATTGTTTGGATGTTTGATTGATTAATGTTTTCAAATAATTCATCGTTTCTTGCCCCTAATTATATTTTTCATGTTTATAATTTGAATTATCTAGTGGAAAAAAAAAATCAAGTGAAAAAATTTCAAATTAAATGCATTGATTCTTGGCCCCGAGTATGGCACAAGCCGGGCCCGGCCCGTTTCAATATGGACATAGACTTTGGACCAGGGCGTGATCAATGATTTTACTAAATAGGCCGGCACGGCACAGCCCGATAAATTAAATGGGCCAACCCGACCAGACACTAAAACGTTAAAAGCACAAGTTTAAATGGGATGGGCCAGCCCAGCACAGCACGGCCCGTTTGAGAACAAAACTTTTTGCAATTGTATCCCTATCAAAAGAACATGAATAAATAAGTCATGTAGAAAGAGATCTACAATCGTAATAAAAACAAATTTACTTTCTACAAAAGGGTTTTAGGAGATTGGGTGAAAAAAGAGGGAGAA\n",
            "\n",
            "Finished processing index 107: Quercus_lobata.ValleyOak3.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "QL01p000012::1:8755-9755(+) \n",
            " GATTTGGATGTGTATTTTGAGTTCGGCCGTGTGTGGCTGTTTCTGTAGGTATGTCACATGTGAGTGAGCAGTGAGCATGAGTTTGTACCTCAAAACAAGGCAGCTTGTGTATTTTTTGAGTATCAGTACAAAATGTATCCTTGTGATGTGTGTGTTCGCACATGAACTAACTGTATAGAAGCATGAATACATAAACAACGTGTATTGAAAGCATGAGAAGGGAATTAATGTTACCCCTTTTCAGTTTCTCCACTTTCCCTCCTTGGTTGTTCTGCTTTCCAATGCTTTAGCTTTAGCTTGCTGCTTCTGTCTTCTTTACTCTTTTCTTTGTTGTTGTTCATTACTGTTGTCTGTGGTTGTTGTTGTCGCTACTGCTAGTGTGGGTTGTAGTTGTTGGGTAAGTGGCAAAAATGAGGGACCTTTGTTGGGTTTTTTTGGGTTGATGGTAGAAAGGAGGGAGCTTTCTATGTTTTGGATCTGCTTTTCGGTGTTTTGGAGGTTGTGGAGATGGTATTGGGAGTTGGCTATGTTTGTAGTTTTCGAATATGTAAAAATGATGGATTTTTATGTGTTTTCTAGTCTATTTGTAAGCTTCTTATTGTTAGGATGTTGGTTTTAGAGCTGTGATTTTGCTCTTTGTTATCGCTGGGATTTTTGGAAGTGTTGATTTTGAGGATGAGAGGGCTTTTTTTTTTGCTTTGACGAGGTAGCTGATGGTTTTGAGATAAGGGACTGTTAGTCCGGGTAAGTGTAGAAAATGATGACTATTGCTTGGGAAGAACAAAAAATGGAGGCATTTTTTTTCGGCTCCATTTTTGGAGCTTTGGAGAAAAAGCTGTGAAATGATGTTCCTTTTTCTGAGTTGTATATGGTTTGCTGCATTGCTGGGGGCGGACAGATTTATAGACTTAAGATATGTATGATAAAAAAAGTATGGAAGTTTGAAAAGAGAAGATTATTTTCCATAAAATATATATATATTATTTCATCAGAAAAGAAA\n",
            "\n",
            "Finished processing index 108: Quercus_suber.CorkOak1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "gene-CFP56_78981::PKMF03000005.1:0-31(+) \n",
            " TATGTATAGCTCGGCACCCCTTGCTAAAAAT\n",
            "\n",
            "Finished processing index 109: Rosa_chinensis.RchiOBHm-V2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "RchiOBHm_Chr1g0312971::1:9707-10707(-) \n",
            " GAAGGATCCTCTAGTTTCGGAAGTTTAGTAGGGCATGCCTCTGGTGGGGGGCTTTCAGGCGACAAAAACCCGCCTCTAAGTCCCTCTTATTTGTTTCCAGAATTGGGGACTACGCTACCTCCCCAGGAGGATCCCATACGACGTGCATTTATGGAAAAGGAGATAATCCAGGTTGTAACAAAAAAACTGGTAGATCTTTATGGGCCCCGGAACCTAATATTATCCTTTCCACTTTTGATTTAAATCTCTTTGAGTGTGGAGGAGACCACCTAGAAGAGATTTATACGTCGGTCTCCCTTGGAACTTGGAAAATAAGTTCTATGCGCGAATAAACGTTTTGCGAATAGATTCGGAGAAATAAGTGGCTCACGAGGAATGGAAAGAAACCAACGATTCTCTTCTCAAAGTAATAGAGAGATTCTTTTTCTTTTTCCGCTCCGCGAGCTAGGAGCGCATCATGGGGGCAGAGCGAAAACGAACATAGGATCATCATCATGGCCCTTTTCTTGTTTCTTTTGTTTGTGTCCGTTGTGTGTGTTTTTTTTAAGGGGGAAATTCTTCTTCTTTCTTTTGCTGCTGATCTCACATCAAGAGCAGCTCCTTCTTGTTCAGGGTCATCAGATGCCTCCGATTCCGAGAAATTGGTCAAGTGTGGGGCTACCCTCGACTCACCTCTCTATCTATAAAAGTACCTCCCCAGAGGAGAGCAGAAGCTCTAGGATGAGTATGTCCTGGATTCCCGGATTCATTCTCGTCCTGATCCACCATGGAATTTTAGGAATCTACAAAAACATTCTAGGAGAGGGCTCGTAATGATCTTTGAGAAGACCATAAGATGCTGAAGTGGCAGGATAGGGGGGGATCCGTAGGTTTTTGTGAGAGGGATGCTCCTATCATGTTCTTGCTGAAAAGAAAAACCGAATTCCTTTATTTGATGTGGATTCTTCCACACTTCCATTCTTTGTAGAGGAAGGCTAACTGCTTGCTGGCTGGGAGCTGT\n",
            "\n",
            "Finished processing index 110: Saccharum_spontaneum.Sspon.HiC_chr_asm.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Sspon.03G0012650-1P::1A:42912-43912(+) \n",
            " CCGAAATGGTAGTTGCACTTTTTGCCCTGTTTCAGTCTATTTTCAGAGAAATTCAGTGTGTTGGAGGTTACTGCCTCCATAAAGTTTTGATAACCACTGTTTTTCCTTCAACAGGATGTGGCGGGGATTTCTTGGCTCAACCCTTTCATCCGTCGATCTCATACTATTTTCATTCTACATGTTTGCCATAGCTCCAAATCAATTGCGAATGAACCTCGTCAGACATCTCCTCTCTGACCCGACTGGATCAACCATGATCAAGACCTACCTGACCTTATAAAACCTTCTGCATCAGGCTGCAAGCCTGCAAGAACTCTTAGAAACTGTACAGTTTTGTAGTTGTACATAAGTTAGAGGGGATCTGGGGGTTCCTTTTCTTATTCCCGTATGGCGGCGGATCTAGGGGGTTACCAGCAATGCTATAATATACTCGAAATCTTTTATGGTTGCTATGGTGATGGGATAAAGAAGCACACCGTGTTGTGCATGATGGAATGGAAGAAGAATAAGAGAGATCAGGTGATGGTCATGGTTCCTGCATTGGCCATTTTTAGGTCTTCTATCCCTTGTTTTTGCTCCATCTCTGTCTTTGCTGTTTCAAGGCTTCTTACATGTCCAATCAATTACACCCTTATTTTAGGTTGCTCAATGCCAAGTCAAAATGATTCCCCCCCCCCCCCCCCCTGTCGAACAATATTTGCAAAAGCAAAAAGGGAAAAAATAATTAGATGTGCAAGAATAGTATGAGGCTGTCCTTGCTCCTGCGCCATGGCATTTAGCCGCAAGATGTTGGGGAAGGAGAAGGCCTGTAGGAGACAAAGGTTGGAGGCAAGGGAGAGAAAGGAAGAAGGGCAGGAGGAGAGGCGGAACAACCCGTCTCCCTCCACCTCTCTGCCTTCTTCTCTTTGCGCCGCCATCCTCACATCCGTCGCACCACCCACTGACAGGCCGGCGGCATCCTTCTCGATCGGCTTATTGCCATATCATCACCGCCGGACGG\n",
            "\n",
            "Finished processing index 111: Secale_cereale.Rye_Lo7_2018_v1p1p1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "SECCEUnv1G0527060::0R:10346-11346(+) \n",
            " CACATCATCGACACAACAAACATATAGATGCCATTGGTAGCGTCACATCAAAATTTCCATTAAATGCCCGTTATACAACCTTAAAACATGATGCATCGAGAGCATCTTTATGTCGTTATCTCTTACTTTATTCTCAAATAGATGCAGGGGTTTGTAAGTTTTGGGAAACAAAAAAGTTAGAGAGAGTTGAAAACAAACAAGCAAAAGGTGGACAAGGGAAACCATCACCAAGTGTTTGGGTGGTGCGGTGTGCTGCAATGCCTGGACACAACTACGCTTTAACTAAAAAAATTTCCAATGTCCCATGTGCACGTTTGAAGGATCTGTCTGTACTTTCAATTTTACTTTGTTGACAAACGTGTGTCTTTGCTGTGCTCTTCAATTTGTTCATGGGGTCTTTGTAAAAAGAAAAATGATTGTTACTGACATAGCCATGAGTGAGGGTGATGTAGAACCGAATATGGCAACTAGAATACGTAGCACTGACATCTGACACATGAGTAGAAACATTTGATTGAGTCATCATGTTAGGTGTGGCAATAAAAGCTTTTCGACACTTGAAAATAGATATCCACCGAAGAGATATTCATTCAATTCCCTCGTTACAACCTTATATAAATACAAGCTAAAGACACCTTATTCATTGAGAATCGACGTGGGGATTTCCCTGCATTTAATGCACGCGAGACAACCTAAAGATACATTACTGGTTACTCCCTCTGTAAACTAATATAAGAGCGTTTAGATCACTACTTTAGTATTCTAAACGCTCTTATATTAGTTTACGGAGGGAGTACTTTGTAAAGGTGAAATAGAGGGGATTGTAAGTTTTGTCAAGCCAAAAAGCCTTCACATTAAGAAGAGTAGTCCAGTATATATGTGTTGGTTGGGTCCTCTGCTTCACCTCTCCTCTGCACCGCACCGCACCGCACCGCACCGGACAGTCAGTCAGAAAGAAAGAAAAGGAAAGCAGAGTAGGAGAAAGGGAAAGTCTTTAGCG\n",
            "\n",
            "Finished processing index 112: Selaginella_moellendorffii.v1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "SELMODRAFT_401996::GL377565:0-42(+) \n",
            " AAACGCGTTAGAGGCCTAACACGAGGCAGCTTCATGCTGCAG\n",
            "\n",
            "Finished processing index 113: Sesamum_indicum.S_indicum_v1.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "SIN_1000325::C00398:6400-7400(-) \n",
            " TGTCATATTCACGTTTAGACAATTAATGCTCTTATAAATTTTTAGGTGAGATTTATAAACTTCTTCTGCCACAGTGCAAGAGCATGTGATCTTAAATTCGATCTCTTGGTTTTGGTGTCTTGAGACACGCATGCTCTGAGAATTTGAGACAATATAAATTCAAATTGTAAACTAAATTCAAATTCATTATTGAAGAAGAAAACAAGATGTTGAGCTCTAACGACACTTGTTTTTCCTTCCAAGTTATGCACAATATAATAACTCAGTTAACATTTTGGCCAGAGAATGGTTCCAGGTATTTGAACTGGACCAACCAACCATGGAATTATGTTCAAATTCTTTCATATTTACTGCTGTACCAAGCAATTTTGTCCTTTGCCATATCTGAAATAGAAATAATGGGGAGGTAAAATGTAAAGAGAAAATGTCTCTGTTTGTTCTATTTCCTGAACATCATATGGTACTTAAACTACACTCATCAAAATTGGATTAACTAGAAGAACAAGGAAATCATAAAGTTTGCCCTTGTACCATAGGTCATGTTACAAAGTAATGAACTTTTGTTAGTAATTTCAGTTGGATGTATGTTTTGCATGGATTCTTTAATGTCCTCATTCTTCCATTTGAAAGTTGTGAACCTACTAAACTAATCTTTCTGTTTCTAACTCATATTATTATGAGAGAGTCTGTCTGACAATTCCTTTACTCATGACTTGGCACAGTAGTCTGCTGAAATTTTCACTTGTATTCTTCACTTTTTTAGTGCTTGCACTTTTCCATTTTGCAGCTTCATTAGTAGACTTCTTTGTCTACTCACAAATTATGCCAACAAATTATCTAAGGCAAAGTGTTTAATTCAAGATCTTGGTTCAAGTTTGTTTCATTACCTTGTTTTGGATGTACATGACATGTTTGACATTTCATCACTCTGTGATGTTTGTGCATGTTTGTTCAGGAGCCACACATACAAGAGAAGCAAATACACTCGTTTTCAGCCTCA\n",
            "\n",
            "Finished processing index 114: Setaria_italica.Setaria_italica_v2.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "SETIT_016158mg::I:9196-10196(+) \n",
            " TAATCATGCAATCCCGTATTCTCAGGATGACTTCACCAAAGCAACGTATATAAACTTTAGTCCAAAAACCATCATTCCACCTTCAGCGTGCACGTCATGCGACATTTGTGGACTGAATTCAATGATACCGTGTCGTTATAGTGGCCGGTCGCCAGGGCAGTCAATTGATGAATAGTGACAAAATTGGACGGACACAGCGCTGATAGCAAAGACTTTGACCCAGACTCCAATGGCGAACTGCCTGTAGCAGCAACATTTGTAGACTGAATTCAAATACCGTCATGTTGTTATATAGTGGACGGCCCAGGGCAGAATAGTGACGAAATTGCACGGACACAACGTTGATTGCGAGACAATCATCAATCATGCACAACTTTTTGTTTCTCGCAATCATGCGTACGAGACACAATCCGATCCTCACAATCGGAAGCACAACTTTTTGTTTCTCGCCGGGGATGCACTACAACACCAGCTCTCTCATAGGCCATGGCAGCAACGATGGCAGTGGTAGCATCCCTGTCGAGAGGGCCGCCTTGCTATCTTTCAAGAAGGGTATCACAAGTGACCCCACCGATCACCTTGCCTCTTGGAGAGGTGTCAAATGCAACAACAAGATATATCCTACCCTGAAGAGAAAAATTACATTGTGCAACTACGAGATATGCATATATATCCTTCAATTTTGTACTGACAGATATATCCTACCCTGAAGAGAAAAATTACATTGCGCAACTGCAGACAGGCCACCAAATAAATGTAACCAACTCGAAAGCAACACAAATCCACGTCAGTCATTGACCAGGAATCAACCCATCGTTGTCCTGATAGGAATTTGCCGTGGGACGACCGCCAATATTCTAGACGACCGCCAATATTCTAATCTCCGCACAAGCTGGCAACTTGCACAAGCGATTCTTTAGTACTCTCTGCAGGTAACGTACAGTACTGTAGGCCTATATATATAACCTATCAGCCCAAAAATACACCACAGCTCCCAACA\n",
            "\n",
            "Finished processing index 115: Setaria_viridis.Setaria_viridis_v2.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "SEVIR_1G000100v2::1:754-1754(+) \n",
            " ACACGGGGTGCGGAGTTGGTCGTCGCGGTCGTGGGCGTGGTCGGCGTGGAGCAGGCGTAGTCGTCGTCCTCATTGTCCGACGACGGCGGTGGCGGCTTGGGCGCCTCCGCAACCGCCAGGTCCAGGCGTAGGAACTGGATAAGCTTCTTGTGGTTCCCCATGGCCGCCACAGTTTGTGTTGCAGGAACAATCGAAGCAAGAAGCTGAATAGGAGAGGCTGTTGGTAGGGAGAAGATGGAAGAAGCGAGGTTGTCTGTCTTGTGAGCTGTGCTTGCGATTCTGTTGAGGTGCTCCACACACCTGCCGAGCGTCTCCTCGCGCCTCCCCACGCCCGCCGATGGTCCCCTCGAGCCGCCGCTGGGGAGTGCCAGCTCTTCCTCGAGTTCGCGCTGAGCGGCTTGGTGAGGGGGCAGCGGGCGCCTGGGCGAGCGCGTCGTCCGAGCGTCATGGCGCCCGAGGTGGCGCAGGGGGAGGAGCAGGGCCCAGCCGCTGTCGTCGGGGCGCTCGGCTGGACGGTCATCGAGATGGCCACGGGCCGCGCGCCGTGGAGCGACGTGGGCGACCTTCTCGCGGCCGTGCACCGGATCTGGTGGCCAAGGAAAGAGGTGGCCAACGAGATGGCCGGCGAAGACAAGCGCCTGTTGGGAGAGAGGAGGAGAGGAGGGGGAAGGGTGGTGGGGCTCGTAGGAGCAAAATTGTCTTTTCGTAAGTTATTTAACAATTTTTGGATGGAAAATCTAACAGAGTGAACTCAGTATCAAATAAGGAATTAAAAAAATTTTAGACCAAGAAAGCAACCGGTAATTGTTCTGATGTCAAATAAGAAATTATTTAAATTCAGAAGGAAATACAAGAGACGAGTAGAGCAACTAGCAAGGGAAAAAAAAAGGAAGAATTGGTAAATTAGGCCCAATAAGCACGGGCCCAAAATGGAGGCCCACAAGGCCTGTCTCATCCAGCCGTGGATAAGATAAAACACAAGCAGAAGGGGAGGATCGAG\n",
            "\n",
            "Finished processing index 116: Solanum_lycopersicum.SL3.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Solyc01g005000.3::1:11922-12922(+) \n",
            " ATTTGAAACAAATATCAACTAGTATATATTAAAAATTCAATACCAAAATATCATTTGTCAGAAAAAGAATAAAAGGCAGAGAGAAACCTGCGTGCAACCAACGGAGCTAAAGGCAAATTAGAAGAGGGAATTATTCATTTGATGAGTAGGCTTTTTATTTGTGGTTACTATTATTAGGAATATAAGAATATATGATAAAGTACGTAGAGAACGTGGCTCTTACTTTTCAAAGAATAAGATATCTTATTTCTCCTCGTGACACTAATAAGTGTATTTGAGAAAAAAGAAATTTGAAATTGTGATTGAAAATATAATTTCTCAATGTGGAATTTAATTATAATTTTAAAAAATTGCCTGATTTTTTTTATCAAACTTGTTATATGGATCTTATTTAATACAATTTATTTTATCGTAATGACGATGATTGATTTCATTCAAAGAGGAATAAATTTGGATGCTTTATTTATCTAACATAGGAAGGTTTACTGTCTCAAATGTCAGAACAAAGCTAAACAAAAATATTACTTTTGGAATAATATTATTTTATTAAAAGCACAAAATATTTCTTGGACTTAGCTCAATATAATAATTATACTTTGAAAATATTATTATTTTTATTACTAAAAGCACACAACAACTAACATATATTTGAGAAAGAATTCCAAGGTCATTTGCTCACAAATATATATGCCTTCTGTAATTCCCTACCAATAGAAGAATTGGAAAATTTTTAACTAAACTAAGTCATTATAAAATAATTTATCAAAGTAATATATTTTTTTAAAATTTTACAAAACTAGTATAAACGTATTCCACGGTAACGTTTTAGGGTATGTTTTATTTTTTAAAAACTAACAGCATTACTGTGAGTAACGTTTTAGGAGTAAAACGTTACTGTGAGTAACGTATATCAATCTGACGCCATNNNNNNNNNNNNNNNNNNNNCATATTATTTTGGTGAATGACTTTATATAATGGGTTAGTTTGGTTAAAACCTGAA\n",
            "\n",
            "Finished processing index 117: Solanum_tuberosum.SolTub_3.0.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "PGSC0003DMG400039401::00:68890-69890(+) \n",
            " TTGCCTCAACATGTACACCTTCTTGAGCACCTCTTGGTAGTAGTTTGGAGTCAAGTATCTCAACTCCATAAGTGCAATCAACTCTTGCTAAGTAGGTAACTCGTAGTCATGATAGCTCTCCCTTTTCCTCCTCTTGGACTCCCACCAAGTTGATGCGTACCCCTCAAATTGCGTGAGAGCATACTTCGCCTTCATAGTTTCTGAGATATTATTTGTGAGAAAAATTCGATCACTTTGAATTTTCCATTCCAAGAACTCATCGGGGTCACTACTTCCTTTGAAACTTGGAAGTGTGACCTTGATAGTGTTCAAGCCCCTATCTTGATGATCATTCCTCTCCCTAAACCTATTTCCCCTATTTACCATCCTATCGCCTCTATATCCCATCCTATCTCCTTTATATCCCATCCTAACTCCTCTATTTCCATATTCTTGATCTCTCATGCCTTGGTCCCACCCCAAGTCCCGAGGTCGTACTCCTCCCCTTTGTACTCCATTTCCATCCCTACCGCCATGCTCCCAAATCTCATCATTATTGAACTCCTCATGAATTTCATCATAGAGAGGATTTTGGTATTGGTAGTCTTGGTAGGACATTTGTGGTGCATTTTGTGGATGTGATGTGTAGGGTTCATGAAAAGATGCATTTGAAGCATTTCGAAATGGTGTTTGGTATTGGACTTGGTTTGCCACAAAAGAAAGATTTTCCTGTGGCGGGTGATAGGTTTGTGGATTTGGATGTGGCGTAGGCTGTCTTAGGTGGTTTGGAGGGTTGGGAGTTTGGAAAGTAGGGCGTTGGTAGTTTCGGGCTTGGCTTTGTGGAGGAGAACATAGATTATGAGCTTGTCCCGAAGCAAGGGTGGTGGAATGCACCGATGCCCTTCTTATCTCATCATATTCGGGTTGAATTGCTTTCCTTCTATTCTCATTTTCCACCCGTACCAACCTTTCCTCCATACCTCCCCATTTCTTCCATTTCATTTCCGCTACTTGCTTAGAG\n",
            "\n",
            "Finished processing index 118: Solanum_tuberosum_rh8903916.ASM1418947v1.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "RHC01H1G0001.2::1:8903-9903(+) \n",
            " CTTATCCTTACCTTTGTGGTGTAGAGAGATGTTGTTTCCGATAGACCCTCGACTCAACAATAAAGAAAAGAATTTGAAGCATGGTAGCAAGAAAATATGACAGCAAAACAGCAAGATACAAATCAAATGAAGCAACAAAGTGCAACAATACTCGGCTACCTACTAACTTTCTAATCTAATCCTCAACTTTCACTCCTTCCTATCTAGGGTCATATCCGGAGTAAAACTTGTAGATTTTGAAATTACCTGACTATAGTTATATTTCGAAGATATGTCCGAAAAGTGGCTCTCTAGTATGAATGTAGTTTGGGCAAGTTGTATGACTTTGGGTCTCTGAAACTATTCCAACTTGCTTTTTGGTACAAGTCTTAAATACAATTTGGGTCTCAATCTGACAAATAATTCAATGAGCAAAACAAGAATTCTCACTTCTACCCTTCCACCAACCCTACCATTACTCTATTGGAGAGAAGCAGTCACAAAGAGAAGACTTCTCTTTACAAACCCTTAAAAGTCTTGCATTCCCATCCATTACACTGCCTTCTTCTTTCATAAAGAATTGGTTTTGAGAATCTAAATCAACAACTCAGAACTTCCTCCTACTGTCCTGTTTAGCAAGAAACTAACAAACAGCGATATGGCGGCGTATGCTGCTGTGACTTCACTTTTGCATACCCTGGACAGTCTATCTCAAACTGATATTCTTTACAAAAAAAAGCAAACTGAAGTCCTCAGTGACAAGTACACCTTCCTTAAAACCTTTCTTGAGGATTTTACTAATATTTTCCATGAAGATGTAAAGATGAAGCATCTCGAAAGGATAATCCAAGAAGCTGCTAATGGAGTGGAAGATACTATTGACTCACATGTATATGATAGACATGTTTTAGTACAAAGCAAGCAAGTACGAAGTAAGGCTGATGCAATTTTCCATCAAAACTTGGAATATGCAATTGAGGAAATTGGTTTGATACAAAAAGAGGTTATGAAAAAGATAAAG\n",
            "\n",
            "Finished processing index 119: Sorghum_bicolor.Sorghum_bicolor_NCBIv3.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "SORBI_3001G000100::1:950-1950(+) \n",
            " ATGCTATAGAAGGGAAGCCGTCAAGCGAAGGGAAGGGAAGCCATATATACCTACCTTGAAGAAGAAAGTATATATCGGATCGGGAGCTAAGCTTCCACTTAGGTCTTGTTTAGATCTTAAAACGTTTTGGATTTTAACATTATAACACTTTTGTTTTTATTTGACAAATATTGTCAAATTATAAAGTAACTATGCTTAAAAGATTCGTATCGTAATTTACAAGTAAACTGTGCAATTAGTTATCTTTTTTATCTATATTTAATGTTCCATACAAGTGCCACAAGATTCGATGTGATGGGAAATCTTGTAAAATTTTGGGTTTTTTGGTGCATTAAAAAAGGCCTTAATAATATAGAGCTAGCACATACTCTCTCCGTCCCTGAAAGAATCAATTTCTAGGATTCGTGTCAGTCAAACTTTTTAAAGTTTGACTAACTTTATAGAAAAGAGTAACAACATTTATAATATAAAATATACATTATATGAAAATATATTTTATGATGAATCTAATAATACTAATTTGATACTATAAATCTTAGTTTTTTTTTCTAGAAATTTGGTTAAAGTTTAAAAAGTTTGACTTAGGACAACTCTAGAAATTGATTCTTTCAGGGACCGAGAGAGTATATTAATTCTGCATTTTGGCACAAGGAACACTAAAGGCAGGCGCCGCAACTAACATGCATATACCCCCCCAGCATCCATCCTCAATCGGGCAACGGCCGGGCATGCAGCAGCCATTATCATTGCCATTGCCAATGCGCCTGCTACACTACACACCAAAACCGTACCACTGATCCATATGCCTCTCTCTCACTCTATATACATGTTACTGTAGTAAGTAGTACTCCAGTAATAAGTGGAGTATAATAAGTAGTGGTCGATCGGTGAGGGAGCTGTTAATAGATCGAGTGAGTACTTATTGTTAACTAACGAGTACCCCAGCTGGCTAGCTAAGCCTATGCTAGCTGCTGGTTAATACGTACTCCAGTTGGTTACC\n",
            "\n",
            "Finished processing index 120: Theobroma_cacao.Theobroma_cacao_20110822.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "TCM_000001::1:368-1368(+) \n",
            " ACATGGAAGCATTTAATAGACGAAAGAGAGACAGCAGGCCAACAAATACATTGGAGGATAGGAAAAGGTGGCAAATTTTTTTGGCATGATGCTTGGATAGGTGATGAGCCACTGGTAAATTCTTTTCCCTCATTTAGTCAATCTATGATGAAGGTTAATTATTTTTTTAATGACAATGCATGGGATGTGGATAAGCTTAAAACTGTTGTTCCTAATGCGATTGTGGATCAGATATTGAAGATTCCCATTAGCAGAGATAAGGAGGACATAGCATATTGGGCCCTTACACCGAATGGAGATTTTTCCACTAAAAGTGCTTGAGAGTTATTGAGACAACGAAAATAGGTGAATCTAGTGGGTTAGCTAATTTGGCATAAGTCTATTCCCCTCACTGTCTCTTTCTTTTTGCGGTGCATGTTGAATAACTAGATTCCAGTGGAGATGTGTATGAAAACCAAAGGCATTCAACTTGCATCAAAATGTCTTTGTCGTCAATTGGAAGAATCATTATTGCATGTTTTATGGGACAGCCCAGTTGCACAGCAGGTTTGGAATTTTTTTACAAAATTCTTTCAAATCTATGTTCATAATCCTCAAAATGTTTTGTAGATCCTTCATTCATGCTACTATTCAAGTGACTTTACTAAACTTGGGCATATTAGAACATTATTGCCTTTGTTTATTTTCTGGTTCTTATGGGTTGAAAGGAATGATGCAAAGCAAAGGGACCTCGACATGTATCCTGATCATGTTATTTGGCAAATTATGAAAATGTTGAGACAGATGTTTCAAGGTCATTTATTGTGTAAATGGCAGTGAAAAAGGGATATTGACATTGCTATACATTGGAGTTTTAATTTTGCACAGGAAAGATAAGCTAGACAAAAAATTATTCATTGGATTAATCCATTGATAGGTGAACTCAAGTTGAATGTCGATGGTAGTTCTAAGGATGAGTTTCAAAATGCTGCTGGTTGGGGTGTTTTGAGGGACCACACAG\n",
            "\n",
            "Finished processing index 121: Theobroma_cacao_criollo.Criollo_cocoa_genome_V2.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Tc01v2_g000010::I:13315-14315(+) \n",
            " TGTGCATACAAAAACCGTACATGAAGTTATCAACTAATTGAATTCACTCTTGGTTATTTACTATGTCTCTTAGTTTTTTTGTTTTGCCACAACATGACACTTATTAAACATATAAGTCAATCAACACCATGCCTCAACTTTTGTATACTATTAATTGTTTCTTTTTCCATTTTGCAGTAACTATTTCAATTGTTTACCTTGCTAAACAATTTTTTTGTATATTCTCATTATATTTATATATCATCATCTTCATATTTCATTGTTAAATATGTATAAATTGAGAAAATGTAATACAATCTCTTCATATAAATAAATAGATGAAAATTTTCTAAATTATAGATCAATACAATTTCTTTACTTGTTTTACAAATTCTCAACCTCTACCATTCTTCTGATCAAATAATTTATTATTTTTGCACGTACACAATACTAAATATAGTGGTCTATGACCGAACTTATGAACTAATCAATTATGCATCCTTTTGTTGCATTGATGACATATAATTAAAAGTAGCCGTTGGTTATGAAGACATTGAGCTCAATTTCACTTGATTGTAAGTCAAGCTTTGAAGAGCAAAGGCTACACTGAAAAAACTAGACAGGGTCACGAGCATAATGTAAATATCTTTTTTTAATGTTGAGAAACCTTTACATTGCCGCAGCCTAATTCAAAATATATACAATATTGATATTTTTCAAAAAAAAAAATTCCAAAGGCTACTTTCTAGTTATAGATCAACATATATTGTATTTGCATTATACATTAATATATAATATATTCTATATAAACATATTAATAATGCTTATATTTTATAAAGTAAAATTTATTATTATTAACTAATAATTTATTTAATTAGTAAAAATTTGTCAATGTTAGGATCGAACTCACGTCTGAAAAGAAAACACTGCAACCACAAGCCATCTTTGCCAACCATACACATCGAGAAGGGCGGAAGTTTAATAAATAAATCTAGCTAATTGCCAAATATAAGACTAACCC\n",
            "\n",
            "Finished processing index 122: Trifolium_pratense.Trpr.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "Tp57577_TGAC_v2_gene4341::FKJA01000040.1:0-115(+) \n",
            " CTTATTTAATCATTAGATATCTTACAGCTAATCAGACCCGTGCACCGCACGGGTCGATGTTCTAGTTTTCACAAAACACATATTACTGAAATGAAACCCTAGTCTCTCTCCCTCA\n",
            "\n",
            "Finished processing index 123: Triticum_aestivum.IWGSC.dna.toplevel.fa\n",
            "First Sequence in the processed file:\n",
            "TraesCS1A02G000300::1A:94905-95905(+) \n",
            " CAAGAGAGTCAAATAGTGCTTGAAATTATCGGGAGGGAAGCGGATGGGGGCCGACGATGCGCCCCGGCCGTATGCGGAACGGCTCTTGCTGGTCCGCCGCTCGGCTCGGGGTGTGGACTGTTGTCGGTCGTGCCGGCGGCCAAAGCCCGGGGGCCCTAGGTGCCCCCGGTGGCCGTCGTCGGCACGGCAGGTACCCGCGCGCTAAAAGGCGTGTCCCTCGAGGTACTGCGCTGCAATGGCCTGCGGGCTCCCCATCTGACCCGTCTTGAAACACGGACCAAGGAGTCTGACATGCGTGCGAGTCGACGGGTTCTGAAACCTGGGATGCGCAAGGAAGCTGACGAGCGGGAGGCCCTCACGGGCCGCACCGCTGGCCGACCCTGATCTTCTGTGAAGGGTTCGAGTTGGAGCATGCCTGTCGGGACCCGAAAGATGGTGAACTATGCCTGAGCGGGGCGATGCCAGAGGAAACTCTGATGGAGGCTCGAAGCGATACTGACGTGCAAATCGTTCGTCTGACTTGGGTATAGGGGCGAAAGACTAATCGAACCATCTAGTAGCTGGTTCCCTCCGAAGTTTCCCTCAGGATAGCTGGAGCCCATTACGAGTTCTATCAGGTAAAGCCAATGATTAGAGGCATCGGGGACGCAACGTCCTCGACCTATTCTCAAACTTTGAATAGGTAGGATGGCGCGGCTGCTTCGGTGAGCCGTGACACGGAATCGGGTGCTCCAAGTGGGCCATTTTTGGTAAGCAGAACTGGCGATGCGGGATGTACCGGAAGCCGGGTTACGGTGCCCAACTGCGCGCTAACCTAGAACCCACAAAGGGTGTTGGTCGATTAAGACAGTAGGATGGTGGTCATGGAAGTCGAAATCCGCTAAGAAGTGTGTAACAACTCATATGCCGAATCAACTAGCCCCGAAAATGGATGGCGCTGAAGCGCGCGACCCACACCCGGCCATCTGGGCGAGCGCCATGCCCCGATGAGTAGGAGGGC\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%run /content/florabert/scripts/0-data-loading-processing/02-download-process-db-data.py Refseq"
      ],
      "metadata": {
        "id": "-D-jABtDWMI-",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "ecd82dca-e421-4931-b524-05dd098ed904"
      },
      "execution_count": 11,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Name of database to be processed: Refseq\n",
            "\n",
            "Finished processing index 0: GCF_003935025.1_Abrus_2018_genomic.fna\n",
            "First Sequence in the processed file:\n",
            "113848124::NW_020874290.1:16265-17265(+) \n",
            " ctttacaaaaaaacaaaaaaaaaggagtaattaactgtgacacgttcactaaaaaaaaatccttttgtagcaaatcatttattaagaataataaataagcttaacgcaaaaggagaaaaagaagtaataataacTTGGTCCAGAACATCTACCATTATACCCACAATGATTGGCCATACCATTGCTATCCATAATGGAAAAGAACATTTACCTATTTATATAACAGATCGTATGGTAGGTCATAAATTGGGAGAATTTTCACCTACTCTAAACTTCCGAGGacatacaaaaaatgataatagatctcttcgttaacattaataacgaacgataaaatgaaaatcatataatttatatatttgaatttatatatatatataagaaataatctatatattaacaaataagatttatatttctaatataaatgtttataaatgtttattcattttttagtgaaaatcgaattttatgataaagaagAACCAATATATAGAAGTATAATCTTTTAGTCAAAAACGATTCATATTTGTTCACAACATAAAGCAcgaatactaatattaattgattagatTTGTGCATAAAGAAAAGGTTATGTGTCGTAAAGTTATACTTCATTGAGTTGAATATCGTATTCAATATTTTGGATTGGTTTATTTTGCAATAGCAAATGCTAGTAACAAtgtaaatttcaacaaattaagtcaatgatttataaagaaatattaaaaaaaaaatacacaaacaaataagacgtatcatttttatgtatagTAGTGGAGAATTAtgggacaaaaaataaatccgCTTGGTTTCAGACTTGGTACAACTCAAAGTCATGATTCTCTTTGGTTTGCACAACCAACAAATTATTCCGAGaatctacaagaagataaaaaaatacgagattggatccaaaattatatacaaaaaaatattagaatatcctCTGGTGTTGAAGGAATTGGACGGATAAAGATTCAAAAAAGAATCGATCTGA\n",
            "\n",
            "Finished processing index 1: GCF_019202715.1_MaoHua_MHT_genomic.fna\n",
            "First Sequence in the processed file:\n",
            "130786625::NC_080133.1:10395-11395(+) \n",
            " gttaaaATGGGTATTAATTGAGTTTTGACTTTTGATGTCATCATGGATGGGGGAGTATGTAACAAAACTCTAAACTTTAGGGTAGGTGATTATATTTCCAAAATATTAGGGGGGCaatgtgtattaaaggcaaacctaaGGTTATTGTAATCTTCCCTATTTATTAGTGTTTAcaagtttttgataatttaagggtaaatttcacttatctccctaagatttctgacaaatacagatttctgataatttaaaaaatagcactaacctttcttacttttttaaaattaatatcaagattcctcctagagttaatttttgtacaaaaataataaatttgtcttttgatttagaattttcaagtttttttctcaatacatctattaaaataaataaatattttttttaaaaaaacaccaCTTATTTataagttattatatttttatttttataaataattattttttataagaaaattaatctaaatagtttactttttaagtactcctattaaaaatatacttaaaaaattaaattttgatattaatattttttttaaaaaaacgtaagttaaatattttactttttaagttctctTAATCTAagcatatttaaaaattaacattcattatcaatttttttaaaaacaaagaagaggatatttattttaattagtgtattgagaaaaaaaataacaaatttattatttttgcacaaaaattagctttagaagagaatcttgacaatagttttaaaagtgaGAGAGGTcgatactatttttgaaattacataaaagatGACTATATTTGCCAAAAATCTCATGATggataagtgaaatttacccataattTAATGAATCATACCAAATATGCGTACAAAATAAACAGAGCGCTATGTACACACATAAGGCAAACATTAAGCCACGTTAGCTGGCCCATCTCAACCAATGAGACCTTATTATTGTACTCCAAATATGCtttattcattcattcattgtCCCAAC\n",
            "\n",
            "Finished processing index 2: GCF_002575655.1_Aet_v4.0_genomic.fna\n",
            "First Sequence in the processed file:\n",
            "120973134::NC_053035.1:1041-2041(-) \n",
            " gtttagggtttagggtatagggtgtttagggtttagggtttagggtttagggttagggttagggtttagggtttagggtttagggtttagggttagggtttagggtttagggtttagggtttagggtttaNNNNNNNNNNGCATAAAATTGATGTCATTAGATTTGTGTTGAAAAATAATAATGACTATTTTTTATGACATAATGAATGACGTCAATCGAGTAATTGTCTGTAAAGTCATGAAATATGCCAAATAAAGCACTGTTTGCTTAGCATAATGCGGATCTCTACTACGGCAAATCTTTGGGTGTAATAGTCCATGATACGGGCCAAAAACGGCTAAAACTGCGGGTGTTTATGACCGATAACTAAACGCACCTCGGGGTTCGTCAATCGTCGAAATCACTCCGGGgccccaaaacagtgagtaatagtccatgaAACGGGCCAGAATGGGCCAGCCAAAACAGCGAGTGtttatgaccgacacgtaaaggcaccccggggttcgtcaatcgtggaaatcactccgggactccaaaacagtgagtaatagtccacaaAATGGGCCAGAATCGGTCAAAACTGTGAGTGTTGATGACCAAAATGTAATTGCACCTTGGGGTTCAAGAACTATGGAAATCACTTCGGGACCCCAAAacggtgagtaatagtccacgaaacgagGCAGAATTGACAATAGTGTGAGTGTTCATGACCAACACGTAAACGCATCCCGGGGTTCATCaaggtttagggtttagggtttagggttagggtttagggttagggtttagggtttagggttagggtttagggtttagggtttagggtttagggtttagggtttagggtttagggttgggtttagggtttagggtttagggtttagNNNNNNNNNNCGATCGATCGAAATACTCCAAGACTCCACCTTTGTCATATATTCCATATATCACATTAGATAGATATCATATTCATGGAATACGATTCACTTTCAAGAT\n",
            "\n",
            "Finished processing index 3: GCF_002575655.2_Aet_v5.0_genomic.fna\n",
            "First Sequence in the processed file:\n",
            "120962283::NC_053035.2:1041-2041(-) \n",
            " gtttagggtttagggtatagggtgtttagggtttagggtttagggtttagggttagggttagggtttagggtttagggtttagggtttagggttagggtttagggtttagggtttagggtttagggtttaNNNNNNNNNNGCATAAAATTGATGTCATTAGATTTGTGTTGAAAAATAATAATGACTATTTTTTATGACATAATGAATGACGTCAATCGAGTAATTGTCTGTAAAGTCATGAAATATGCCAAATAAAGCACTGTTTGCTTAGCATAATGCGGATCTCTACTACGGCAAATCTTTGGGTGTAATAGTCCATGATACGGGCCAAAAACGGCTAAAACTGCGGGTGTTTATGACCGATAACTAAACGCACCTCGGGGTTCGTCAATCGTCGAAATCACTCCGGGgccccaaaacagtgagtaatagtccatgaAACGGGCCAGAATGGGCCAGCCAAAACAGCGAGTGtttatgaccgacacgtaaaggcaccccggggttcgtcaatcgtggaaatcactccgggactccaaaacagtgagtaatagtccacaaAATGGGCCAGAATCGGTCAAAACTGTGAGTGTTGATGACCAAAATGTAATTGCACCTTGGGGTTCAAGAACTATGGAAATCACTTCGGGACCCCAAAacggtgagtaatagtccacgaaacgagGCAGAATTGACAATAGTGTGAGTGTTCATGACCAACACGTAAACGCATCCCGGGGTTCATCaaggtttagggtttagggtttagggttagggtttagggttagggtttagggtttagggttagggtttagggtttagggtttagggtttagggtttagggtttagggtttagggttgggtttagggtttagggtttagggtttagNNNNNNNNNNCGATCGATCGAAATACTCCAAGACTCCACCTTTGTCATATATTCCATATATCACATTAGATAGATATCATATTCATGGAATACGATTCACTTTCAAGAT\n",
            "\n",
            "Finished processing index 4: GCF_026212465.1_ASM2621246v1_genomic.fna\n",
            "First Sequence in the processed file:\n",
            "130800972::NC_080047.1:22424-23424(-) \n",
            " TGATAGTCTAACAATATAAATGACTGattttaataatcaaataaaaattgaattttctgtcattttagttttaaataacaattgatGAAAAAAGAGGCTATTATGTACTTCAAGTAATTTCCTAAATTGATGTATAATTTTATACACAGTATGATTttgcagtataattttctaaattgtgaAATTTCAATTACACTGTAATAAATGAAATTTATTCCGTAGTATAATAATCTATGTGGTGTAAGTCCAAAATTTTGTCAGAAAATTTATAATGTTCTACACAGCACATTATATATtttccattataattttttgaataGTGCATTTATATACAGTTGTGTAATTGGGATATTTGTGATTTATTGCATCCATTAATGTATGAGCTTTtccgaaaatataaatttgtaaatatggtaagtgtaaaaaatgcaaatcagtcAAATTTCATTGATACTAACCAATGAAAGAAAGACAAATGACATGCTCACAATCAAACTTCCTTTTTGCAATTATACggtaaaattaatagaatgtatgatagaaaaatatttttataaaggtGGGAAACATGTAGGGCCcacaagaaatataaaaatgttaaaataatgttACTGGAAGATAAGTAGggatcataagcattattaaattataaaaatgaggataaataatattatttttttccaaaatttgtgacattaataaaaagatttttttttataagaacaacaaatataaacaccaaaaaatgacaaatggaaACTTTTTAAGGAATGAGAGGAGTAATTAGTTTCCACTAAATGGGATTTCACCAATTTTAAGAAAATGGGAGGGACTAATTTTACACTCTTTATTTTATGTgatcttattttattagttttgttgCATATAAAAGATCAATAAACGcattaaaatgtgaaaaaaagtgCGTGGgggaatataaattaattaatctttaaagaatataaatataataataatactaaatacTAATAGCCTAATG\n",
            "\n",
            "Finished processing index 5: GCF_000471905.1_AMTR1.0_genomic.fna\n",
            "First Sequence in the processed file:\n",
            "18449118::NW_006500357.1:0-482(+) \n",
            " AAAACCGTAGATTAAATAAACTGTCGGATTAAATATTGATTAGTTAATATTTtagttactttttttatttattagtagTAATGGTTACAATAGAACCCCATGAAAACAAATGATAATGGCCGCCACAAACATTTGAAGTTCAACCCCACACTAcacctttttactttttacctTACAAAGTATTGTAACAATAAAGGATGAGGAGTCCGGGTGTGGAATGAATAAAGTAGTAAGGTCCACTACGCCCCTCCTCAAATAGGTAACGGCTATTTTGTTTTACTTCAAACTGTTCAAGCGCCCTTCCCTtctatttgtctctctctccatggcAACAATGGGTAGCACATCCAAATTCCAAATGAGCAGGaaacaaactctctctttcaaattccAAAAGAGCAggaaataatctctctctaacccaaagAAAACCCATATTTTAGGTCAATTCTCAGTACTTGCCTGGACCTGATTTCATGACC\n",
            "\n",
            "Finished processing index 6: GCF_000471905.2_AMTR1.0_genomic.fna\n",
            "First Sequence in the processed file:\n",
            "18448976::NW_006500357.1:0-300(+) \n",
            " AAAACCGTAGATTAAATAAACTGTCGGATTAAATATTGATTAGTTAATATTTtagttactttttttatttattagtagTAATGGTTACAATAGAACCCCATGAAAACAAATGATAATGGCCGCCACAAACATTTGAAGTTCAACCCCACACTAcacctttttactttttacctTACAAAGTATTGTAACAATAAAGGATGAGGAGTCCGGGTGTGGAATGAATAAAGTAGTAAGGTCCACTACGCCCCTCCTCAAATAGGTAACGGCTATTTTGTTTTACTTCAAACTGTTCAAGCGCCC\n",
            "\n",
            "Finished processing index 7: GCF_001540865.1_ASM154086v1_genomic.fna\n",
            "First Sequence in the processed file:\n",
            "109710808::NC_033621.1:1096-2096(+) \n",
            " AGGAATAAGCTATTCCTAGTTATTCCTGGTATAGGTACAACTCTAATTTTgcttttggataaaaattgggttattctCTAGAATAGGGAAAATAGTGTTTCAAAGTTAGATTGAAATAATGACAATAAGAGTAATTATagttctaaaataataatatttttctaattaattaacataaaaatattaatgaaataataatttaacttatttaacCACAAAATAGTGGTGGGAAGGGTGACACAGTTTTGTCTCTTTCATCCGCCATTCACACATGAGgcatattataaatttgagctTGTTCTACTTATATGAGAgttaatttgttgaaatttacTATCCTATTGCTCTTTCAGTTGGACATCTATTTTTTAGTTAGATTGATGAGAATGTATCTTGGTTATTTGATGAATTACATTTTGCTGTCCTGTTAGTTCAGCTTGCCATTGTATGGTGCTGTATTTGAGTTAGTTTGGTAAGTATACAGGCTAGTTATTTGTTCAGTTTGCTGTTATTTCCATCGCCGCTTGCTATAACCACCCGCACATTCTCTCTAATTTTGTCCTGTGACTCACAGCTTGGCTTATGAGATGATGAAAGGGATCACattaaccattaaaattgtaGGAGTAAAAATGAGCAGTGCTATTTGTAGAAGTACTAACTGCTTTCTTGTTAAACTCTTCCTCATTTCTTGTTAAAAACCTCCAAAGTGTATTTTCTGCTTTTGAAAATAACTTTCAAGCATAGCAAGTGCTCCAAACTTTTTCATTTGTCAAATATTTATTGGTTGCTGCTTCTGAAAGTCAAagcagaagctaggccaaacgggTACTAGGCACCGATTTGATCCGGTCCGTTTTTGCAGTGTCTTTTATGGCATTGTTTGCTAGATTGAGAAAGGTTGGTCTAGTGCCCATTTGGCGTAGTGTGAGACCTAAATTCTACCTTTGCTTTCAGCTGCAGGAACCTGTACCAGAacatttagaaagaaaaatttggA\n",
            "\n",
            "Finished processing index 8: GCF_009805555.1_ASM980555v1_genomic.fna\n",
            "First Sequence in the processed file:\n",
            "127239155::NW_026137370.1:0-691(+) \n",
            " GGGGGGACCACCCTCTAAGCCTAAGTATTCCTCAATGACCGATATAGCGTACAAGTACCGTGAGGGAAAGGTGAAAGAAAACCCTATTTAGGGAGTGCAATAGAGAACTGAGATCCGATGCGAACAATCAGTCGAAGGAGCGTAGCTTAGAGCCTTTACTTGATAAGATATTAGTAAAGTAGTAAAGCGCACTCACCACTCTAACGGCGTACCTTTTGCATGATGGGTCAGCGAGGAAATGGGAAGAGCGGCTTAAGCCATTAGGTGTAGGCGCTTTCCAGAGGTTGAATCTTCTAGTTCTTCCTATTTGACCCGAAACCGATCGATCTAGCCATGAGCAGGTTGAAGAGAGCTCTAACAGGCCTTGGAGGACCGAACCCACGTATGTGGCAAAATACGGGGATGACTTGTGGCTAGGGGTGAAAGGCCAACCAAGATCGGATATAGCTGGTTTTCCGCGAAATCGATTTCAGTAGAGCGTATGATGTCGATGGCCCGAGGTAGAGCACTCAATGGGCTAGGGTGGCCCCATTTCGCCTTACCAACCCCAAGGAAACTCCGAATACAGGCCGTCATCGTTTGTACAGACAGACTTTTTGGGTGCTAAGATCCAAAGTCGAGAGGGAAACAGCCCAGATCGTACGCTAAGGTCCCTAAGCAATCACTTAGTGGAAAAGGAAGTGATCGAGCG\n",
            "\n",
            "Finished processing index 9: GCF_000004255.1_v.1.0_genomic.fna\n",
            "First Sequence in the processed file:\n",
            "9330760::NW_003302555.1:2523-3310(-) \n",
            " AAACAGTGATATGATCATTACATCTTAACTAAATAATATCTGCGTGcatatttaataaaaaccaaaaatataacatatatatatatatatatatatttgggtGCATCATCAAATCAAAGTAGTTGCAAAAACTGGACGAGGTTTTACTTAAATGGTCCTTACTCATGCATGGTCCATTTGCTACCTAATTAAGGATTGGTAGGGTGCGTATACATCTTATATAATTTGTGGCGGTGGGAGATGGATGGAGTTACTAAAAACGAAACGTATACAAGTACTATTCAAGGCTCGCTTAAGGGGTTTGCCCTCTGAGTATTAGATCTAGATATTTTTACGTTTCTTTCATTTAATGTCGGAACAACAGACACTAGCTGGCGCTTCAACGTGCATGATCTTGATTGGCTAGTAGATTCCAAGCATCAATACCTATCACATGCCCAACTTGGCTATGCATTCATGGGTAAATACCCTTACTTTCAATTATATctagaaataaatataagaagCCATCCATTAACCAGTGCATTTCCTCAACGCATGGATATGATCAGAACATCGATGAAGGTGGGAGGGGGTAATTAGCTGAGTGTCATAAATGAGGATCCATGTGGAGAGCATCCAATGGTAGTAGTACATGTTTGGTCTTAGCTGGCCCTACCACAAGGAATTAAACTGGTGGGACGATAGGGGTGGTTTCGTCATTCCACATATCTACCGATTAAGGAGTTTGATACCTTGCTATATAATGTACCTTGGCTCACAAGAGTTGGAGAGACAGTGCACAGAC\n",
            "\n",
            "Finished processing index 10: GCF_000004255.2_v.1.0_genomic.fna\n",
            "First Sequence in the processed file:\n",
            "9330760::NW_003302555.1:2879-3312(-) \n",
            " AGAAACAGTGATATGATCATTACATCTTAACTAAATAATATCTGCGTGcatatttaataaaaaccaaaaatataacatatatatatatatatatatatttgggtGCATCATCAAATCAAAGTAGTTGCAAAAACTGGACGAGGTTTTACTTAAATGGTCCTTACTCATGCATGGTCCATTTGCTACCTAATTAAGGATTGGTAGGGTGCGTATACATCTTATATAATTTGTGGCGGTGGGAGATGGATGGAGTTACTAAAAACGAAACGTATACAAGTACTATTCAAGGCTCGCTTAAGGGGTTTGCCCTCTGAGTATTAGATCTAGATATTTTTACGTTTCTTTCATTTAATGTCGGAACAACAGACACTAGCTGGCGCTTCAACGTGCATGATCTTGATTGGCTAGTAGATTCCAAGCATCAATACCTATC\n"
          ]
        },
        {
          "output_type": "error",
          "ename": "AssertionError",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mAssertionError\u001b[0m                            Traceback (most recent call last)",
            "\u001b[0;32m/content/florabert/scripts/0-data-loading-processing/02-download-process-db-data.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m     56\u001b[0m         \u001b[0;31m# Test run the main function\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     57\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 58\u001b[0;31m             gene_db_io.generate_sequence_for_species(\n\u001b[0m\u001b[1;32m     59\u001b[0m                 \u001b[0mdna_name\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     60\u001b[0m                 \u001b[0mannot_name\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/content/florabert/module/florabert/gene_db_io.py\u001b[0m in \u001b[0;36mgenerate_sequence_for_species\u001b[0;34m(dna_name, annot_name, dna_url, annot_url, dna_path, annot_path, save_path, db_name, species_name, regulatory_len)\u001b[0m\n\u001b[1;32m    222\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    223\u001b[0m     \u001b[0;31m# Unzip the gz files\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 224\u001b[0;31m     \u001b[0mutils\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0munzip\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdna_path_s\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdna_name\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    225\u001b[0m     \u001b[0mutils\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0munzip\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mannot_path_s\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mannot_name\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    226\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/content/florabert/module/florabert/utils.py\u001b[0m in \u001b[0;36munzip\u001b[0;34m(folder_path, file_name)\u001b[0m\n\u001b[1;32m     63\u001b[0m     \u001b[0min_path\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mout_path\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\".gz\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     64\u001b[0m     \u001b[0mexe_str\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34mf\"gunzip  -c {in_path} > {out_path}\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 65\u001b[0;31m     \u001b[0mexecute\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mexe_str\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     66\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     67\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/content/florabert/module/florabert/utils.py\u001b[0m in \u001b[0;36mexecute\u001b[0;34m(exe_str)\u001b[0m\n\u001b[1;32m     51\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0;34m\"grep\"\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mexe_str\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     52\u001b[0m         \u001b[0;31m# The response should be 0 in all cases other than grep\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 53\u001b[0;31m         \u001b[0;32massert\u001b[0m \u001b[0mresponse\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     54\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     55\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mAssertionError\u001b[0m: "
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!git init"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "nFygSvlR4Euo",
        "outputId": "c697d434-2f19-4522-ace3-6a0ff458ce83"
      },
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[33mhint: Using 'master' as the name for the initial branch. This default branch name\u001b[m\n",
            "\u001b[33mhint: is subject to change. To configure the initial branch name to use in all\u001b[m\n",
            "\u001b[33mhint: of your new repositories, which will suppress this warning, call:\u001b[m\n",
            "\u001b[33mhint: \u001b[m\n",
            "\u001b[33mhint: \tgit config --global init.defaultBranch <name>\u001b[m\n",
            "\u001b[33mhint: \u001b[m\n",
            "\u001b[33mhint: Names commonly chosen instead of 'master' are 'main', 'trunk' and\u001b[m\n",
            "\u001b[33mhint: 'development'. The just-created branch can be renamed via this command:\u001b[m\n",
            "\u001b[33mhint: \u001b[m\n",
            "\u001b[33mhint: \tgit branch -m <name>\u001b[m\n",
            "Initialized empty Git repository in /content/.git/\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!git config --global user.email \"gurveervirk.15517@gmail.com\"\n",
        "!git config --global user.name \"gurveervirk\""
      ],
      "metadata": {
        "id": "lpFSPJRP4IpX"
      },
      "execution_count": 5,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!git add -A"
      ],
      "metadata": {
        "id": "WHV9Zej15O9R"
      },
      "execution_count": 12,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!git commit -m \"first commit\""
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "tIoHaeJs5QLq",
        "outputId": "6c3d437e-9748-4283-bdb1-a487161223ed"
      },
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[main 7f58523] first commit\n",
            " 2302 files changed, 11764043 insertions(+), 16 deletions(-)\n",
            " create mode 100644 bedtools2/.github/workflows/main.yml\n",
            " create mode 100644 bedtools2/.gitignore\n",
            " create mode 100644 bedtools2/LICENSE\n",
            " create mode 100644 bedtools2/Makefile\n",
            " create mode 100644 bedtools2/README.md\n",
            " create mode 100644 bedtools2/data/aluY.chr1.bed.gz\n",
            " create mode 100644 bedtools2/data/gerp.chr1.bed.gz\n",
            " create mode 100755 bedtools2/data/knownGene.hg18.chr21.bed\n",
            " create mode 100644 bedtools2/data/knownGene.hg18.chr21.short.bed\n",
            " create mode 100644 bedtools2/data/refseq.chr1.exons.bed.gz\n",
            " create mode 100644 bedtools2/data/simpleRepeats.chr1.bed.gz\n",
            " create mode 100755 bedtools2/docs/Makefile\n",
            " create mode 100644 bedtools2/docs/_static/labibi.css\n",
            " create mode 100644 bedtools2/docs/_static/labibi.js\n",
            " create mode 100644 bedtools2/docs/bedtools.png\n",
            " create mode 100644 bedtools2/docs/bedtools.swiss.png\n",
            " create mode 100755 bedtools2/docs/build.sh\n",
            " create mode 100755 bedtools2/docs/conf.py\n",
            " create mode 100755 bedtools2/docs/content/advanced-usage.rst\n",
            " create mode 100755 bedtools2/docs/content/bedtools-suite.rst\n",
            " create mode 100755 bedtools2/docs/content/example-usage.rst\n",
            " create mode 100644 bedtools2/docs/content/faq.rst\n",
            " create mode 100755 bedtools2/docs/content/general-usage.rst\n",
            " create mode 100644 bedtools2/docs/content/history.rst\n",
            " create mode 100755 bedtools2/docs/content/installation.rst\n",
            " create mode 100755 bedtools2/docs/content/overview.rst\n",
            " create mode 100755 bedtools2/docs/content/quick-start.rst\n",
            " create mode 100644 bedtools2/docs/content/related-tools.rst\n",
            " create mode 100644 bedtools2/docs/content/tips-and-tricks.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/annotate.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/bamtobed.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/bamtofastq.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/bed12tobed6.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/bedpetobam.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/bedtobam.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/closest.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/cluster.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/complement.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/coverage.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/expand.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/fisher.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/flank.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/genomecov.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/getfasta.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/groupby.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/igv.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/intersect.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/jaccard.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/links.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/makewindows.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/map.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/maskfasta.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/merge.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/multicov.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/multiinter.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/nuc.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/overlap.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/pairtobed.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/pairtopair.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/random.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/reldist.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/shift.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/shuffle.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/slop.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/sort.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/subtract.rst\n",
            " create mode 100644 bedtools2/docs/content/tools/tag.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/unionbedg.rst\n",
            " create mode 100755 bedtools2/docs/content/tools/window.rst\n",
            " create mode 100755 bedtools2/docs/index.rst\n",
            " create mode 100755 bedtools2/docs/make.bat\n",
            " create mode 100644 bedtools2/docs/templates/page.html\n",
            " create mode 100644 bedtools2/docs/templates/sidebar-intro.html\n",
            " create mode 100755 bedtools2/docs/themes/rtd/static/rtd.css_t\n",
            " create mode 100755 bedtools2/docs/themes/rtd/theme.conf\n",
            " create mode 100644 bedtools2/genomes/human.hg18.genome\n",
            " create mode 100644 bedtools2/genomes/human.hg19.genome\n",
            " create mode 100644 bedtools2/genomes/human.hg38.genome\n",
            " create mode 100644 bedtools2/genomes/mouse.mm10.genome\n",
            " create mode 100644 bedtools2/genomes/mouse.mm8.genome\n",
            " create mode 100644 bedtools2/genomes/mouse.mm9.genome\n",
            " create mode 100755 bedtools2/scripts/makeBashScripts.py\n",
            " create mode 100755 bedtools2/scripts/vcfsort.sh\n",
            " create mode 100644 bedtools2/src/annotateBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/annotateBed/annotateBed.cpp\n",
            " create mode 100644 bedtools2/src/annotateBed/annotateBed.h\n",
            " create mode 100644 bedtools2/src/annotateBed/annotateMain.cpp\n",
            " create mode 100644 bedtools2/src/bamToBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/bamToBed/bamToBed.cpp\n",
            " create mode 100644 bedtools2/src/bamToFastq/Makefile.frag\n",
            " create mode 100644 bedtools2/src/bamToFastq/bamToFastq.cpp\n",
            " create mode 100644 bedtools2/src/bamToFastq/bamToFastq.h\n",
            " create mode 100644 bedtools2/src/bamToFastq/bamToFastqMain.cpp\n",
            " create mode 100644 bedtools2/src/bed12ToBed6/Makefile.frag\n",
            " create mode 100644 bedtools2/src/bed12ToBed6/bed12ToBed6.cpp\n",
            " create mode 100644 bedtools2/src/bedToBam/Makefile.frag\n",
            " create mode 100644 bedtools2/src/bedToBam/bedToBam.cpp\n",
            " create mode 100644 bedtools2/src/bedToIgv/Makefile.frag\n",
            " create mode 100644 bedtools2/src/bedToIgv/bedToIgv.cpp\n",
            " create mode 100644 bedtools2/src/bedpeToBam/Makefile.frag\n",
            " create mode 100644 bedtools2/src/bedpeToBam/bedpeToBam.cpp\n",
            " create mode 100644 bedtools2/src/bedpeToBam/notes.txt\n",
            " create mode 100644 bedtools2/src/bedtools.cpp\n",
            " create mode 100644 bedtools2/src/closestFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/closestFile/closestFile.cpp\n",
            " create mode 100644 bedtools2/src/closestFile/closestFile.h\n",
            " create mode 100644 bedtools2/src/closestFile/closestHelp.cpp\n",
            " create mode 100644 bedtools2/src/clusterBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/clusterBed/clusterBed.cpp\n",
            " create mode 100644 bedtools2/src/clusterBed/clusterBed.h\n",
            " create mode 100644 bedtools2/src/clusterBed/clusterMain.cpp\n",
            " create mode 100644 bedtools2/src/complementFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/complementFile/complementFile.cpp\n",
            " create mode 100644 bedtools2/src/complementFile/complementFile.h\n",
            " create mode 100644 bedtools2/src/complementFile/complementHelp.cpp\n",
            " create mode 100644 bedtools2/src/coverageFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/coverageFile/coverageFile.cpp\n",
            " create mode 100644 bedtools2/src/coverageFile/coverageFile.h\n",
            " create mode 100644 bedtools2/src/coverageFile/coverageHelp.cpp\n",
            " create mode 100644 bedtools2/src/expand/Makefile.frag\n",
            " create mode 100644 bedtools2/src/expand/expand.cpp\n",
            " create mode 100644 bedtools2/src/fastaFromBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/fastaFromBed/fastaFromBed.cpp\n",
            " create mode 100644 bedtools2/src/fastaFromBed/fastaFromBed.h\n",
            " create mode 100644 bedtools2/src/fastaFromBed/fastaFromBedMain.cpp\n",
            " create mode 100644 bedtools2/src/fisher/Makefile.frag\n",
            " create mode 100644 bedtools2/src/fisher/fisher.cpp\n",
            " create mode 100644 bedtools2/src/fisher/fisher.h\n",
            " create mode 100644 bedtools2/src/fisher/fisherHelp.cpp\n",
            " create mode 100644 bedtools2/src/fisher/kfunc.cpp\n",
            " create mode 100644 bedtools2/src/fisher/kfunc.h\n",
            " create mode 100644 bedtools2/src/flankBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/flankBed/flankBed.cpp\n",
            " create mode 100644 bedtools2/src/flankBed/flankBed.h\n",
            " create mode 100644 bedtools2/src/flankBed/flankBedMain.cpp\n",
            " create mode 100644 bedtools2/src/flankBed/test.fa\n",
            " create mode 100644 bedtools2/src/flankBed/test.fa.fai\n",
            " create mode 100644 bedtools2/src/flankBed/test.gff\n",
            " create mode 100644 bedtools2/src/genomeCoverageBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/genomeCoverageBed/genomeCoverageBed.cpp\n",
            " create mode 100644 bedtools2/src/genomeCoverageBed/genomeCoverageBed.h\n",
            " create mode 100644 bedtools2/src/genomeCoverageBed/genomeCoverageMain.cpp\n",
            " create mode 100644 bedtools2/src/getOverlap/Makefile.frag\n",
            " create mode 100644 bedtools2/src/getOverlap/getOverlap.cpp\n",
            " create mode 100644 bedtools2/src/groupBy/Makefile.frag\n",
            " create mode 100644 bedtools2/src/groupBy/groupBy.cpp\n",
            " create mode 100644 bedtools2/src/groupBy/groupBy.h\n",
            " create mode 100644 bedtools2/src/groupBy/groupByHelp.cpp\n",
            " create mode 100644 bedtools2/src/intersectFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/intersectFile/intersectFile.cpp\n",
            " create mode 100644 bedtools2/src/intersectFile/intersectFile.h\n",
            " create mode 100644 bedtools2/src/intersectFile/intersectHelp.cpp\n",
            " create mode 100644 bedtools2/src/jaccard/Makefile.frag\n",
            " create mode 100644 bedtools2/src/jaccard/jaccard.cpp\n",
            " create mode 100644 bedtools2/src/jaccard/jaccard.h\n",
            " create mode 100644 bedtools2/src/jaccard/jaccardHelp.cpp\n",
            " create mode 100644 bedtools2/src/linksBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/linksBed/linksBed.cpp\n",
            " create mode 100644 bedtools2/src/linksBed/linksBed.h\n",
            " create mode 100644 bedtools2/src/linksBed/linksMain.cpp\n",
            " create mode 100644 bedtools2/src/mapFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/mapFile/mapFile.cpp\n",
            " create mode 100644 bedtools2/src/mapFile/mapFile.h\n",
            " create mode 100644 bedtools2/src/mapFile/mapHelp.cpp\n",
            " create mode 100644 bedtools2/src/maskFastaFromBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/maskFastaFromBed/maskFastaFromBed.cpp\n",
            " create mode 100644 bedtools2/src/maskFastaFromBed/maskFastaFromBed.h\n",
            " create mode 100644 bedtools2/src/maskFastaFromBed/maskFastaFromBedMain.cpp\n",
            " create mode 100644 bedtools2/src/mergeFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/mergeFile/mergeFile.cpp\n",
            " create mode 100644 bedtools2/src/mergeFile/mergeFile.h\n",
            " create mode 100644 bedtools2/src/mergeFile/mergeHelp.cpp\n",
            " create mode 100644 bedtools2/src/multiBamCov/Makefile.frag\n",
            " create mode 100644 bedtools2/src/multiBamCov/multiBamCov.cpp\n",
            " create mode 100644 bedtools2/src/multiBamCov/multiBamCov.h\n",
            " create mode 100644 bedtools2/src/multiBamCov/multiBamCovMain.cpp\n",
            " create mode 100644 bedtools2/src/multiIntersectBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/multiIntersectBed/multiIntersectBed.cpp\n",
            " create mode 100644 bedtools2/src/multiIntersectBed/multiIntersectBed.h\n",
            " create mode 100644 bedtools2/src/multiIntersectBed/multiIntersectBedMain.cpp\n",
            " create mode 100644 bedtools2/src/nekSandbox1/FormatGuess.h\n",
            " create mode 100644 bedtools2/src/nekSandbox1/Makefile\n",
            " create mode 100644 bedtools2/src/nekSandbox1/nekSandboxMain.cpp\n",
            " create mode 100644 bedtools2/src/nucBed/LargeFileSupport.h\n",
            " create mode 100644 bedtools2/src/nucBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/nucBed/nucBed.cpp\n",
            " create mode 100644 bedtools2/src/nucBed/nucBed.h\n",
            " create mode 100644 bedtools2/src/nucBed/nucBedMain.cpp\n",
            " create mode 100644 bedtools2/src/pairToBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/pairToBed/pairToBed.cpp\n",
            " create mode 100644 bedtools2/src/pairToBed/pairToBed.h\n",
            " create mode 100644 bedtools2/src/pairToBed/pairToBedMain.cpp\n",
            " create mode 100644 bedtools2/src/pairToPair/Makefile.frag\n",
            " create mode 100644 bedtools2/src/pairToPair/pairToPair.cpp\n",
            " create mode 100644 bedtools2/src/pairToPair/pairToPair.h\n",
            " create mode 100644 bedtools2/src/pairToPair/pairToPairMain.cpp\n",
            " create mode 100644 bedtools2/src/randomBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/randomBed/randomBed.cpp\n",
            " create mode 100644 bedtools2/src/randomBed/randomBed.h\n",
            " create mode 100644 bedtools2/src/randomBed/randomBedMain.cpp\n",
            " create mode 100644 bedtools2/src/regressTest/Makefile.frag\n",
            " create mode 100644 bedtools2/src/regressTest/RegressTest.cpp\n",
            " create mode 100644 bedtools2/src/regressTest/RegressTest.h\n",
            " create mode 100644 bedtools2/src/regressTest/regressTestMain.cpp\n",
            " create mode 100644 bedtools2/src/reldist/Makefile.frag\n",
            " create mode 100644 bedtools2/src/reldist/reldist.cpp\n",
            " create mode 100644 bedtools2/src/reldist/reldist.h\n",
            " create mode 100644 bedtools2/src/reldist/reldistMain.cpp\n",
            " create mode 100644 bedtools2/src/sampleFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/sampleFile/sampleFile.cpp\n",
            " create mode 100644 bedtools2/src/sampleFile/sampleFile.h\n",
            " create mode 100644 bedtools2/src/sampleFile/sampleHelp.cpp\n",
            " create mode 100644 bedtools2/src/shiftBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/shiftBed/shiftBed.cpp\n",
            " create mode 100644 bedtools2/src/shiftBed/shiftBed.h\n",
            " create mode 100644 bedtools2/src/shiftBed/shiftBedMain.cpp\n",
            " create mode 100644 bedtools2/src/shuffleBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/shuffleBed/shuffleBed.cpp\n",
            " create mode 100644 bedtools2/src/shuffleBed/shuffleBed.h\n",
            " create mode 100644 bedtools2/src/shuffleBed/shuffleBedMain.cpp\n",
            " create mode 100644 bedtools2/src/slopBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/slopBed/slopBed.cpp\n",
            " create mode 100644 bedtools2/src/slopBed/slopBed.h\n",
            " create mode 100644 bedtools2/src/slopBed/slopBedMain.cpp\n",
            " create mode 100644 bedtools2/src/sortBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/sortBed/sortBed.cpp\n",
            " create mode 100644 bedtools2/src/sortBed/sortBed.h\n",
            " create mode 100644 bedtools2/src/sortBed/sortMain.cpp\n",
            " create mode 100644 bedtools2/src/spacingFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/spacingFile/spacingFile.cpp\n",
            " create mode 100644 bedtools2/src/spacingFile/spacingFile.h\n",
            " create mode 100644 bedtools2/src/spacingFile/spacingHelp.cpp\n",
            " create mode 100644 bedtools2/src/split/Makefile.frag\n",
            " create mode 100644 bedtools2/src/split/splitBed.cpp\n",
            " create mode 100644 bedtools2/src/split/splitBed.h\n",
            " create mode 100644 bedtools2/src/split/splitBedMain.cpp\n",
            " create mode 100644 bedtools2/src/subtractFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/subtractFile/subtractFile.cpp\n",
            " create mode 100644 bedtools2/src/subtractFile/subtractFile.h\n",
            " create mode 100644 bedtools2/src/subtractFile/subtractHelp.cpp\n",
            " create mode 100644 bedtools2/src/summaryFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/summaryFile/summaryFile.cpp\n",
            " create mode 100644 bedtools2/src/summaryFile/summaryFile.h\n",
            " create mode 100644 bedtools2/src/summaryFile/summaryHelp.cpp\n",
            " create mode 100644 bedtools2/src/tagBam/Makefile.frag\n",
            " create mode 100644 bedtools2/src/tagBam/tagBam.cpp\n",
            " create mode 100644 bedtools2/src/tagBam/tagBam.h\n",
            " create mode 100644 bedtools2/src/tagBam/tagBamMain.cpp\n",
            " create mode 100644 bedtools2/src/unionBedGraphs/Makefile.frag\n",
            " create mode 100644 bedtools2/src/unionBedGraphs/unionBedGraphs.cpp\n",
            " create mode 100644 bedtools2/src/unionBedGraphs/unionBedGraphs.h\n",
            " create mode 100644 bedtools2/src/unionBedGraphs/unionBedGraphsMain.cpp\n",
            " create mode 100644 bedtools2/src/utils/BamTools-Ancillary/BamAncillary.cpp\n",
            " create mode 100644 bedtools2/src/utils/BamTools-Ancillary/BamAncillary.h\n",
            " create mode 100644 bedtools2/src/utils/BamTools-Ancillary/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/BamTools/.gitignore\n",
            " create mode 100644 bedtools2/src/utils/BamTools/Makefile.frag\n",
            " create mode 100755 bedtools2/src/utils/BamTools/include/BamAlignment.hpp\n",
            " create mode 100644 bedtools2/src/utils/BamTools/include/BamAlignment.mapping.hpp\n",
            " create mode 100644 bedtools2/src/utils/BamTools/include/BamAncillary.h\n",
            " create mode 100644 bedtools2/src/utils/BamTools/include/BamReader.hpp\n",
            " create mode 100644 bedtools2/src/utils/BamTools/include/BamWriter.hpp\n",
            " create mode 100644 bedtools2/src/utils/BamTools/include/SamHeader.hpp\n",
            " create mode 100644 bedtools2/src/utils/BamTools/include/api/BamAlignment.h\n",
            " create mode 100644 bedtools2/src/utils/BamTools/include/api/BamAux.h\n",
            " create mode 100644 bedtools2/src/utils/BamTools/include/api/BamMultiReader.h\n",
            " create mode 100644 bedtools2/src/utils/BamTools/include/api/BamReader.h\n",
            " create mode 100644 bedtools2/src/utils/BamTools/include/api/BamWriter.h\n",
            " create mode 100644 bedtools2/src/utils/BamTools/include/api/SamConstants.h\n",
            " create mode 100644 bedtools2/src/utils/BamTools/mapping/BamAlignment.map\n",
            " create mode 100644 bedtools2/src/utils/BamTools/mapping/BamAlignment.py\n",
            " create mode 100644 bedtools2/src/utils/BinTree/BinTree.cpp\n",
            " create mode 100644 bedtools2/src/utils/BinTree/BinTree.h\n",
            " create mode 100644 bedtools2/src/utils/BinTree/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/BlockedIntervals/BlockedIntervals.cpp\n",
            " create mode 100644 bedtools2/src/utils/BlockedIntervals/BlockedIntervals.h\n",
            " create mode 100644 bedtools2/src/utils/BlockedIntervals/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextBase.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextBase.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextClosest.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextClosest.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextComplement.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextComplement.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextCoverage.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextCoverage.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextFisher.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextFisher.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextGroupBy.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextGroupBy.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextIntersect.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextIntersect.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextJaccard.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextJaccard.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextMap.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextMap.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextMerge.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextMerge.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextSample.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextSample.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextSpacing.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextSpacing.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextSubtract.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextSubtract.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextSummary.cpp\n",
            " create mode 100644 bedtools2/src/utils/Contexts/ContextSummary.h\n",
            " create mode 100644 bedtools2/src/utils/Contexts/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/Fasta/Fasta.cpp\n",
            " create mode 100644 bedtools2/src/utils/Fasta/Fasta.h\n",
            " create mode 100644 bedtools2/src/utils/Fasta/LargeFileSupport.h\n",
            " create mode 100644 bedtools2/src/utils/Fasta/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/Fasta/split.cpp\n",
            " create mode 100644 bedtools2/src/utils/Fasta/split.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/BamFileReader.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/BamFileReader.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/BufferedStreamMgr.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/BufferedStreamMgr.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/FileReader.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/FileReader.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/InputStreamMgr.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/InputStreamMgr.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/SingleLineDelimTextFileReader.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/SingleLineDelimTextFileReader.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/SingleLineDelimTransferBuffer.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileReaders/SingleLineDelimTransferBuffer.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileRecordMergeMgr.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileRecordMergeMgr.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileRecordMgr.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/FileRecordMgr.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/BamRecord.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/BamRecord.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Bed12Interval.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Bed12Interval.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Bed3Interval.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Bed3Interval.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Bed4Interval.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Bed4Interval.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Bed5Interval.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Bed5Interval.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Bed6Interval.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Bed6Interval.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/BedGraphInterval.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/BedGraphInterval.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/BedPlusInterval.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/BedPlusInterval.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/BlockMgr.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/BlockMgr.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/EmptyRecord.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/EmptyRecord.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/GffPlusRecord.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/GffPlusRecord.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/GffRecord.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/GffRecord.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/NoPosPlusRecord.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/NoPosPlusRecord.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/PlusFields.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/PlusFields.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Record.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/Record.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/RecordKeyList.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/RecordKeyList.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/RecordKeyVector.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/RecordKeyVector.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/RecordList.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/RecordList.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/RecordMgr.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/RecordMgr.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/StrandQueue.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/StrandQueue.h\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/VcfRecord.cpp\n",
            " create mode 100644 bedtools2/src/utils/FileRecordTools/Records/VcfRecord.h\n",
            " create mode 100644 bedtools2/src/utils/GenomeFile/GenomeFile.cpp\n",
            " create mode 100644 bedtools2/src/utils/GenomeFile/GenomeFile.h\n",
            " create mode 100644 bedtools2/src/utils/GenomeFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/GenomeFile/NewGenomeFile.cpp\n",
            " create mode 100644 bedtools2/src/utils/GenomeFile/NewGenomeFile.h\n",
            " create mode 100644 bedtools2/src/utils/KeyListOps/KeyListOps.cpp\n",
            " create mode 100644 bedtools2/src/utils/KeyListOps/KeyListOps.h\n",
            " create mode 100644 bedtools2/src/utils/KeyListOps/KeyListOpsMethods.cpp\n",
            " create mode 100644 bedtools2/src/utils/KeyListOps/KeyListOpsMethods.h\n",
            " create mode 100644 bedtools2/src/utils/KeyListOps/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/NewChromsweep/CloseSweep.cpp\n",
            " create mode 100644 bedtools2/src/utils/NewChromsweep/CloseSweep.h\n",
            " create mode 100644 bedtools2/src/utils/NewChromsweep/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/NewChromsweep/NewChromsweep.cpp\n",
            " create mode 100644 bedtools2/src/utils/NewChromsweep/NewChromsweep.h\n",
            " create mode 100644 bedtools2/src/utils/Point/Point.h\n",
            " create mode 100644 bedtools2/src/utils/RecordOutputMgr/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/RecordOutputMgr/RecordOutputMgr.cpp\n",
            " create mode 100644 bedtools2/src/utils/RecordOutputMgr/RecordOutputMgr.h\n",
            " create mode 100644 bedtools2/src/utils/ToolBase/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/ToolBase/ToolBase.cpp\n",
            " create mode 100644 bedtools2/src/utils/ToolBase/ToolBase.h\n",
            " create mode 100644 bedtools2/src/utils/VectorOps/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/VectorOps/VectorOps.cpp\n",
            " create mode 100644 bedtools2/src/utils/VectorOps/VectorOps.h\n",
            " create mode 100644 bedtools2/src/utils/bedFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/bedFile/bedFile.cpp\n",
            " create mode 100644 bedtools2/src/utils/bedFile/bedFile.h\n",
            " create mode 100644 bedtools2/src/utils/bedFilePE/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/bedFilePE/bedFilePE.cpp\n",
            " create mode 100644 bedtools2/src/utils/bedFilePE/bedFilePE.h\n",
            " create mode 100644 bedtools2/src/utils/bedGraphFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/bedGraphFile/bedGraphFile.cpp\n",
            " create mode 100644 bedtools2/src/utils/bedGraphFile/bedGraphFile.h\n",
            " create mode 100644 bedtools2/src/utils/chromsweep/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/chromsweep/chromsweep.cpp\n",
            " create mode 100644 bedtools2/src/utils/chromsweep/chromsweep.h\n",
            " create mode 100644 bedtools2/src/utils/driver/BedtoolsDriver.cpp\n",
            " create mode 100644 bedtools2/src/utils/driver/BedtoolsDriver.h\n",
            " create mode 100644 bedtools2/src/utils/driver/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/fileType/FileRecordTypeChecker.cpp\n",
            " create mode 100644 bedtools2/src/utils/fileType/FileRecordTypeChecker.h\n",
            " create mode 100644 bedtools2/src/utils/fileType/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/fileType/fileType.cpp\n",
            " create mode 100644 bedtools2/src/utils/fileType/fileType.h\n",
            " create mode 100644 bedtools2/src/utils/general/BTlist.h\n",
            " create mode 100644 bedtools2/src/utils/general/BedtoolsTypes.h\n",
            " create mode 100644 bedtools2/src/utils/general/CommonHelp.cpp\n",
            " create mode 100644 bedtools2/src/utils/general/CommonHelp.h\n",
            " create mode 100644 bedtools2/src/utils/general/CompressionTools.cpp\n",
            " create mode 100644 bedtools2/src/utils/general/CompressionTools.h\n",
            " create mode 100644 bedtools2/src/utils/general/ErrorMsg.cpp\n",
            " create mode 100644 bedtools2/src/utils/general/ErrorMsg.h\n",
            " create mode 100644 bedtools2/src/utils/general/FreeList.h\n",
            " create mode 100644 bedtools2/src/utils/general/InflateStreamBuf.h\n",
            " create mode 100644 bedtools2/src/utils/general/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/general/ParseTools.cpp\n",
            " create mode 100644 bedtools2/src/utils/general/ParseTools.h\n",
            " create mode 100644 bedtools2/src/utils/general/PushBackStreamBuf.cpp\n",
            " create mode 100644 bedtools2/src/utils/general/PushBackStreamBuf.h\n",
            " create mode 100644 bedtools2/src/utils/general/QuickString.cpp\n",
            " create mode 100644 bedtools2/src/utils/general/QuickString.h\n",
            " create mode 100644 bedtools2/src/utils/general/Tokenizer.cpp\n",
            " create mode 100644 bedtools2/src/utils/general/Tokenizer.h\n",
            " create mode 100644 bedtools2/src/utils/gzstream/COPYING.LIB\n",
            " create mode 100644 bedtools2/src/utils/gzstream/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/gzstream/README\n",
            " create mode 100644 bedtools2/src/utils/gzstream/gzstream.C\n",
            " create mode 100644 bedtools2/src/utils/gzstream/gzstream.h\n",
            " create mode 100644 bedtools2/src/utils/gzstream/version\n",
            " create mode 100644 bedtools2/src/utils/htslib/.gitattributes\n",
            " create mode 100644 bedtools2/src/utils/htslib/.gitignore\n",
            " create mode 100644 bedtools2/src/utils/htslib/INSTALL\n",
            " create mode 100644 bedtools2/src/utils/htslib/LICENSE\n",
            " create mode 100644 bedtools2/src/utils/htslib/Makefile\n",
            " create mode 100644 bedtools2/src/utils/htslib/NEWS\n",
            " create mode 100644 bedtools2/src/utils/htslib/README\n",
            " create mode 100644 bedtools2/src/utils/htslib/README.md\n",
            " create mode 100644 bedtools2/src/utils/htslib/bcf_sr_sort.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/bcf_sr_sort.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/bgzf.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/bgzip.1\n",
            " create mode 100644 bedtools2/src/utils/htslib/bgzip.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/config.mk.in\n",
            " create mode 100644 bedtools2/src/utils/htslib/configure.ac\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_codecs.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_codecs.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_decode.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_decode.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_encode.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_encode.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_external.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_index.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_index.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_io.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_io.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_samtools.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_samtools.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_stats.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_stats.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/cram_structs.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/files.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/mFILE.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/mFILE.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/misc.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/open_trace_file.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/open_trace_file.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/os.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/pooled_alloc.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/pooled_alloc.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/rANS_byte.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/rANS_static.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/rANS_static.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/sam_header.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/sam_header.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/string_alloc.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/cram/string_alloc.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/errmod.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/faidx.5\n",
            " create mode 100644 bedtools2/src/utils/htslib/faidx.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/hfile.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/hfile_gcs.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/hfile_internal.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/hfile_libcurl.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/hfile_net.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/hfile_s3.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/hts.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/hts_internal.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/hts_os.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/htsfile.1\n",
            " create mode 100644 bedtools2/src/utils/htslib/htsfile.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib.mk\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib.pc.in\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/bgzf.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/cram.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/faidx.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/hfile.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/hts.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/hts_defs.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/hts_endian.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/hts_log.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/hts_os.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/kbitset.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/kfunc.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/khash.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/khash_str2int.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/klist.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/knetfile.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/kseq.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/ksort.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/kstring.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/regidx.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/sam.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/synced_bcf_reader.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/tbx.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/thread_pool.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/vcf.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/vcf_sweep.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib/vcfutils.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/htslib_vars.mk\n",
            " create mode 100644 bedtools2/src/utils/htslib/kfunc.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/knetfile.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/kstring.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/m4/hts_prog_cc_warnings.m4\n",
            " create mode 100644 bedtools2/src/utils/htslib/md5.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/multipart.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/os/lzma_stub.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/os/rand.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/plugin.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/probaln.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/realn.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/regidx.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/sam.5\n",
            " create mode 100644 bedtools2/src/utils/htslib/sam.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/synced_bcf_reader.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/tabix.1\n",
            " create mode 100644 bedtools2/src/utils/htslib/tabix.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/tbx.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/textutils.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/textutils_internal.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/thread_pool.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/thread_pool_internal.h\n",
            " create mode 100644 bedtools2/src/utils/htslib/vcf.5\n",
            " create mode 100644 bedtools2/src/utils/htslib/vcf.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/vcf_sweep.c\n",
            " create mode 100644 bedtools2/src/utils/htslib/vcfutils.c\n",
            " create mode 100755 bedtools2/src/utils/htslib/version.sh\n",
            " create mode 100644 bedtools2/src/utils/lineFileUtilities/lineFileUtilities.h\n",
            " create mode 100644 bedtools2/src/utils/sequenceUtilities/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/sequenceUtilities/sequenceUtils.cpp\n",
            " create mode 100644 bedtools2/src/utils/sequenceUtilities/sequenceUtils.h\n",
            " create mode 100644 bedtools2/src/utils/stringUtilities/stringUtilities.h\n",
            " create mode 100644 bedtools2/src/utils/tabFile/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/tabFile/tabFile.cpp\n",
            " create mode 100644 bedtools2/src/utils/tabFile/tabFile.h\n",
            " create mode 100644 bedtools2/src/utils/version/Makefile.frag\n",
            " create mode 100644 bedtools2/src/utils/version/version.cpp\n",
            " create mode 100644 bedtools2/src/utils/version/version.h\n",
            " create mode 100644 bedtools2/src/utils/version/version_release.txt\n",
            " create mode 100644 bedtools2/src/windowBed/Makefile.frag\n",
            " create mode 100644 bedtools2/src/windowBed/windowBed.cpp\n",
            " create mode 100644 bedtools2/src/windowBed/windowBed.h\n",
            " create mode 100644 bedtools2/src/windowBed/windowMain.cpp\n",
            " create mode 100644 bedtools2/src/windowMaker/Makefile.frag\n",
            " create mode 100644 bedtools2/src/windowMaker/windowMaker.cpp\n",
            " create mode 100644 bedtools2/src/windowMaker/windowMaker.h\n",
            " create mode 100644 bedtools2/src/windowMaker/windowMakerMain.cpp\n",
            " create mode 100644 bedtools2/test/bamtobed/extra-long-header.sam.gz\n",
            " create mode 100644 bedtools2/test/bamtobed/numeric_tag.sam\n",
            " create mode 100644 bedtools2/test/bamtobed/one_block.sam\n",
            " create mode 100644 bedtools2/test/bamtobed/test-bamtobed.sh\n",
            " create mode 100644 bedtools2/test/bamtobed/three_blocks.sam\n",
            " create mode 100644 bedtools2/test/bamtobed/two_blocks.sam\n",
            " create mode 100644 bedtools2/test/bamtobed/two_blocks_w_D.sam\n",
            " create mode 100644 bedtools2/test/bamtofastq/golden.fq\n",
            " create mode 100644 bedtools2/test/bamtofastq/golden.fq2\n",
            " create mode 100755 bedtools2/test/bamtofastq/test-bamtofastq.sh\n",
            " create mode 100755 bedtools2/test/bamtofastq/test.sam\n",
            " create mode 100644 bedtools2/test/bamtofastq/test2.sam\n",
            " create mode 100644 bedtools2/test/bed12tobed6/bed6.bed\n",
            " create mode 100644 bedtools2/test/bed12tobed6/one_blocks.bed\n",
            " create mode 100644 bedtools2/test/bed12tobed6/test-bed12tobed6.sh\n",
            " create mode 100644 bedtools2/test/bed12tobed6/three_blocks.bed\n",
            " create mode 100644 bedtools2/test/bed12tobed6/two_blocks.bed\n",
            " create mode 100644 bedtools2/test/bedtobam/chrsize.tmp\n",
            " create mode 100644 bedtools2/test/bedtobam/test-bedtobam.sh\n",
            " create mode 100644 bedtools2/test/bigchroms/abig.bed\n",
            " create mode 100644 bedtools2/test/bigchroms/big4.bed\n",
            " create mode 100644 bedtools2/test/bigchroms/big4c.bed\n",
            " create mode 100644 bedtools2/test/bigchroms/bigx.bed\n",
            " create mode 100644 bedtools2/test/bigchroms/make-big-chrom.py\n",
            " create mode 100644 bedtools2/test/bigchroms/test-bigchroms.sh\n",
            " create mode 100644 bedtools2/test/closest/a.bed\n",
            " create mode 100644 bedtools2/test/closest/a.names.bed\n",
            " create mode 100644 bedtools2/test/closest/a2.bed\n",
            " create mode 100644 bedtools2/test/closest/b-one-bp-closer.bed\n",
            " create mode 100644 bedtools2/test/closest/b.bed\n",
            " create mode 100644 bedtools2/test/closest/b.names.bed\n",
            " create mode 100644 bedtools2/test/closest/b2.bed\n",
            " create mode 100644 bedtools2/test/closest/bug157_a.bed\n",
            " create mode 100644 bedtools2/test/closest/bug157_b.bed\n",
            " create mode 100644 bedtools2/test/closest/bug281_a.flip.medium.bed\n",
            " create mode 100644 bedtools2/test/closest/bug281_a.medium.bed\n",
            " create mode 100644 bedtools2/test/closest/bug281_b.bed\n",
            " create mode 100644 bedtools2/test/closest/bug281_b.flip.bed\n",
            " create mode 100644 bedtools2/test/closest/close-a.bed\n",
            " create mode 100644 bedtools2/test/closest/close-b.bed\n",
            " create mode 100644 bedtools2/test/closest/d.bed\n",
            " create mode 100644 bedtools2/test/closest/d4.bed\n",
            " create mode 100644 bedtools2/test/closest/d_d1F.bed\n",
            " create mode 100644 bedtools2/test/closest/d_d1R.bed\n",
            " create mode 100644 bedtools2/test/closest/d_d2F.bed\n",
            " create mode 100644 bedtools2/test/closest/d_d2R.bed\n",
            " create mode 100644 bedtools2/test/closest/d_id.bed\n",
            " create mode 100644 bedtools2/test/closest/d_iu.bed\n",
            " create mode 100644 bedtools2/test/closest/d_q1.bed\n",
            " create mode 100644 bedtools2/test/closest/d_q2.bed\n",
            " create mode 100644 bedtools2/test/closest/dmr.bed\n",
            " create mode 100644 bedtools2/test/closest/islands.2.bed\n",
            " create mode 100644 bedtools2/test/closest/islands.bed\n",
            " create mode 100644 bedtools2/test/closest/kclosest/bug471_a.bed\n",
            " create mode 100644 bedtools2/test/closest/kclosest/bug471_b.bed\n",
            " create mode 100644 bedtools2/test/closest/kclosest/d1.bed\n",
            " create mode 100644 bedtools2/test/closest/kclosest/d2.bed\n",
            " create mode 100644 bedtools2/test/closest/kclosest/d3.bed\n",
            " create mode 100644 bedtools2/test/closest/kclosest/q1.bed\n",
            " create mode 100644 bedtools2/test/closest/kclosest/q2.bed\n",
            " create mode 100644 bedtools2/test/closest/kclosest/test-kclosest.sh\n",
            " create mode 100644 bedtools2/test/closest/mdb1.bed\n",
            " create mode 100644 bedtools2/test/closest/mdb2.bed\n",
            " create mode 100644 bedtools2/test/closest/mdb3.bed\n",
            " create mode 100644 bedtools2/test/closest/mq1.bed\n",
            " create mode 100644 bedtools2/test/closest/null_a.bed\n",
            " create mode 100644 bedtools2/test/closest/null_b.bed\n",
            " create mode 100644 bedtools2/test/closest/null_c.bed\n",
            " create mode 100644 bedtools2/test/closest/shores.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/alpha_all.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/alpha_missing.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/db1_gls.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/db1_leadingZero.txt\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/db1_noChr.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/db1_num.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/db2_num.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/db2_numBackwards.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/db3_numBackwards.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/num_all.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/num_missing.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/q1_gls.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/q1_num.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/q1a_num.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/sdb1.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/sq1.bed\n",
            " create mode 100644 bedtools2/test/closest/sortAndNaming/test-sort-and-naming.sh\n",
            " create mode 100644 bedtools2/test/closest/strand-test-a.bed\n",
            " create mode 100644 bedtools2/test/closest/strand-test-b.bed\n",
            " create mode 100644 bedtools2/test/closest/strand-test-c.bed\n",
            " create mode 100644 bedtools2/test/closest/strand-test-d.bed\n",
            " create mode 100644 bedtools2/test/closest/test-closest.sh\n",
            " create mode 100644 bedtools2/test/closest/tfbs.bed\n",
            " create mode 100644 bedtools2/test/cluster/in.bed\n",
            " create mode 100644 bedtools2/test/cluster/test-cluster.sh\n",
            " create mode 100644 bedtools2/test/complement/issue_503.bed\n",
            " create mode 100644 bedtools2/test/complement/issue_503.genome\n",
            " create mode 100644 bedtools2/test/complement/test-complement.sh\n",
            " create mode 100644 bedtools2/test/coverage/a.bed\n",
            " create mode 100644 bedtools2/test/coverage/b.bed\n",
            " create mode 100644 bedtools2/test/coverage/c.bed\n",
            " create mode 100644 bedtools2/test/coverage/chr_0-100.bed\n",
            " create mode 100644 bedtools2/test/coverage/one_block.bam\n",
            " create mode 100644 bedtools2/test/coverage/one_block.bam.bai\n",
            " create mode 100644 bedtools2/test/coverage/one_block.sam\n",
            " create mode 100644 bedtools2/test/coverage/sam-w-del.bam\n",
            " create mode 100644 bedtools2/test/coverage/sam-w-del.bam.bai\n",
            " create mode 100644 bedtools2/test/coverage/sam-w-del.sam\n",
            " create mode 100755 bedtools2/test/coverage/test-coverage.sh\n",
            " create mode 100644 bedtools2/test/coverage/three_blocks.bam\n",
            " create mode 100644 bedtools2/test/coverage/three_blocks.bam.bai\n",
            " create mode 100644 bedtools2/test/coverage/three_blocks.sam\n",
            " create mode 100644 bedtools2/test/coverage/three_blocks_match.bam\n",
            " create mode 100644 bedtools2/test/coverage/three_blocks_match.bed\n",
            " create mode 100644 bedtools2/test/coverage/three_blocks_match_1bp.bed\n",
            " create mode 100644 bedtools2/test/coverage/three_blocks_nomatch.bed\n",
            " create mode 100644 bedtools2/test/coverage/two_blocks.bam\n",
            " create mode 100644 bedtools2/test/coverage/two_blocks.bam.bai\n",
            " create mode 100644 bedtools2/test/coverage/two_blocks.sam\n",
            " create mode 100644 bedtools2/test/coverage/x.bed\n",
            " create mode 100644 bedtools2/test/coverage/y.bed\n",
            " create mode 100644 bedtools2/test/expand/expand.txt\n",
            " create mode 100644 bedtools2/test/expand/test-expand.sh\n",
            " create mode 100644 bedtools2/test/fisher/README.md\n",
            " create mode 100644 bedtools2/test/fisher/a.bed\n",
            " create mode 100644 bedtools2/test/fisher/a_merge.bed\n",
            " create mode 100644 bedtools2/test/fisher/b.bed\n",
            " create mode 100644 bedtools2/test/fisher/cmp.sh\n",
            " create mode 100644 bedtools2/test/fisher/dm6.fai\n",
            " create mode 100644 bedtools2/test/fisher/get.sh\n",
            " create mode 100644 bedtools2/test/fisher/plot.py\n",
            " create mode 100644 bedtools2/test/fisher/shuf.sh\n",
            " create mode 100644 bedtools2/test/fisher/sim.py\n",
            " create mode 100644 bedtools2/test/fisher/t.500.genome\n",
            " create mode 100644 bedtools2/test/fisher/t.60.genome\n",
            " create mode 100644 bedtools2/test/fisher/test-fisher.sh\n",
            " create mode 100644 bedtools2/test/fisher/test.bed\n",
            " create mode 100644 bedtools2/test/fisher/tumor.gff\n",
            " create mode 100644 bedtools2/test/flank/a.bed\n",
            " create mode 100644 bedtools2/test/flank/test-flank.sh\n",
            " create mode 100644 bedtools2/test/flank/tiny.genome\n",
            " create mode 100644 bedtools2/test/general/a.chromheader.bed\n",
            " create mode 100644 bedtools2/test/general/a.trackheader.bed\n",
            " create mode 100644 bedtools2/test/general/empty.bed\n",
            " create mode 100644 bedtools2/test/general/hg19.fa.fai\n",
            " create mode 100644 bedtools2/test/general/non-empty.bed.gz\n",
            " create mode 100644 bedtools2/test/general/t.bed\n",
            " create mode 100644 bedtools2/test/general/test-general.sh\n",
            " create mode 100644 bedtools2/test/genomecov/chip.sam\n",
            " create mode 100644 bedtools2/test/genomecov/empty.bam\n",
            " create mode 100644 bedtools2/test/genomecov/empty.cram\n",
            " create mode 100644 bedtools2/test/genomecov/genome.txt\n",
            " create mode 100644 bedtools2/test/genomecov/one_block.sam\n",
            " create mode 100644 bedtools2/test/genomecov/pair-chip.sam\n",
            " create mode 100644 bedtools2/test/genomecov/sam-w-del.sam\n",
            " create mode 100755 bedtools2/test/genomecov/test-genomecov.sh\n",
            " create mode 100644 bedtools2/test/genomecov/test_ref.fa\n",
            " create mode 100644 bedtools2/test/genomecov/test_ref.fa.fai\n",
            " create mode 100644 bedtools2/test/genomecov/three_blocks.sam\n",
            " create mode 100644 bedtools2/test/genomecov/three_blocks_match.bed\n",
            " create mode 100644 bedtools2/test/genomecov/three_blocks_match_1bp.bed\n",
            " create mode 100644 bedtools2/test/genomecov/three_blocks_nomatch.bed\n",
            " create mode 100644 bedtools2/test/genomecov/two_blocks.sam\n",
            " create mode 100644 bedtools2/test/genomecov/y.bam\n",
            " create mode 100644 bedtools2/test/genomecov/y.bed\n",
            " create mode 100644 bedtools2/test/getfasta/blocks.bed\n",
            " create mode 100644 bedtools2/test/getfasta/t.bed\n",
            " create mode 100644 bedtools2/test/getfasta/t.fa\n",
            " create mode 100644 bedtools2/test/getfasta/t.fa.fai\n",
            " create mode 100644 bedtools2/test/getfasta/t_fH.fa\n",
            " create mode 100644 bedtools2/test/getfasta/t_fH.fa.fai\n",
            " create mode 100644 bedtools2/test/getfasta/test-getfasta.sh\n",
            " create mode 100644 bedtools2/test/getfasta/test.iupac.bed\n",
            " create mode 100644 bedtools2/test/getfasta/test.iupac.fa\n",
            " create mode 100644 bedtools2/test/groupby/a_vcfSVtest.vcf\n",
            " create mode 100644 bedtools2/test/groupby/bug569_problem.txt\n",
            " create mode 100644 bedtools2/test/groupby/gdc.bam\n",
            " create mode 100644 bedtools2/test/groupby/noPosvalues.header.bed\n",
            " create mode 100644 bedtools2/test/groupby/test-groupby.sh\n",
            " create mode 100644 bedtools2/test/groupby/test.bed\n",
            " create mode 100644 bedtools2/test/groupby/values3.7fields.header.bed\n",
            " create mode 100644 bedtools2/test/groupby/values3.header.bed\n",
            " create mode 100644 bedtools2/test/groupby/values3.no_header.bed\n",
            " create mode 100644 bedtools2/test/groupby/values3.unmarked_header.bed\n",
            " create mode 100644 bedtools2/test/groupby/values3.unmarked_header.bed.2\n",
            " create mode 100644 bedtools2/test/groupby/values3_case.header.bed\n",
            " create mode 100644 bedtools2/test/intersect/a.bam\n",
            " create mode 100644 bedtools2/test/intersect/a.bed\n",
            " create mode 100644 bedtools2/test/intersect/a.cram\n",
            " create mode 100644 bedtools2/test/intersect/a.issue311.bed\n",
            " create mode 100644 bedtools2/test/intersect/aVSb.bam\n",
            " create mode 100644 bedtools2/test/intersect/aWithHeaderVsB.txt\n",
            " create mode 100644 bedtools2/test/intersect/a_bgzipped.bed.gz\n",
            " create mode 100644 bedtools2/test/intersect/a_gzipped.bed.gz\n",
            " create mode 100644 bedtools2/test/intersect/a_testZeroLen.bed\n",
            " create mode 100644 bedtools2/test/intersect/a_vcfSVtest.vcf\n",
            " create mode 100644 bedtools2/test/intersect/a_withLargeHeader.bed\n",
            " create mode 100644 bedtools2/test/intersect/a_withLargeHeader_bgzipped.bed.gz\n",
            " create mode 100644 bedtools2/test/intersect/a_withLargeHeader_gzipped.bed.gz\n",
            " create mode 100644 bedtools2/test/intersect/a_with_bothUnmapped.bam\n",
            " create mode 100644 bedtools2/test/intersect/b.bed\n",
            " create mode 100644 bedtools2/test/intersect/b.cram\n",
            " create mode 100644 bedtools2/test/intersect/b.issue311.gff\n",
            " create mode 100644 bedtools2/test/intersect/b.issue311.vcf\n",
            " create mode 100644 bedtools2/test/intersect/b_testZeroLen.bed\n",
            " create mode 100644 bedtools2/test/intersect/b_vcfSVtest.vcf\n",
            " create mode 100644 bedtools2/test/intersect/bed3.bed\n",
            " create mode 100644 bedtools2/test/intersect/bed4.bed\n",
            " create mode 100644 bedtools2/test/intersect/bed5.bed\n",
            " create mode 100644 bedtools2/test/intersect/bed6.bed\n",
            " create mode 100644 bedtools2/test/intersect/bed6.strand.bed\n",
            " create mode 100644 bedtools2/test/intersect/bed6.strand2.bed\n",
            " create mode 100644 bedtools2/test/intersect/bedplus.bed\n",
            " create mode 100644 bedtools2/test/intersect/blocks.bed12\n",
            " create mode 100644 bedtools2/test/intersect/bug150_a.bed\n",
            " create mode 100644 bedtools2/test/intersect/bug150_b.bed\n",
            " create mode 100644 bedtools2/test/intersect/bug167_strandSweep.bed\n",
            " create mode 100644 bedtools2/test/intersect/bug187_a.bed\n",
            " create mode 100644 bedtools2/test/intersect/bug187_b.bed\n",
            " create mode 100644 bedtools2/test/intersect/bug223_d.vcf\n",
            " create mode 100644 bedtools2/test/intersect/bug223_e.vcf\n",
            " create mode 100644 bedtools2/test/intersect/bug223_f.vcf\n",
            " create mode 100644 bedtools2/test/intersect/bug223_sv1_a.vcf\n",
            " create mode 100644 bedtools2/test/intersect/bug223_sv1_b.vcf\n",
            " create mode 100644 bedtools2/test/intersect/bug44_a.vcf.gz\n",
            " create mode 100644 bedtools2/test/intersect/bug44_b.bed\n",
            " create mode 100644 bedtools2/test/intersect/c.bed\n",
            " create mode 100644 bedtools2/test/intersect/chromOrderA.bed\n",
            " create mode 100644 bedtools2/test/intersect/chromOrderB.bed\n",
            " create mode 100644 bedtools2/test/intersect/chromsOutOfOrder.bed\n",
            " create mode 100644 bedtools2/test/intersect/d.bed\n",
            " create mode 100644 bedtools2/test/intersect/dosLineCharWithExtraTab_b.bed\n",
            " create mode 100644 bedtools2/test/intersect/dosLineChar_a.bed\n",
            " create mode 100644 bedtools2/test/intersect/e.bed\n",
            " create mode 100644 bedtools2/test/intersect/exons.issue750.bed\n",
            " create mode 100644 bedtools2/test/intersect/foo\n",
            " create mode 100644 bedtools2/test/intersect/gdc.bam\n",
            " create mode 100644 bedtools2/test/intersect/gdc.gff\n",
            " create mode 100644 bedtools2/test/intersect/gdc_exp\n",
            " create mode 100644 bedtools2/test/intersect/gdc_one.bed\n",
            " create mode 100644 bedtools2/test/intersect/headerOnly.vcf\n",
            " create mode 100644 bedtools2/test/intersect/human.hg19.genome\n",
            " create mode 100644 bedtools2/test/intersect/human.hg19.vSort.genome\n",
            " create mode 100644 bedtools2/test/intersect/issue_773_b.bed\n",
            " create mode 100644 bedtools2/test/intersect/issue_773_y.bed\n",
            " create mode 100644 bedtools2/test/intersect/j1.bed\n",
            " create mode 100644 bedtools2/test/intersect/jim.bed\n",
            " create mode 100644 bedtools2/test/intersect/jim.vcf\n",
            " create mode 100644 bedtools2/test/intersect/large_a.bed\n",
            " create mode 100644 bedtools2/test/intersect/large_b.bed\n",
            " create mode 100644 bedtools2/test/intersect/mapped_and_unmapped.sam\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/d1.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/d2.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/d3.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/d4.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/d5.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/d6.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/d7.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/d8.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/d9.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/g.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/query.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/query2.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/query3.bed\n",
            " create mode 100644 bedtools2/test/intersect/multi_intersect/test-multi_intersect.sh\n",
            " create mode 100755 bedtools2/test/intersect/new_test-intersect.sh\n",
            " create mode 100755 bedtools2/test/intersect/new_test-intersect.sh~\n",
            " create mode 100644 bedtools2/test/intersect/nonamecheck_a.bed\n",
            " create mode 100644 bedtools2/test/intersect/nonamecheck_b.bed\n",
            " create mode 100644 bedtools2/test/intersect/notexthdr.bam\n",
            " create mode 100644 bedtools2/test/intersect/null_a.bed\n",
            " create mode 100644 bedtools2/test/intersect/null_b.bed\n",
            " create mode 100644 bedtools2/test/intersect/null_c.bed\n",
            " create mode 100644 bedtools2/test/intersect/oneRecordNoNewline.bed\n",
            " create mode 100644 bedtools2/test/intersect/oneUnmapped.bam\n",
            " create mode 100644 bedtools2/test/intersect/one_block.sam\n",
            " create mode 100755 bedtools2/test/intersect/performanceTest.sh\n",
            " create mode 100644 bedtools2/test/intersect/queryUnmappedMateMappedCoordsInvalid.bam\n",
            " create mode 100644 bedtools2/test/intersect/recordsOutOfOrder.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/alpha_all.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/alpha_missing.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/bigTests/2.21_500Kvs500K.txt\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/bigTests/2.21_5v5.txt\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/bigTests/2.21_chr1.txt\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/bigTests/curr500Kvs500K.txt\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/bigTests/curr5v5.txt\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/bigTests/currChr1.txt\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/bigTests/db500K.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/bigTests/dbchr1.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/bigTests/diffs.txt\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/bigTests/q500K.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/bigTests/qchr1.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/db1_gls.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/db1_leadingZero.txt\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/db1_noChr.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/db1_num.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/db2_num.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/db2_numBackwards.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/db3_numBackwards.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/diffs.txt\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/exp1\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/num_all.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/num_missing.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/q1_gls.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/q1_num.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/q1a_num.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/sdb1.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/sq1.bed\n",
            " create mode 100644 bedtools2/test/intersect/sortAndNaming/test-sort-and-naming.sh\n",
            " create mode 100644 bedtools2/test/intersect/split.issue750.bam\n",
            " create mode 100644 bedtools2/test/intersect/split.issue750.sam\n",
            " create mode 100644 bedtools2/test/intersect/splitBug155_a.bed\n",
            " create mode 100644 bedtools2/test/intersect/splitBug155_b.bed\n",
            " create mode 100644 bedtools2/test/intersect/svlen.vcf\n",
            " create mode 100644 bedtools2/test/intersect/test-intersect.sh\n",
            " create mode 100644 bedtools2/test/intersect/test_ref.fa\n",
            " create mode 100644 bedtools2/test/intersect/test_ref.fa.fai\n",
            " create mode 100644 bedtools2/test/intersect/three_blocks.sam\n",
            " create mode 100644 bedtools2/test/intersect/three_blocks_match.bam\n",
            " create mode 100644 bedtools2/test/intersect/three_blocks_match.bed\n",
            " create mode 100644 bedtools2/test/intersect/three_blocks_match_1bp.bed\n",
            " create mode 100644 bedtools2/test/intersect/three_blocks_nomatch.bed\n",
            " create mode 100644 bedtools2/test/intersect/two_blocks.sam\n",
            " create mode 100644 bedtools2/test/intersect/two_blocks_partial.bam\n",
            " create mode 100644 bedtools2/test/intersect/two_blocks_partial.bam.bai\n",
            " create mode 100644 bedtools2/test/intersect/two_blocks_partial.bed\n",
            " create mode 100644 bedtools2/test/intersect/x.bam\n",
            " create mode 100644 bedtools2/test/intersect/x.bed\n",
            " create mode 100644 bedtools2/test/intersect/y.bed\n",
            " create mode 100644 bedtools2/test/jaccard/a.bam\n",
            " create mode 100644 bedtools2/test/jaccard/a.bed\n",
            " create mode 100644 bedtools2/test/jaccard/a645.bed\n",
            " create mode 100644 bedtools2/test/jaccard/aMixedStrands.bed\n",
            " create mode 100644 bedtools2/test/jaccard/b.bed\n",
            " create mode 100644 bedtools2/test/jaccard/b645.bed\n",
            " create mode 100644 bedtools2/test/jaccard/bMixedStrands.bed\n",
            " create mode 100644 bedtools2/test/jaccard/c.bed\n",
            " create mode 100644 bedtools2/test/jaccard/e.bed\n",
            " create mode 100644 bedtools2/test/jaccard/long.bed\n",
            " create mode 100644 bedtools2/test/jaccard/short.bed\n",
            " create mode 100644 bedtools2/test/jaccard/test-jaccard.sh\n",
            " create mode 100644 bedtools2/test/jaccard/three_blocks_match.bam\n",
            " create mode 100644 bedtools2/test/jaccard/three_blocks_match.bed\n",
            " create mode 100644 bedtools2/test/makewindows/a.19bp.bed\n",
            " create mode 100644 bedtools2/test/makewindows/a.20bp.bed\n",
            " create mode 100644 bedtools2/test/makewindows/a.33bp.bed\n",
            " create mode 100644 bedtools2/test/makewindows/input.bed\n",
            " create mode 100644 bedtools2/test/makewindows/test-makewindows.sh\n",
            " create mode 100644 bedtools2/test/map/.RData\n",
            " create mode 100644 bedtools2/test/map/.Rhistory\n",
            " create mode 100644 bedtools2/test/map/a.vsorted.bed\n",
            " create mode 100644 bedtools2/test/map/b.vsorted.bed\n",
            " create mode 100644 bedtools2/test/map/bug262_a.bed\n",
            " create mode 100644 bedtools2/test/map/bug262_b.bed\n",
            " create mode 100644 bedtools2/test/map/d.bed\n",
            " create mode 100644 bedtools2/test/map/fullFields.bam\n",
            " create mode 100644 bedtools2/test/map/genome\n",
            " create mode 100644 bedtools2/test/map/ivls.bed\n",
            " create mode 100644 bedtools2/test/map/ivls2.bed\n",
            " create mode 100644 bedtools2/test/map/t.bed\n",
            " create mode 100644 bedtools2/test/map/test-map.sh\n",
            " create mode 100644 bedtools2/test/map/test.gff\n",
            " create mode 100644 bedtools2/test/map/test.vcf\n",
            " create mode 100644 bedtools2/test/map/three_blocks_match.bed\n",
            " create mode 100644 bedtools2/test/map/three_blocks_nomatch.bed\n",
            " create mode 100644 bedtools2/test/map/values.bam\n",
            " create mode 100644 bedtools2/test/map/values.bed\n",
            " create mode 100644 bedtools2/test/map/values2.bed\n",
            " create mode 100644 bedtools2/test/map/values3.bed\n",
            " create mode 100644 bedtools2/test/map/values4.bed\n",
            " create mode 100644 bedtools2/test/map/values5.bed\n",
            " create mode 100644 bedtools2/test/merge/a.bed\n",
            " create mode 100644 bedtools2/test/merge/a.full.bam\n",
            " create mode 100644 bedtools2/test/merge/a.full.bed\n",
            " create mode 100644 bedtools2/test/merge/a.gff\n",
            " create mode 100644 bedtools2/test/merge/a.names.bed\n",
            " create mode 100644 bedtools2/test/merge/b.bed\n",
            " create mode 100644 bedtools2/test/merge/bamCol10Collapse.txt\n",
            " create mode 100644 bedtools2/test/merge/bamCol11Collapse.txt\n",
            " create mode 100644 bedtools2/test/merge/bamCol1Collapse.txt\n",
            " create mode 100644 bedtools2/test/merge/bamCol3Collapse.txt\n",
            " create mode 100644 bedtools2/test/merge/bamCol4Mean.txt\n",
            " create mode 100644 bedtools2/test/merge/bamCol5Mean.txt\n",
            " create mode 100644 bedtools2/test/merge/bamCol6Collapse.txt\n",
            " create mode 100644 bedtools2/test/merge/bamCol7Collapse.txt\n",
            " create mode 100644 bedtools2/test/merge/bamCol8Mean.txt\n",
            " create mode 100644 bedtools2/test/merge/bamCol9Mean.txt\n",
            " create mode 100644 bedtools2/test/merge/bug254_c.bed\n",
            " create mode 100644 bedtools2/test/merge/bug254_d.bed\n",
            " create mode 100644 bedtools2/test/merge/bug254_e.bed\n",
            " create mode 100644 bedtools2/test/merge/expFormat.bed\n",
            " create mode 100644 bedtools2/test/merge/fullFields.bam\n",
            " create mode 100644 bedtools2/test/merge/mergeBugTest.bed\n",
            " create mode 100644 bedtools2/test/merge/mixedStrands.bed\n",
            " create mode 100644 bedtools2/test/merge/precisionTest.bed\n",
            " create mode 100644 bedtools2/test/merge/precisionTest2.bed\n",
            " create mode 100644 bedtools2/test/merge/test-merge.sh\n",
            " create mode 100644 bedtools2/test/merge/testA.vcf\n",
            " create mode 100644 bedtools2/test/merge/unsorted.bed\n",
            " create mode 100644 bedtools2/test/merge/vcfSVtest.vcf\n",
            " create mode 100644 bedtools2/test/multicov/multicov.bed\n",
            " create mode 100644 bedtools2/test/multicov/one_block.sam\n",
            " create mode 100644 bedtools2/test/multicov/test-multi.2.sam\n",
            " create mode 100644 bedtools2/test/multicov/test-multi.bed\n",
            " create mode 100644 bedtools2/test/multicov/test-multi.sam\n",
            " create mode 100644 bedtools2/test/multicov/test-multicov.sh\n",
            " create mode 100644 bedtools2/test/multicov/two_blocks.sam\n",
            " create mode 100644 bedtools2/test/negativecontrol/test-negativecontrol.sh\n",
            " create mode 100644 bedtools2/test/reldist/issue_711.a.bed\n",
            " create mode 100644 bedtools2/test/reldist/issue_711.b.bed\n",
            " create mode 100644 bedtools2/test/reldist/test-reldist.sh\n",
            " create mode 100644 bedtools2/test/sample/human.hg19.genome\n",
            " create mode 100644 bedtools2/test/sample/test-sample.sh\n",
            " create mode 100644 bedtools2/test/shift/a.bed\n",
            " create mode 100644 bedtools2/test/shift/b.bed\n",
            " create mode 100644 bedtools2/test/shift/huge.genome\n",
            " create mode 100644 bedtools2/test/shift/test-shift.sh\n",
            " create mode 100644 bedtools2/test/shift/tiny.genome\n",
            " create mode 100644 bedtools2/test/shuffle/excl.bed\n",
            " create mode 100644 bedtools2/test/shuffle/incl.bed\n",
            " create mode 100644 bedtools2/test/shuffle/incl.sizedif.bed\n",
            " create mode 100644 bedtools2/test/shuffle/simrep.bed\n",
            " create mode 100644 bedtools2/test/shuffle/test-shuffle.sh\n",
            " create mode 100644 bedtools2/test/slop/a.bed\n",
            " create mode 100644 bedtools2/test/slop/test-slop.sh\n",
            " create mode 100644 bedtools2/test/slop/tiny.genome\n",
            " create mode 100644 bedtools2/test/sort/a.bed\n",
            " create mode 100755 bedtools2/test/sort/b.bed\n",
            " create mode 100644 bedtools2/test/sort/names.txt\n",
            " create mode 100644 bedtools2/test/sort/test-sort.sh\n",
            " create mode 100644 bedtools2/test/spacing/a.bed\n",
            " create mode 100644 bedtools2/test/spacing/test-spacing.sh\n",
            " create mode 100644 bedtools2/test/split/randData.bed\n",
            " create mode 100644 bedtools2/test/split/test-split.sh\n",
            " create mode 100644 bedtools2/test/subtract/a.bed\n",
            " create mode 100644 bedtools2/test/subtract/b.bed\n",
            " create mode 100644 bedtools2/test/subtract/b2.bed\n",
            " create mode 100644 bedtools2/test/subtract/c.bed\n",
            " create mode 100644 bedtools2/test/subtract/d.bed\n",
            " create mode 100644 bedtools2/test/subtract/test-subtract.sh\n",
            " create mode 100755 bedtools2/test/test.sh\n",
            " create mode 100644 bedtools2/tutorial/Makefile\n",
            " create mode 100644 bedtools2/tutorial/answers.html\n",
            " create mode 100644 bedtools2/tutorial/answers.md\n",
            " create mode 100644 bedtools2/tutorial/bedtools.html\n",
            " create mode 100644 bedtools2/tutorial/bedtools.md\n",
            " create mode 100644 bedtools2/tutorial/bootstrap.css\n",
            " create mode 100644 bedtools2/tutorial/template.class.html\n",
            " create mode 100644 bedtools2/tutorial/template.html\n",
            " create mode 100755 bin/ace2sam\n",
            " create mode 100755 bin/blast2sam.pl\n",
            " create mode 100755 bin/bowtie2sam.pl\n",
            " create mode 100755 bin/export2sam.pl\n",
            " create mode 100755 bin/interpolate_sam.pl\n",
            " create mode 100755 bin/maq2sam-long\n",
            " create mode 100755 bin/maq2sam-short\n",
            " create mode 100755 bin/md5fa\n",
            " create mode 100755 bin/md5sum-lite\n",
            " create mode 100755 bin/novo2sam.pl\n",
            " create mode 100755 bin/plot-ampliconstats\n",
            " create mode 100755 bin/plot-bamstats\n",
            " create mode 100755 bin/psl2sam.pl\n",
            " create mode 100755 bin/sam2vcf.pl\n",
            " create mode 100755 bin/samtools\n",
            " create mode 100755 bin/samtools.pl\n",
            " create mode 100755 bin/seq_cache_populate.pl\n",
            " create mode 100755 bin/soap2sam.pl\n",
            " create mode 100755 bin/wgsim\n",
            " create mode 100755 bin/wgsim_eval.pl\n",
            " create mode 100755 bin/zoom2sam.pl\n",
            " create mode 100644 data/processed/Refseq/gcf_000004255.1_v.1.0/GCF_000004255.1_v.1.0_genomic.gene.1kbup.nov.final.fna\n",
            " create mode 100644 data/processed/Refseq/gcf_000004255.2_v.1.0/GCF_000004255.2_v.1.0_genomic.gene.1kbup.nov.final.fna\n",
            " create mode 100644 data/processed/Refseq/gcf_000471905.1_amtr1.0/GCF_000471905.1_AMTR1.0_genomic.gene.1kbup.nov.final.fna\n",
            " create mode 100644 data/processed/Refseq/gcf_000471905.2_amtr1.0/GCF_000471905.2_AMTR1.0_genomic.gene.1kbup.nov.final.fna\n",
            " create mode 100644 data/processed/Refseq/gcf_001540865.1_asm154086v1/GCF_001540865.1_ASM154086v1_genomic.gene.1kbup.nov.final.fna\n",
            " create mode 100644 data/processed/Refseq/gcf_002575655.1_aet_v4.0/GCF_002575655.1_Aet_v4.0_genomic.gene.1kbup.nov.final.fna\n",
            " create mode 100644 data/processed/Refseq/gcf_002575655.2_aet_v5.0/GCF_002575655.2_Aet_v5.0_genomic.gene.1kbup.nov.final.fna\n",
            " create mode 100644 data/processed/Refseq/gcf_003935025.1_abrus_2018/GCF_003935025.1_Abrus_2018_genomic.gene.1kbup.nov.final.fna\n",
            " create mode 100644 data/processed/Refseq/gcf_009805555.1_asm980555v1/GCF_009805555.1_ASM980555v1_genomic.gene.1kbup.nov.final.fna\n",
            " create mode 100644 data/processed/Refseq/gcf_019202715.1_maohua_mht/GCF_019202715.1_MaoHua_MHT_genomic.gene.1kbup.nov.final.fna\n",
            " create mode 100644 data/processed/Refseq/gcf_026212465.1_asm2621246v1/GCF_026212465.1_ASM2621246v1_genomic.gene.1kbup.nov.final.fna\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000001735.1_tair8/GCF_000001735.1_TAIR8_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.1_v.1.0/GCF_000004255.1_v.1.0_genomic.gene.1kbup.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.1_v.1.0/GCF_000004255.1_v.1.0_genomic.gene.1kbup.nov.final.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.1_v.1.0/GCF_000004255.1_v.1.0_genomic.gene.1kbup.nov.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.1_v.1.0/GCF_000004255.1_v.1.0_genomic.gene.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.1_v.1.0/GCF_000004255.1_v.1.0_genomic.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.1_v.1.0/GCF_000004255.1_v.1.0_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.2_v.1.0/GCF_000004255.2_v.1.0_genomic.gene.1kbup.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.2_v.1.0/GCF_000004255.2_v.1.0_genomic.gene.1kbup.nov.final.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.2_v.1.0/GCF_000004255.2_v.1.0_genomic.gene.1kbup.nov.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.2_v.1.0/GCF_000004255.2_v.1.0_genomic.gene.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.2_v.1.0/GCF_000004255.2_v.1.0_genomic.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000004255.2_v.1.0/GCF_000004255.2_v.1.0_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.1_amtr1.0/GCF_000471905.1_AMTR1.0_genomic.gene.1kbup.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.1_amtr1.0/GCF_000471905.1_AMTR1.0_genomic.gene.1kbup.nov.final.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.1_amtr1.0/GCF_000471905.1_AMTR1.0_genomic.gene.1kbup.nov.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.1_amtr1.0/GCF_000471905.1_AMTR1.0_genomic.gene.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.1_amtr1.0/GCF_000471905.1_AMTR1.0_genomic.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.1_amtr1.0/GCF_000471905.1_AMTR1.0_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.2_amtr1.0/GCF_000471905.2_AMTR1.0_genomic.gene.1kbup.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.2_amtr1.0/GCF_000471905.2_AMTR1.0_genomic.gene.1kbup.nov.final.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.2_amtr1.0/GCF_000471905.2_AMTR1.0_genomic.gene.1kbup.nov.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.2_amtr1.0/GCF_000471905.2_AMTR1.0_genomic.gene.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.2_amtr1.0/GCF_000471905.2_AMTR1.0_genomic.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_000471905.2_amtr1.0/GCF_000471905.2_AMTR1.0_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_001540865.1_asm154086v1/GCF_001540865.1_ASM154086v1_genomic.gene.1kbup.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_001540865.1_asm154086v1/GCF_001540865.1_ASM154086v1_genomic.gene.1kbup.nov.final.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_001540865.1_asm154086v1/GCF_001540865.1_ASM154086v1_genomic.gene.1kbup.nov.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_001540865.1_asm154086v1/GCF_001540865.1_ASM154086v1_genomic.gene.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_001540865.1_asm154086v1/GCF_001540865.1_ASM154086v1_genomic.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_001540865.1_asm154086v1/GCF_001540865.1_ASM154086v1_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.1_aet_v4.0/GCF_002575655.1_Aet_v4.0_genomic.gene.1kbup.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.1_aet_v4.0/GCF_002575655.1_Aet_v4.0_genomic.gene.1kbup.nov.final.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.1_aet_v4.0/GCF_002575655.1_Aet_v4.0_genomic.gene.1kbup.nov.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.1_aet_v4.0/GCF_002575655.1_Aet_v4.0_genomic.gene.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.1_aet_v4.0/GCF_002575655.1_Aet_v4.0_genomic.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.1_aet_v4.0/GCF_002575655.1_Aet_v4.0_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.2_aet_v5.0/GCF_002575655.2_Aet_v5.0_genomic.gene.1kbup.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.2_aet_v5.0/GCF_002575655.2_Aet_v5.0_genomic.gene.1kbup.nov.final.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.2_aet_v5.0/GCF_002575655.2_Aet_v5.0_genomic.gene.1kbup.nov.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.2_aet_v5.0/GCF_002575655.2_Aet_v5.0_genomic.gene.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.2_aet_v5.0/GCF_002575655.2_Aet_v5.0_genomic.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_002575655.2_aet_v5.0/GCF_002575655.2_Aet_v5.0_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_003935025.1_abrus_2018/GCF_003935025.1_Abrus_2018_genomic.gene.1kbup.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_003935025.1_abrus_2018/GCF_003935025.1_Abrus_2018_genomic.gene.1kbup.nov.final.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_003935025.1_abrus_2018/GCF_003935025.1_Abrus_2018_genomic.gene.1kbup.nov.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_003935025.1_abrus_2018/GCF_003935025.1_Abrus_2018_genomic.gene.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_003935025.1_abrus_2018/GCF_003935025.1_Abrus_2018_genomic.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_003935025.1_abrus_2018/GCF_003935025.1_Abrus_2018_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_009805555.1_asm980555v1/GCF_009805555.1_ASM980555v1_genomic.gene.1kbup.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_009805555.1_asm980555v1/GCF_009805555.1_ASM980555v1_genomic.gene.1kbup.nov.final.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_009805555.1_asm980555v1/GCF_009805555.1_ASM980555v1_genomic.gene.1kbup.nov.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_009805555.1_asm980555v1/GCF_009805555.1_ASM980555v1_genomic.gene.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_009805555.1_asm980555v1/GCF_009805555.1_ASM980555v1_genomic.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_009805555.1_asm980555v1/GCF_009805555.1_ASM980555v1_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_019202715.1_maohua_mht/GCF_019202715.1_MaoHua_MHT_genomic.gene.1kbup.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_019202715.1_maohua_mht/GCF_019202715.1_MaoHua_MHT_genomic.gene.1kbup.nov.final.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_019202715.1_maohua_mht/GCF_019202715.1_MaoHua_MHT_genomic.gene.1kbup.nov.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_019202715.1_maohua_mht/GCF_019202715.1_MaoHua_MHT_genomic.gene.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_019202715.1_maohua_mht/GCF_019202715.1_MaoHua_MHT_genomic.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_019202715.1_maohua_mht/GCF_019202715.1_MaoHua_MHT_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_026212465.1_asm2621246v1/GCF_026212465.1_ASM2621246v1_genomic.gene.1kbup.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_026212465.1_asm2621246v1/GCF_026212465.1_ASM2621246v1_genomic.gene.1kbup.nov.final.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_026212465.1_asm2621246v1/GCF_026212465.1_ASM2621246v1_genomic.gene.1kbup.nov.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_026212465.1_asm2621246v1/GCF_026212465.1_ASM2621246v1_genomic.gene.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_026212465.1_asm2621246v1/GCF_026212465.1_ASM2621246v1_genomic.gff\n",
            " create mode 100644 data/raw/Refseq/annot/gcf_026212465.1_asm2621246v1/GCF_026212465.1_ASM2621246v1_genomic.gff.gz\n",
            " create mode 100644 data/raw/Refseq/dna/gcf_000001735.1_tair8/GCF_000001735.1_TAIR8_genomic.fna\n",
            " create mode 100644 data/raw/Refseq/dna/gcf_000001735.1_tair8/GCF_000001735.1_TAIR8_genomic.fna.gz\n",
            " create mode 100644 data/raw/gz_link/ensembl_link.csv\n",
            " create mode 100644 data/raw/gz_link/refseq_link.csv\n",
            " create mode 120000 libs/bedtools\n",
            " create mode 120000 libs/samtools\n",
            " create mode 100644 module/florabert.egg-info/PKG-INFO\n",
            " create mode 100644 module/florabert.egg-info/SOURCES.txt\n",
            " create mode 100644 module/florabert.egg-info/dependency_links.txt\n",
            " create mode 100644 module/florabert.egg-info/top_level.txt\n",
            " rename module/{inari => florabert}/__init__.py (100%)\n",
            " create mode 100644 module/florabert/__pycache__/__init__.cpython-310.pyc\n",
            " create mode 100644 module/florabert/__pycache__/config.cpython-310.pyc\n",
            " create mode 100644 module/florabert/__pycache__/gene_db_io.cpython-310.pyc\n",
            " create mode 100644 module/florabert/__pycache__/utils.cpython-310.pyc\n",
            " rename module/{inari => florabert}/config.py (100%)\n",
            " rename module/{inari => florabert}/dataio.py (100%)\n",
            " rename module/{inari => florabert}/exceptions.py (100%)\n",
            " rename module/{inari => florabert}/gene_db_io.py (100%)\n",
            " rename module/{inari => florabert}/metrics.py (100%)\n",
            " rename module/{inari => florabert}/models.py (100%)\n",
            " rename module/{inari => florabert}/nlp.py (100%)\n",
            " rename module/{inari => florabert}/training.py (100%)\n",
            " rename module/{inari => florabert}/transformers.py (100%)\n",
            " rename module/{inari => florabert}/utils.py (100%)\n",
            " rename module/{inari => florabert}/visualization.py (100%)\n",
            " create mode 100644 samtools-1.11/AUTHORS\n",
            " create mode 100644 samtools-1.11/ChangeLog.old\n",
            " create mode 100644 samtools-1.11/INSTALL\n",
            " create mode 100644 samtools-1.11/LICENSE\n",
            " create mode 100644 samtools-1.11/Makefile\n",
            " create mode 100644 samtools-1.11/NEWS\n",
            " create mode 100644 samtools-1.11/README\n",
            " create mode 100644 samtools-1.11/amplicon_stats.c\n",
            " create mode 100644 samtools-1.11/amplicon_stats.o\n",
            " create mode 100644 samtools-1.11/bam.c\n",
            " create mode 100644 samtools-1.11/bam.h\n",
            " create mode 100644 samtools-1.11/bam.o\n",
            " create mode 100644 samtools-1.11/bam2bcf.c\n",
            " create mode 100644 samtools-1.11/bam2bcf.h\n",
            " create mode 100644 samtools-1.11/bam2bcf.o\n",
            " create mode 100644 samtools-1.11/bam2bcf_indel.c\n",
            " create mode 100644 samtools-1.11/bam2bcf_indel.o\n",
            " create mode 100644 samtools-1.11/bam2depth.c\n",
            " create mode 100644 samtools-1.11/bam2depth.o\n",
            " create mode 100644 samtools-1.11/bam_addrprg.c\n",
            " create mode 100644 samtools-1.11/bam_addrprg.o\n",
            " create mode 100644 samtools-1.11/bam_ampliconclip.c\n",
            " create mode 100644 samtools-1.11/bam_ampliconclip.h\n",
            " create mode 100644 samtools-1.11/bam_ampliconclip.o\n",
            " create mode 100644 samtools-1.11/bam_aux.c\n",
            " create mode 100644 samtools-1.11/bam_aux.o\n",
            " create mode 100644 samtools-1.11/bam_cat.c\n",
            " create mode 100644 samtools-1.11/bam_cat.o\n",
            " create mode 100644 samtools-1.11/bam_color.c\n",
            " create mode 100644 samtools-1.11/bam_color.o\n",
            " create mode 100644 samtools-1.11/bam_endian.h\n",
            " create mode 100644 samtools-1.11/bam_fastq.c\n",
            " create mode 100644 samtools-1.11/bam_fastq.o\n",
            " create mode 100644 samtools-1.11/bam_flags.c\n",
            " create mode 100644 samtools-1.11/bam_flags.o\n",
            " create mode 100644 samtools-1.11/bam_index.c\n",
            " create mode 100644 samtools-1.11/bam_index.o\n",
            " create mode 100644 samtools-1.11/bam_lpileup.c\n",
            " create mode 100644 samtools-1.11/bam_lpileup.h\n",
            " create mode 100644 samtools-1.11/bam_lpileup.o\n",
            " create mode 100644 samtools-1.11/bam_markdup.c\n",
            " create mode 100644 samtools-1.11/bam_markdup.o\n",
            " create mode 100644 samtools-1.11/bam_mate.c\n",
            " create mode 100644 samtools-1.11/bam_mate.o\n",
            " create mode 100644 samtools-1.11/bam_md.c\n",
            " create mode 100644 samtools-1.11/bam_md.o\n",
            " create mode 100644 samtools-1.11/bam_plbuf.c\n",
            " create mode 100644 samtools-1.11/bam_plbuf.h\n",
            " create mode 100644 samtools-1.11/bam_plbuf.o\n",
            " create mode 100644 samtools-1.11/bam_plcmd.c\n",
            " create mode 100644 samtools-1.11/bam_plcmd.o\n",
            " create mode 100644 samtools-1.11/bam_quickcheck.c\n",
            " create mode 100644 samtools-1.11/bam_quickcheck.o\n",
            " create mode 100644 samtools-1.11/bam_reheader.c\n",
            " create mode 100644 samtools-1.11/bam_reheader.o\n",
            " create mode 100644 samtools-1.11/bam_rmdup.c\n",
            " create mode 100644 samtools-1.11/bam_rmdup.o\n",
            " create mode 100644 samtools-1.11/bam_rmdupse.c\n",
            " create mode 100644 samtools-1.11/bam_rmdupse.o\n",
            " create mode 100644 samtools-1.11/bam_sort.c\n",
            " create mode 100644 samtools-1.11/bam_sort.o\n",
            " create mode 100644 samtools-1.11/bam_split.c\n",
            " create mode 100644 samtools-1.11/bam_split.o\n",
            " create mode 100644 samtools-1.11/bam_stat.c\n",
            " create mode 100644 samtools-1.11/bam_stat.o\n",
            " create mode 100644 samtools-1.11/bam_tview.c\n",
            " create mode 100644 samtools-1.11/bam_tview.h\n",
            " create mode 100644 samtools-1.11/bam_tview.o\n",
            " create mode 100644 samtools-1.11/bam_tview_curses.c\n",
            " create mode 100644 samtools-1.11/bam_tview_curses.o\n",
            " create mode 100644 samtools-1.11/bam_tview_html.c\n",
            " create mode 100644 samtools-1.11/bam_tview_html.o\n",
            " create mode 100644 samtools-1.11/bamshuf.c\n",
            " create mode 100644 samtools-1.11/bamshuf.o\n",
            " create mode 100644 samtools-1.11/bamtk.c\n",
            " create mode 100644 samtools-1.11/bamtk.o\n",
            " create mode 100644 samtools-1.11/bedcov.c\n",
            " create mode 100644 samtools-1.11/bedcov.o\n",
            " create mode 100644 samtools-1.11/bedidx.c\n",
            " create mode 100644 samtools-1.11/bedidx.h\n",
            " create mode 100644 samtools-1.11/bedidx.o\n",
            " create mode 100644 samtools-1.11/config.h\n",
            " create mode 100644 samtools-1.11/config.h.in\n",
            " create mode 100644 samtools-1.11/config.log\n",
            " create mode 100644 samtools-1.11/config.mk\n",
            " create mode 100644 samtools-1.11/config.mk.in\n",
            " create mode 100755 samtools-1.11/config.status\n",
            " create mode 100755 samtools-1.11/configure\n",
            " create mode 100644 samtools-1.11/configure.ac\n",
            " create mode 100644 samtools-1.11/coverage.c\n",
            " create mode 100644 samtools-1.11/coverage.o\n",
            " create mode 100644 samtools-1.11/cut_target.c\n",
            " create mode 100644 samtools-1.11/cut_target.o\n",
            " create mode 100644 samtools-1.11/dict.c\n",
            " create mode 100644 samtools-1.11/dict.o\n",
            " create mode 100644 samtools-1.11/doc/samtools-addreplacerg.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-ampliconclip.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-ampliconstats.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-bedcov.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-calmd.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-cat.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-collate.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-coverage.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-depad.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-depth.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-dict.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-faidx.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-fasta.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-fastq.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-fixmate.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-flags.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-flagstat.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-fqidx.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-idxstats.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-index.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-markdup.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-merge.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-mpileup.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-phase.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-quickcheck.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-reheader.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-rmdup.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-sort.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-split.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-stats.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-targetcut.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-tview.1\n",
            " create mode 100644 samtools-1.11/doc/samtools-view.1\n",
            " create mode 100644 samtools-1.11/doc/samtools.1\n",
            " create mode 100644 samtools-1.11/examples/00README.txt\n",
            " create mode 100644 samtools-1.11/examples/ex1.fa\n",
            " create mode 100644 samtools-1.11/examples/ex1.sam.gz\n",
            " create mode 100644 samtools-1.11/examples/toy.fa\n",
            " create mode 100644 samtools-1.11/examples/toy.sam\n",
            " create mode 100644 samtools-1.11/faidx.c\n",
            " create mode 100644 samtools-1.11/faidx.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/INSTALL\n",
            " create mode 100644 samtools-1.11/htslib-1.11/LICENSE\n",
            " create mode 100644 samtools-1.11/htslib-1.11/Makefile\n",
            " create mode 100644 samtools-1.11/htslib-1.11/NEWS\n",
            " create mode 100644 samtools-1.11/htslib-1.11/README\n",
            " create mode 100644 samtools-1.11/htslib-1.11/README.large_positions.md\n",
            " create mode 100644 samtools-1.11/htslib-1.11/bcf_sr_sort.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/bcf_sr_sort.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/bcf_sr_sort.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/bgzf.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/bgzf.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/bgzip.1\n",
            " create mode 100644 samtools-1.11/htslib-1.11/bgzip.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/config.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/config.h.in\n",
            " create mode 100644 samtools-1.11/htslib-1.11/config.log\n",
            " create mode 100644 samtools-1.11/htslib-1.11/config.mk\n",
            " create mode 100644 samtools-1.11/htslib-1.11/config.mk.in\n",
            " create mode 100755 samtools-1.11/htslib-1.11/config.status\n",
            " create mode 100755 samtools-1.11/htslib-1.11/configure\n",
            " create mode 100644 samtools-1.11/htslib-1.11/configure.ac\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_codecs.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_codecs.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_codecs.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_decode.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_decode.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_decode.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_encode.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_encode.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_encode.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_external.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_external.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_index.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_index.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_index.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_io.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_io.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_io.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_samtools.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_samtools.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_samtools.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_stats.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_stats.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_stats.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/cram_structs.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/mFILE.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/mFILE.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/mFILE.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/misc.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/open_trace_file.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/open_trace_file.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/open_trace_file.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/os.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/pooled_alloc.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/pooled_alloc.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/pooled_alloc.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/rANS_byte.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/rANS_static.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/rANS_static.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/rANS_static.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/string_alloc.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/string_alloc.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/cram/string_alloc.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/errmod.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/errmod.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/faidx.5\n",
            " create mode 100644 samtools-1.11/htslib-1.11/faidx.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/faidx.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/header.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/header.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/header.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile_gcs.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile_gcs.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile_internal.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile_libcurl.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile_libcurl.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile_net.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile_net.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile_s3.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile_s3.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile_s3_write.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hfile_s3_write.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hts-object-files\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hts.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hts.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hts_internal.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hts_os.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/hts_os.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htsfile.1\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htsfile.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib-s3-plugin.7\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib.mk\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib.pc.in\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib.pc.tmp\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/bgzf.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/cram.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/faidx.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/hfile.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/hts.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/hts_defs.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/hts_endian.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/hts_log.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/hts_os.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/kbitset.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/kfunc.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/khash.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/khash_str2int.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/klist.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/knetfile.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/kroundup.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/kseq.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/ksort.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/kstring.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/regidx.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/sam.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/synced_bcf_reader.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/tbx.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/thread_pool.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/vcf.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/vcf_sweep.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib/vcfutils.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib_static.mk\n",
            " create mode 100644 samtools-1.11/htslib-1.11/htslib_vars.mk\n",
            " create mode 100644 samtools-1.11/htslib-1.11/kfunc.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/kfunc.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/knetfile.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/knetfile.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/kstring.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/kstring.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/libhts.a\n",
            " create mode 100644 samtools-1.11/htslib-1.11/m4/hts_hide_dynamic_syms.m4\n",
            " create mode 100644 samtools-1.11/htslib-1.11/m4/hts_prog_cc_warnings.m4\n",
            " create mode 100644 samtools-1.11/htslib-1.11/m4/pkg.m4\n",
            " create mode 100644 samtools-1.11/htslib-1.11/md5.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/md5.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/multipart.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/multipart.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/os/lzma_stub.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/os/rand.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/plugin.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/probaln.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/probaln.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/realn.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/realn.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/regidx.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/regidx.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/region.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/region.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/sam.5\n",
            " create mode 100644 samtools-1.11/htslib-1.11/sam.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/sam.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/sam_internal.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/synced_bcf_reader.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/synced_bcf_reader.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/tabix.1\n",
            " create mode 100644 samtools-1.11/htslib-1.11/tabix.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/tbx.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/tbx.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/auxf#values.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/auxf#values_java.cram\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/auxf.fa\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/auxf.fa.fai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/bcf-sr/merge.noidx.a.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/bcf-sr/merge.noidx.abc.expected.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/bcf-sr/merge.noidx.b.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/bcf-sr/merge.noidx.c.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/bcf-sr/merge.noidx.hdr_order.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/bcf-sr/merge.noidx.rec_order.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/bgziptest.txt\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/bgziptest.txt.gz\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/bgziptest.txt.gz.gzi\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c1#bounds.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c1#clip.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c1#noseq.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c1#pad1.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c1#pad2.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c1#pad3.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c1#unknown.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c1.fa\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c1.fa.fai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c2#pad.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c2.fa\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/c2.fa.fai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#1.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#1000.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#2.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#5.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#5b.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#5b_java.cram\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#large_seq.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#supp.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#tag_depadded.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#tag_padded.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#unmap.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#unmap1.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce#unmap2.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce.fa\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/ce.fa.fai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/colons.bam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/colons.bam.bai\n",
            " create mode 100755 samtools-1.11/htslib-1.11/test/compare_sam.pl\n",
            " create mode 100755 samtools-1.11/htslib-1.11/test/cross_validate.sh\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/emptyfile\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/faidx.fa\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/fastqs.fq\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/fastqs_README.txt\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/fieldarith.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/fieldarith.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/formatcols.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/formatmissing-out.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/formatmissing.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/fuzz/hts_open_fuzzer.c\n",
            " create mode 100755 samtools-1.11/htslib-1.11/test/header_syms.pl\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/hfile.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/hts_endian.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/index.bam.bai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/index.bam.csi\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/index.bcf.csi\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/index.cram.crai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/index.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/index.sam.gz.bai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/index.sam.gz.csi\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/index.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/index.vcf.gz.csi\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/index.vcf.gz.tbi\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/longrefs/index.expected1.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/longrefs/index.expected2.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/longrefs/index.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/longrefs/longref.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/longrefs/longref_itr.expected.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/longrefs/longref_multi.expected.sam\n",
            " create mode 100755 samtools-1.11/htslib-1.11/test/maintainer/check_copyright.pl\n",
            " create mode 100755 samtools-1.11/htslib-1.11/test/maintainer/check_spaces.pl\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/md#1.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/md.fa\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/md.fa.fai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/c1#pad1.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/c1#pad1.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/c1#pad2.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/c1#pad2.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/c1#pad3.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/c1#pad3.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_D.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_D.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_DI.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_DI.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_I.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_I.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_ID.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_ID.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_N.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_N.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_N2.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_N2.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_P.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mp_P.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/mpileup.tst\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/small.bam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/mpileup/small.out\n",
            " create mode 100755 samtools-1.11/htslib-1.11/test/mpileup/test-pileup.sh\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/no_hdr_sq_1.bam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/no_hdr_sq_1.bam.csi\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/no_hdr_sq_1.expected.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/noroundtrip-out.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/noroundtrip.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/pileup.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/plugins-dlhts.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/range.bam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/range.bam.bai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/range.cram\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/range.cram.crai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/range.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn01.fa\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn01.fa.fai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn01.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn01_exp-a.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn01_exp-e.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn01_exp.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn02-r.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn02.fa\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn02.fa.fai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn02.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn02_exp-a.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn02_exp-e.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/realn02_exp.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/sam.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/simple_test_driver.sh\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/bed_file.Y.100200.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/bed_file.bed\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/bed_file.separate.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/gff_file.X.2934832.2935190.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/gff_file.gff\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/large_chr.20.1.2147483647.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/large_chr.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/tabix.tst\n",
            " create mode 100755 samtools-1.11/htslib-1.11/test/tabix/test-tabix.sh\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/vcf_file.1.3000151.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/vcf_file.2.3199812.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/vcf_file.bcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/tabix/vcf_file.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test-bcf-sr.c\n",
            " create mode 100755 samtools-1.11/htslib-1.11/test/test-bcf-sr.pl\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test-bcf-translate.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test-bcf-translate.out\n",
            " create mode 100755 samtools-1.11/htslib-1.11/test/test-logging.pl\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test-parse-reg.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test-regidx.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test-vcf-api.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test-vcf-api.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test-vcf-hdr-in.vcf\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test-vcf-hdr.out\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test-vcf-sweep.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test-vcf-sweep.out\n",
            " create mode 100755 samtools-1.11/htslib-1.11/test/test.pl\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test_bgzf.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test_index.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test_kfunc.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test_kstring.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test_realn.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test_str2int.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/test_view.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/thrash_threads1.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/thrash_threads2.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/thrash_threads3.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/thrash_threads4.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/thrash_threads5.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/thrash_threads6.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/thrash_threads7.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/thread_pool.md\n",
            " create mode 100755 samtools-1.11/htslib-1.11/test/with-shlib.sh\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#MD.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#MD2.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#blank.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#large_aux.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#large_aux2.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#large_aux_java.cram\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#minimal.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#pair.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#repeated.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#rg.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#tlen.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#tlen2.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#triplet.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx#unsorted.sam\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx.fa\n",
            " create mode 100644 samtools-1.11/htslib-1.11/test/xx.fa.fai\n",
            " create mode 100644 samtools-1.11/htslib-1.11/textutils.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/textutils.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/textutils_internal.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/thread_pool.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/thread_pool.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/thread_pool_internal.h\n",
            " create mode 100644 samtools-1.11/htslib-1.11/vcf.5\n",
            " create mode 100644 samtools-1.11/htslib-1.11/vcf.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/vcf.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/vcf_sweep.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/vcf_sweep.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/vcfutils.c\n",
            " create mode 100644 samtools-1.11/htslib-1.11/vcfutils.o\n",
            " create mode 100644 samtools-1.11/htslib-1.11/version.h\n",
            " create mode 100755 samtools-1.11/htslib-1.11/version.sh\n",
            " create mode 100755 samtools-1.11/install-sh\n",
            " create mode 100644 samtools-1.11/libbam.a\n",
            " create mode 100644 samtools-1.11/libst.a\n",
            " create mode 100644 samtools-1.11/lz4/LICENSE\n",
            " create mode 100644 samtools-1.11/lz4/lz4.c\n",
            " create mode 100644 samtools-1.11/lz4/lz4.h\n",
            " create mode 100644 samtools-1.11/lz4/lz4.o\n",
            " create mode 100644 samtools-1.11/m4/ax_with_htslib.m4\n",
            " create mode 100644 samtools-1.11/m4/hts_prog_cc_warnings.m4\n",
            " create mode 100644 samtools-1.11/m4/hts_with_curses.m4\n",
            " create mode 100644 samtools-1.11/misc/HmmGlocal.java\n",
            " create mode 100755 samtools-1.11/misc/ace2sam\n",
            " create mode 100644 samtools-1.11/misc/ace2sam.c\n",
            " create mode 100644 samtools-1.11/misc/ace2sam.o\n",
            " create mode 100755 samtools-1.11/misc/blast2sam.pl\n",
            " create mode 100755 samtools-1.11/misc/bowtie2sam.pl\n",
            " create mode 100755 samtools-1.11/misc/export2sam.pl\n",
            " create mode 100755 samtools-1.11/misc/interpolate_sam.pl\n",
            " create mode 100755 samtools-1.11/misc/maq2sam-long\n",
            " create mode 100644 samtools-1.11/misc/maq2sam-long.o\n",
            " create mode 100755 samtools-1.11/misc/maq2sam-short\n",
            " create mode 100644 samtools-1.11/misc/maq2sam-short.o\n",
            " create mode 100644 samtools-1.11/misc/maq2sam.c\n",
            " create mode 100755 samtools-1.11/misc/md5fa\n",
            " create mode 100644 samtools-1.11/misc/md5fa.c\n",
            " create mode 100644 samtools-1.11/misc/md5fa.o\n",
            " create mode 100755 samtools-1.11/misc/md5sum-lite\n",
            " create mode 100644 samtools-1.11/misc/md5sum-lite.c\n",
            " create mode 100644 samtools-1.11/misc/md5sum-lite.o\n",
            " create mode 100755 samtools-1.11/misc/novo2sam.pl\n",
            " create mode 100755 samtools-1.11/misc/plot-ampliconstats\n",
            " create mode 100755 samtools-1.11/misc/plot-bamstats\n",
            " create mode 100755 samtools-1.11/misc/psl2sam.pl\n",
            " create mode 100755 samtools-1.11/misc/r2plot.lua\n",
            " create mode 100755 samtools-1.11/misc/sam2vcf.pl\n",
            " create mode 100755 samtools-1.11/misc/samtools.pl\n",
            " create mode 100644 samtools-1.11/misc/samtools_tab_completion\n",
            " create mode 100755 samtools-1.11/misc/seq_cache_populate.pl\n",
            " create mode 100755 samtools-1.11/misc/soap2sam.pl\n",
            " create mode 100755 samtools-1.11/misc/vcfutils.lua\n",
            " create mode 100755 samtools-1.11/misc/wgsim\n",
            " create mode 100644 samtools-1.11/misc/wgsim.1\n",
            " create mode 100644 samtools-1.11/misc/wgsim.c\n",
            " create mode 100644 samtools-1.11/misc/wgsim.o\n",
            " create mode 100755 samtools-1.11/misc/wgsim_eval.pl\n",
            " create mode 100755 samtools-1.11/misc/zoom2sam.pl\n",
            " create mode 100644 samtools-1.11/padding.c\n",
            " create mode 100644 samtools-1.11/padding.o\n",
            " create mode 100644 samtools-1.11/phase.c\n",
            " create mode 100644 samtools-1.11/phase.o\n",
            " create mode 100644 samtools-1.11/sam.c\n",
            " create mode 100644 samtools-1.11/sam.h\n",
            " create mode 100644 samtools-1.11/sam.o\n",
            " create mode 100644 samtools-1.11/sam_opts.c\n",
            " create mode 100644 samtools-1.11/sam_opts.h\n",
            " create mode 100644 samtools-1.11/sam_opts.o\n",
            " create mode 100644 samtools-1.11/sam_utils.c\n",
            " create mode 100644 samtools-1.11/sam_utils.o\n",
            " create mode 100644 samtools-1.11/sam_view.c\n",
            " create mode 100644 samtools-1.11/sam_view.o\n",
            " create mode 100644 samtools-1.11/sample.c\n",
            " create mode 100644 samtools-1.11/sample.h\n",
            " create mode 100644 samtools-1.11/sample.o\n",
            " create mode 100755 samtools-1.11/samtools\n",
            " create mode 100644 samtools-1.11/samtools.h\n",
            " create mode 100644 samtools-1.11/stats.c\n",
            " create mode 100644 samtools-1.11/stats.o\n",
            " create mode 100644 samtools-1.11/stats_isize.c\n",
            " create mode 100644 samtools-1.11/stats_isize.h\n",
            " create mode 100644 samtools-1.11/stats_isize.o\n",
            " create mode 100644 samtools-1.11/test/addrprg/1_fixup.sam\n",
            " create mode 100644 samtools-1.11/test/addrprg/1_fixup.sam.expected\n",
            " create mode 100644 samtools-1.11/test/addrprg/1_fixup.sam.expected.err\n",
            " create mode 100644 samtools-1.11/test/addrprg/2_fixup_orphan.sam\n",
            " create mode 100644 samtools-1.11/test/addrprg/2_fixup_orphan.sam.expected\n",
            " create mode 100644 samtools-1.11/test/addrprg/2_fixup_orphan.sam.expected.err\n",
            " create mode 100644 samtools-1.11/test/addrprg/3_fixup.sam.expected\n",
            " create mode 100644 samtools-1.11/test/addrprg/3_fixup.sam.expected.err\n",
            " create mode 100644 samtools-1.11/test/addrprg/4_fixup_norg.sam\n",
            " create mode 100644 samtools-1.11/test/addrprg/4_fixup_norg.sam.expected\n",
            " create mode 100644 samtools-1.11/test/addrprg/4_fixup_norg.sam.expected.err\n",
            " create mode 100644 samtools-1.11/test/ampliconclip/1_delete_tag.expected.sam\n",
            " create mode 100644 samtools-1.11/test/ampliconclip/1_fail.expected.sam\n",
            " create mode 100644 samtools-1.11/test/ampliconclip/1_filter.expected.sam\n",
            " create mode 100644 samtools-1.11/test/ampliconclip/1_hard_clipped.expected.sam\n",
            " create mode 100644 samtools-1.11/test/ampliconclip/1_original_tag.expected.sam\n",
            " create mode 100644 samtools-1.11/test/ampliconclip/1_soft_clipped.expected.sam\n",
            " create mode 100644 samtools-1.11/test/ampliconclip/1_soft_clipped_strand.expected.sam\n",
            " create mode 100644 samtools-1.11/test/ampliconclip/1_test_data.sam\n",
            " create mode 100644 samtools-1.11/test/ampliconclip/2_both_clipped.expected.sam\n",
            " create mode 100644 samtools-1.11/test/ampliconclip/2_both_test_data.sam\n",
            " create mode 100755 samtools-1.11/test/ampliconclip/ac_test.bed\n",
            " create mode 100644 samtools-1.11/test/ampliconstats/stats.expected.txt\n",
            " create mode 100644 samtools-1.11/test/bam2fq/1.1.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/1.2.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/1.stdout.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/10.1.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/10.2.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/10.s.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/11.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/12.1.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/12.2.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/12.i.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/12.s.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/13.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/2.1.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/2.2.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/2.s.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/2.stdout.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/3.1.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/3.2.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/3.s.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/3.stdout.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/4.1.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/4.2.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/4.s.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/4.stdout.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/5.1.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/5.2.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/5.s.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/6.1.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/6.2.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/6.s.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/7.1.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/7.2.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/7.s.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/8.1.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/8.2.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/8.i.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/8.s.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/9.1.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/9.2.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/bc.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/bc10.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bam2fq/bc_split.fq.expected\n",
            " create mode 100644 samtools-1.11/test/bedcov/bedcov.bam\n",
            " create mode 100644 samtools-1.11/test/bedcov/bedcov.bam.bai\n",
            " create mode 100644 samtools-1.11/test/bedcov/bedcov.bed\n",
            " create mode 100644 samtools-1.11/test/bedcov/bedcov.expected\n",
            " create mode 100644 samtools-1.11/test/bedcov/bedcov_gG.bed\n",
            " create mode 100644 samtools-1.11/test/bedcov/bedcov_gG.expected\n",
            " create mode 100644 samtools-1.11/test/bedcov/bedcov_j.expected\n",
            " create mode 100644 samtools-1.11/test/collate/1_fast_collate.sam.expected\n",
            " create mode 100644 samtools-1.11/test/collate/2_fast_collate_with_tmp_used.sam.expected\n",
            " create mode 100644 samtools-1.11/test/collate/collate.expected.sam\n",
            " create mode 100644 samtools-1.11/test/collate/fast_collate.sam\n",
            " create mode 100644 samtools-1.11/test/dat/bam2fq.001.sam\n",
            " create mode 100644 samtools-1.11/test/dat/bam2fq.002.sam\n",
            " create mode 100644 samtools-1.11/test/dat/bam2fq.003.sam\n",
            " create mode 100644 samtools-1.11/test/dat/bam2fq.004.sam\n",
            " create mode 100644 samtools-1.11/test/dat/bam2fq.005.sam\n",
            " create mode 100644 samtools-1.11/test/dat/bam2fq.010.sam\n",
            " create mode 100644 samtools-1.11/test/dat/bam2fq.013.sam\n",
            " create mode 100644 samtools-1.11/test/dat/bam2fq.703.sam\n",
            " create mode 100644 samtools-1.11/test/dat/cat.hdr\n",
            " create mode 100644 samtools-1.11/test/dat/cram_md5/08c04d512d4797d9ba2a156c1daba468\n",
            " create mode 100644 samtools-1.11/test/dat/cram_md5/7c35feac7036c1cdef3bee0cc4b21437\n",
            " create mode 100644 samtools-1.11/test/dat/cram_md5/89b96de7acc5918c2858ef92710be7d4\n",
            " create mode 100644 samtools-1.11/test/dat/cram_md5/8a3d8deb9edd4ce3346b6061fcee6d0b\n",
            " create mode 100644 samtools-1.11/test/dat/depad.001.fa\n",
            " create mode 100644 samtools-1.11/test/dat/depad.001.fa.fai\n",
            " create mode 100644 samtools-1.11/test/dat/depad.001p.sam\n",
            " create mode 100644 samtools-1.11/test/dat/depad.001u.sam\n",
            " create mode 100644 samtools-1.11/test/dat/dict.alias.fa\n",
            " create mode 100644 samtools-1.11/test/dat/dict.alias.out\n",
            " create mode 100644 samtools-1.11/test/dat/dict.fa\n",
            " create mode 100644 samtools-1.11/test/dat/dict.out\n",
            " create mode 100644 samtools-1.11/test/dat/empty.expected\n",
            " create mode 100644 samtools-1.11/test/dat/large_chrom.out\n",
            " create mode 100644 samtools-1.11/test/dat/large_chrom.sam\n",
            " create mode 100644 samtools-1.11/test/dat/large_chrom_bai_index.err\n",
            " create mode 100644 samtools-1.11/test/dat/mpileup.1.sam\n",
            " create mode 100644 samtools-1.11/test/dat/mpileup.2.sam\n",
            " create mode 100644 samtools-1.11/test/dat/mpileup.3.sam\n",
            " create mode 100644 samtools-1.11/test/dat/mpileup.err.1\n",
            " create mode 100644 samtools-1.11/test/dat/mpileup.out.1\n",
            " create mode 100644 samtools-1.11/test/dat/mpileup.out.2\n",
            " create mode 100644 samtools-1.11/test/dat/mpileup.out.3\n",
            " create mode 100644 samtools-1.11/test/dat/mpileup.out.4\n",
            " create mode 100644 samtools-1.11/test/dat/mpileup.out.5\n",
            " create mode 100644 samtools-1.11/test/dat/mpileup.ref.fa\n",
            " create mode 100644 samtools-1.11/test/dat/mpileup.ref.fa.fai\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_a.bam\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_a.bam.bai.expected\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_a.cram\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_a.sam\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_a_regex.sam\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_ab.X.expected\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_b.X.expected\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_b.bam\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_b.sam\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_b_regex.sam\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_c.bam\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_c.sam\n",
            " create mode 100644 samtools-1.11/test/dat/test_input_1_d.sam\n",
            " create mode 100644 samtools-1.11/test/dat/view.001.01.bed\n",
            " create mode 100644 samtools-1.11/test/dat/view.001.02.bed\n",
            " create mode 100644 samtools-1.11/test/dat/view.001.fa\n",
            " create mode 100644 samtools-1.11/test/dat/view.001.fa.fai\n",
            " create mode 100644 samtools-1.11/test/dat/view.001.sam\n",
            " create mode 100644 samtools-1.11/test/dat/view.002.01.bed\n",
            " create mode 100644 samtools-1.11/test/dat/view.002.02.bed\n",
            " create mode 100644 samtools-1.11/test/dat/view.002.fa\n",
            " create mode 100644 samtools-1.11/test/dat/view.002.fa.fai\n",
            " create mode 100644 samtools-1.11/test/dat/view.002.sam\n",
            " create mode 100644 samtools-1.11/test/dat/view.003.expected.sam\n",
            " create mode 100644 samtools-1.11/test/dat/view.003.sam\n",
            " create mode 100644 samtools-1.11/test/dat/view.004.expected.sam\n",
            " create mode 100644 samtools-1.11/test/fixmate/1_coord_sort.sam\n",
            " create mode 100644 samtools-1.11/test/fixmate/1_coord_sort.sam.expected\n",
            " create mode 100644 samtools-1.11/test/fixmate/1_coord_sort.sam.expected.err\n",
            " create mode 100644 samtools-1.11/test/fixmate/2_isize_overflow.sam\n",
            " create mode 100644 samtools-1.11/test/fixmate/2_isize_overflow.sam.expected\n",
            " create mode 100644 samtools-1.11/test/fixmate/3_reverse_read_pp_lt.sam\n",
            " create mode 100644 samtools-1.11/test/fixmate/3_reverse_read_pp_lt.sam.expected\n",
            " create mode 100644 samtools-1.11/test/fixmate/4_reverse_read_pp_equal.sam\n",
            " create mode 100644 samtools-1.11/test/fixmate/4_reverse_read_pp_equal.sam.expected\n",
            " create mode 100644 samtools-1.11/test/fixmate/5_ct.sam\n",
            " create mode 100644 samtools-1.11/test/fixmate/5_ct.sam.expected\n",
            " create mode 100644 samtools-1.11/test/fixmate/6_ct_replace.sam\n",
            " create mode 100644 samtools-1.11/test/fixmate/6_ct_replace.sam.expected\n",
            " create mode 100644 samtools-1.11/test/fixmate/7_two_read_mapped.sam\n",
            " create mode 100644 samtools-1.11/test/fixmate/7_two_read_mapped.sam.expected\n",
            " create mode 100644 samtools-1.11/test/fixmate/8_isize_overflow_64bit.sam\n",
            " create mode 100644 samtools-1.11/test/fixmate/8_isize_overflow_64bit.sam.expected\n",
            " create mode 100644 samtools-1.11/test/idxstats/test_input_1_a.bam.expected\n",
            " create mode 100644 samtools-1.11/test/idxstats/test_input_1_a.bam.expected.err\n",
            " create mode 100644 samtools-1.11/test/large_pos/depth.expected.out\n",
            " create mode 100644 samtools-1.11/test/large_pos/depth_bed.expected.out\n",
            " create mode 100644 samtools-1.11/test/large_pos/longref.sam\n",
            " create mode 100644 samtools-1.11/test/large_pos/longref2.sam\n",
            " create mode 100644 samtools-1.11/test/large_pos/longref3.expected.sam\n",
            " create mode 100644 samtools-1.11/test/large_pos/longref3.sam\n",
            " create mode 100644 samtools-1.11/test/large_pos/longref_idx.expected.sam\n",
            " create mode 100644 samtools-1.11/test/large_pos/longref_name.sam\n",
            " create mode 100644 samtools-1.11/test/large_pos/merge.expected.sam\n",
            " create mode 100644 samtools-1.11/test/large_pos/test.bed\n",
            " create mode 100644 samtools-1.11/test/large_pos/tview.expected.out\n",
            " create mode 100755 samtools-1.11/test/maintainer/check_copyright.pl\n",
            " create mode 100755 samtools-1.11/test/maintainer/check_spaces.pl\n",
            " create mode 100644 samtools-1.11/test/markdup/1_name_sort.expected.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/1_name_sort.expected.sam.err\n",
            " create mode 100644 samtools-1.11/test/markdup/1_name_sort.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/2_bad_order.expected.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/2_bad_order.expected.sam.err\n",
            " create mode 100644 samtools-1.11/test/markdup/2_bad_order.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/3_missing_mc.expected.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/3_missing_mc.expected.sam.err\n",
            " create mode 100644 samtools-1.11/test/markdup/3_missing_mc.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/4_missing_ms.expected.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/4_missing_ms.expected.sam.err\n",
            " create mode 100644 samtools-1.11/test/markdup/4_missing_ms.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/5_markdup.expected.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/5_markdup.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/6_remove_dups.expected.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/6_remove_dups.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/7_mark_supp_dup.expected.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/7_mark_supp_dup.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/8_optical_dup.expected.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/8_optical_dup.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/9_optical_dup_qcfail.expected.sam\n",
            " create mode 100644 samtools-1.11/test/markdup/9_optical_dup_qcfail.sam\n",
            " create mode 100644 samtools-1.11/test/merge/2.merge.expected.sam\n",
            " create mode 100644 samtools-1.11/test/merge/3.merge.expected.err\n",
            " create mode 100644 samtools-1.11/test/merge/3.merge.expected.sam\n",
            " create mode 100644 samtools-1.11/test/merge/4.merge.expected.sam\n",
            " create mode 100644 samtools-1.11/test/merge/5.merge.expected.sam\n",
            " create mode 100644 samtools-1.11/test/merge/6.merge.expected.sam\n",
            " create mode 100644 samtools-1.11/test/merge/7.merge.expected.sam\n",
            " create mode 100644 samtools-1.11/test/merge/merge.bed\n",
            " create mode 100644 samtools-1.11/test/merge/merge.bed.1.sam\n",
            " create mode 100644 samtools-1.11/test/merge/merge.bed.2.sam\n",
            " create mode 100644 samtools-1.11/test/merge/merge.bed.expected.sam\n",
            " create mode 100644 samtools-1.11/test/merge/tag.pg.merge.expected.sam\n",
            " create mode 100644 samtools-1.11/test/merge/tag.pg.n.merge.expected.sam\n",
            " create mode 100644 samtools-1.11/test/merge/test_3.fofn\n",
            " create mode 100755 samtools-1.11/test/merge/test_bam_translate\n",
            " create mode 100644 samtools-1.11/test/merge/test_bam_translate.c\n",
            " create mode 100644 samtools-1.11/test/merge/test_bam_translate.o\n",
            " create mode 100644 samtools-1.11/test/merge/test_header.sam\n",
            " create mode 100644 samtools-1.11/test/merge/test_no_pg_rg_co.sam\n",
            " create mode 100755 samtools-1.11/test/merge/test_rtrans_build\n",
            " create mode 100644 samtools-1.11/test/merge/test_rtrans_build.c\n",
            " create mode 100644 samtools-1.11/test/merge/test_rtrans_build.o\n",
            " create mode 100755 samtools-1.11/test/merge/test_trans_tbl_init\n",
            " create mode 100644 samtools-1.11/test/merge/test_trans_tbl_init.c\n",
            " create mode 100644 samtools-1.11/test/merge/test_trans_tbl_init.o\n",
            " create mode 100644 samtools-1.11/test/mpileup/1read.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/1read.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/anomalous.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/c1#clip.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/c1#pad1.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/c1#pad2.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/c1#pad3.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/c1.fa\n",
            " create mode 100644 samtools-1.11/test/mpileup/c1.fa.fai\n",
            " create mode 100644 samtools-1.11/test/mpileup/ce#5b.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/ce#5b.bam.bai\n",
            " create mode 100644 samtools-1.11/test/mpileup/ce#5b.cram\n",
            " create mode 100644 samtools-1.11/test/mpileup/ce#5b.cram.crai\n",
            " create mode 100644 samtools-1.11/test/mpileup/ce#5b.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/ce#large_seq.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/ce#unmap.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/ce#unmap1.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/ce#unmap2.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/ce.fa\n",
            " create mode 100644 samtools-1.11/test/mpileup/ce.fa.fai\n",
            " create mode 100644 samtools-1.11/test/mpileup/deep.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/depth.reg\n",
            " create mode 100644 samtools-1.11/test/mpileup/depth_del.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/1.out.f3-6.gz\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/10.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/11.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/12.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/13.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/14.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/15.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/16.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/17.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/18.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/19.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/2.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/20.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/21.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/22.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/23.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/24.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/25.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/26.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/27.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/28.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/29.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/3.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/30.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/31.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/32.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/33.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/34.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/35.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/37.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/38.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/39.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/4.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/40.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/41.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/42.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/43.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/44.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/45.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/46.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/47.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/48.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/49.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/5.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/50.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/51.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/52.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/53.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/54.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/55.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/56.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/57.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/58.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/59.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/6.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/60.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/61.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/62.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/63.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/64.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/65.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/66.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/67.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/68.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/69.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/7.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/70.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/71.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/72.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/73.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/74.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/75.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/76.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/77.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/78.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/79.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/8.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/9.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a0.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a1.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a10.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a11.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a13.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a14.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a15.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a16.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a17.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a2.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a3.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a4.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a5.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a6.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a8.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/a9.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d1_1.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d1_12.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d1_2.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d2_12r.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d3_12.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d3_12r1a.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d3_12r2a.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d3_3.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d4_12.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d4_12b.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d4_12r.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_all1.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_all2.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_all3.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_all4.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_b3.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_b3a.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_b3aa.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_blank.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_xx1.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_xx2.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_xx3.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d5_xy1.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d6_nodel.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d6_wdel.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/d7_nodup.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/m5_all1.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/m5_all2.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/m5_all3.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/m5_b3.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/m5_b3a.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/m5_b3aa.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/m5_blank.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/m5_xx1.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/m5_xx2.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/m5_xx3.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/m5_xy1.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/mp_D.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/mp_DI.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/mp_I.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/mp_ID.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/mp_ID_rev.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/mp_N.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/mp_N2.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/expected/mp_P.out\n",
            " create mode 100644 samtools-1.11/test/mpileup/indels.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/md5/02099b2aad2cfe707759353724fb40ae\n",
            " create mode 100644 samtools-1.11/test/mpileup/md5/0d4dfc45807d6a11439e858605cb364d\n",
            " create mode 100644 samtools-1.11/test/mpileup/md5/351f64d4f4f9ddd45b35336ad97aa6de\n",
            " create mode 100644 samtools-1.11/test/mpileup/md5/6ec590eb82c361a61e6baabce20109bc\n",
            " create mode 100644 samtools-1.11/test/mpileup/md5/6f60cdf4c739651ae72d53e2e94825c1\n",
            " create mode 100644 samtools-1.11/test/mpileup/md5/73d180d5cb3860a2713927b6812eab0e\n",
            " create mode 100644 samtools-1.11/test/mpileup/mp.fa\n",
            " create mode 100644 samtools-1.11/test/mpileup/mp.fa.fai\n",
            " create mode 100644 samtools-1.11/test/mpileup/mp_D.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mp_DI.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mp_I.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mp_ID.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mp_ID_rev.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mp_N.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mp_N2.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mp_P.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mpileup-E.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mpileup.1.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mpileup.1.bam.bai\n",
            " create mode 100644 samtools-1.11/test/mpileup/mpileup.2.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mpileup.3.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/mpileup.ref.fa\n",
            " create mode 100644 samtools-1.11/test/mpileup/mpileup.ref.fa.fai\n",
            " create mode 100644 samtools-1.11/test/mpileup/mpileup.reg\n",
            " create mode 100644 samtools-1.11/test/mpileup/overlap.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/overlap50.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/overlapIllumina.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/regions\n",
            " create mode 100755 samtools-1.11/test/mpileup/regression.sh\n",
            " create mode 100644 samtools-1.11/test/mpileup/xx#depth1.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/xx#depth2.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/xx#depth3.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/xx#depth4.sam\n",
            " create mode 100644 samtools-1.11/test/mpileup/xx#minimal.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/xx#triplet.bam\n",
            " create mode 100644 samtools-1.11/test/mpileup/xx.bed\n",
            " create mode 100644 samtools-1.11/test/mpileup/xx.bed2\n",
            " create mode 100644 samtools-1.11/test/mpileup/xx.bed3\n",
            " create mode 100644 samtools-1.11/test/mpileup/xx.fa\n",
            " create mode 100644 samtools-1.11/test/mpileup/xx.fa.fai\n",
            " create mode 100644 samtools-1.11/test/quickcheck/1.quickcheck.badeof.bam\n",
            " create mode 100644 samtools-1.11/test/quickcheck/10.quickcheck.notargets.bam\n",
            " create mode 100644 samtools-1.11/test/quickcheck/2.quickcheck.badheader.bam\n",
            " create mode 100644 samtools-1.11/test/quickcheck/3.quickcheck.ok.bam\n",
            " create mode 100644 samtools-1.11/test/quickcheck/4.quickcheck.ok.bam\n",
            " create mode 100644 samtools-1.11/test/quickcheck/5.quickcheck.scramble30.truncated.cram\n",
            " create mode 100644 samtools-1.11/test/quickcheck/6.quickcheck.cram21.ok.cram\n",
            " create mode 100644 samtools-1.11/test/quickcheck/7.quickcheck.cram30.ok.cram\n",
            " create mode 100644 samtools-1.11/test/quickcheck/8.quickcheck.cram21.truncated.cram\n",
            " create mode 100644 samtools-1.11/test/quickcheck/9.quickcheck.cram30.truncated.cram\n",
            " create mode 100644 samtools-1.11/test/quickcheck/all.expected\n",
            " create mode 100644 samtools-1.11/test/reheader/1_view1.sam.expected\n",
            " create mode 100644 samtools-1.11/test/reheader/1_view1.sam.expected.err\n",
            " create mode 100644 samtools-1.11/test/reheader/2_view1.sam.expected\n",
            " create mode 100644 samtools-1.11/test/reheader/2_view1.sam.expected.err\n",
            " create mode 100644 samtools-1.11/test/reheader/3_view1.sam.expected\n",
            " create mode 100644 samtools-1.11/test/reheader/3_view1.sam.expected.err\n",
            " create mode 100644 samtools-1.11/test/reheader/4_view1.sam.expected\n",
            " create mode 100644 samtools-1.11/test/reheader/5_view1.sam.expected\n",
            " create mode 100644 samtools-1.11/test/reheader/hdr.sam\n",
            " create mode 100644 samtools-1.11/test/sort/name.sort.expected.sam\n",
            " create mode 100644 samtools-1.11/test/sort/pos.sort.expected.sam\n",
            " create mode 100644 samtools-1.11/test/sort/tag.as.sort.expected.sam\n",
            " create mode 100644 samtools-1.11/test/sort/tag.fi.sort.expected.sam\n",
            " create mode 100644 samtools-1.11/test/sort/tag.rg.n.sort.expected.sam\n",
            " create mode 100644 samtools-1.11/test/sort/tag.rg.sort.expected.sam\n",
            " create mode 100644 samtools-1.11/test/split/split.expected.grp1.sam\n",
            " create mode 100644 samtools-1.11/test/split/split.expected.grp2.sam\n",
            " create mode 100644 samtools-1.11/test/split/split.expected.unk.sam\n",
            " create mode 100644 samtools-1.11/test/split/split.sam\n",
            " create mode 100755 samtools-1.11/test/split/test_count_rg\n",
            " create mode 100644 samtools-1.11/test/split/test_count_rg.c\n",
            " create mode 100644 samtools-1.11/test/split/test_count_rg.o\n",
            " create mode 100755 samtools-1.11/test/split/test_expand_format_string\n",
            " create mode 100644 samtools-1.11/test/split/test_expand_format_string.c\n",
            " create mode 100644 samtools-1.11/test/split/test_expand_format_string.o\n",
            " create mode 100755 samtools-1.11/test/split/test_filter_header_rg\n",
            " create mode 100644 samtools-1.11/test/split/test_filter_header_rg.c\n",
            " create mode 100644 samtools-1.11/test/split/test_filter_header_rg.o\n",
            " create mode 100755 samtools-1.11/test/split/test_parse_args\n",
            " create mode 100644 samtools-1.11/test/split/test_parse_args.c\n",
            " create mode 100644 samtools-1.11/test/split/test_parse_args.o\n",
            " create mode 100644 samtools-1.11/test/stat/1.stats.expected\n",
            " create mode 100644 samtools-1.11/test/stat/1.stats.large.expected\n",
            " create mode 100644 samtools-1.11/test/stat/10.stats.expected\n",
            " create mode 100644 samtools-1.11/test/stat/10_map_cigar.sam\n",
            " create mode 100644 samtools-1.11/test/stat/10_map_cigar.sam_s1_a_1.expected.bamstat\n",
            " create mode 100644 samtools-1.11/test/stat/10_map_cigar.sam_s1_b_1.expected.bamstat\n",
            " create mode 100644 samtools-1.11/test/stat/11.stats.expected\n",
            " create mode 100644 samtools-1.11/test/stat/11.stats.g4.expected\n",
            " create mode 100644 samtools-1.11/test/stat/11.stats.targets\n",
            " create mode 100644 samtools-1.11/test/stat/11_target.bam\n",
            " create mode 100644 samtools-1.11/test/stat/11_target.bam.bai\n",
            " create mode 100644 samtools-1.11/test/stat/11_target.sam\n",
            " create mode 100644 samtools-1.11/test/stat/12.2reads.nooverlap.expected\n",
            " create mode 100644 samtools-1.11/test/stat/12.2reads.overlap.expected\n",
            " create mode 100644 samtools-1.11/test/stat/12.3reads.nooverlap.expected\n",
            " create mode 100644 samtools-1.11/test/stat/12.3reads.overlap.expected\n",
            " create mode 100644 samtools-1.11/test/stat/12_2reads.bed\n",
            " create mode 100644 samtools-1.11/test/stat/12_3reads.bed\n",
            " create mode 100644 samtools-1.11/test/stat/12_overlaps.bam\n",
            " create mode 100644 samtools-1.11/test/stat/13.barcodes.bc.ok.expected\n",
            " create mode 100644 samtools-1.11/test/stat/13.barcodes.ox.ok.expected\n",
            " create mode 100644 samtools-1.11/test/stat/13_barcodes_fail_bc_length.sam\n",
            " create mode 100644 samtools-1.11/test/stat/13_barcodes_fail_hyphen.sam\n",
            " create mode 100644 samtools-1.11/test/stat/13_barcodes_fail_qt_length.sam\n",
            " create mode 100644 samtools-1.11/test/stat/13_barcodes_ok.sam\n",
            " create mode 100644 samtools-1.11/test/stat/13_barcodes_ok_ox_bz.sam\n",
            " create mode 100644 samtools-1.11/test/stat/1_map_cigar.sam\n",
            " create mode 100644 samtools-1.11/test/stat/1_map_cigar.sam_s1_a_1.expected.bamstat\n",
            " create mode 100644 samtools-1.11/test/stat/1_map_cigar_large.sam\n",
            " create mode 100644 samtools-1.11/test/stat/2.stats.expected\n",
            " create mode 100644 samtools-1.11/test/stat/2.stats.large.expected\n",
            " create mode 100644 samtools-1.11/test/stat/2_equal_cigar_full_seq.sam\n",
            " create mode 100644 samtools-1.11/test/stat/2_equal_cigar_full_seq_large.sam\n",
            " create mode 100644 samtools-1.11/test/stat/3.stats.expected\n",
            " create mode 100644 samtools-1.11/test/stat/3.stats.large.expected\n",
            " create mode 100644 samtools-1.11/test/stat/3_map_cigar_equal_seq.sam\n",
            " create mode 100644 samtools-1.11/test/stat/3_map_cigar_equal_seq_large.sam\n",
            " create mode 100644 samtools-1.11/test/stat/4.stats.expected\n",
            " create mode 100644 samtools-1.11/test/stat/4.stats.large.expected\n",
            " create mode 100644 samtools-1.11/test/stat/4_X_cigar_full_seq.sam\n",
            " create mode 100644 samtools-1.11/test/stat/4_X_cigar_full_seq_large.sam\n",
            " create mode 100644 samtools-1.11/test/stat/5.stats.expected\n",
            " create mode 100644 samtools-1.11/test/stat/5.stats.large.expected\n",
            " create mode 100644 samtools-1.11/test/stat/5_insert_cigar.sam\n",
            " create mode 100644 samtools-1.11/test/stat/5_insert_cigar_large.sam\n",
            " create mode 100644 samtools-1.11/test/stat/6.stats.expected\n",
            " create mode 100644 samtools-1.11/test/stat/7.stats.expected\n",
            " create mode 100644 samtools-1.11/test/stat/7.stats.large.expected\n",
            " create mode 100644 samtools-1.11/test/stat/7_supp.sam\n",
            " create mode 100644 samtools-1.11/test/stat/7_supp_large.sam\n",
            " create mode 100644 samtools-1.11/test/stat/8.stats.expected\n",
            " create mode 100644 samtools-1.11/test/stat/8.stats.large.expected\n",
            " create mode 100644 samtools-1.11/test/stat/8_secondary.sam\n",
            " create mode 100644 samtools-1.11/test/stat/8_secondary_large.sam\n",
            " create mode 100644 samtools-1.11/test/stat/9.stats.expected\n",
            " create mode 100644 samtools-1.11/test/stat/test.fa\n",
            " create mode 100644 samtools-1.11/test/stat/test.fa.fai\n",
            " create mode 100644 samtools-1.11/test/test.c\n",
            " create mode 100644 samtools-1.11/test/test.h\n",
            " create mode 100644 samtools-1.11/test/test.o\n",
            " create mode 100755 samtools-1.11/test/test.pl\n",
            " create mode 100644 samtools-1.11/test/tview/test_get_rg_sample.c\n",
            " create mode 100755 samtools-1.11/test/vcf-miniview\n",
            " create mode 100644 samtools-1.11/test/vcf-miniview.c\n",
            " create mode 100644 samtools-1.11/test/vcf-miniview.o\n",
            " create mode 100644 samtools-1.11/tmp_file.c\n",
            " create mode 100644 samtools-1.11/tmp_file.h\n",
            " create mode 100644 samtools-1.11/tmp_file.o\n",
            " create mode 100644 samtools-1.11/version.h\n",
            " create mode 100755 samtools-1.11/version.sh\n",
            " mode change 100755 => 100644 scripts/0-data-loading-processing/05-cluster-maize_seq.sh\n",
            " create mode 100644 share/man/man1/samtools-addreplacerg.1\n",
            " create mode 100644 share/man/man1/samtools-ampliconclip.1\n",
            " create mode 100644 share/man/man1/samtools-ampliconstats.1\n",
            " create mode 100644 share/man/man1/samtools-bedcov.1\n",
            " create mode 100644 share/man/man1/samtools-calmd.1\n",
            " create mode 100644 share/man/man1/samtools-cat.1\n",
            " create mode 100644 share/man/man1/samtools-collate.1\n",
            " create mode 100644 share/man/man1/samtools-coverage.1\n",
            " create mode 100644 share/man/man1/samtools-depad.1\n",
            " create mode 100644 share/man/man1/samtools-depth.1\n",
            " create mode 100644 share/man/man1/samtools-dict.1\n",
            " create mode 100644 share/man/man1/samtools-faidx.1\n",
            " create mode 100644 share/man/man1/samtools-fasta.1\n",
            " create mode 100644 share/man/man1/samtools-fastq.1\n",
            " create mode 100644 share/man/man1/samtools-fixmate.1\n",
            " create mode 100644 share/man/man1/samtools-flags.1\n",
            " create mode 100644 share/man/man1/samtools-flagstat.1\n",
            " create mode 100644 share/man/man1/samtools-fqidx.1\n",
            " create mode 100644 share/man/man1/samtools-idxstats.1\n",
            " create mode 100644 share/man/man1/samtools-index.1\n",
            " create mode 100644 share/man/man1/samtools-markdup.1\n",
            " create mode 100644 share/man/man1/samtools-merge.1\n",
            " create mode 100644 share/man/man1/samtools-mpileup.1\n",
            " create mode 100644 share/man/man1/samtools-phase.1\n",
            " create mode 100644 share/man/man1/samtools-quickcheck.1\n",
            " create mode 100644 share/man/man1/samtools-reheader.1\n",
            " create mode 100644 share/man/man1/samtools-rmdup.1\n",
            " create mode 100644 share/man/man1/samtools-sort.1\n",
            " create mode 100644 share/man/man1/samtools-split.1\n",
            " create mode 100644 share/man/man1/samtools-stats.1\n",
            " create mode 100644 share/man/man1/samtools-targetcut.1\n",
            " create mode 100644 share/man/man1/samtools-tview.1\n",
            " create mode 100644 share/man/man1/samtools-view.1\n",
            " create mode 100644 share/man/man1/samtools.1\n",
            " create mode 100644 share/man/man1/wgsim.1\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!git push https://gurveervirk:Gurdarsh0501*8@github.com/gurveervirk/florabert.git --all"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "iVr2Ckqe5XDi",
        "outputId": "c2d794a6-93e4-421c-956a-488126725e1a"
      },
      "execution_count": 15,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "remote: Support for password authentication was removed on August 13, 2021.\n",
            "remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.\n",
            "fatal: Authentication failed for 'https://github.com/gurveervirk/florabert.git/'\n"
          ]
        }
      ]
    }
  ]
}